{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "accessory-elephant",
   "metadata": {},
   "source": [
    "# Phylogeny reconstruction"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "parental-earth",
   "metadata": {},
   "source": [
    "## 1. Alignment\n",
    "\n",
    "MAFFT"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "b0449d56-1905-47f7-9d6b-e3e7814b83f3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates'"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "id": "accepted-morning",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.490\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "20 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "   30 / 40\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP    35 /39 (thread    8) \n",
      "Reallocating (by thread 3) ..done. *alloclen = 2344\n",
      "STEP    39 /39 (thread   15) \n",
      "done.\n",
      "tbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "16 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "   30 / 40\n",
      "Segment   1/  1    1- 829\n",
      "009-0076-1 (thread    8) worse         001-0063-0 (thread    6) identical     003-0046-0 (thread    6) identical     004-0071-0 (thread    1) identical     006-0048-1 (thread    8) identical     008-0076-1 (thread    6) identical     \n",
      "Converged.\n",
      "\n",
      "Converged2.\n",
      "done\n",
      "dvtditr (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "    0 / 18 (by thread   0) \r"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alignment complete for file: combined_ITS_sequences.fasta. Progress: 33.33%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "tbfast-pair (nuc) Version 7.490\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "20 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "   10 / 18\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP    15 /17 (thread   15) \n",
      "Reallocating (by thread 13) ..done. *alloclen = 2544\n",
      "STEP    17 /17 (thread    2) \n",
      "done.\n",
      "tbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "16 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "   10 / 18\n",
      "Segment   1/  1    1-1011\n",
      "003-0032-1 (thread    7) worse         001-0026-1 (thread    7) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alignment complete for file: combined_Btub_sequences.fasta. Progress: 66.67%\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.490\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "20 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "   10 / 16\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP    15 /15 (thread   14) \n",
      "done.\n",
      "tbfast (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "16 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "   10 / 16\n",
      "Segment   1/  1    1-1341\n",
      "001-0023-0 (thread    6) identical     "
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alignment complete for file: combined_RPBII_sequences.fasta. Progress: 100.00%\n",
      "All alignments are complete.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "001-0028-0 (thread    4) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.490\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import subprocess\n",
    "\n",
    "# Define the directories containing the sequence files\n",
    "seq_directory = 'All seq'  # Replace with your path\n",
    "# Define the output directories for aligned files\n",
    "aligned_seq_directory = 'aligned'  # Replace with your path\n",
    "\n",
    "mafft_thread = 20\n",
    "\n",
    "def align_sequences(file_path, output_dir):\n",
    "    \"\"\"Align sequences in a file using MAFFT and save to the specified output directory.\"\"\"\n",
    "    file_name = os.path.basename(file_path)\n",
    "    base_name, ext = os.path.splitext(file_name)\n",
    "    # Ensure the output file ends with '_aligned.fasta'\n",
    "    output_file_name = f\"{base_name}_aligned.fasta\"\n",
    "    output_path = os.path.join(output_dir, output_file_name)\n",
    "    \n",
    "    mafft_command = ['mafft', '--thread', str(mafft_thread), '--auto', file_path]\n",
    "    with open(output_path, 'w') as output_file:\n",
    "        subprocess.run(mafft_command, stdout=output_file)\n",
    "\n",
    "def process_directory(input_dir, output_dir):\n",
    "    \"\"\"Process all sequence files in a given input directory and store aligned files in the output directory.\"\"\"\n",
    "    # Create the output directory if it does not exist\n",
    "    os.makedirs(output_dir, exist_ok=True)\n",
    "    \n",
    "    # Get a list of all files that will be processed\n",
    "    valid_extensions = ('.fasta', '.fas', '.faa', '.fna')\n",
    "    files_to_process = [f for f in os.listdir(input_dir) if f.endswith(valid_extensions)]\n",
    "    total_files = len(files_to_process)\n",
    "    \n",
    "    for index, file_name in enumerate(files_to_process, start=1):\n",
    "        file_path = os.path.join(input_dir, file_name)\n",
    "        align_sequences(file_path, output_dir)\n",
    "        progress_percentage = (index / total_files) * 100\n",
    "        print(f\"Alignment complete for file: {file_name}. Progress: {progress_percentage:.2f}%\")\n",
    "\n",
    "# Perform alignment for the sequence directory\n",
    "process_directory(seq_directory, aligned_seq_directory)\n",
    "\n",
    "print(\"All alignments are complete.\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "vital-daniel",
   "metadata": {},
   "source": [
    "## 2. Trimming \n",
    "\n",
    "Clipkit"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "id": "brown-system",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "-------------\n",
      "| Arguments |\n",
      "-------------\n",
      "Input file: aligned/combined_ITS_sequences_aligned.fasta (format: fasta)\n",
      "Output file: trimmed/combined_ITS_sequences_trimmed.fasta (format: fasta)\n",
      "Sequence type: Nucleotides\n",
      "Gaps threshold: 0.95\n",
      "Gap characters: ['-', '?', '*', 'X', 'x', 'N', 'n']\n",
      "Trimming mode: smart-gap\n",
      "Create complementary output: False\n",
      "Process as codons: False\n",
      "Create log file: False\n",
      "\n",
      "\n",
      "------------------------\n",
      "| Writing output files |\n",
      "------------------------\n",
      "Trimmed alignment: trimmed/combined_ITS_sequences_trimmed.fasta\n",
      "Complement file: False\n",
      "Log file: False\n",
      "\n",
      "\n",
      "---------------------\n",
      "| Output Statistics |\n",
      "---------------------\n",
      "Original length: 817\n",
      "Number of sites kept: 738\n",
      "Number of sites trimmed: 79\n",
      "Percentage of alignment trimmed: 9.67%\n",
      "\n",
      "Execution time: 0.017s\n",
      "\n",
      "Trimming complete for file: combined_ITS_sequences_aligned.fasta\n",
      "\n",
      "-------------\n",
      "| Arguments |\n",
      "-------------\n",
      "Input file: aligned/combined_Btub_sequences_aligned.fasta (format: fasta)\n",
      "Output file: trimmed/combined_Btub_sequences_trimmed.fasta (format: fasta)\n",
      "Sequence type: Nucleotides\n",
      "Gaps threshold: 0.6667\n",
      "Gap characters: ['-', '?', '*', 'X', 'x', 'N', 'n']\n",
      "Trimming mode: smart-gap\n",
      "Create complementary output: False\n",
      "Process as codons: False\n",
      "Create log file: False\n",
      "\n",
      "\n",
      "------------------------\n",
      "| Writing output files |\n",
      "------------------------\n",
      "Trimmed alignment: trimmed/combined_Btub_sequences_trimmed.fasta\n",
      "Complement file: False\n",
      "Log file: False\n",
      "\n",
      "\n",
      "---------------------\n",
      "| Output Statistics |\n",
      "---------------------\n",
      "Original length: 1012\n",
      "Number of sites kept: 614\n",
      "Number of sites trimmed: 398\n",
      "Percentage of alignment trimmed: 39.328%\n",
      "\n",
      "Execution time: 0.016s\n",
      "\n",
      "Trimming complete for file: combined_Btub_sequences_aligned.fasta\n",
      "\n",
      "-------------\n",
      "| Arguments |\n",
      "-------------\n",
      "Input file: aligned/combined_RPBII_sequences_aligned.fasta (format: fasta)\n",
      "Output file: trimmed/combined_RPBII_sequences_trimmed.fasta (format: fasta)\n",
      "Sequence type: Nucleotides\n",
      "Gaps threshold: 0.6875\n",
      "Gap characters: ['-', '?', '*', 'X', 'x', 'N', 'n']\n",
      "Trimming mode: smart-gap\n",
      "Create complementary output: False\n",
      "Process as codons: False\n",
      "Create log file: False\n",
      "\n",
      "\n",
      "------------------------\n",
      "| Writing output files |\n",
      "------------------------\n",
      "Trimmed alignment: trimmed/combined_RPBII_sequences_trimmed.fasta\n",
      "Complement file: False\n",
      "Log file: False\n",
      "\n",
      "\n",
      "---------------------\n",
      "| Output Statistics |\n",
      "---------------------\n",
      "Original length: 1340\n",
      "Number of sites kept: 1214\n",
      "Number of sites trimmed: 126\n",
      "Percentage of alignment trimmed: 9.403%\n",
      "\n",
      "Execution time: 0.012s\n",
      "\n",
      "Trimming complete for file: combined_RPBII_sequences_aligned.fasta\n",
      "All trimming operations for .fa files are complete.\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import subprocess\n",
    "\n",
    "# Define the directories containing the aligned .fa files\n",
    "aligned_seq_directory = 'aligned'  # Replace with your path\n",
    "\n",
    "# Define the output directories for trimmed files\n",
    "trimmed_seq_directory = 'trimmed'  # Replace with your path\n",
    "\n",
    "def trim_sequences(file_path, output_dir):\n",
    "    \"\"\"Trim sequences in a file using ClipKIT and save to the specified output directory.\"\"\"\n",
    "    file_name = os.path.basename(file_path)\n",
    "    output_path = os.path.join(output_dir, file_name.replace('_aligned.fasta', '_trimmed.fasta'))\n",
    "    \n",
    "#    clipkit_command = ['clipkit', file_path, '-m', 'gappy', '-g' '1','-o', output_path]\n",
    "    clipkit_command = ['clipkit', file_path, '-m', 'smart-gap', '-o', output_path]\n",
    "    subprocess.run(clipkit_command)\n",
    "\n",
    "def process_directory(input_dir, output_dir):\n",
    "    \"\"\"Process all aligned files in a given input directory and store trimmed files in the output directory.\"\"\"\n",
    "    # Create the output directory if it does not exist\n",
    "    os.makedirs(output_dir, exist_ok=True)\n",
    "\n",
    "    for file_name in os.listdir(input_dir):\n",
    "        if '_aligned.fasta' in file_name:\n",
    "            file_path = os.path.join(input_dir, file_name)\n",
    "            trim_sequences(file_path, output_dir)\n",
    "            print(f\"Trimming complete for file: {file_name}\")\n",
    "\n",
    "# Perform trimming for the .fa directory\n",
    "process_directory(aligned_seq_directory, trimmed_seq_directory)\n",
    "\n",
    "print(\"All trimming operations for .fa files are complete.\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "rapid-secretariat",
   "metadata": {},
   "source": [
    "## 3. Modeltest\n",
    "\n",
    "modeltest-ng"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "id": "shared-institution",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                             _      _ _            _      _   _  _____ \n",
      "                            | |    | | |          | |    | \\ | |/ ____|\n",
      "         _ __ ___   ___   __| | ___| | |_ ___  ___| |_   |  \\| | |  __ \n",
      "        | '_ ` _ \\ / _ \\ / _` |/ _ \\ | __/ _ \\/ __| __|  | . ` | | |_ |\n",
      "        | | | | | | (_) | (_| |  __/ | ||  __/\\__ \\ |_   | |\\  | |__| |\n",
      "        |_| |_| |_|\\___/ \\__,_|\\___|_|\\__\\___||___/\\__|  |_| \\_|\\_____|\n",
      "--------------------------------------------------------------------------------\n",
      "ModelTest-NG v0.1.7 released on 17.03.2021 by The Exelixis Lab.\n",
      "Written by Diego Darriba.\n",
      "Contributors: Tomas Flouri, Alexey Kozlov, Benoit Morel, David Posada, \n",
      "              Alexandros Stamatakis.\n",
      "Latest version: https://github.com/ddarriba/modeltest\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Physical cores: 16\n",
      "Logical cores:  32\n",
      "Memory:         126GB\n",
      "Extensions:     AVX\n",
      "\n",
      "WARNING: Partition DATA: Sequences Ophiostoma_quercus_CMW43227 and Ophiostoma_quercus_CMW40208 are identical\n",
      "WARNING: Partition DATA: Sequences Ophiostoma_quercus_CMW43227 and Ophiostoma_quercus_CMW22909 are identical\n",
      "WARNING: MSA has not enough sites to infer reliable results\n",
      "Creating new checkpoint file: /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/0.ckp\n",
      "--------------------------------------------------------------------------------\n",
      "ModelTest-NG v0.1.7\n",
      "\n",
      "Input data:\n",
      "  MSA:        /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta\n",
      "  Tree:       Maximum parsimony\n",
      "    file:           -\n",
      "  #taxa:            40\n",
      "  #sites:           738\n",
      "  #patterns:        514\n",
      "  Max. thread mem:  9 MB\n",
      "\n",
      "Output:\n",
      "  Log:           /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/0.log\n",
      "  Starting tree: /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/0.tree\n",
      "  Results:       /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/0.out\n",
      "\n",
      "Selection options:\n",
      "  # dna schemes:      11\n",
      "  # dna models:       88\n",
      "  include model parameters:\n",
      "    Uniform:         true\n",
      "    p-inv (+I):      true\n",
      "    gamma (+G):      true\n",
      "    both (+I+G):     true\n",
      "    free rates (+R): false\n",
      "    fixed freqs:     true\n",
      "    estimated freqs: true\n",
      "    #categories:     4\n",
      "  gamma rates mode:   mean\n",
      "  asc bias:           none\n",
      "  epsilon (opt):      0.01\n",
      "  epsilon (par):      0.05\n",
      "  keep branches:      false\n",
      "\n",
      "Additional options:\n",
      "  verbosity:        very low\n",
      "  threads:          1/16\n",
      "  RNG seed:         12345\n",
      "  subtree repeats:  enabled\n",
      "--------------------------------------------------------------------------------\n",
      "modeltest-ng was called as follows: \n",
      ">> modeltest-ng -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -d nt -o /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/0 \n",
      "\n",
      "\n",
      "modeltest-ng: Warning: You are using one single thread out of 16 physical cores.\n",
      "modeltest-ng:          You can set the number of threads with -p argument.\n",
      "modeltest-ng: Try 'modeltest-ng --help' or 'modeltest-ng --usage' for more information\n",
      "\n",
      "Partition 1/1\n",
      "\n",
      "    1/88   JC             0h:00:00   0h:00:00           -7504.9713       -       -\n",
      "    2/88   JC+I           0h:00:00   0h:00:00           -7184.2047       -  0.2973\n",
      "    3/88   JC+G4          0h:00:00   0h:00:00           -7121.5065  0.6257       -\n",
      "    4/88   JC+I+G4        0h:00:00   0h:00:00           -7108.0659  1.5013  0.2375\n",
      "    5/88   F81            0h:00:00   0h:00:00           -7438.7106       -       -\n",
      "    6/88   F81+I          0h:00:00   0h:00:00           -7089.6503       -  0.2982\n",
      "    7/88   F81+G4         0h:00:00   0h:00:00           -7018.8730  0.5647       -\n",
      "    8/88   F81+I+G4       0h:00:01   0h:00:01           -7005.4064  1.4478  0.2475\n",
      "    9/88   K80            0h:00:00   0h:00:01           -7485.4129       -       -\n",
      "   10/88   K80+I          0h:00:00   0h:00:01           -7162.9817       -  0.2975\n",
      "   11/88   K80+G4         0h:00:00   0h:00:01           -7097.0406  0.5944       -\n",
      "   12/88   K80+I+G4       0h:00:00   0h:00:01           -7083.9958  1.4333  0.2342\n",
      "   13/88   HKY            0h:00:00   0h:00:01           -7411.3583       -       -\n",
      "   14/88   HKY+I          0h:00:00   0h:00:01           -7062.9008       -  0.2983\n",
      "   15/88   HKY+G4         0h:00:00   0h:00:01           -6985.7984  0.5647       -\n",
      "   16/88   HKY+I+G4       0h:00:01   0h:00:02           -6973.3460  1.3136  0.2372\n",
      "   17/88   TrNef          0h:00:00   0h:00:02           -7455.4560       -       -\n",
      "   18/88   TrNef+I        0h:00:00   0h:00:02           -7152.7314       -  0.2966\n",
      "   19/88   TrNef+G4       0h:00:00   0h:00:02           -7073.5749  0.5923       -\n",
      "   20/88   TrNef+I+G4     0h:00:00   0h:00:02           -7064.2206  1.2031  0.2088\n",
      "   21/88   TrN            0h:00:00   0h:00:02           -7391.8243       -       -\n",
      "   22/88   TrN+I          0h:00:00   0h:00:02           -7060.4839       -  0.2983\n",
      "   23/88   TrN+G4         0h:00:00   0h:00:02           -6976.3324  0.5647       -\n",
      "   24/88   TrN+I+G4       0h:00:00   0h:00:02           -6966.0878  1.3024  0.2341\n",
      "   25/88   TPM2           0h:00:00   0h:00:02           -7483.2168       -       -\n",
      "   26/88   TPM2+I         0h:00:00   0h:00:02           -7160.9451       -  0.2974\n",
      "   27/88   TPM2+G4        0h:00:00   0h:00:02           -7094.5380  0.5944       -\n",
      "   28/88   TPM2+I+G4      0h:00:00   0h:00:02           -7081.5226  1.4360  0.2345\n",
      "   29/88   TPM2uf         0h:00:01   0h:00:03           -7410.7030       -       -\n",
      "   30/88   TPM2uf+I       0h:00:00   0h:00:03           -7061.0167       -  0.2978\n",
      "   31/88   TPM2uf+G4      0h:00:00   0h:00:03           -6979.7432  0.5364       -\n",
      "   32/88   TPM2uf+I+G4    0h:00:00   0h:00:03           -6967.3156  1.2722  0.2365\n",
      "   33/88   TPM3           0h:00:00   0h:00:03           -7461.5416       -       -\n",
      "   34/88   TPM3+I         0h:00:00   0h:00:03           -7149.4169       -  0.2966\n",
      "   35/88   TPM3+G4        0h:00:00   0h:00:03           -7075.1209  0.5928       -\n",
      "   36/88   TPM3+I+G4      0h:00:01   0h:00:04           -7063.5829  1.3348  0.2280\n",
      "   37/88   TPM3uf         0h:00:00   0h:00:04           -7405.0376       -       -\n",
      "   38/88   TPM3uf+I       0h:00:00   0h:00:04           -7061.1939       -  0.2987\n",
      "   39/88   TPM3uf+G4      0h:00:00   0h:00:04           -6983.9347  0.5647       -\n",
      "   40/88   TPM3uf+I+G4    0h:00:00   0h:00:04           -6971.9763  1.3544  0.2431\n",
      "   41/88   TPM1           0h:00:00   0h:00:04           -7476.3429       -       -\n",
      "   42/88   TPM1+I         0h:00:00   0h:00:04           -7153.2651       -  0.2975\n",
      "   43/88   TPM1+G4        0h:00:01   0h:00:05           -7085.5578  0.5944       -\n",
      "   44/88   TPM1+I+G4      0h:00:00   0h:00:05           -7072.5536  1.4355  0.2352\n",
      "   45/88   TPM1uf         0h:00:00   0h:00:05           -7408.7403       -       -\n",
      "   46/88   TPM1uf+I       0h:00:00   0h:00:05           -7061.2160       -  0.2980\n",
      "   47/88   TPM1uf+G4      0h:00:00   0h:00:05           -6984.1230  0.5647       -\n",
      "   48/88   TPM1uf+I+G4    0h:00:00   0h:00:05           -6971.7696  1.2969  0.2358\n",
      "   49/88   TIM2ef         0h:00:00   0h:00:05           -7453.2326       -       -\n",
      "   50/88   TIM2ef+I       0h:00:00   0h:00:05           -7150.6068       -  0.2964\n",
      "   51/88   TIM2ef+G4      0h:00:01   0h:00:06           -7071.4927  0.5938       -\n",
      "   52/88   TIM2ef+I+G4    0h:00:00   0h:00:06           -7061.9879  1.2579  0.2147\n",
      "   53/88   TIM2           0h:00:00   0h:00:06           -7391.4130       -       -\n",
      "   54/88   TIM2+I         0h:00:00   0h:00:06           -7058.7518       -  0.2983\n",
      "   55/88   TIM2+G4        0h:00:00   0h:00:06           -6972.2571  0.5364       -\n",
      "   56/88   TIM2+I+G4      0h:00:01   0h:00:07           -6961.7925  1.2581  0.2315\n",
      "   57/88   TIM3ef         0h:00:00   0h:00:07           -7431.3065       -       -\n",
      "   58/88   TIM3ef+I       0h:00:00   0h:00:07           -7139.2715       -  0.2955\n",
      "   59/88   TIM3ef+G4      0h:00:00   0h:00:07           -7048.1651  0.5944       -\n",
      "   60/88   TIM3ef+I+G4    0h:00:00   0h:00:07           -7040.7418  1.1171  0.2004\n",
      "   61/88   TIM3           0h:00:00   0h:00:07           -7386.9516       -       -\n",
      "   62/88   TIM3+I         0h:00:00   0h:00:07           -7058.9510       -  0.2990\n",
      "   63/88   TIM3+G4        0h:00:01   0h:00:08           -6975.2593  0.5600       -\n",
      "   64/88   TIM3+I+G4      0h:00:00   0h:00:08           -6965.3439  1.2975  0.2363\n",
      "   65/88   TIM1ef         0h:00:00   0h:00:08           -7446.4051       -       -\n",
      "   66/88   TIM1ef+I       0h:00:00   0h:00:08           -7143.1150       -  0.2966\n",
      "   67/88   TIM1ef+G4      0h:00:00   0h:00:08           -7062.4713  0.5906       -\n",
      "   68/88   TIM1ef+I+G4    0h:00:00   0h:00:08           -7053.0624  1.1996  0.2092\n",
      "   69/88   TIM1           0h:00:01   0h:00:09           -7389.2004       -       -\n",
      "   70/88   TIM1+I         0h:00:00   0h:00:09           -7058.7933       -  0.2984\n",
      "   71/88   TIM1+G4        0h:00:00   0h:00:09           -6974.6678  0.5647       -\n",
      "   72/88   TIM1+I+G4      0h:00:00   0h:00:09           -6964.4802  1.2901  0.2329\n",
      "   73/88   TVMef          0h:00:00   0h:00:09           -7454.6268       -       -\n",
      "   74/88   TVMef+I        0h:00:00   0h:00:09           -7141.4243       -  0.2965\n",
      "   75/88   TVMef+G4       0h:00:01   0h:00:10           -7066.3347  0.5923       -\n",
      "   76/88   TVMef+I+G4     0h:00:00   0h:00:10           -7054.6707  1.3226  0.2267\n",
      "   77/88   TVM            0h:00:00   0h:00:10           -7401.7474       -       -\n",
      "   78/88   TVM+I          0h:00:00   0h:00:10           -7056.5597       -  0.2989\n",
      "   79/88   TVM+G4         0h:00:00   0h:00:10           -6975.0189  0.5364       -\n",
      "   80/88   TVM+I+G4       0h:00:01   0h:00:11           -6962.9041  1.3306  0.2437\n",
      "   81/88   SYM            0h:00:00   0h:00:11           -7424.3344       -       -\n",
      "   82/88   SYM+I          0h:00:00   0h:00:11           -7131.2695       -  0.2945\n",
      "   83/88   SYM+G4         0h:00:00   0h:00:11           -7040.1043  0.5944       -\n",
      "   84/88   SYM+I+G4       0h:00:00   0h:00:11           -7032.7374  1.0000  0.1771\n",
      "   85/88   GTR            0h:00:00   0h:00:11           -7383.8717       -       -\n",
      "   86/88   GTR+I          0h:00:00   0h:00:11           -7054.4751       -  0.2981\n",
      "   87/88   GTR+G4         0h:00:01   0h:00:12           -6968.0928  0.5364       -\n",
      "   88/88   GTR+I+G4       0h:00:00   0h:00:12           -6957.8020  1.1947  0.2236\n",
      " ----ID---  ----MODEL---- ---Time--- -Elapsed--- -------LnL------- -Alpha- -P-inv-\n",
      "\n",
      "Computation of likelihood scores completed. It took 0h:00:12\n",
      "\n",
      "BIC       model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  TIM2+I+G4          8     -6961.7925     14484.9202         0.0000    0.5982\n",
      "       2  TrN+I+G4           7     -6966.0878     14486.9069         1.9868    0.2215\n",
      "       3  TPM2uf+I+G4        7     -6967.3156     14489.3626         4.4424    0.0649\n",
      "       4  GTR+I+G4          10     -6957.8020     14490.1471         5.2270    0.0438\n",
      "       5  TIM1+I+G4          8     -6964.4802     14490.2956         5.3754    0.0407\n",
      "       6  TIM3+I+G4          8     -6965.3439     14492.0230         7.1028    0.0172\n",
      "       7  TVM+I+G4           9     -6962.9041     14493.7473         8.8271    0.0072\n",
      "       8  HKY+I+G4           6     -6973.3460     14494.8194         9.8993    0.0042\n",
      "       9  TPM1uf+I+G4        7     -6971.7696     14498.2705        13.3504    0.0008\n",
      "      10  TPM3uf+I+G4        7     -6971.9763     14498.6838        13.7637    0.0006\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to BIC\n",
      "---------------------------\n",
      "Model:              TIM2+I+G4\n",
      "lnL:                -6961.7925\n",
      "Frequencies:        0.1629 0.3757 0.2795 0.1819\n",
      "Subst. Rates:       1.4585 1.5918 1.4585 1.0000 2.7103 1.0000 \n",
      "Inv. sites prop:    0.2315\n",
      "Gamma shape:        1.2581\n",
      "Score:              14484.9202\n",
      "Weight:             0.5982\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.0008\n",
      "Gamma-Inv:          0.9992\n",
      "Frequencies:        1.0000\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.2983\n",
      "Alpha:              0.5463\n",
      "Alpha-P.Inv:        1.2689\n",
      "P.Inv-Alpha:        0.2323\n",
      "Frequencies:        0.1666 0.3761 0.2740 0.1833 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m 010232 -f m -v e -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m GTRGAMMAIX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta --model TIM2+I+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m TIM2+I+G4\n",
      "\n",
      "AIC       model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  GTR+I+G4          10     -6957.8020     14089.6040         0.0000    0.8525\n",
      "       2  TIM2+I+G4          8     -6961.7925     14093.5849         3.9809    0.1165\n",
      "       3  TVM+I+G4           9     -6962.9041     14097.8081         8.2041    0.0141\n",
      "       4  TIM1+I+G4          8     -6964.4802     14098.9603         9.3563    0.0079\n",
      "       5  TrN+I+G4           7     -6966.0878     14100.1757        10.5716    0.0043\n",
      "       6  TIM3+I+G4          8     -6965.3439     14100.6877        11.0837    0.0033\n",
      "       7  TPM2uf+I+G4        7     -6967.3156     14102.6313        13.0273    0.0013\n",
      "       8  GTR+G4             9     -6968.0928     14108.1857        18.5817    0.0001\n",
      "       9  TPM1uf+I+G4        7     -6971.7696     14111.5392        21.9352    0.0000\n",
      "      10  TPM3uf+I+G4        7     -6971.9763     14111.9525        22.3485    0.0000\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to AIC\n",
      "---------------------------\n",
      "Model:              GTR+I+G4\n",
      "lnL:                -6957.8020\n",
      "Frequencies:        0.1643 0.3698 0.2789 0.1870\n",
      "Subst. Rates:       1.8721 2.1749 2.2057 1.5801 3.6838 1.0000 \n",
      "Inv. sites prop:    0.2236\n",
      "Gamma shape:        1.1947\n",
      "Score:              14089.6040\n",
      "Weight:             0.8525\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.0001\n",
      "Gamma-Inv:          0.9999\n",
      "Frequencies:        1.0000\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.2983\n",
      "Alpha:              0.5369\n",
      "Alpha-P.Inv:        1.2056\n",
      "P.Inv-Alpha:        0.2250\n",
      "Frequencies:        0.1642 0.3707 0.2785 0.1865 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m 012345 -f m -v e -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m GTRGAMMAIX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta --model GTR+I+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m GTR+I+G4\n",
      "\n",
      "AICc      model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  GTR+I+G4          10     -6957.8020     14112.6040         0.0000    0.7737\n",
      "       2  TIM2+I+G4          8     -6961.7925     14115.5849         2.9809    0.1743\n",
      "       3  TVM+I+G4           9     -6962.9041     14119.8081         7.2041    0.0211\n",
      "       4  TIM1+I+G4          8     -6964.4802     14120.9603         8.3563    0.0119\n",
      "       5  TrN+I+G4           7     -6966.0878     14121.1757         8.5716    0.0106\n",
      "       6  TIM3+I+G4          8     -6965.3439     14122.6877        10.0837    0.0050\n",
      "       7  TPM2uf+I+G4        7     -6967.3156     14123.6313        11.0273    0.0031\n",
      "       8  GTR+G4             9     -6968.0928     14130.1857        17.5817    0.0001\n",
      "       9  TPM1uf+I+G4        7     -6971.7696     14132.5392        19.9352    0.0000\n",
      "      10  TPM3uf+I+G4        7     -6971.9763     14132.9525        20.3485    0.0000\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to AICc\n",
      "---------------------------\n",
      "Model:              GTR+I+G4\n",
      "lnL:                -6957.8020\n",
      "Frequencies:        0.1643 0.3698 0.2789 0.1870\n",
      "Subst. Rates:       1.8721 2.1749 2.2057 1.5801 3.6838 1.0000 \n",
      "Inv. sites prop:    0.2236\n",
      "Gamma shape:        1.1947\n",
      "Score:              14112.6040\n",
      "Weight:             0.7737\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.0001\n",
      "Gamma-Inv:          0.9999\n",
      "Frequencies:        1.0000\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.2983\n",
      "Alpha:              0.5372\n",
      "Alpha-P.Inv:        1.2116\n",
      "P.Inv-Alpha:        0.2257\n",
      "Frequencies:        0.1642 0.3712 0.2783 0.1863 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m 012345 -f m -v e -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m GTRGAMMAIX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta --model GTR+I+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m GTR+I+G4\n",
      "Summary:\n",
      "\n",
      "Partition 1/1:\n",
      "                         Model         Score        Weight\n",
      "----------------------------------------------------------\n",
      "       BIC           TIM2+I+G4    14484.9202        0.5982\n",
      "       AIC            GTR+I+G4    14089.6040        0.8525\n",
      "      AICc            GTR+I+G4    14112.6040        0.7737\n",
      "\n",
      "Execution results written to /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/0.out\n",
      "Starting tree written to /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/0.tree\n",
      "                             _      _ _            _      _   _  _____ \n",
      "                            | |    | | |          | |    | \\ | |/ ____|\n",
      "         _ __ ___   ___   __| | ___| | |_ ___  ___| |_   |  \\| | |  __ \n",
      "        | '_ ` _ \\ / _ \\ / _` |/ _ \\ | __/ _ \\/ __| __|  | . ` | | |_ |\n",
      "        | | | | | | (_) | (_| |  __/ | ||  __/\\__ \\ |_   | |\\  | |__| |\n",
      "        |_| |_| |_|\\___/ \\__,_|\\___|_|\\__\\___||___/\\__|  |_| \\_|\\_____|\n",
      "--------------------------------------------------------------------------------\n",
      "ModelTest-NG v0.1.7 released on 17.03.2021 by The Exelixis Lab.\n",
      "Written by Diego Darriba.\n",
      "Contributors: Tomas Flouri, Alexey Kozlov, Benoit Morel, David Posada, \n",
      "              Alexandros Stamatakis.\n",
      "Latest version: https://github.com/ddarriba/modeltest\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Physical cores: 16\n",
      "Logical cores:  32\n",
      "Memory:         126GB\n",
      "Extensions:     AVX\n",
      "\n",
      "WARNING: Partition DATA: Sequences Ophiostoma_quercus_CMW43227 and Ophiostoma_quercus_CMW40208 are identical\n",
      "WARNING: Partition DATA: Sequences Ophiostoma_quercus_CMW43227 and Ophiostoma_quercus_CMW22909 are identical\n",
      "WARNING: MSA has not enough sites to infer reliable results\n",
      "Creating new checkpoint file: /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb0.ckp\n",
      "--------------------------------------------------------------------------------\n",
      "ModelTest-NG v0.1.7\n",
      "\n",
      "Input data:\n",
      "  MSA:        /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta\n",
      "  Tree:       Maximum parsimony\n",
      "    file:           -\n",
      "  #taxa:            40\n",
      "  #sites:           738\n",
      "  #patterns:        514\n",
      "  Max. thread mem:  9 MB\n",
      "\n",
      "Output:\n",
      "  Log:           /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb0.log\n",
      "  Starting tree: /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb0.tree\n",
      "  Results:       /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb0.out\n",
      "\n",
      "Selection options:\n",
      "  # dna schemes:      3\n",
      "  # dna models:       24\n",
      "  include model parameters:\n",
      "    Uniform:         true\n",
      "    p-inv (+I):      true\n",
      "    gamma (+G):      true\n",
      "    both (+I+G):     true\n",
      "    free rates (+R): false\n",
      "    fixed freqs:     true\n",
      "    estimated freqs: true\n",
      "    #categories:     4\n",
      "  gamma rates mode:   mean\n",
      "  asc bias:           none\n",
      "  epsilon (opt):      0.01\n",
      "  epsilon (par):      0.05\n",
      "  keep branches:      false\n",
      "\n",
      "Additional options:\n",
      "  verbosity:        very low\n",
      "  threads:          1/16\n",
      "  RNG seed:         12345\n",
      "  subtree repeats:  enabled\n",
      "--------------------------------------------------------------------------------\n",
      "modeltest-ng was called as follows: \n",
      ">> modeltest-ng -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -d nt -o /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb0 -T mrbayes \n",
      "\n",
      "\n",
      "modeltest-ng: Warning: You are using one single thread out of 16 physical cores.\n",
      "modeltest-ng:          You can set the number of threads with -p argument.\n",
      "modeltest-ng: Try 'modeltest-ng --help' or 'modeltest-ng --usage' for more information\n",
      "\n",
      "Partition 1/1\n",
      "\n",
      "    1/24   JC             0h:00:00   0h:00:00           -7504.9713       -       -\n",
      "    2/24   JC+I           0h:00:00   0h:00:00           -7184.2047       -  0.2973\n",
      "    3/24   JC+G4          0h:00:00   0h:00:00           -7121.5065  0.6257       -\n",
      "    4/24   JC+I+G4        0h:00:00   0h:00:00           -7108.0659  1.5013  0.2375\n",
      "    5/24   F81            0h:00:00   0h:00:00           -7438.7106       -       -\n",
      "    6/24   F81+I          0h:00:01   0h:00:01           -7089.6503       -  0.2982\n",
      "    7/24   F81+G4         0h:00:00   0h:00:01           -7018.8730  0.5647       -\n",
      "    8/24   F81+I+G4       0h:00:00   0h:00:01           -7005.4064  1.4478  0.2475\n",
      "    9/24   K80            0h:00:00   0h:00:01           -7485.4129       -       -\n",
      "   10/24   K80+I          0h:00:00   0h:00:01           -7162.9817       -  0.2975\n",
      "   11/24   K80+G4         0h:00:00   0h:00:01           -7097.0406  0.5944       -\n",
      "   12/24   K80+I+G4       0h:00:00   0h:00:01           -7083.9958  1.4333  0.2342\n",
      "   13/24   HKY            0h:00:00   0h:00:01           -7411.3583       -       -\n",
      "   14/24   HKY+I          0h:00:00   0h:00:01           -7062.9008       -  0.2983\n",
      "   15/24   HKY+G4         0h:00:01   0h:00:02           -6985.7984  0.5647       -\n",
      "   16/24   HKY+I+G4       0h:00:00   0h:00:02           -6973.3460  1.3136  0.2372\n",
      "   17/24   SYM            0h:00:00   0h:00:02           -7424.3344       -       -\n",
      "   18/24   SYM+I          0h:00:00   0h:00:02           -7131.2695       -  0.2945\n",
      "   19/24   SYM+G4         0h:00:00   0h:00:02           -7040.1043  0.5944       -\n",
      "   20/24   SYM+I+G4       0h:00:00   0h:00:02           -7032.7374  1.0000  0.1771\n",
      "   21/24   GTR            0h:00:00   0h:00:02           -7383.8717       -       -\n",
      "   22/24   GTR+I          0h:00:00   0h:00:02           -7054.4751       -  0.2981\n",
      "   23/24   GTR+G4         0h:00:01   0h:00:03           -6968.0928  0.5364       -\n",
      "   24/24   GTR+I+G4       0h:00:00   0h:00:03           -6957.8020  1.1947  0.2236\n",
      " ----ID---  ----MODEL---- ---Time--- -Elapsed--- -------LnL------- -Alpha- -P-inv-\n",
      "\n",
      "Computation of likelihood scores completed. It took 0h:00:03\n",
      "\n",
      "BIC       model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  GTR+I+G4          10     -6957.8020     14490.1471         0.0000    0.9111\n",
      "       2  HKY+I+G4           6     -6973.3460     14494.8194         4.6723    0.0881\n",
      "       3  GTR+G4             9     -6968.0928     14504.1249        13.9777    0.0008\n",
      "       4  HKY+G4             5     -6985.7984     14513.1201        22.9730    0.0000\n",
      "       5  F81+I+G4           5     -7005.4064     14552.3361        62.1890    0.0000\n",
      "       6  F81+G4             4     -7018.8730     14572.6654        82.5183    0.0000\n",
      "       7  SYM+I+G4           7     -7032.7374     14620.2062       130.0590    0.0000\n",
      "       8  SYM+G4             6     -7040.1043     14628.3359       138.1888    0.0000\n",
      "       9  HKY+I              5     -7062.9008     14667.3250       177.1779    0.0000\n",
      "      10  GTR+I              9     -7054.4751     14676.8894       186.7422    0.0000\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to BIC\n",
      "---------------------------\n",
      "Model:              GTR+I+G4\n",
      "lnL:                -6957.8020\n",
      "Frequencies:        0.1643 0.3698 0.2789 0.1870\n",
      "Subst. Rates:       1.8721 2.1749 2.2057 1.5801 3.6838 1.0000 \n",
      "Inv. sites prop:    0.2236\n",
      "Gamma shape:        1.1947\n",
      "Score:              14490.1471\n",
      "Weight:             0.9111\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.0008\n",
      "Gamma-Inv:          0.9992\n",
      "Frequencies:        1.0000\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.2983\n",
      "Alpha:              0.5367\n",
      "Alpha-P.Inv:        1.2052\n",
      "P.Inv-Alpha:        0.2248\n",
      "Frequencies:        0.1644 0.3717 0.2762 0.1877 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m 012345 -f m -v e -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m GTRGAMMAIX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta --model GTR+I+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m GTR+I+G4\n",
      "\n",
      "AIC       model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  GTR+I+G4          10     -6957.8020     14089.6040         0.0000    0.9999\n",
      "       2  GTR+G4             9     -6968.0928     14108.1857        18.5817    0.0001\n",
      "       3  HKY+I+G4           6     -6973.3460     14112.6921        23.0881    0.0000\n",
      "       4  HKY+G4             5     -6985.7984     14135.5967        45.9927    0.0000\n",
      "       5  F81+I+G4           5     -7005.4064     14174.8127        85.2087    0.0000\n",
      "       6  F81+G4             4     -7018.8730     14199.7460       110.1420    0.0000\n",
      "       7  SYM+I+G4           7     -7032.7374     14233.4749       143.8709    0.0000\n",
      "       8  SYM+G4             6     -7040.1043     14246.2086       156.6045    0.0000\n",
      "       9  GTR+I              9     -7054.4751     14280.9502       191.3462    0.0000\n",
      "      10  HKY+I              5     -7062.9008     14289.8016       200.1976    0.0000\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to AIC\n",
      "---------------------------\n",
      "Model:              GTR+I+G4\n",
      "lnL:                -6957.8020\n",
      "Frequencies:        0.1643 0.3698 0.2789 0.1870\n",
      "Subst. Rates:       1.8721 2.1749 2.2057 1.5801 3.6838 1.0000 \n",
      "Inv. sites prop:    0.2236\n",
      "Gamma shape:        1.1947\n",
      "Score:              14089.6040\n",
      "Weight:             0.9999\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.0001\n",
      "Gamma-Inv:          0.9999\n",
      "Frequencies:        1.0000\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.2981\n",
      "Alpha:              0.5364\n",
      "Alpha-P.Inv:        1.1947\n",
      "P.Inv-Alpha:        0.2236\n",
      "Frequencies:        0.1643 0.3698 0.2789 0.1870 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m 012345 -f m -v e -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m GTRGAMMAIX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta --model GTR+I+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m GTR+I+G4\n",
      "\n",
      "AICc      model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  GTR+I+G4          10     -6957.8020     14112.6040         0.0000    0.9998\n",
      "       2  GTR+G4             9     -6968.0928     14130.1857        17.5817    0.0002\n",
      "       3  HKY+I+G4           6     -6973.3460     14133.6921        21.0881    0.0000\n",
      "       4  HKY+G4             5     -6985.7984     14155.5967        42.9927    0.0000\n",
      "       5  F81+I+G4           5     -7005.4064     14194.8127        82.2087    0.0000\n",
      "       6  F81+G4             4     -7018.8730     14219.7460       107.1420    0.0000\n",
      "       7  SYM+I+G4           7     -7032.7374     14254.4749       141.8709    0.0000\n",
      "       8  SYM+G4             6     -7040.1043     14267.2086       154.6045    0.0000\n",
      "       9  GTR+I              9     -7054.4751     14302.9502       190.3462    0.0000\n",
      "      10  HKY+I              5     -7062.9008     14309.8016       197.1976    0.0000\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to AICc\n",
      "---------------------------\n",
      "Model:              GTR+I+G4\n",
      "lnL:                -6957.8020\n",
      "Frequencies:        0.1643 0.3698 0.2789 0.1870\n",
      "Subst. Rates:       1.8721 2.1749 2.2057 1.5801 3.6838 1.0000 \n",
      "Inv. sites prop:    0.2236\n",
      "Gamma shape:        1.1947\n",
      "Score:              14112.6040\n",
      "Weight:             0.9998\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.0002\n",
      "Gamma-Inv:          0.9998\n",
      "Frequencies:        1.0000\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.2981\n",
      "Alpha:              0.5364\n",
      "Alpha-P.Inv:        1.1947\n",
      "P.Inv-Alpha:        0.2236\n",
      "Frequencies:        0.1643 0.3698 0.2789 0.1870 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m 012345 -f m -v e -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m GTRGAMMAIX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta --model GTR+I+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_ITS_sequences_trimmed.fasta -m GTR+I+G4\n",
      "Summary:\n",
      "\n",
      "Partition 1/1:\n",
      "                         Model         Score        Weight\n",
      "----------------------------------------------------------\n",
      "       BIC            GTR+I+G4    14490.1471        0.9111\n",
      "       AIC            GTR+I+G4    14089.6040        0.9999\n",
      "      AICc            GTR+I+G4    14112.6040        0.9998\n",
      "\n",
      "Execution results written to /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb0.out\n",
      "Starting tree written to /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb0.tree\n",
      "                             _      _ _            _      _   _  _____ \n",
      "                            | |    | | |          | |    | \\ | |/ ____|\n",
      "         _ __ ___   ___   __| | ___| | |_ ___  ___| |_   |  \\| | |  __ \n",
      "        | '_ ` _ \\ / _ \\ / _` |/ _ \\ | __/ _ \\/ __| __|  | . ` | | |_ |\n",
      "        | | | | | | (_) | (_| |  __/ | ||  __/\\__ \\ |_   | |\\  | |__| |\n",
      "        |_| |_| |_|\\___/ \\__,_|\\___|_|\\__\\___||___/\\__|  |_| \\_|\\_____|\n",
      "--------------------------------------------------------------------------------\n",
      "ModelTest-NG v0.1.7 released on 17.03.2021 by The Exelixis Lab.\n",
      "Written by Diego Darriba.\n",
      "Contributors: Tomas Flouri, Alexey Kozlov, Benoit Morel, David Posada, \n",
      "              Alexandros Stamatakis.\n",
      "Latest version: https://github.com/ddarriba/modeltest\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Physical cores: 16\n",
      "Logical cores:  32\n",
      "Memory:         126GB\n",
      "Extensions:     AVX\n",
      "\n",
      "WARNING: Partition DATA: Sequences Leptographium_procerum_CMW_34542 and Leptographium_procerum_CMW_29994 are identical\n",
      "WARNING: Partition DATA: Sequences Ophiostoma_floccosum_CMW12623 and Ophiostoma_floccosum_CMW12622 are identical\n",
      "WARNING: MSA has not enough sites to infer reliable results\n",
      "Creating new checkpoint file: /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/1.ckp\n",
      "--------------------------------------------------------------------------------\n",
      "ModelTest-NG v0.1.7\n",
      "\n",
      "Input data:\n",
      "  MSA:        /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta\n",
      "  Tree:       Maximum parsimony\n",
      "    file:           -\n",
      "  #taxa:            18\n",
      "  #sites:           614\n",
      "  #patterns:        379\n",
      "  Max. thread mem:  3 MB\n",
      "\n",
      "Output:\n",
      "  Log:           /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/1.log\n",
      "  Starting tree: /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/1.tree\n",
      "  Results:       /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/1.out\n",
      "\n",
      "Selection options:\n",
      "  # dna schemes:      11\n",
      "  # dna models:       88\n",
      "  include model parameters:\n",
      "    Uniform:         true\n",
      "    p-inv (+I):      true\n",
      "    gamma (+G):      true\n",
      "    both (+I+G):     true\n",
      "    free rates (+R): false\n",
      "    fixed freqs:     true\n",
      "    estimated freqs: true\n",
      "    #categories:     4\n",
      "  gamma rates mode:   mean\n",
      "  asc bias:           none\n",
      "  epsilon (opt):      0.01\n",
      "  epsilon (par):      0.05\n",
      "  keep branches:      false\n",
      "\n",
      "Additional options:\n",
      "  verbosity:        very low\n",
      "  threads:          1/16\n",
      "  RNG seed:         12345\n",
      "  subtree repeats:  enabled\n",
      "--------------------------------------------------------------------------------\n",
      "modeltest-ng was called as follows: \n",
      ">> modeltest-ng -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -d nt -o /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/1 \n",
      "\n",
      "\n",
      "modeltest-ng: Warning: You are using one single thread out of 16 physical cores.\n",
      "modeltest-ng:          You can set the number of threads with -p argument.\n",
      "modeltest-ng: Try 'modeltest-ng --help' or 'modeltest-ng --usage' for more information\n",
      "\n",
      "Partition 1/1\n",
      "\n",
      "    1/88   JC             0h:00:00   0h:00:00           -4312.2330       -       -\n",
      "    2/88   JC+I           0h:00:00   0h:00:00           -4081.7552       -  0.4079\n",
      "    3/88   JC+G4          0h:00:00   0h:00:00           -4049.3810  0.3559       -\n",
      "    4/88   JC+I+G4        0h:00:00   0h:00:00           -4047.0958  1.0000  0.3283\n",
      "    5/88   F81            0h:00:00   0h:00:00           -4300.0637       -       -\n",
      "    6/88   F81+I          0h:00:00   0h:00:00           -4061.2916       -  0.4076\n",
      "    7/88   F81+G4         0h:00:00   0h:00:00           -4028.3514  0.3272       -\n",
      "    8/88   F81+I+G4       0h:00:00   0h:00:00           -4025.2104  0.9493  0.3287\n",
      "    9/88   K80            0h:00:01   0h:00:01           -4271.2813       -       -\n",
      "   10/88   K80+I          0h:00:00   0h:00:01           -4035.7826       -  0.4081\n",
      "   11/88   K80+G4         0h:00:00   0h:00:01           -3994.3796  0.3214       -\n",
      "   12/88   K80+I+G4       0h:00:00   0h:00:01           -3993.0101  0.6027  0.2559\n",
      "   13/88   HKY            0h:00:00   0h:00:01           -4260.2486       -       -\n",
      "   14/88   HKY+I          0h:00:00   0h:00:01           -4023.2500       -  0.4077\n",
      "   15/88   HKY+G4         0h:00:00   0h:00:01           -3981.4090  0.3171       -\n",
      "   16/88   HKY+I+G4       0h:00:00   0h:00:01           -3980.3045  0.7893  0.3145\n",
      "   17/88   TrNef          0h:00:00   0h:00:01           -4253.5255       -       -\n",
      "   18/88   TrNef+I        0h:00:00   0h:00:01           -4030.7624       -  0.4079\n",
      "   19/88   TrNef+G4       0h:00:00   0h:00:01           -3980.0036  0.3086       -\n",
      "   20/88   TrNef+I+G4     0h:00:00   0h:00:01           -3980.6152  0.4272  0.1434\n",
      "   21/88   TrN            0h:00:00   0h:00:01           -4250.5139       -       -\n",
      "   22/88   TrN+I          0h:00:00   0h:00:01           -4022.8038       -  0.4075\n",
      "   23/88   TrN+G4         0h:00:00   0h:00:01           -3976.1083  0.2951       -\n",
      "   24/88   TrN+I+G4       0h:00:01   0h:00:02           -3977.3367  0.6814  0.2806\n",
      "   25/88   TPM2           0h:00:00   0h:00:02           -4263.8611       -       -\n",
      "   26/88   TPM2+I         0h:00:00   0h:00:02           -4032.5689       -  0.4080\n",
      "   27/88   TPM2+G4        0h:00:00   0h:00:02           -3990.3026  0.3225       -\n",
      "   28/88   TPM2+I+G4      0h:00:00   0h:00:02           -3989.5554  0.8009  0.3108\n",
      "   29/88   TPM2uf         0h:00:00   0h:00:02           -4248.9763       -       -\n",
      "   30/88   TPM2uf+I       0h:00:00   0h:00:02           -4019.6960       -  0.4077\n",
      "   31/88   TPM2uf+G4      0h:00:00   0h:00:02           -3977.0402  0.3192       -\n",
      "   32/88   TPM2uf+I+G4    0h:00:00   0h:00:02           -3976.3725  0.7759  0.3081\n",
      "   33/88   TPM3           0h:00:00   0h:00:02           -4267.2177       -       -\n",
      "   34/88   TPM3+I         0h:00:00   0h:00:02           -4033.3163       -  0.4080\n",
      "   35/88   TPM3+G4        0h:00:00   0h:00:02           -3993.5529  0.3233       -\n",
      "   36/88   TPM3+I+G4      0h:00:00   0h:00:02           -3991.9330  0.6382  0.2678\n",
      "   37/88   TPM3uf         0h:00:00   0h:00:02           -4252.8939       -       -\n",
      "   38/88   TPM3uf+I       0h:00:00   0h:00:02           -4019.6297       -  0.4080\n",
      "   39/88   TPM3uf+G4      0h:00:00   0h:00:02           -3980.1077  0.3215       -\n",
      "   40/88   TPM3uf+I+G4    0h:00:00   0h:00:02           -3978.6398  0.8649  0.3285\n",
      "   41/88   TPM1           0h:00:00   0h:00:02           -4271.2772       -       -\n",
      "   42/88   TPM1+I         0h:00:00   0h:00:02           -4035.7748       -  0.4081\n",
      "   43/88   TPM1+G4        0h:00:00   0h:00:03           -3994.2646  0.3210       -\n",
      "   44/88   TPM1+I+G4      0h:00:00   0h:00:03           -3993.2084  0.8012  0.3142\n",
      "   45/88   TPM1uf         0h:00:00   0h:00:03           -4260.2474       -       -\n",
      "   46/88   TPM1uf+I       0h:00:00   0h:00:03           -4023.2427       -  0.4078\n",
      "   47/88   TPM1uf+G4      0h:00:00   0h:00:03           -3981.2730  0.3167       -\n",
      "   48/88   TPM1uf+I+G4    0h:00:00   0h:00:03           -3980.1530  0.7751  0.3109\n",
      "   49/88   TIM2ef         0h:00:00   0h:00:03           -4246.6260       -       -\n",
      "   50/88   TIM2ef+I       0h:00:00   0h:00:03           -4027.4720       -  0.4079\n",
      "   51/88   TIM2ef+G4      0h:00:00   0h:00:03           -3975.9952  0.3086       -\n",
      "   52/88   TIM2ef+I+G4    0h:00:00   0h:00:03           -3976.6213  0.4290  0.1437\n",
      "   53/88   TIM2           0h:00:00   0h:00:03           -4240.0667       -       -\n",
      "   54/88   TIM2+I         0h:00:00   0h:00:03           -4019.0788       -  0.4075\n",
      "   55/88   TIM2+G4        0h:00:00   0h:00:03           -3971.8732  0.2965       -\n",
      "   56/88   TIM2+I+G4      0h:00:00   0h:00:03           -3973.0866  0.6507  0.2676\n",
      "   57/88   TIM3ef         0h:00:00   0h:00:03           -4249.0335       -       -\n",
      "   58/88   TIM3ef+I       0h:00:00   0h:00:03           -4028.2887       -  0.4080\n",
      "   59/88   TIM3ef+G4      0h:00:00   0h:00:03           -3979.2471  0.3107       -\n",
      "   60/88   TIM3ef+I+G4    0h:00:00   0h:00:03           -3979.7349  0.4331  0.1461\n",
      "   61/88   TIM3           0h:00:00   0h:00:03           -4242.4459       -       -\n",
      "   62/88   TIM3+I         0h:00:00   0h:00:03           -4019.2333       -  0.4077\n",
      "   63/88   TIM3+G4        0h:00:01   0h:00:04           -3974.8170  0.2988       -\n",
      "   64/88   TIM3+I+G4      0h:00:00   0h:00:04           -3975.6647  0.7103  0.2849\n",
      "   65/88   TIM1ef         0h:00:00   0h:00:04           -4253.5185       -       -\n",
      "   66/88   TIM1ef+I       0h:00:00   0h:00:04           -4030.7515       -  0.4079\n",
      "   67/88   TIM1ef+G4      0h:00:00   0h:00:04           -3979.8316  0.3080       -\n",
      "   68/88   TIM1ef+I+G4    0h:00:00   0h:00:04           -3980.4838  0.4341  0.1493\n",
      "   69/88   TIM1           0h:00:00   0h:00:04           -4250.5143       -       -\n",
      "   70/88   TIM1+I         0h:00:00   0h:00:04           -4022.7969       -  0.4075\n",
      "   71/88   TIM1+G4        0h:00:00   0h:00:04           -3975.9387  0.2944       -\n",
      "   72/88   TIM1+I+G4      0h:00:00   0h:00:04           -3977.2068  0.6801  0.2805\n",
      "   73/88   TVMef          0h:00:00   0h:00:04           -4259.7514       -       -\n",
      "   74/88   TVMef+I        0h:00:01   0h:00:05           -4030.1206       -  0.4079\n",
      "   75/88   TVMef+G4       0h:00:00   0h:00:05           -3989.1245  0.3229       -\n",
      "   76/88   TVMef+I+G4     0h:00:00   0h:00:05           -3987.8067  0.5729  0.2290\n",
      "   77/88   TVM            0h:00:00   0h:00:05           -4241.5703       -       -\n",
      "   78/88   TVM+I          0h:00:00   0h:00:05           -4016.0778       -  0.4078\n",
      "   79/88   TVM+G4         0h:00:00   0h:00:05           -3975.2183  0.3215       -\n",
      "   80/88   TVM+I+G4       0h:00:00   0h:00:05           -3973.9228  0.5696  0.2289\n",
      "   81/88   SYM            0h:00:00   0h:00:05           -4242.0882       -       -\n",
      "   82/88   SYM+I          0h:00:00   0h:00:05           -4025.0205       -  0.4072\n",
      "   83/88   SYM+G4         0h:00:00   0h:00:05           -3974.9036  0.3096       -\n",
      "   84/88   SYM+I+G4       0h:00:00   0h:00:05           -3975.4735  0.4340  0.1465\n",
      "   85/88   GTR            0h:00:01   0h:00:06           -4231.9747       -       -\n",
      "   86/88   GTR+I          0h:00:00   0h:00:06           -4015.5134       -  0.4085\n",
      "   87/88   GTR+G4         0h:00:00   0h:00:06           -3970.0640  0.2974       -\n",
      "   88/88   GTR+I+G4       0h:00:00   0h:00:06           -3971.3109  0.6963  0.2796\n",
      " ----ID---  ----MODEL---- ---Time--- -Elapsed--- -------LnL------- -Alpha- -P-inv-\n",
      "\n",
      "Computation of likelihood scores completed. It took 0h:00:06\n",
      "\n",
      "BIC       model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  TIM2ef+G4          4     -3975.9952      8189.5303         0.0000    0.6431\n",
      "       2  TrNef+G4           3     -3980.0036      8191.1271         1.5968    0.2894\n",
      "       3  TIM3ef+G4          4     -3979.2471      8196.0340         6.5037    0.0249\n",
      "       4  TIM2ef+I+G4        5     -3976.6213      8197.2025         7.6722    0.0139\n",
      "       5  TIM1ef+G4          4     -3979.8316      8197.2030         7.6727    0.0139\n",
      "       6  TrNef+I+G4         4     -3980.6152      8198.7702         9.2399    0.0063\n",
      "       7  SYM+G4             6     -3974.9036      8200.1869        10.6567    0.0031\n",
      "       8  TIM2+G4            7     -3971.8732      8200.5461        11.0159    0.0026\n",
      "       9  TrN+G4             6     -3976.1083      8202.5964        13.0661    0.0009\n",
      "      10  TIM3ef+I+G4        5     -3979.7349      8203.4295        13.8993    0.0006\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to BIC\n",
      "---------------------------\n",
      "Model:              TIM2ef+G4\n",
      "lnL:                -3975.9952\n",
      "Frequencies:        0.2500 0.2500 0.2500 0.2500\n",
      "Subst. Rates:       1.6312 2.2519 1.6312 1.0000 5.9828 1.0000 \n",
      "Inv. sites prop:    -\n",
      "Gamma shape:        0.3086\n",
      "Score:              8189.5303\n",
      "Weight:             0.6431\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.9787\n",
      "Gamma-Inv:          0.0213\n",
      "Frequencies:        0.0044\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.4078\n",
      "Alpha:              0.3086\n",
      "Alpha-P.Inv:        0.4300\n",
      "P.Inv-Alpha:        0.1444\n",
      "Frequencies:        0.2240 0.2797 0.2324 0.2639 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m 010232 -f 0.25,0.25,0.25,0.25 -v 0 -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m GTRGAMMA -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta --model TIM2ef+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m TIM2ef+G4\n",
      "\n",
      "AIC       model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  TIM2+G4            7     -3971.8732      8023.7463         0.0000    0.3574\n",
      "       2  GTR+G4             9     -3970.0640      8024.1280         0.3817    0.2953\n",
      "       3  TIM2ef+G4          4     -3975.9952      8025.9904         2.2441    0.1164\n",
      "       4  SYM+G4             6     -3974.9036      8027.8071         4.0608    0.0469\n",
      "       5  TIM2+I+G4          8     -3973.0866      8028.1732         4.4269    0.0391\n",
      "       6  GTR+I+G4          10     -3971.3109      8028.6218         4.8754    0.0312\n",
      "       7  TIM2ef+I+G4        5     -3976.6213      8029.2427         5.4964    0.0229\n",
      "       8  TIM3+G4            7     -3974.8170      8029.6340         5.8877    0.0188\n",
      "       9  TrN+G4             6     -3976.1083      8030.2166         6.4703    0.0141\n",
      "      10  SYM+I+G4           7     -3975.4735      8030.9471         7.2007    0.0098\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to AIC\n",
      "---------------------------\n",
      "Model:              TIM2+G4\n",
      "lnL:                -3971.8732\n",
      "Frequencies:        0.2218 0.2777 0.2381 0.2624\n",
      "Subst. Rates:       1.8196 2.7608 1.8196 1.0000 5.5686 1.0000 \n",
      "Inv. sites prop:    -\n",
      "Gamma shape:        0.2965\n",
      "Score:              8023.7463\n",
      "Weight:             0.3574\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.8784\n",
      "Gamma-Inv:          0.1216\n",
      "Frequencies:        0.7887\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.4080\n",
      "Alpha:              0.2996\n",
      "Alpha-P.Inv:        0.6015\n",
      "P.Inv-Alpha:        0.2353\n",
      "Frequencies:        0.2214 0.2815 0.2372 0.2599 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m 010232 -f m -v 0 -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m GTRGAMMAX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta --model TIM2+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m TIM2+G4\n",
      "\n",
      "AICc      model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  TIM2+G4            7     -3971.8732      8028.7463         0.0000    0.3822\n",
      "       2  TIM2ef+G4          4     -3975.9952      8029.9904         1.2441    0.2052\n",
      "       3  GTR+G4             9     -3970.0640      8030.1280         1.3817    0.1915\n",
      "       4  SYM+G4             6     -3974.9036      8032.8071         4.0608    0.0502\n",
      "       5  TIM2+I+G4          8     -3973.0866      8034.1732         5.4269    0.0253\n",
      "       6  TIM2ef+I+G4        5     -3976.6213      8034.2427         5.4964    0.0245\n",
      "       7  GTR+I+G4          10     -3971.3109      8034.6218         5.8754    0.0203\n",
      "       8  TIM3+G4            7     -3974.8170      8034.6340         5.8877    0.0201\n",
      "       9  TrN+G4             6     -3976.1083      8035.2166         6.4703    0.0150\n",
      "      10  SYM+I+G4           7     -3975.4735      8035.9471         7.2007    0.0104\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to AICc\n",
      "---------------------------\n",
      "Model:              TIM2+G4\n",
      "lnL:                -3971.8732\n",
      "Frequencies:        0.2218 0.2777 0.2381 0.2624\n",
      "Subst. Rates:       1.8196 2.7608 1.8196 1.0000 5.5686 1.0000 \n",
      "Inv. sites prop:    -\n",
      "Gamma shape:        0.2965\n",
      "Score:              8028.7463\n",
      "Weight:             0.3822\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.9029\n",
      "Gamma-Inv:          0.0971\n",
      "Frequencies:        0.6836\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.4079\n",
      "Alpha:              0.3007\n",
      "Alpha-P.Inv:        0.5781\n",
      "P.Inv-Alpha:        0.2225\n",
      "Frequencies:        0.2218 0.2807 0.2370 0.2606 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m 010232 -f m -v 0 -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m GTRGAMMAX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta --model TIM2+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m TIM2+G4\n",
      "Summary:\n",
      "\n",
      "Partition 1/1:\n",
      "                         Model         Score        Weight\n",
      "----------------------------------------------------------\n",
      "       BIC           TIM2ef+G4     8189.5303        0.6431\n",
      "       AIC             TIM2+G4     8023.7463        0.3574\n",
      "      AICc             TIM2+G4     8028.7463        0.3822\n",
      "\n",
      "Execution results written to /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/1.out\n",
      "Starting tree written to /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/1.tree\n",
      "                             _      _ _            _      _   _  _____ \n",
      "                            | |    | | |          | |    | \\ | |/ ____|\n",
      "         _ __ ___   ___   __| | ___| | |_ ___  ___| |_   |  \\| | |  __ \n",
      "        | '_ ` _ \\ / _ \\ / _` |/ _ \\ | __/ _ \\/ __| __|  | . ` | | |_ |\n",
      "        | | | | | | (_) | (_| |  __/ | ||  __/\\__ \\ |_   | |\\  | |__| |\n",
      "        |_| |_| |_|\\___/ \\__,_|\\___|_|\\__\\___||___/\\__|  |_| \\_|\\_____|\n",
      "--------------------------------------------------------------------------------\n",
      "ModelTest-NG v0.1.7 released on 17.03.2021 by The Exelixis Lab.\n",
      "Written by Diego Darriba.\n",
      "Contributors: Tomas Flouri, Alexey Kozlov, Benoit Morel, David Posada, \n",
      "              Alexandros Stamatakis.\n",
      "Latest version: https://github.com/ddarriba/modeltest\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Physical cores: 16\n",
      "Logical cores:  32\n",
      "Memory:         126GB\n",
      "Extensions:     AVX\n",
      "\n",
      "WARNING: Partition DATA: Sequences Leptographium_procerum_CMW_34542 and Leptographium_procerum_CMW_29994 are identical\n",
      "WARNING: Partition DATA: Sequences Ophiostoma_floccosum_CMW12623 and Ophiostoma_floccosum_CMW12622 are identical\n",
      "WARNING: MSA has not enough sites to infer reliable results\n",
      "Creating new checkpoint file: /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb1.ckp\n",
      "--------------------------------------------------------------------------------\n",
      "ModelTest-NG v0.1.7\n",
      "\n",
      "Input data:\n",
      "  MSA:        /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta\n",
      "  Tree:       Maximum parsimony\n",
      "    file:           -\n",
      "  #taxa:            18\n",
      "  #sites:           614\n",
      "  #patterns:        379\n",
      "  Max. thread mem:  3 MB\n",
      "\n",
      "Output:\n",
      "  Log:           /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb1.log\n",
      "  Starting tree: /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb1.tree\n",
      "  Results:       /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb1.out\n",
      "\n",
      "Selection options:\n",
      "  # dna schemes:      3\n",
      "  # dna models:       24\n",
      "  include model parameters:\n",
      "    Uniform:         true\n",
      "    p-inv (+I):      true\n",
      "    gamma (+G):      true\n",
      "    both (+I+G):     true\n",
      "    free rates (+R): false\n",
      "    fixed freqs:     true\n",
      "    estimated freqs: true\n",
      "    #categories:     4\n",
      "  gamma rates mode:   mean\n",
      "  asc bias:           none\n",
      "  epsilon (opt):      0.01\n",
      "  epsilon (par):      0.05\n",
      "  keep branches:      false\n",
      "\n",
      "Additional options:\n",
      "  verbosity:        very low\n",
      "  threads:          1/16\n",
      "  RNG seed:         12345\n",
      "  subtree repeats:  enabled\n",
      "--------------------------------------------------------------------------------\n",
      "modeltest-ng was called as follows: \n",
      ">> modeltest-ng -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -d nt -o /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb1 -T mrbayes \n",
      "\n",
      "\n",
      "modeltest-ng: Warning: You are using one single thread out of 16 physical cores.\n",
      "modeltest-ng:          You can set the number of threads with -p argument.\n",
      "modeltest-ng: Try 'modeltest-ng --help' or 'modeltest-ng --usage' for more information\n",
      "\n",
      "Partition 1/1\n",
      "\n",
      "    1/24   JC             0h:00:00   0h:00:00           -4312.2330       -       -\n",
      "    2/24   JC+I           0h:00:00   0h:00:00           -4081.7552       -  0.4079\n",
      "    3/24   JC+G4          0h:00:00   0h:00:00           -4049.3810  0.3559       -\n",
      "    4/24   JC+I+G4        0h:00:00   0h:00:00           -4047.0958  1.0000  0.3283\n",
      "    5/24   F81            0h:00:00   0h:00:00           -4300.0637       -       -\n",
      "    6/24   F81+I          0h:00:00   0h:00:00           -4061.2916       -  0.4076\n",
      "    7/24   F81+G4         0h:00:00   0h:00:00           -4028.3514  0.3272       -\n",
      "    8/24   F81+I+G4       0h:00:00   0h:00:00           -4025.2104  0.9493  0.3287\n",
      "    9/24   K80            0h:00:00   0h:00:00           -4271.2813       -       -\n",
      "   10/24   K80+I          0h:00:00   0h:00:00           -4035.7826       -  0.4081\n",
      "   11/24   K80+G4         0h:00:00   0h:00:00           -3994.3796  0.3214       -\n",
      "   12/24   K80+I+G4       0h:00:00   0h:00:00           -3993.0101  0.6027  0.2559\n",
      "   13/24   HKY            0h:00:00   0h:00:00           -4260.2486       -       -\n",
      "   14/24   HKY+I          0h:00:00   0h:00:00           -4023.2500       -  0.4077\n",
      "   15/24   HKY+G4         0h:00:00   0h:00:00           -3981.4090  0.3171       -\n",
      "   16/24   HKY+I+G4       0h:00:01   0h:00:01           -3980.3045  0.7893  0.3145\n",
      "   17/24   SYM            0h:00:00   0h:00:01           -4242.0882       -       -\n",
      "   18/24   SYM+I          0h:00:00   0h:00:01           -4025.0205       -  0.4072\n",
      "   19/24   SYM+G4         0h:00:00   0h:00:01           -3974.9036  0.3096       -\n",
      "   20/24   SYM+I+G4       0h:00:00   0h:00:01           -3975.4735  0.4340  0.1465\n",
      "   21/24   GTR            0h:00:00   0h:00:01           -4231.9747       -       -\n",
      "   22/24   GTR+I          0h:00:00   0h:00:01           -4015.5134       -  0.4085\n",
      "   23/24   GTR+G4         0h:00:00   0h:00:01           -3970.0640  0.2974       -\n",
      "   24/24   GTR+I+G4       0h:00:00   0h:00:01           -3971.3109  0.6963  0.2796\n",
      " ----ID---  ----MODEL---- ---Time--- -Elapsed--- -------LnL------- -Alpha- -P-inv-\n",
      "\n",
      "Computation of likelihood scores completed. It took 0h:00:01\n",
      "\n",
      "BIC       model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  SYM+G4             6     -3974.9036      8200.1869         0.0000    0.9308\n",
      "       2  HKY+G4             5     -3981.4090      8206.7779         6.5909    0.0345\n",
      "       3  SYM+I+G4           7     -3975.4735      8207.7469         7.5599    0.0212\n",
      "       4  GTR+G4             9     -3970.0640      8209.7678         9.5809    0.0077\n",
      "       5  HKY+I+G4           6     -3980.3045      8210.9888        10.8018    0.0042\n",
      "       6  K80+G4             2     -3994.3796      8213.4590        13.2720    0.0012\n",
      "       7  K80+I+G4           3     -3993.0101      8217.1401        16.9532    0.0002\n",
      "       8  GTR+I+G4          10     -3971.3109      8218.6816        18.4946    0.0001\n",
      "       9  HKY+I              5     -4023.2500      8290.4599        90.2729    0.0000\n",
      "      10  F81+G4             4     -4028.3514      8294.2427        94.0557    0.0000\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to BIC\n",
      "---------------------------\n",
      "Model:              SYM+G4\n",
      "lnL:                -3974.9036\n",
      "Frequencies:        0.2500 0.2500 0.2500 0.2500\n",
      "Subst. Rates:       1.2248 2.1263 1.9043 0.8809 5.6451 1.0000 \n",
      "Inv. sites prop:    -\n",
      "Gamma shape:        0.3096\n",
      "Score:              8200.1869\n",
      "Weight:             0.9308\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.9743\n",
      "Gamma-Inv:          0.0257\n",
      "Frequencies:        0.0465\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.4078\n",
      "Alpha:              0.3098\n",
      "Alpha-P.Inv:        0.4941\n",
      "P.Inv-Alpha:        0.1752\n",
      "Frequencies:        0.2215 0.2895 0.2163 0.2727 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m 012345 -f 0.25,0.25,0.25,0.25 -v 0 -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m GTRGAMMA -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta --model SYM+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m SYM+G4\n",
      "\n",
      "AIC       model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  GTR+G4             9     -3970.0640      8024.1280         0.0000    0.7698\n",
      "       2  SYM+G4             6     -3974.9036      8027.8071         3.6791    0.1223\n",
      "       3  GTR+I+G4          10     -3971.3109      8028.6218         4.4937    0.0814\n",
      "       4  SYM+I+G4           7     -3975.4735      8030.9471         6.8190    0.0254\n",
      "       5  HKY+I+G4           6     -3980.3045      8038.6090        14.4809    0.0006\n",
      "       6  HKY+G4             5     -3981.4090      8038.8181        14.6900    0.0005\n",
      "       7  K80+I+G4           3     -3993.0101      8058.0203        33.8923    0.0000\n",
      "       8  K80+G4             2     -3994.3796      8058.7591        34.6311    0.0000\n",
      "       9  GTR+I              9     -4015.5134      8115.0268        90.8988    0.0000\n",
      "      10  HKY+I              5     -4023.2500      8122.5000        98.3720    0.0000\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to AIC\n",
      "---------------------------\n",
      "Model:              GTR+G4\n",
      "lnL:                -3970.0640\n",
      "Frequencies:        0.2209 0.2843 0.2391 0.2557\n",
      "Subst. Rates:       1.2249 2.4914 2.1918 0.7868 5.0310 1.0000 \n",
      "Inv. sites prop:    -\n",
      "Gamma shape:        0.2974\n",
      "Score:              8024.1280\n",
      "Weight:             0.7698\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.8926\n",
      "Gamma-Inv:          0.1074\n",
      "Frequencies:        0.8522\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.4085\n",
      "Alpha:              0.2991\n",
      "Alpha-P.Inv:        0.6346\n",
      "P.Inv-Alpha:        0.2482\n",
      "Frequencies:        0.2205 0.2847 0.2388 0.2559 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m 012345 -f m -v 0 -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m GTRGAMMAX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta --model GTR+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m GTR+G4\n",
      "\n",
      "AICc      model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  GTR+G4             9     -3970.0640      8030.1280         0.0000    0.7020\n",
      "       2  SYM+G4             6     -3974.9036      8032.8071         2.6791    0.1839\n",
      "       3  GTR+I+G4          10     -3971.3109      8034.6218         4.4937    0.0742\n",
      "       4  SYM+I+G4           7     -3975.4735      8035.9471         5.8190    0.0383\n",
      "       5  HKY+I+G4           6     -3980.3045      8043.6090        13.4809    0.0008\n",
      "       6  HKY+G4             5     -3981.4090      8043.8181        13.6900    0.0007\n",
      "       7  K80+I+G4           3     -3993.0101      8062.0203        31.8923    0.0000\n",
      "       8  K80+G4             2     -3994.3796      8062.7591        32.6311    0.0000\n",
      "       9  GTR+I              9     -4015.5134      8121.0268        90.8988    0.0000\n",
      "      10  HKY+I              5     -4023.2500      8127.5000        97.3720    0.0000\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to AICc\n",
      "---------------------------\n",
      "Model:              GTR+G4\n",
      "lnL:                -3970.0640\n",
      "Frequencies:        0.2209 0.2843 0.2391 0.2557\n",
      "Subst. Rates:       1.2249 2.4914 2.1918 0.7868 5.0310 1.0000 \n",
      "Inv. sites prop:    -\n",
      "Gamma shape:        0.2974\n",
      "Score:              8030.1280\n",
      "Weight:             0.7020\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.8867\n",
      "Gamma-Inv:          0.1133\n",
      "Frequencies:        0.7778\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.4084\n",
      "Alpha:              0.2999\n",
      "Alpha-P.Inv:        0.6084\n",
      "P.Inv-Alpha:        0.2349\n",
      "Frequencies:        0.2205 0.2847 0.2388 0.2560 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m 012345 -f m -v 0 -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m GTRGAMMAX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta --model GTR+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_Btub_sequences_trimmed.fasta -m GTR+G4\n",
      "Summary:\n",
      "\n",
      "Partition 1/1:\n",
      "                         Model         Score        Weight\n",
      "----------------------------------------------------------\n",
      "       BIC              SYM+G4     8200.1869        0.9308\n",
      "       AIC              GTR+G4     8024.1280        0.7698\n",
      "      AICc              GTR+G4     8030.1280        0.7020\n",
      "\n",
      "Execution results written to /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb1.out\n",
      "Starting tree written to /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb1.tree\n",
      "                             _      _ _            _      _   _  _____ \n",
      "                            | |    | | |          | |    | \\ | |/ ____|\n",
      "         _ __ ___   ___   __| | ___| | |_ ___  ___| |_   |  \\| | |  __ \n",
      "        | '_ ` _ \\ / _ \\ / _` |/ _ \\ | __/ _ \\/ __| __|  | . ` | | |_ |\n",
      "        | | | | | | (_) | (_| |  __/ | ||  __/\\__ \\ |_   | |\\  | |__| |\n",
      "        |_| |_| |_|\\___/ \\__,_|\\___|_|\\__\\___||___/\\__|  |_| \\_|\\_____|\n",
      "--------------------------------------------------------------------------------\n",
      "ModelTest-NG v0.1.7 released on 17.03.2021 by The Exelixis Lab.\n",
      "Written by Diego Darriba.\n",
      "Contributors: Tomas Flouri, Alexey Kozlov, Benoit Morel, David Posada, \n",
      "              Alexandros Stamatakis.\n",
      "Latest version: https://github.com/ddarriba/modeltest\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Physical cores: 16\n",
      "Logical cores:  32\n",
      "Memory:         126GB\n",
      "Extensions:     AVX\n",
      "\n",
      "WARNING: MSA has not enough sites to infer reliable results\n",
      "Creating new checkpoint file: /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/2.ckp\n",
      "--------------------------------------------------------------------------------\n",
      "ModelTest-NG v0.1.7\n",
      "\n",
      "Input data:\n",
      "  MSA:        /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta\n",
      "  Tree:       Maximum parsimony\n",
      "    file:           -\n",
      "  #taxa:            16\n",
      "  #sites:           1214\n",
      "  #patterns:        490\n",
      "  Max. thread mem:  3 MB\n",
      "\n",
      "Output:\n",
      "  Log:           /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/2.log\n",
      "  Starting tree: /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/2.tree\n",
      "  Results:       /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/2.out\n",
      "\n",
      "Selection options:\n",
      "  # dna schemes:      11\n",
      "  # dna models:       88\n",
      "  include model parameters:\n",
      "    Uniform:         true\n",
      "    p-inv (+I):      true\n",
      "    gamma (+G):      true\n",
      "    both (+I+G):     true\n",
      "    free rates (+R): false\n",
      "    fixed freqs:     true\n",
      "    estimated freqs: true\n",
      "    #categories:     4\n",
      "  gamma rates mode:   mean\n",
      "  asc bias:           none\n",
      "  epsilon (opt):      0.01\n",
      "  epsilon (par):      0.05\n",
      "  keep branches:      false\n",
      "\n",
      "Additional options:\n",
      "  verbosity:        very low\n",
      "  threads:          1/16\n",
      "  RNG seed:         12345\n",
      "  subtree repeats:  enabled\n",
      "--------------------------------------------------------------------------------\n",
      "modeltest-ng was called as follows: \n",
      ">> modeltest-ng -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -d nt -o /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/2 \n",
      "\n",
      "\n",
      "modeltest-ng: Warning: You are using one single thread out of 16 physical cores.\n",
      "modeltest-ng:          You can set the number of threads with -p argument.\n",
      "modeltest-ng: Try 'modeltest-ng --help' or 'modeltest-ng --usage' for more information\n",
      "\n",
      "Partition 1/1\n",
      "\n",
      "    1/88   JC             0h:00:00   0h:00:00           -7709.2923       -       -\n",
      "    2/88   JC+I           0h:00:00   0h:00:00           -7374.6111       -  0.4949\n",
      "    3/88   JC+G4          0h:00:00   0h:00:00           -7303.4068  0.3035       -\n",
      "    4/88   JC+I+G4        0h:00:00   0h:00:00           -7305.1619  0.4314  0.1639\n",
      "    5/88   F81            0h:00:00   0h:00:00           -7701.3212       -       -\n",
      "    6/88   F81+I          0h:00:01   0h:00:01           -7355.4103       -  0.4984\n",
      "    7/88   F81+G4         0h:00:00   0h:00:01           -7276.0305  0.2772       -\n",
      "    8/88   F81+I+G4       0h:00:00   0h:00:01           -7278.5616  0.4173  0.1734\n",
      "    9/88   K80            0h:00:00   0h:00:01           -7560.6435       -       -\n",
      "   10/88   K80+I          0h:00:00   0h:00:01           -7209.2987       -  0.4978\n",
      "   11/88   K80+G4         0h:00:00   0h:00:01           -7135.6623  0.2945       -\n",
      "   12/88   K80+I+G4       0h:00:00   0h:00:01           -7136.2787  0.5448  0.2740\n",
      "   13/88   HKY            0h:00:00   0h:00:01           -7545.3516       -       -\n",
      "   14/88   HKY+I          0h:00:00   0h:00:01           -7167.8016       -  0.5015\n",
      "   15/88   HKY+G4         0h:00:00   0h:00:01           -7078.4160  0.2536       -\n",
      "   16/88   HKY+I+G4       0h:00:00   0h:00:01           -7078.8883  0.4339  0.2374\n",
      "   17/88   TrNef          0h:00:00   0h:00:01           -7525.7859       -       -\n",
      "   18/88   TrNef+I        0h:00:00   0h:00:01           -7182.5177       -  0.4936\n",
      "   19/88   TrNef+G4       0h:00:00   0h:00:01           -7115.8265  0.3050       -\n",
      "   20/88   TrNef+I+G4     0h:00:00   0h:00:01           -7115.0487  0.6361  0.3063\n",
      "   21/88   TrN            0h:00:00   0h:00:02           -7509.3678       -       -\n",
      "   22/88   TrN+I          0h:00:00   0h:00:02           -7138.7964       -  0.4992\n",
      "   23/88   TrN+G4         0h:00:00   0h:00:02           -7057.2938  0.2721       -\n",
      "   24/88   TrN+I+G4       0h:00:00   0h:00:02           -7055.3211  0.6065  0.3378\n",
      "   25/88   TPM2           0h:00:00   0h:00:02           -7535.6849       -       -\n",
      "   26/88   TPM2+I         0h:00:00   0h:00:02           -7194.2733       -  0.4956\n",
      "   27/88   TPM2+G4        0h:00:00   0h:00:02           -7123.0904  0.3019       -\n",
      "   28/88   TPM2+I+G4      0h:00:00   0h:00:02           -7123.5133  0.5665  0.2758\n",
      "   29/88   TPM2uf         0h:00:00   0h:00:02           -7525.6889       -       -\n",
      "   30/88   TPM2uf+I       0h:00:00   0h:00:02           -7161.9982       -  0.5022\n",
      "   31/88   TPM2uf+G4      0h:00:00   0h:00:02           -7077.2604  0.2584       -\n",
      "   32/88   TPM2uf+I+G4    0h:00:00   0h:00:02           -7078.6566  0.5460  0.3153\n",
      "   33/88   TPM3           0h:00:00   0h:00:02           -7519.6839       -       -\n",
      "   34/88   TPM3+I         0h:00:00   0h:00:02           -7184.0663       -  0.4926\n",
      "   35/88   TPM3+G4        0h:00:01   0h:00:03           -7116.4001  0.3094       -\n",
      "   36/88   TPM3+I+G4      0h:00:00   0h:00:03           -7117.0334  0.6417  0.3059\n",
      "   37/88   TPM3uf         0h:00:00   0h:00:03           -7509.8187       -       -\n",
      "   38/88   TPM3uf+I       0h:00:00   0h:00:03           -7153.6554       -  0.4993\n",
      "   39/88   TPM3uf+G4      0h:00:00   0h:00:03           -7073.0280  0.2494       -\n",
      "   40/88   TPM3uf+I+G4    0h:00:00   0h:00:03           -7074.3024  0.5607  0.3178\n",
      "   41/88   TPM1           0h:00:00   0h:00:03           -7553.5433       -       -\n",
      "   42/88   TPM1+I         0h:00:00   0h:00:03           -7200.1528       -  0.4981\n",
      "   43/88   TPM1+G4        0h:00:00   0h:00:03           -7128.4641  0.2945       -\n",
      "   44/88   TPM1+I+G4      0h:00:00   0h:00:03           -7129.3459  0.5528  0.2754\n",
      "   45/88   TPM1uf         0h:00:00   0h:00:03           -7539.8755       -       -\n",
      "   46/88   TPM1uf+I       0h:00:00   0h:00:03           -7164.3111       -  0.5025\n",
      "   47/88   TPM1uf+G4      0h:00:00   0h:00:03           -7077.5331  0.2540       -\n",
      "   48/88   TPM1uf+I+G4    0h:00:00   0h:00:03           -7079.1347  0.5414  0.3153\n",
      "   49/88   TIM2ef         0h:00:00   0h:00:03           -7501.6002       -       -\n",
      "   50/88   TIM2ef+I       0h:00:00   0h:00:03           -7167.2529       -  0.4899\n",
      "   51/88   TIM2ef+G4      0h:00:00   0h:00:03           -7104.6192  0.3190       -\n",
      "   52/88   TIM2ef+I+G4    0h:00:00   0h:00:03           -7103.5132  0.6548  0.3047\n",
      "   53/88   TIM2           0h:00:00   0h:00:03           -7488.5412       -       -\n",
      "   54/88   TIM2+I         0h:00:01   0h:00:04           -7130.1775       -  0.4967\n",
      "   55/88   TIM2+G4        0h:00:00   0h:00:04           -7054.4713  0.2711       -\n",
      "   56/88   TIM2+I+G4      0h:00:00   0h:00:04           -7052.6489  0.6382  0.3418\n",
      "   57/88   TIM3ef         0h:00:00   0h:00:04           -7483.2542       -       -\n",
      "   58/88   TIM3ef+I       0h:00:00   0h:00:04           -7157.0638       -  0.4864\n",
      "   59/88   TIM3ef+G4      0h:00:00   0h:00:04           -7094.7813  0.3212       -\n",
      "   60/88   TIM3ef+I+G4    0h:00:00   0h:00:04           -7093.6904  0.7072  0.3165\n",
      "   61/88   TIM3           0h:00:00   0h:00:04           -7475.7648       -       -\n",
      "   62/88   TIM3+I         0h:00:00   0h:00:04           -7127.7222       -  0.4936\n",
      "   63/88   TIM3+G4        0h:00:00   0h:00:04           -7054.2603  0.2797       -\n",
      "   64/88   TIM3+I+G4      0h:00:00   0h:00:04           -7052.9189  0.6447  0.3408\n",
      "   65/88   TIM1ef         0h:00:00   0h:00:04           -7518.6889       -       -\n",
      "   66/88   TIM1ef+I       0h:00:00   0h:00:04           -7173.5164       -  0.4937\n",
      "   67/88   TIM1ef+G4      0h:00:00   0h:00:04           -7109.0036  0.3051       -\n",
      "   68/88   TIM1ef+I+G4    0h:00:00   0h:00:04           -7108.5474  0.6504  0.3089\n",
      "   69/88   TIM1           0h:00:01   0h:00:05           -7503.8446       -       -\n",
      "   70/88   TIM1+I         0h:00:00   0h:00:05           -7135.2708       -  0.4990\n",
      "   71/88   TIM1+G4        0h:00:00   0h:00:05           -7056.1122  0.2623       -\n",
      "   72/88   TIM1+I+G4      0h:00:00   0h:00:05           -7054.7849  0.6160  0.3381\n",
      "   73/88   TVMef          0h:00:00   0h:00:05           -7494.2682       -       -\n",
      "   74/88   TVMef+I        0h:00:00   0h:00:05           -7167.1919       -  0.4886\n",
      "   75/88   TVMef+G4       0h:00:00   0h:00:05           -7102.0937  0.3176       -\n",
      "   76/88   TVMef+I+G4     0h:00:00   0h:00:05           -7102.4590  0.6509  0.3008\n",
      "   77/88   TVM            0h:00:00   0h:00:05           -7490.1382       -       -\n",
      "   78/88   TVM+I          0h:00:00   0h:00:05           -7147.3213       -  0.4965\n",
      "   79/88   TVM+G4         0h:00:00   0h:00:05           -7071.5426  0.2555       -\n",
      "   80/88   TVM+I+G4       0h:00:00   0h:00:05           -7072.7949  0.5833  0.3201\n",
      "   81/88   SYM            0h:00:00   0h:00:05           -7458.5917       -       -\n",
      "   82/88   SYM+I          0h:00:00   0h:00:05           -7140.0015       -  0.4823\n",
      "   83/88   SYM+G4         0h:00:00   0h:00:05           -7081.6920  0.3311       -\n",
      "   84/88   SYM+I+G4       0h:00:00   0h:00:05           -7080.8509  0.8229  0.3447\n",
      "   85/88   GTR            0h:00:00   0h:00:05           -7454.9183       -       -\n",
      "   86/88   GTR+I          0h:00:00   0h:00:05           -7118.7305       -  0.4909\n",
      "   87/88   GTR+G4         0h:00:00   0h:00:05           -7051.0601  0.2770       -\n",
      "   88/88   GTR+I+G4       0h:00:01   0h:00:06           -7050.1195  0.7262  0.3617\n",
      " ----ID---  ----MODEL---- ---Time--- -Elapsed--- -------LnL------- -Alpha- -P-inv-\n",
      "\n",
      "Computation of likelihood scores completed. It took 0h:00:06\n",
      "\n",
      "BIC       model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  TrN+G4             6     -7057.2938     14363.1463         0.0000    0.3917\n",
      "       2  TIM3+G4            7     -7054.2603     14364.1810         1.0347    0.2335\n",
      "       3  TIM2+G4            7     -7054.4713     14364.6029         1.4566    0.1891\n",
      "       4  TrN+I+G4           7     -7055.3211     14366.3026         3.1563    0.0808\n",
      "       5  TIM1+G4            7     -7056.1122     14367.8847         4.7384    0.0366\n",
      "       6  TIM2+I+G4          8     -7052.6489     14368.0599         4.9136    0.0336\n",
      "       7  TIM3+I+G4          8     -7052.9189     14368.5999         5.4536    0.0256\n",
      "       8  GTR+G4             9     -7051.0601     14371.9840         8.8377    0.0047\n",
      "       9  TIM1+I+G4          8     -7054.7849     14372.3318         9.1856    0.0040\n",
      "      10  GTR+I+G4          10     -7050.1195     14377.2044        14.0582    0.0003\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to BIC\n",
      "---------------------------\n",
      "Model:              TrN+G4\n",
      "lnL:                -7057.2938\n",
      "Frequencies:        0.2146 0.3049 0.3003 0.1802\n",
      "Subst. Rates:       1.0000 2.6038 1.0000 1.0000 6.8042 1.0000 \n",
      "Inv. sites prop:    -\n",
      "Gamma shape:        0.2721\n",
      "Score:              14363.1463\n",
      "Weight:             0.3917\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.8557\n",
      "Gamma-Inv:          0.1443\n",
      "Frequencies:        1.0000\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.4913\n",
      "Alpha:              0.2736\n",
      "Alpha-P.Inv:        0.6212\n",
      "P.Inv-Alpha:        0.3393\n",
      "Frequencies:        0.2170 0.3026 0.2983 0.1821 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m 010020 -f m -v 0 -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m GTRGAMMAX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta --model TrN+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m TrN+G4\n",
      "\n",
      "AIC       model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  GTR+G4             9     -7051.0601     14178.1203         0.0000    0.2686\n",
      "       2  GTR+I+G4          10     -7050.1195     14178.2391         0.1188    0.2531\n",
      "       3  TIM2+I+G4          8     -7052.6489     14179.2978         1.1776    0.1491\n",
      "       4  TIM3+I+G4          8     -7052.9189     14179.8378         1.7176    0.1138\n",
      "       5  TIM3+G4            7     -7054.2603     14180.5206         2.4003    0.0809\n",
      "       6  TIM2+G4            7     -7054.4713     14180.9426         2.8223    0.0655\n",
      "       7  TrN+I+G4           7     -7055.3211     14182.6423         4.5220    0.0280\n",
      "       8  TIM1+I+G4          8     -7054.7849     14183.5698         5.4495    0.0176\n",
      "       9  TIM1+G4            7     -7056.1122     14184.2243         6.1041    0.0127\n",
      "      10  TrN+G4             6     -7057.2938     14184.5876         6.4673    0.0106\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to AIC\n",
      "---------------------------\n",
      "Model:              GTR+G4\n",
      "lnL:                -7051.0601\n",
      "Frequencies:        0.2252 0.2959 0.2900 0.1888\n",
      "Subst. Rates:       1.1532 3.1215 0.6396 1.6458 7.8660 1.0000 \n",
      "Inv. sites prop:    -\n",
      "Gamma shape:        0.2770\n",
      "Score:              14178.1203\n",
      "Weight:             0.2686\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.4383\n",
      "Gamma-Inv:          0.5617\n",
      "Frequencies:        1.0000\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.4909\n",
      "Alpha:              0.2761\n",
      "Alpha-P.Inv:        0.6769\n",
      "P.Inv-Alpha:        0.3502\n",
      "Frequencies:        0.2230 0.2982 0.2940 0.1848 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m 012345 -f m -v 0 -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m GTRGAMMAX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta --model GTR+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m GTR+G4\n",
      "\n",
      "AICc      model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  GTR+G4             9     -7051.0601     14180.1203         0.0000    0.2686\n",
      "       2  GTR+I+G4          10     -7050.1195     14180.2391         0.1188    0.2531\n",
      "       3  TIM2+I+G4          8     -7052.6489     14181.2978         1.1776    0.1491\n",
      "       4  TIM3+I+G4          8     -7052.9189     14181.8378         1.7176    0.1138\n",
      "       5  TIM3+G4            7     -7054.2603     14182.5206         2.4003    0.0809\n",
      "       6  TIM2+G4            7     -7054.4713     14182.9426         2.8223    0.0655\n",
      "       7  TrN+I+G4           7     -7055.3211     14184.6423         4.5220    0.0280\n",
      "       8  TIM1+I+G4          8     -7054.7849     14185.5698         5.4495    0.0176\n",
      "       9  TIM1+G4            7     -7056.1122     14186.2243         6.1041    0.0127\n",
      "      10  TrN+G4             6     -7057.2938     14186.5876         6.4673    0.0106\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to AICc\n",
      "---------------------------\n",
      "Model:              GTR+G4\n",
      "lnL:                -7051.0601\n",
      "Frequencies:        0.2252 0.2959 0.2900 0.1888\n",
      "Subst. Rates:       1.1532 3.1215 0.6396 1.6458 7.8660 1.0000 \n",
      "Inv. sites prop:    -\n",
      "Gamma shape:        0.2770\n",
      "Score:              14180.1203\n",
      "Weight:             0.2686\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.4383\n",
      "Gamma-Inv:          0.5617\n",
      "Frequencies:        1.0000\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.4909\n",
      "Alpha:              0.2761\n",
      "Alpha-P.Inv:        0.6769\n",
      "P.Inv-Alpha:        0.3502\n",
      "Frequencies:        0.2230 0.2982 0.2940 0.1848 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m 012345 -f m -v 0 -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m GTRGAMMAX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta --model GTR+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m GTR+G4\n",
      "Summary:\n",
      "\n",
      "Partition 1/1:\n",
      "                         Model         Score        Weight\n",
      "----------------------------------------------------------\n",
      "       BIC              TrN+G4    14363.1463        0.3917\n",
      "       AIC              GTR+G4    14178.1203        0.2686\n",
      "      AICc              GTR+G4    14180.1203        0.2686\n",
      "\n",
      "Execution results written to /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/2.out\n",
      "Starting tree written to /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/2.tree\n",
      "                             _      _ _            _      _   _  _____ \n",
      "                            | |    | | |          | |    | \\ | |/ ____|\n",
      "         _ __ ___   ___   __| | ___| | |_ ___  ___| |_   |  \\| | |  __ \n",
      "        | '_ ` _ \\ / _ \\ / _` |/ _ \\ | __/ _ \\/ __| __|  | . ` | | |_ |\n",
      "        | | | | | | (_) | (_| |  __/ | ||  __/\\__ \\ |_   | |\\  | |__| |\n",
      "        |_| |_| |_|\\___/ \\__,_|\\___|_|\\__\\___||___/\\__|  |_| \\_|\\_____|\n",
      "--------------------------------------------------------------------------------\n",
      "ModelTest-NG v0.1.7 released on 17.03.2021 by The Exelixis Lab.\n",
      "Written by Diego Darriba.\n",
      "Contributors: Tomas Flouri, Alexey Kozlov, Benoit Morel, David Posada, \n",
      "              Alexandros Stamatakis.\n",
      "Latest version: https://github.com/ddarriba/modeltest\n",
      "--------------------------------------------------------------------------------\n",
      "\n",
      "Physical cores: 16\n",
      "Logical cores:  32\n",
      "Memory:         126GB\n",
      "Extensions:     AVX\n",
      "\n",
      "WARNING: MSA has not enough sites to infer reliable results\n",
      "Creating new checkpoint file: /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb2.ckp\n",
      "--------------------------------------------------------------------------------\n",
      "ModelTest-NG v0.1.7\n",
      "\n",
      "Input data:\n",
      "  MSA:        /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta\n",
      "  Tree:       Maximum parsimony\n",
      "    file:           -\n",
      "  #taxa:            16\n",
      "  #sites:           1214\n",
      "  #patterns:        490\n",
      "  Max. thread mem:  3 MB\n",
      "\n",
      "Output:\n",
      "  Log:           /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb2.log\n",
      "  Starting tree: /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb2.tree\n",
      "  Results:       /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb2.out\n",
      "\n",
      "Selection options:\n",
      "  # dna schemes:      3\n",
      "  # dna models:       24\n",
      "  include model parameters:\n",
      "    Uniform:         true\n",
      "    p-inv (+I):      true\n",
      "    gamma (+G):      true\n",
      "    both (+I+G):     true\n",
      "    free rates (+R): false\n",
      "    fixed freqs:     true\n",
      "    estimated freqs: true\n",
      "    #categories:     4\n",
      "  gamma rates mode:   mean\n",
      "  asc bias:           none\n",
      "  epsilon (opt):      0.01\n",
      "  epsilon (par):      0.05\n",
      "  keep branches:      false\n",
      "\n",
      "Additional options:\n",
      "  verbosity:        very low\n",
      "  threads:          1/16\n",
      "  RNG seed:         12345\n",
      "  subtree repeats:  enabled\n",
      "--------------------------------------------------------------------------------\n",
      "modeltest-ng was called as follows: \n",
      ">> modeltest-ng -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -d nt -o /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb2 -T mrbayes \n",
      "\n",
      "\n",
      "modeltest-ng: Warning: You are using one single thread out of 16 physical cores.\n",
      "modeltest-ng:          You can set the number of threads with -p argument.\n",
      "modeltest-ng: Try 'modeltest-ng --help' or 'modeltest-ng --usage' for more information\n",
      "\n",
      "Partition 1/1\n",
      "\n",
      "    1/24   JC             0h:00:00   0h:00:00           -7709.2923       -       -\n",
      "    2/24   JC+I           0h:00:00   0h:00:00           -7374.6111       -  0.4949\n",
      "    3/24   JC+G4          0h:00:00   0h:00:00           -7303.4068  0.3035       -\n",
      "    4/24   JC+I+G4        0h:00:00   0h:00:00           -7305.1619  0.4314  0.1639\n",
      "    5/24   F81            0h:00:00   0h:00:00           -7701.3212       -       -\n",
      "    6/24   F81+I          0h:00:00   0h:00:00           -7355.4103       -  0.4984\n",
      "    7/24   F81+G4         0h:00:00   0h:00:00           -7276.0305  0.2772       -\n",
      "    8/24   F81+I+G4       0h:00:00   0h:00:00           -7278.5616  0.4173  0.1734\n",
      "    9/24   K80            0h:00:00   0h:00:00           -7560.6435       -       -\n",
      "   10/24   K80+I          0h:00:00   0h:00:00           -7209.2987       -  0.4978\n",
      "   11/24   K80+G4         0h:00:00   0h:00:00           -7135.6623  0.2945       -\n",
      "   12/24   K80+I+G4       0h:00:00   0h:00:00           -7136.2787  0.5448  0.2740\n",
      "   13/24   HKY            0h:00:00   0h:00:00           -7545.3516       -       -\n",
      "   14/24   HKY+I          0h:00:00   0h:00:00           -7167.8016       -  0.5015\n",
      "   15/24   HKY+G4         0h:00:00   0h:00:00           -7078.4160  0.2536       -\n",
      "   16/24   HKY+I+G4       0h:00:01   0h:00:01           -7078.8883  0.4339  0.2374\n",
      "   17/24   SYM            0h:00:00   0h:00:01           -7458.5917       -       -\n",
      "   18/24   SYM+I          0h:00:00   0h:00:01           -7140.0015       -  0.4823\n",
      "   19/24   SYM+G4         0h:00:00   0h:00:01           -7081.6920  0.3311       -\n",
      "   20/24   SYM+I+G4       0h:00:00   0h:00:01           -7080.8509  0.8229  0.3447\n",
      "   21/24   GTR            0h:00:00   0h:00:01           -7454.9183       -       -\n",
      "   22/24   GTR+I          0h:00:00   0h:00:01           -7118.7305       -  0.4909\n",
      "   23/24   GTR+G4         0h:00:00   0h:00:01           -7051.0601  0.2770       -\n",
      "   24/24   GTR+I+G4       0h:00:00   0h:00:01           -7050.1195  0.7262  0.3617\n",
      " ----ID---  ----MODEL---- ---Time--- -Elapsed--- -------LnL------- -Alpha- -P-inv-\n",
      "\n",
      "Computation of likelihood scores completed. It took 0h:00:01\n",
      "\n",
      "BIC       model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  GTR+G4             9     -7051.0601     14371.9840         0.0000    0.9315\n",
      "       2  GTR+I+G4          10     -7050.1195     14377.2044         5.2205    0.0685\n",
      "       3  HKY+G4             5     -7078.4160     14398.2890        26.3051    0.0000\n",
      "       4  HKY+I+G4           6     -7078.8883     14406.3352        34.3512    0.0000\n",
      "       5  SYM+G4             6     -7081.6920     14411.9426        39.9586    0.0000\n",
      "       6  SYM+I+G4           7     -7080.8509     14417.3621        45.3781    0.0000\n",
      "       7  K80+G4             2     -7135.6623     14491.4766       119.4926    0.0000\n",
      "       8  K80+I+G4           3     -7136.2787     14499.8111       127.8271    0.0000\n",
      "       9  GTR+I              9     -7118.7305     14507.3246       135.3407    0.0000\n",
      "      10  SYM+I              6     -7140.0015     14528.5616       156.5776    0.0000\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to BIC\n",
      "---------------------------\n",
      "Model:              GTR+G4\n",
      "lnL:                -7051.0601\n",
      "Frequencies:        0.2252 0.2959 0.2900 0.1888\n",
      "Subst. Rates:       1.1532 3.1215 0.6396 1.6458 7.8660 1.0000 \n",
      "Inv. sites prop:    -\n",
      "Gamma shape:        0.2770\n",
      "Score:              14371.9840\n",
      "Weight:             0.9315\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.9315\n",
      "Gamma-Inv:          0.0685\n",
      "Frequencies:        1.0000\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.4909\n",
      "Alpha:              0.2770\n",
      "Alpha-P.Inv:        0.7262\n",
      "P.Inv-Alpha:        0.3617\n",
      "Frequencies:        0.2253 0.2959 0.2902 0.1886 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m 012345 -f m -v 0 -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m GTRGAMMAX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta --model GTR+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m GTR+G4\n",
      "\n",
      "AIC       model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  GTR+G4             9     -7051.0601     14178.1203         0.0000    0.5148\n",
      "       2  GTR+I+G4          10     -7050.1195     14178.2391         0.1188    0.4852\n",
      "       3  HKY+G4             5     -7078.4160     14224.8321        46.7118    0.0000\n",
      "       4  HKY+I+G4           6     -7078.8883     14227.7766        49.6563    0.0000\n",
      "       5  SYM+G4             6     -7081.6920     14233.3839        55.2636    0.0000\n",
      "       6  SYM+I+G4           7     -7080.8509     14233.7018        55.5815    0.0000\n",
      "       7  GTR+I              9     -7118.7305     14313.4609       135.3407    0.0000\n",
      "       8  K80+G4             2     -7135.6623     14333.3246       155.2043    0.0000\n",
      "       9  K80+I+G4           3     -7136.2787     14336.5574       158.4371    0.0000\n",
      "      10  SYM+I              6     -7140.0015     14350.0029       171.8826    0.0000\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to AIC\n",
      "---------------------------\n",
      "Model:              GTR+G4\n",
      "lnL:                -7051.0601\n",
      "Frequencies:        0.2252 0.2959 0.2900 0.1888\n",
      "Subst. Rates:       1.1532 3.1215 0.6396 1.6458 7.8660 1.0000 \n",
      "Inv. sites prop:    -\n",
      "Gamma shape:        0.2770\n",
      "Score:              14178.1203\n",
      "Weight:             0.5148\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.5148\n",
      "Gamma-Inv:          0.4852\n",
      "Frequencies:        1.0000\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.4909\n",
      "Alpha:              0.2770\n",
      "Alpha-P.Inv:        0.7262\n",
      "P.Inv-Alpha:        0.3617\n",
      "Frequencies:        0.2258 0.2957 0.2910 0.1875 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m 012345 -f m -v 0 -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m GTRGAMMAX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta --model GTR+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m GTR+G4\n",
      "\n",
      "AICc      model              K            lnL          score          delta    weight\n",
      "--------------------------------------------------------------------------------\n",
      "       1  GTR+G4             9     -7051.0601     14180.1203         0.0000    0.5148\n",
      "       2  GTR+I+G4          10     -7050.1195     14180.2391         0.1188    0.4852\n",
      "       3  HKY+G4             5     -7078.4160     14226.8321        46.7118    0.0000\n",
      "       4  HKY+I+G4           6     -7078.8883     14229.7766        49.6563    0.0000\n",
      "       5  SYM+G4             6     -7081.6920     14235.3839        55.2636    0.0000\n",
      "       6  SYM+I+G4           7     -7080.8509     14235.7018        55.5815    0.0000\n",
      "       7  GTR+I              9     -7118.7305     14315.4609       135.3407    0.0000\n",
      "       8  K80+G4             2     -7135.6623     14334.3246       154.2043    0.0000\n",
      "       9  K80+I+G4           3     -7136.2787     14337.5574       157.4371    0.0000\n",
      "      10  SYM+I              6     -7140.0015     14352.0029       171.8826    0.0000\n",
      "--------------------------------------------------------------------------------\n",
      "Best model according to AICc\n",
      "---------------------------\n",
      "Model:              GTR+G4\n",
      "lnL:                -7051.0601\n",
      "Frequencies:        0.2252 0.2959 0.2900 0.1888\n",
      "Subst. Rates:       1.1532 3.1215 0.6396 1.6458 7.8660 1.0000 \n",
      "Inv. sites prop:    -\n",
      "Gamma shape:        0.2770\n",
      "Score:              14180.1203\n",
      "Weight:             0.5148\n",
      "---------------------------\n",
      "Parameter importances\n",
      "---------------------------\n",
      "P.Inv:              0.0000\n",
      "Gamma:              0.5148\n",
      "Gamma-Inv:          0.4852\n",
      "Frequencies:        1.0000\n",
      "---------------------------\n",
      "Model averaged estimates\n",
      "---------------------------\n",
      "P.Inv:              0.4909\n",
      "Alpha:              0.2770\n",
      "Alpha-P.Inv:        0.7262\n",
      "P.Inv-Alpha:        0.3617\n",
      "Frequencies:        0.2258 0.2957 0.2910 0.1875 \n",
      "\n",
      "Commands:\n",
      "  > phyml  -i /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m 012345 -f m -v 0 -a e -c 4 -o tlr\n",
      "  > raxmlHPC-SSE3 -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m GTRGAMMAX -n EXEC_NAME -p PARSIMONY_SEED\n",
      "  > raxml-ng --msa /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta --model GTR+G4\n",
      "  > paup -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta\n",
      "  > iqtree -s /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/trimmed/combined_RPBII_sequences_trimmed.fasta -m GTR+G4\n",
      "Summary:\n",
      "\n",
      "Partition 1/1:\n",
      "                         Model         Score        Weight\n",
      "----------------------------------------------------------\n",
      "       BIC              GTR+G4    14371.9840        0.9315\n",
      "       AIC              GTR+G4    14178.1203        0.5148\n",
      "      AICc              GTR+G4    14180.1203        0.5148\n",
      "\n",
      "Execution results written to /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb2.out\n",
      "Starting tree written to /mnt/Huanglab/Data/Khaled Dimorphism/Molecular identification of our isolates/modeltest_seqs/mb2.tree\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "139"
      ]
     },
     "execution_count": 58,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import os\n",
    "import re\n",
    "import subprocess\n",
    "from Bio import AlignIO\n",
    "from pathlib import Path\n",
    "\n",
    "# Set up base directory and subdirectories\n",
    "base_dir = os.getcwd()\n",
    "trimmed_seq_folder = os.path.join(base_dir, \"trimmed\")\n",
    "modeltest_seq_folder = os.path.join(base_dir, \"modeltest_seqs\")\n",
    "\n",
    "# Create the modeltest_fa directory if it doesn't exist\n",
    "os.makedirs(modeltest_seq_folder, exist_ok=True)\n",
    "\n",
    "def mb_model(model, index):\n",
    "    return f\"lset applyto=(part{index}) model={model};\\n\"\n",
    "\n",
    "# List all the clipped files in the directory\n",
    "clips = list(Path(trimmed_seq_folder).glob(\"*.fasta\"))\n",
    "\n",
    "start = 1\n",
    "partition = \"\"\n",
    "mb_charset = \"\"\n",
    "mb_lset = \"\"\n",
    "\n",
    "for index, clip in enumerate(clips):\n",
    "    msa = AlignIO.read(str(clip), \"fasta\")\n",
    "    msa_len = len(msa[0].seq)\n",
    "\n",
    "    # Model test for iqtree and RAxML\n",
    "    modeltest_path = os.path.join(modeltest_seq_folder, str(index))\n",
    "    subprocess.run([\"modeltest-ng\", \"-i\", str(clip), \"-d\", \"nt\", \"-o\", modeltest_path])\n",
    "    \n",
    "    try:\n",
    "        with open(modeltest_path + \".out\", 'r') as f:\n",
    "            out = f.read()\n",
    "        regexp = re.search(\"raxml-ng --msa.*--model (.*)\", out)\n",
    "        model = regexp.group(1)\n",
    "        partition += f\"{model}, gene{index}={start}-{start+msa_len-1}\\n\"\n",
    "    except Exception as e:\n",
    "        print(f\"Error processing {clip}: {e}\")\n",
    "\n",
    "    # Model test for MrBayes\n",
    "    modeltest_path_mb = os.path.join(modeltest_seq_folder, \"mb\" + str(index))\n",
    "    subprocess.run([\"modeltest-ng\", \"-i\", str(clip), \"-d\", \"nt\", \"-o\", modeltest_path_mb, \"-T\", \"mrbayes\"])\n",
    "\n",
    "    try:\n",
    "        with open(modeltest_path_mb + \".out\", 'r') as f:\n",
    "            out = f.read()\n",
    "        regexp = re.search(\"Model:              (.*)\", out)\n",
    "        model = regexp.group(1)\n",
    "        mb_charset += f\"charset part{index}={start}-{start+msa_len-1};\\n\"\n",
    "        mb_lset += mb_model(model, index+1)\n",
    "    except Exception as e:\n",
    "        print(f\"Error processing MrBayes model for {clip}: {e}\")\n",
    "\n",
    "    start += msa_len\n",
    "\n",
    "mb_charset += f\"partition favored = {len(clips)}: {', '.join(['part'+str(i) for i in range(len(clips))])};\\n\"\n",
    "mb_charset += \"set partition = favored;\\n\"\n",
    "\n",
    "# Save files\n",
    "Path(os.path.join(modeltest_seq_folder, 'part.txt')).write_text(partition)\n",
    "Path(os.path.join(modeltest_seq_folder, 'mb_lset.txt')).write_text(mb_lset)\n",
    "Path(os.path.join(modeltest_seq_folder, 'mb_charset.txt')).write_text(mb_charset)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "leading-grenada",
   "metadata": {},
   "source": [
    "## 4. Concatnate sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "capital-belize",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'concatenated.fasta'"
      ]
     },
     "execution_count": 59,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import os\n",
    "from Bio import SeqIO\n",
    "\n",
    "# Define the directory containing the FASTA files\n",
    "directory = \"trimmed\"\n",
    "\n",
    "# Collect all sequences from the files\n",
    "all_sequences = {}\n",
    "lengths = {}\n",
    "\n",
    "# Loop through all files in the directory\n",
    "for filename in os.listdir(directory):\n",
    "    if filename.endswith(\".fasta\"):\n",
    "        file_path = os.path.join(directory, filename)\n",
    "        sequences = {record.id: str(record.seq) for record in SeqIO.parse(file_path, \"fasta\")}\n",
    "        for seq_id, sequence in sequences.items():\n",
    "            if seq_id not in all_sequences:\n",
    "                all_sequences[seq_id] = {}\n",
    "            all_sequences[seq_id][filename] = sequence\n",
    "            if filename not in lengths:\n",
    "                lengths[filename] = len(sequence)\n",
    "            else:\n",
    "                lengths[filename] = max(lengths[filename], len(sequence))\n",
    "\n",
    "# Get the set of all sequence IDs\n",
    "all_titles = set(all_sequences.keys())\n",
    "\n",
    "# Concatenate sequences and fill missing sequences with '?'\n",
    "concatenated_sequences = {}\n",
    "for title in all_titles:\n",
    "    concatenated_sequence = \"\"\n",
    "    for filename in os.listdir(directory):\n",
    "        if filename.endswith(\".fasta\"):\n",
    "            seq = all_sequences[title].get(filename, '?' * lengths[filename])\n",
    "            concatenated_sequence += seq\n",
    "    concatenated_sequences[title] = concatenated_sequence\n",
    "\n",
    "# Convert the result to a FASTA formatted string\n",
    "concatenated_fasta = \"\"\n",
    "for title, sequence in concatenated_sequences.items():\n",
    "    concatenated_fasta += f\">{title}\\n{sequence}\\n\"\n",
    "\n",
    "# Save the concatenated sequences to a new FASTA file\n",
    "output_file = \"concatenated.fasta\"\n",
    "with open(output_file, 'w') as file:\n",
    "    file.write(concatenated_fasta)\n",
    "\n",
    "output_file\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "lonely-southwest",
   "metadata": {},
   "source": [
    "## 5. Tree reconstruction"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "defdba75-7221-4806-977f-ca81ece19283",
   "metadata": {},
   "source": [
    "### 5.1 IQtree\n",
    "\n",
    "Maximum likelihood (ML)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "special-effort",
   "metadata": {},
   "source": [
    "#### 5.1.1 Concatenated supermatrix"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "manual-focus",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IQ-TREE multicore version 2.0.7 for Linux 64-bit built Jan 21 2022\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams.\n",
      "\n",
      "Host:    huanglabserver-w70 (AVX2, FMA3, 125 GB RAM)\n",
      "Command: iqtree2 -s concatenated.fasta -p modeltest_seqs/part.txt --prefix iqtree_supermatrix/concat -B 1000 -nt AUTO -alrt 1000 --seed 5566\n",
      "Seed:    5566 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Jul 14 01:24:53 2024\n",
      "Kernel:  AVX+FMA - auto-detect threads (32 CPU cores detected)\n",
      "\n",
      "Reading partition model file modeltest_seqs/part.txt ...\n",
      "Reading alignment file concatenated.fasta ... Fasta format detected\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 50 sequences with 2566 columns, 1383 distinct patterns\n",
      "1012 parsimony-informative, 363 singleton sites, 1191 constant sites\n",
      "                                               Gap/Ambiguity  Composition  p-value\n",
      "   1  Ophiostoma_quercus_CMW22909                     77.24%    passed     62.95%\n",
      "   2  2022_000207                                     31.92%    failed      0.01%\n",
      "   3  Heinzbutinia_microspora_CMW17152                58.42%    failed      0.00%\n",
      "   4  Leptographium_procerum_CMW_34542                59.24%    passed     73.72%\n",
      "   5  Heinzbutinia_microspora_CBS_440_69              79.31%    passed     47.91%\n",
      "   6  Harringtonia_lauricola_CBS_121567               81.49%    failed      3.66%\n",
      "   7  Raffaelea_tritirachium_CBS_726_69               77.28%    failed      0.72%\n",
      "   8  2021_000001                                      6.51%    passed     35.81%\n",
      "   9  Harringtonia_lauricola_voucher_Hulcr7164        82.74%    passed     41.93%\n",
      "  10  Ophiostoma_noisomeae_CMW40326                   79.31%    passed     53.64%\n",
      "  11  Ophiostoma_novo_ulmi_subsp_novo_ulmi_CMW43870   37.49%    passed     53.11%\n",
      "  12  Raffaelea_xyleborini_CMW_45859                  79.07%    failed      2.58%\n",
      "  13  Heinzbutinia_grandicarpa_CMW41131               78.84%    passed     38.80%\n",
      "  14  Raffaelea_canadensis_CBS_168_66                 79.89%    failed      0.01%\n",
      "  15  Leptographium_procerum_CMW45                    85.70%    failed      0.15%\n",
      "  16  2023_000307                                      8.81%    passed     94.63%\n",
      "  17  Ophiostoma_floccosum_CBS_799_73                 77.28%    passed     44.28%\n",
      "  18  2021_000003                                     42.09%    failed      3.30%\n",
      "  19  Raffaelea_canadensis_CBS_805_70                 79.42%    failed      0.00%\n",
      "  20  2023_000565                                     54.44%    passed      5.07%\n",
      "  21  2021_000005                                      8.03%    passed     34.28%\n",
      "  22  Raffaelea_crossotarsa_CMW44793                  61.07%    failed      0.76%\n",
      "  23  Ophiostoma_floccosum_CMW12623                   65.24%    passed     18.91%\n",
      "  24  Aspergillus_egyptiacus_CBS_656_73               31.18%    failed      0.98%\n",
      "  25  Raffaelea_ambrosiae_CBS185_64                   83.13%    passed     81.59%\n",
      "  26  Leptographium_procerum_CBS_594_69               77.75%    passed      7.18%\n",
      "  27  Raffaelea_sp_CMW51555                           80.71%    failed      1.10%\n",
      "  28  Hawksworthiomyces_ciconiae_CBS_147964           77.63%    passed     16.64%\n",
      "  29  Raffaelea_cyclorhipidia_CMW44790                78.64%    failed      0.03%\n",
      "  30  Heinzbutinia_grandicarpa_CBS_250_88             79.58%    passed     27.88%\n",
      "  31  Ophiostoma_quercus_CMW2465                      58.38%    passed      6.62%\n",
      "  32  Ophiostoma_quercus_CMW43227                     77.24%    passed     62.95%\n",
      "  33  Ophiostoma_quercus_CMW40208                     60.76%    passed      7.94%\n",
      "  34  Hawksworthiomyces_hibbettii_CMW_37663           78.88%    failed      5.00%\n",
      "  35  Esteya_vermicola_CBS_115803                     83.98%    failed      0.05%\n",
      "  36  Leptographium_latens_CBS_124023                 79.07%    passed     86.22%\n",
      "  37  Leptographium_procerum_CBS_929_69               80.51%    passed     16.87%\n",
      "  38  2023_000248                                     33.44%    failed      0.36%\n",
      "  39  Harringtonia_lauricola_isolate_CV2018003        86.40%    passed     49.95%\n",
      "  40  Ophiostoma_ulmi_CBS_106_63                      74.90%    passed     78.22%\n",
      "  41  Raffaelea_arxii_CBS_273_70                      42.71%    failed      1.83%\n",
      "  42  2023_000566                                      8.69%    passed     95.98%\n",
      "  43  Leptographium_procerum_CMW_29994                81.84%    failed      1.65%\n",
      "  44  2022_000206                                     32.70%    passed     44.90%\n",
      "  45  Raffaelea_scolytodis_CBS_124461                 80.09%    passed     47.76%\n",
      "  46  Ophiostoma_floccosum_CMW12622                   65.86%    passed     20.09%\n",
      "  47  Ophiostoma_quercus_CMW41732                     77.24%    passed     61.93%\n",
      "  48  Ophiostoma_ulmi_CBS_151_55                      74.75%    passed     73.34%\n",
      "  49  Harringtonia_lauricola_CMW36261                 45.32%    failed      0.00%\n",
      "  50  Heinzbutinia_grandicarpa_CMW1600                40.37%    passed     20.34%\n",
      "WARNING: 37 sequences contain more than 50% gaps/ambiguity\n",
      "****  TOTAL                                           63.25%  19 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "Partition file is not in NEXUS format, assuming RAxML-style partition file...\n",
      "Subset\tType\tSeqs\tSites\tInfor\tInvar\tModel\tName\n",
      "1\tAA\t40\t738\t375\t267\tTIM2+I+G4\tgene0\n",
      "2\tAA\t18\t614\t262\t266\tTIM2ef+G4\tgene1\n",
      "3\tAA\t16\t1214\t375\t658\tTrN+G4\tgene2\n",
      "Degree of missing data: 0.532\n",
      "Info: multi-threading strategy over partitions\n",
      "\n",
      "NOTE: Ophiostoma_quercus_CMW43227 is identical to Ophiostoma_quercus_CMW22909 but kept for subsequent analysis\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.012 seconds\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 5566)...\n",
      "\n",
      "NOTE: 9 MB RAM (0 GB) is required!\n",
      "Measuring multi-threading efficiency up to 32 CPU cores\n",
      "Increase to 10 rounds for branch lengths\n",
      "246 trees examined\n",
      "Threads: 1 / Time: 32.038 sec / Speedup: 1.000 / Efficiency: 100% / LogL: -26366\n",
      "Threads: 2 / Time: 24.531 sec / Speedup: 1.306 / Efficiency: 65% / LogL: -26366\n",
      "Threads: 3 / Time: 19.560 sec / Speedup: 1.638 / Efficiency: 55% / LogL: -26366\n",
      "Threads: 4 / Time: 140.761 sec / Speedup: 0.228 / Efficiency: 6% / LogL: -26366\n",
      "BEST NUMBER OF THREADS: 3\n",
      "\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "Initial log-likelihood: -19199.339\n",
      "Current log-likelihood at step 1: -18271.906\n",
      "Current log-likelihood at step 2: -18240.268\n",
      "Current log-likelihood at step 3: -18237.705\n",
      "Current log-likelihood at step 4: -18236.692\n",
      "Current log-likelihood at step 5: -18235.738\n",
      "Current log-likelihood at step 6: -18234.755\n",
      "Current log-likelihood at step 7: -18233.832\n",
      "Current log-likelihood at step 8: -18233.063\n",
      "Current log-likelihood at step 9: -18232.518\n",
      "Current log-likelihood at step 10: -18232.200\n",
      "Current log-likelihood at step 11: -18232.046\n",
      "Current log-likelihood at step 12: -18231.977\n",
      "Partition-specific rates:  0.942 1.957 0.551\n",
      "Parameters optimization took 11 rounds (0.605 sec)\n",
      "\n",
      "Computing ML distances based on estimated model parameters... 0.035 sec\n",
      "WARNING: Some pairwise ML distances are too long (saturated)\n",
      "Computing BIONJ tree...\n",
      "0.007 seconds\n",
      "Log-likelihood of BIONJ tree: -19403.396\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 98 parsimony trees... 1.344 second\n",
      "Computing log-likelihood of 98 initial trees ... 1.155 seconds\n",
      "Current best score: -18231.977\n",
      "\n",
      "Do NNI search on 20 best initial trees\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "Initial log-likelihood: -18177.587\n",
      "Current log-likelihood at step 1: -18177.408\n",
      "Current log-likelihood at step 2: -18177.394\n",
      "Parameters optimization took 1 rounds (0.068 sec)\n",
      "\n",
      "BETTER TREE FOUND at iteration 1: -18177.394\n",
      "UPDATE BEST LOG-LIKELIHOOD: -18177.386\n",
      "BETTER TREE FOUND at iteration 3: -18177.384\n",
      "Iteration 10 / LogL: -18187.913 / Time: 0h:3m:44s\n",
      "BETTER TREE FOUND at iteration 14: -18177.382\n",
      "BETTER TREE FOUND at iteration 19: -18177.381\n",
      "Iteration 20 / LogL: -18177.382 / Time: 0h:3m:49s\n",
      "Finish initializing candidate tree set (17)\n",
      "Current best tree score: -18177.381 / CPU time: 11.935\n",
      "Number of iterations: 20\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 30 / LogL: -18191.571 / Time: 0h:3m:55s (0h:12m:1s left)\n",
      "BETTER TREE FOUND at iteration 40: -18177.380\n",
      "Iteration 40 / LogL: -18177.380 / Time: 0h:4m:0s (0h:10m:17s left)\n",
      "Iteration 50 / LogL: -18177.423 / Time: 0h:4m:5s (0h:7m:30s left)\n",
      "Log-likelihood cutoff on original alignment: -18211.369\n",
      "Iteration 60 / LogL: -18178.607 / Time: 0h:4m:10s (0h:5m:39s left)\n",
      "BETTER TREE FOUND at iteration 62: -18177.380\n",
      "UPDATE BEST LOG-LIKELIHOOD: -18177.380\n",
      "Iteration 70 / LogL: -18177.385 / Time: 0h:4m:15s (0h:5m:40s left)\n",
      "Iteration 80 / LogL: -18199.987 / Time: 0h:4m:20s (0h:4m:29s left)\n",
      "Iteration 90 / LogL: -18177.384 / Time: 0h:4m:24s (0h:3m:34s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -18177.380\n",
      "Iteration 100 / LogL: -18177.380 / Time: 0h:4m:30s (0h:2m:49s left)\n",
      "Log-likelihood cutoff on original alignment: -18212.341\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.999\n",
      "Iteration 110 / LogL: -18178.039 / Time: 0h:4m:36s (0h:3m:48s left)\n",
      "Iteration 120 / LogL: -18177.383 / Time: 0h:4m:41s (0h:3m:9s left)\n",
      "Iteration 130 / LogL: -18177.873 / Time: 0h:4m:45s (0h:2m:35s left)\n",
      "Iteration 140 / LogL: -18177.381 / Time: 0h:4m:51s (0h:2m:5s left)\n",
      "Iteration 150 / LogL: -18177.384 / Time: 0h:4m:56s (0h:1m:39s left)\n",
      "Log-likelihood cutoff on original alignment: -18211.725\n",
      "Iteration 160 / LogL: -18177.425 / Time: 0h:5m:0s (0h:1m:15s left)\n",
      "TREE SEARCH COMPLETED AFTER 163 ITERATIONS / Time: 0h:5m:1s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "Initial log-likelihood: -18177.380\n",
      "Current log-likelihood at step 1: -18177.376\n",
      "Partition-specific rates:  0.934 1.989 0.540\n",
      "Parameters optimization took 0 rounds (0.026 sec)\n",
      "\n",
      "Partition information was printed to iqtree_supermatrix/concat.best_model.nex\n",
      "BEST SCORE FOUND : -18177.376\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.963 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file iqtree_supermatrix/concat.splits.nex\n",
      "Total tree length: 6.679\n",
      "\n",
      "Total number of iterations: 163\n",
      "CPU time used for tree search: 222.015 sec (0h:3m:42s)\n",
      "Wall-clock time used for tree search: 83.975 sec (0h:1m:23s)\n",
      "Total CPU time used: 423.199 sec (0h:7m:3s)\n",
      "Total wall-clock time used: 302.790 sec (0h:5m:2s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file iqtree_supermatrix/concat.splits.nex...\n",
      "50 taxa and 296 splits.\n",
      "Consensus tree written to iqtree_supermatrix/concat.contree\n",
      "Reading input trees file iqtree_supermatrix/concat.contree\n",
      "Log-likelihood of consensus tree: -18177.397\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                iqtree_supermatrix/concat.iqtree\n",
      "  Maximum-likelihood tree:       iqtree_supermatrix/concat.treefile\n",
      "  Likelihood distances:          iqtree_supermatrix/concat.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          iqtree_supermatrix/concat.splits.nex\n",
      "  Consensus tree:                iqtree_supermatrix/concat.contree\n",
      "  Screen log file:               iqtree_supermatrix/concat.log\n",
      "\n",
      "Date and Time: Sun Jul 14 01:29:56 2024\n"
     ]
    }
   ],
   "source": [
    "!mkdir iqtree_supermatrix\n",
    "\n",
    "!iqtree2 \\\n",
    "-s concatenated.fasta \\\n",
    "-p modeltest_seqs/part.txt \\\n",
    "--prefix iqtree_supermatrix/concat \\\n",
    "-B 1000 \\\n",
    "-nt AUTO \\\n",
    "-alrt 1000 \\\n",
    "--seed 5566"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "judicial-understanding",
   "metadata": {},
   "source": [
    "#### 5.1.2 Genetree\n",
    "\n",
    "reconstruct tree individually and reconcile"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "saved-study",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IQ-TREE multicore version 2.0.7 for Linux 64-bit built Jan 21 2022\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams.\n",
      "\n",
      "Host:    huanglabserver-w70 (AVX2, FMA3, 125 GB RAM)\n",
      "Command: iqtree2 -s trimmed/combined_ITS_sequences_trimmed.fasta --prefix iqtree_genetree/combined_ITS_sequences -B 1000 -nt AUTO -alrt 1000 --seed 5566\n",
      "Seed:    5566 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Jul 14 02:05:58 2024\n",
      "Kernel:  AVX+FMA - auto-detect threads (32 CPU cores detected)\n",
      "\n",
      "Reading alignment file trimmed/combined_ITS_sequences_trimmed.fasta ... Fasta format detected\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 40 sequences with 738 columns, 514 distinct patterns\n",
      "375 parsimony-informative, 96 singleton sites, 267 constant sites\n",
      "                                               Gap/Ambiguity  Composition  p-value\n",
      "   1  2021_000001                                     22.36%    passed     82.52%\n",
      "   2  2021_000003                                     29.81%    failed      0.15%\n",
      "   3  2021_000005                                     22.36%    passed     75.18%\n",
      "   4  2022_000206                                     30.35%    passed     99.48%\n",
      "   5  2023_000248                                     32.52%    failed      2.21%\n",
      "   6  2023_000307                                     28.86%    passed     18.54%\n",
      "   7  2023_000565                                     23.04%    passed     90.40%\n",
      "   8  2023_000566                                     23.17%    passed     40.24%\n",
      "   9  Ophiostoma_ulmi_CBS_106_63                      12.74%    passed     51.87%\n",
      "  10  Ophiostoma_ulmi_CBS_151_55                      12.20%    passed     32.57%\n",
      "  11  Harringtonia_lauricola_CBS_121567               35.64%    passed     34.82%\n",
      "  12  Ophiostoma_novo_ulmi_subsp_novo_ulmi_CMW43870   21.54%    passed     71.57%\n",
      "  13  Harringtonia_lauricola_CMW36261                 50.54%    passed     23.63%\n",
      "  14  Raffaelea_scolytodis_CBS_124461                 30.76%    passed     98.17%\n",
      "  15  Raffaelea_arxii_CBS_273_70                      44.99%    passed     33.03%\n",
      "  16  Leptographium_latens_CBS_124023                 27.24%    passed     65.58%\n",
      "  17  Ophiostoma_quercus_CMW41732                     20.87%    passed     49.14%\n",
      "  18  Ophiostoma_quercus_CMW43227                     20.87%    passed     47.05%\n",
      "  19  Ophiostoma_quercus_CMW40208                     20.87%    passed     47.05%\n",
      "  20  Ophiostoma_quercus_CMW22909                     20.87%    passed     47.05%\n",
      "  21  Ophiostoma_floccosum_CBS_799_73                 21.00%    passed     80.84%\n",
      "  22  Ophiostoma_floccosum_CMW12622                   23.98%    passed     95.58%\n",
      "  23  Ophiostoma_floccosum_CMW12623                   21.82%    passed     89.01%\n",
      "  24  Leptographium_procerum_CBS_594_69               22.63%    passed     87.61%\n",
      "  25  Leptographium_procerum_CMW_34542                21.41%    passed     92.59%\n",
      "  26  Leptographium_procerum_CBS_929_69               32.25%    passed     90.40%\n",
      "  27  Heinzbutinia_grandicarpa_CBS_250_88             29.00%    passed     98.60%\n",
      "  28  Heinzbutinia_grandicarpa_CMW41131               26.42%    passed     94.47%\n",
      "  29  Heinzbutinia_grandicarpa_CMW1600                28.18%    passed     95.36%\n",
      "  30  Heinzbutinia_microspora_CBS_440_69              28.05%    passed     97.86%\n",
      "  31  Hawksworthiomyces_ciconiae_CBS_147964           22.22%    failed      1.31%\n",
      "  32  Hawksworthiomyces_hibbettii_CMW_37663           26.56%    failed      0.45%\n",
      "  33  Ophiostoma_noisomeae_CMW40326                   28.05%    passed     71.83%\n",
      "  34  Raffaelea_sp_CMW51555                           32.93%    passed     23.55%\n",
      "  35  Raffaelea_canadensis_CBS_805_70                 28.46%    failed      1.65%\n",
      "  36  Raffaelea_canadensis_CBS_168_66                 30.08%    failed      2.32%\n",
      "  37  Raffaelea_xyleborini_CMW_45859                  27.24%    passed     24.51%\n",
      "  38  Raffaelea_cyclorhipidia_CMW44790                25.75%    failed      4.02%\n",
      "  39  Raffaelea_tritirachium_CBS_726_69               21.00%    passed     23.72%\n",
      "  40  Aspergillus_egyptiacus_CBS_656_73               35.91%    passed     34.11%\n",
      "WARNING: 1 sequences contain more than 50% gaps/ambiguity\n",
      "****  TOTAL                                           26.61%  7 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "NOTE: Ophiostoma_quercus_CMW40208 is identical to Ophiostoma_quercus_CMW43227 but kept for subsequent analysis\n",
      "NOTE: 1 identical sequences (see below) will be ignored for subsequent analysis\n",
      "NOTE: Ophiostoma_quercus_CMW22909 (identical to Ophiostoma_quercus_CMW43227) is ignored but added at the end\n",
      "Alignment was printed to iqtree_genetree/combined_ITS_sequences.uniqueseq.phy\n",
      "\n",
      "For your convenience alignment with unique sequences printed to iqtree_genetree/combined_ITS_sequences.uniqueseq.phy\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.002 seconds\n",
      "Measuring multi-threading efficiency up to 32 CPU cores\n",
      "Increase to 10 rounds for branch lengths\n",
      "1322 trees examined\n",
      "Threads: 1 / Time: 32.024 sec / Speedup: 1.000 / Efficiency: 100% / LogL: -11015\n",
      "Threads: 2 / Time: 45.532 sec / Speedup: 0.703 / Efficiency: 35% / LogL: -11015\n",
      "BEST NUMBER OF THREADS: 1\n",
      "\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -6948.266\n",
      "Optimal log-likelihood: -6947.846\n",
      "Rate parameters:  A-C: 1.86849  A-G: 2.12191  A-T: 2.06535  C-G: 1.87029  C-T: 4.05406  G-T: 1.00000\n",
      "Base frequencies:  A: 0.186  C: 0.348  G: 0.276  T: 0.190\n",
      "Proportion of invariable sites: 0.192\n",
      "Gamma shape alpha: 1.070\n",
      "Parameters optimization took 1 rounds (0.065 sec)\n",
      "Time for fast ML tree search: 0.342 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 8 MB RAM!\n",
      "ModelFinder will test up to 286 DNA models (sample size: 738) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "  1  GTR+F         7369.851     83  14905.702    14927.023    15287.829\n",
      "  2  GTR+F+I       7040.882     84  14249.765    14271.633    14636.496\n",
      "  3  GTR+F+G4      6956.665     84  14081.330    14103.198    14468.061\n",
      "  4  GTR+F+I+G4    6947.174     85  14064.348    14086.771    14455.683\n",
      "  5  GTR+F+R2      6985.242     85  14140.483    14162.906    14531.818\n",
      "  6  GTR+F+R3      6956.785     87  14087.570    14111.127    14488.113\n",
      "  7  GTR+F+R4      6948.612     89  14075.224    14099.946    14484.975\n",
      "  8  GTR+F+R5      6946.492     91  14074.985    14100.904    14493.944\n",
      " 17  SYM+I+G4      7018.450     82  14200.900    14221.681    14578.423\n",
      " 30  TVM+F+I+G4    6959.625     84  14087.251    14109.119    14473.982\n",
      " 43  TVMe+I+G4     7038.544     81  14239.087    14259.337    14612.007\n",
      " 56  TIM3+F+I+G4   6953.431     83  14072.862    14094.183    14454.989\n",
      " 69  TIM3e+I+G4    7027.962     80  14215.924    14235.650    14584.239\n",
      " 82  TIM2+F+I+G4   6953.831     83  14073.662    14094.983    14455.789\n",
      " 95  TIM2e+I+G4    7050.367     80  14260.734    14280.460    14629.050\n",
      "108  TIM+F+I+G4    6953.739     83  14073.478    14094.799    14455.605\n",
      "121  TIMe+I+G4     7040.446     80  14240.893    14260.619    14609.208\n",
      "134  TPM3u+F+I+G4  6966.585     82  14097.171    14117.953    14474.694\n",
      "147  TPM3+F+I+G4   6966.585     82  14097.171    14117.953    14474.694\n",
      "160  TPM2u+F+I+G4  6966.654     82  14097.307    14118.089    14474.831\n",
      "173  TPM2+F+I+G4   6966.654     82  14097.307    14118.089    14474.831\n",
      "186  K3Pu+F+I+G4   6966.961     82  14097.923    14118.704    14475.446\n",
      "199  K3P+I+G4      7058.099     79  14274.197    14293.407    14637.909\n",
      "212  TN+F+I+G4     6956.139     82  14076.279    14097.061    14453.802\n",
      "225  TNe+I+G4      7052.962     79  14263.924    14283.134    14627.636\n",
      "238  HKY+F+I+G4    6969.507     81  14101.013    14121.263    14473.933\n",
      "251  K2P+I+G4      7070.751     78  14297.502    14316.203    14656.609\n",
      "264  F81+F+I+G4    7005.814     80  14171.628    14191.354    14539.943\n",
      "277  JC+I+G4       7094.823     77  14343.646    14361.846    14698.150\n",
      "Akaike Information Criterion:           GTR+F+I+G4\n",
      "Corrected Akaike Information Criterion: GTR+F+I+G4\n",
      "Bayesian Information Criterion:         TN+F+I+G4\n",
      "Best-fit model: TN+F+I+G4 chosen according to BIC\n",
      "\n",
      "All model information printed to iqtree_genetree/combined_ITS_sequences.model.gz\n",
      "CPU time for ModelFinder: 126.740 seconds (0h:2m:6s)\n",
      "Wall-clock time for ModelFinder: 82.790 seconds (0h:1m:22s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 5566)...\n",
      "\n",
      "NOTE: 4 MB RAM (0 GB) is required!\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "Thoroughly optimizing +I+G parameters from 10 start values...\n",
      "Init pinv, alpha: 0.000, 1.278 / Estimate: 0.000, 0.562 / LogL: -6965.630\n",
      "Init pinv, alpha: 0.040, 1.278 / Estimate: 0.216, 1.221 / LogL: -6956.241\n",
      "Init pinv, alpha: 0.080, 1.278 / Estimate: 0.217, 1.230 / LogL: -6956.219\n",
      "Init pinv, alpha: 0.121, 1.278 / Estimate: 0.215, 1.217 / LogL: -6956.252\n",
      "Init pinv, alpha: 0.161, 1.278 / Estimate: 0.216, 1.225 / LogL: -6956.233\n",
      "Init pinv, alpha: 0.201, 1.278 / Estimate: 0.216, 1.226 / LogL: -6956.230\n",
      "Init pinv, alpha: 0.241, 1.278 / Estimate: 0.234, 1.345 / LogL: -6956.155\n",
      "Init pinv, alpha: 0.281, 1.278 / Estimate: 0.236, 1.364 / LogL: -6956.181\n",
      "Init pinv, alpha: 0.322, 1.278 / Estimate: 0.236, 1.359 / LogL: -6956.172\n",
      "Init pinv, alpha: 0.362, 1.278 / Estimate: 0.237, 1.367 / LogL: -6956.187\n",
      "Optimal pinv,alpha: 0.234, 1.345 / LogL: -6956.155\n",
      "\n",
      "Parameters optimization took 2.712 sec\n",
      "Computing ML distances based on estimated model parameters... 0.020 sec\n",
      "Computing BIONJ tree...\n",
      "0.007 seconds\n",
      "Log-likelihood of BIONJ tree: -6978.777\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 98 parsimony trees... 0.359 second\n",
      "Computing log-likelihood of 98 initial trees ... 0.604 seconds\n",
      "Current best score: -6956.155\n",
      "\n",
      "Do NNI search on 20 best initial trees\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "BETTER TREE FOUND at iteration 1: -6956.140\n",
      "Iteration 10 / LogL: -6959.542 / Time: 0h:0m:5s\n",
      "Iteration 20 / LogL: -6959.542 / Time: 0h:0m:6s\n",
      "Finish initializing candidate tree set (12)\n",
      "Current best tree score: -6956.140 / CPU time: 3.888\n",
      "Number of iterations: 20\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 30 / LogL: -6956.159 / Time: 0h:0m:8s (0h:0m:20s left)\n",
      "Iteration 40 / LogL: -6956.142 / Time: 0h:0m:10s (0h:0m:15s left)\n",
      "Iteration 50 / LogL: -6957.257 / Time: 0h:0m:11s (0h:0m:12s left)\n",
      "Log-likelihood cutoff on original alignment: -6986.620\n",
      "Iteration 60 / LogL: -6956.163 / Time: 0h:0m:13s (0h:0m:9s left)\n",
      "Iteration 70 / LogL: -6959.706 / Time: 0h:0m:15s (0h:0m:6s left)\n",
      "Iteration 80 / LogL: -6956.155 / Time: 0h:0m:16s (0h:0m:4s left)\n",
      "Iteration 90 / LogL: -6956.150 / Time: 0h:0m:18s (0h:0m:2s left)\n",
      "Iteration 100 / LogL: -6959.638 / Time: 0h:0m:20s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -6986.026\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.997\n",
      "TREE SEARCH COMPLETED AFTER 102 ITERATIONS / Time: 0h:0m:20s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -6956.140\n",
      "Optimal log-likelihood: -6956.133\n",
      "Rate parameters:  A-C: 1.00000  A-G: 1.24191  A-T: 1.00000  C-G: 1.00000  C-T: 2.34368  G-T: 1.00000\n",
      "Base frequencies:  A: 0.186  C: 0.348  G: 0.276  T: 0.190\n",
      "Proportion of invariable sites: 0.231\n",
      "Gamma shape alpha: 1.325\n",
      "Parameters optimization took 1 rounds (0.024 sec)\n",
      "BEST SCORE FOUND : -6956.133\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.634 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file iqtree_genetree/combined_ITS_sequences.splits.nex\n",
      "Total tree length: 4.733\n",
      "\n",
      "Total number of iterations: 102\n",
      "CPU time used for tree search: 17.970 sec (0h:0m:17s)\n",
      "Wall-clock time used for tree search: 18.106 sec (0h:0m:18s)\n",
      "Total CPU time used: 21.514 sec (0h:0m:21s)\n",
      "Total wall-clock time used: 21.677 sec (0h:0m:21s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file iqtree_genetree/combined_ITS_sequences.splits.nex...\n",
      "39 taxa and 203 splits.\n",
      "Consensus tree written to iqtree_genetree/combined_ITS_sequences.contree\n",
      "Reading input trees file iqtree_genetree/combined_ITS_sequences.contree\n",
      "Log-likelihood of consensus tree: -6956.968\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                iqtree_genetree/combined_ITS_sequences.iqtree\n",
      "  Maximum-likelihood tree:       iqtree_genetree/combined_ITS_sequences.treefile\n",
      "  Likelihood distances:          iqtree_genetree/combined_ITS_sequences.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          iqtree_genetree/combined_ITS_sequences.splits.nex\n",
      "  Consensus tree:                iqtree_genetree/combined_ITS_sequences.contree\n",
      "  Screen log file:               iqtree_genetree/combined_ITS_sequences.log\n",
      "\n",
      "Date and Time: Sun Jul 14 02:07:43 2024\n",
      "IQ-TREE multicore version 2.0.7 for Linux 64-bit built Jan 21 2022\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams.\n",
      "\n",
      "Host:    huanglabserver-w70 (AVX2, FMA3, 125 GB RAM)\n",
      "Command: iqtree2 -s trimmed/combined_Btub_sequences_trimmed.fasta --prefix iqtree_genetree/combined_Btub_sequences -B 1000 -nt AUTO -alrt 1000 --seed 5566\n",
      "Seed:    5566 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Jul 14 02:07:43 2024\n",
      "Kernel:  AVX+FMA - auto-detect threads (32 CPU cores detected)\n",
      "\n",
      "Reading alignment file trimmed/combined_Btub_sequences_trimmed.fasta ... Fasta format detected\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 18 sequences with 614 columns, 379 distinct patterns\n",
      "262 parsimony-informative, 86 singleton sites, 266 constant sites\n",
      "                                          Gap/Ambiguity  Composition  p-value\n",
      "   1  2021_000001                                 0.33%    passed     31.82%\n",
      "   2  2021_000003                                 4.07%    passed     92.31%\n",
      "   3  2021_000005                                 6.68%    passed     44.00%\n",
      "   4  2022_000207                                12.05%    passed     42.25%\n",
      "   5  2023_000307                                 1.95%    passed     94.44%\n",
      "   6  2023_000565                                 2.12%    passed     69.94%\n",
      "   7  2023_000566                                 8.47%    passed     46.26%\n",
      "   8  Leptographium_procerum_CMW_34542           24.10%    passed     24.93%\n",
      "   9  Leptographium_procerum_CMW_29994           24.10%    passed     24.93%\n",
      "  10  Ophiostoma_floccosum_CMW12623              48.70%    passed     68.17%\n",
      "  11  Ophiostoma_floccosum_CMW12622              48.70%    passed     68.17%\n",
      "  12  Ophiostoma_quercus_CMW40208                31.11%    passed     28.95%\n",
      "  13  Raffaelea_ambrosiae_CBS185_64              29.48%    passed     68.61%\n",
      "  14  Esteya_vermicola_CBS_115803                33.06%    passed      7.64%\n",
      "  15  Leptographium_procerum_CMW45               40.23%    failed      2.61%\n",
      "  16  Harringtonia_lauricola_isolate_CV2018003   43.16%    passed     59.60%\n",
      "  17  Harringtonia_lauricola_voucher_Hulcr7164   27.85%    passed     52.33%\n",
      "  18  Aspergillus_egyptiacus_CBS_656_73          41.37%    passed     73.33%\n",
      "****  TOTAL                                      23.75%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "NOTE: Leptographium_procerum_CMW_29994 is identical to Leptographium_procerum_CMW_34542 but kept for subsequent analysis\n",
      "NOTE: Ophiostoma_floccosum_CMW12622 is identical to Ophiostoma_floccosum_CMW12623 but kept for subsequent analysis\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.001 seconds\n",
      "Measuring multi-threading efficiency up to 32 CPU cores\n",
      "Increase to 10 rounds for branch lengths\n",
      "1858 trees examined\n",
      "Threads: 1 / Time: 32.017 sec / Speedup: 1.000 / Efficiency: 100% / LogL: -5378\n",
      "Threads: 2 / Time: 22.581 sec / Speedup: 1.418 / Efficiency: 71% / LogL: -5378\n",
      "Threads: 3 / Time: 18.108 sec / Speedup: 1.768 / Efficiency: 59% / LogL: -5378\n",
      "Threads: 4 / Time: 16.486 sec / Speedup: 1.942 / Efficiency: 49% / LogL: -5378\n",
      "BEST NUMBER OF THREADS: 3\n",
      "\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -3972.400\n",
      "Optimal log-likelihood: -3971.959\n",
      "Rate parameters:  A-C: 1.15303  A-G: 2.36366  A-T: 2.37414  C-G: 0.66553  C-T: 4.29143  G-T: 1.00000\n",
      "Base frequencies:  A: 0.208  C: 0.298  G: 0.256  T: 0.238\n",
      "Proportion of invariable sites: 0.205\n",
      "Gamma shape alpha: 0.544\n",
      "Parameters optimization took 1 rounds (0.011 sec)\n",
      "Time for fast ML tree search: 0.072 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 3 MB RAM!\n",
      "ModelFinder will test up to 286 DNA models (sample size: 614) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "  1  GTR+F         4232.515     41  8547.030     8553.051     8728.250\n",
      "  2  GTR+F+I       4017.122     42  8118.244     8124.570     8303.884\n",
      "  3  GTR+F+G4      3971.450     42  8026.901     8033.227     8212.541\n",
      "  4  GTR+F+I+G4    3971.813     43  8029.626     8036.264     8219.685\n",
      "  5  GTR+F+R2      4008.306     43  8102.611     8109.250     8292.671\n",
      "  6  GTR+F+R3      3988.220     45  8066.441     8073.730     8265.341\n",
      "  7  GTR+F+R4      3980.911     47  8055.822     8063.793     8263.561\n",
      "  8  GTR+F+R5      3976.621     49  8051.243     8059.931     8267.822\n",
      " 16  SYM+G4        3974.100     39  8026.201     8031.636     8198.580\n",
      " 17  SYM+I+G4      3975.247     40  8030.495     8036.219     8207.294\n",
      " 29  TVM+F+G4      3978.609     41  8039.218     8045.239     8220.438\n",
      " 30  TVM+F+I+G4    3977.359     42  8038.718     8045.044     8224.358\n",
      " 42  TVMe+G4       3987.772     38  8051.544     8056.699     8219.504\n",
      " 43  TVMe+I+G4     3986.705     39  8051.410     8056.846     8223.790\n",
      " 55  TIM3+F+G4     3981.032     40  8042.063     8047.787     8218.863\n",
      " 56  TIM3+F+I+G4   3981.139     41  8044.278     8050.299     8225.497\n",
      " 68  TIM3e+G4      3978.456     37  8030.912     8035.794     8194.452\n",
      " 69  TIM3e+I+G4    3979.475     38  8034.949     8040.104     8202.909\n",
      " 81  TIM2+F+G4     3975.512     40  8031.023     8036.748     8207.823\n",
      " 82  TIM2+F+I+G4   3975.779     41  8033.559     8039.580     8214.778\n",
      " 94  TIM2e+G4      3975.345     37  8024.691     8029.572     8188.230\n",
      " 95  TIM2e+I+G4    3976.556     38  8029.111     8034.266     8197.071\n",
      "107  TIM+F+G4      3984.205     40  8048.410     8054.134     8225.210\n",
      "108  TIM+F+I+G4    3984.585     41  8051.169     8057.190     8232.389\n",
      "120  TIMe+G4       3979.012     37  8032.023     8036.905     8195.563\n",
      "121  TIMe+I+G4     3980.215     38  8036.430     8041.585     8204.390\n",
      "133  TPM3u+F+G4    3988.903     39  8055.807     8061.243     8228.187\n",
      "134  TPM3u+F+I+G4  3987.181     40  8054.361     8060.085     8231.161\n",
      "146  TPM3+F+G4     3988.903     39  8055.807     8061.243     8228.187\n",
      "147  TPM3+F+I+G4   3987.181     40  8054.361     8060.085     8231.161\n",
      "159  TPM2u+F+G4    3982.556     39  8043.113     8048.548     8215.493\n",
      "160  TPM2u+F+I+G4  3981.269     40  8042.539     8048.263     8219.338\n",
      "172  TPM2+F+G4     3982.556     39  8043.113     8048.548     8215.493\n",
      "173  TPM2+F+I+G4   3981.269     40  8042.539     8048.263     8219.338\n",
      "185  K3Pu+F+G4     3991.956     39  8061.912     8067.348     8234.292\n",
      "186  K3Pu+F+I+G4   3990.514     40  8061.027     8066.752     8237.827\n",
      "198  K3P+G4        3992.878     36  8057.756     8062.373     8216.876\n",
      "199  K3P+I+G4      3991.825     37  8057.650     8062.532     8221.190\n",
      "211  TN+F+G4       3984.353     39  8046.706     8052.142     8219.086\n",
      "212  TN+F+I+G4     3984.728     40  8049.456     8055.181     8226.256\n",
      "224  TNe+G4        3979.249     36  8030.497     8035.114     8189.617\n",
      "225  TNe+I+G4      3980.452     37  8034.903     8039.785     8198.443\n",
      "237  HKY+F+G4      3992.073     38  8060.145     8065.300     8228.105\n",
      "238  HKY+F+I+G4    3990.639     39  8059.278     8064.714     8231.658\n",
      "250  K2P+G4        3993.048     35  8056.096     8060.456     8210.796\n",
      "251  K2P+I+G4      3992.016     36  8056.033     8060.649     8215.152\n",
      "263  F81+F+G4      4042.127     37  8158.255     8163.137     8321.794\n",
      "264  F81+F+I+G4    4039.893     38  8155.786     8160.940     8323.745\n",
      "276  JC+G4         4047.415     34  8162.830     8166.940     8313.110\n",
      "277  JC+I+G4       4045.868     35  8161.737     8166.096     8316.436\n",
      "Akaike Information Criterion:           TIM2e+G4\n",
      "Corrected Akaike Information Criterion: TIM2e+G4\n",
      "Bayesian Information Criterion:         TIM2e+G4\n",
      "Best-fit model: TIM2e+G4 chosen according to BIC\n",
      "\n",
      "All model information printed to iqtree_genetree/combined_Btub_sequences.model.gz\n",
      "CPU time for ModelFinder: 199.161 seconds (0h:3m:19s)\n",
      "Wall-clock time for ModelFinder: 90.493 seconds (0h:1m:30s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 5566)...\n",
      "\n",
      "NOTE: 2 MB RAM (0 GB) is required!\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -3975.345\n",
      "Optimal log-likelihood: -3975.342\n",
      "Rate parameters:  A-C: 1.62472  A-G: 2.25871  A-T: 1.62472  C-G: 1.00000  C-T: 5.97881  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Gamma shape alpha: 0.299\n",
      "Parameters optimization took 1 rounds (0.004 sec)\n",
      "Computing ML distances based on estimated model parameters... 0.010 sec\n",
      "Computing BIONJ tree...\n",
      "0.005 seconds\n",
      "Log-likelihood of BIONJ tree: -3975.346\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 98 parsimony trees... 0.104 second\n",
      "Computing log-likelihood of 98 initial trees ... 0.138 seconds\n",
      "Current best score: -3975.342\n",
      "\n",
      "Do NNI search on 20 best initial trees\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "BETTER TREE FOUND at iteration 1: -3974.346\n",
      "UPDATE BEST LOG-LIKELIHOOD: -3974.346\n",
      "UPDATE BEST LOG-LIKELIHOOD: -3974.346\n",
      "Iteration 10 / LogL: -3974.346 / Time: 0h:0m:0s\n",
      "Iteration 20 / LogL: -3974.346 / Time: 0h:0m:1s\n",
      "Finish initializing candidate tree set (2)\n",
      "Current best tree score: -3974.346 / CPU time: 0.991\n",
      "Number of iterations: 20\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "UPDATE BEST LOG-LIKELIHOOD: -3974.346\n",
      "Iteration 30 / LogL: -3974.414 / Time: 0h:0m:1s (0h:0m:4s left)\n",
      "Iteration 40 / LogL: -3974.347 / Time: 0h:0m:2s (0h:0m:3s left)\n",
      "Iteration 50 / LogL: -3974.422 / Time: 0h:0m:2s (0h:0m:2s left)\n",
      "Log-likelihood cutoff on original alignment: -3994.360\n",
      "Iteration 60 / LogL: -3974.351 / Time: 0h:0m:3s (0h:0m:2s left)\n",
      "Iteration 70 / LogL: -3974.346 / Time: 0h:0m:3s (0h:0m:1s left)\n",
      "Iteration 80 / LogL: -3974.415 / Time: 0h:0m:3s (0h:0m:1s left)\n",
      "Iteration 90 / LogL: -3974.347 / Time: 0h:0m:4s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -3974.346\n",
      "Iteration 100 / LogL: -3974.347 / Time: 0h:0m:4s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -3994.360\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.998\n",
      "TREE SEARCH COMPLETED AFTER 102 ITERATIONS / Time: 0h:0m:4s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -3974.346\n",
      "Optimal log-likelihood: -3974.344\n",
      "Rate parameters:  A-C: 1.58165  A-G: 2.24716  A-T: 1.58165  C-G: 1.00000  C-T: 6.01018  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Gamma shape alpha: 0.295\n",
      "Parameters optimization took 1 rounds (0.004 sec)\n",
      "BEST SCORE FOUND : -3974.344\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.075 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file iqtree_genetree/combined_Btub_sequences.splits.nex\n",
      "Total tree length: 4.963\n",
      "\n",
      "Total number of iterations: 102\n",
      "CPU time used for tree search: 13.142 sec (0h:0m:13s)\n",
      "Wall-clock time used for tree search: 4.840 sec (0h:0m:4s)\n",
      "Total CPU time used: 13.538 sec (0h:0m:13s)\n",
      "Total wall-clock time used: 5.009 sec (0h:0m:5s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file iqtree_genetree/combined_Btub_sequences.splits.nex...\n",
      "18 taxa and 113 splits.\n",
      "Consensus tree written to iqtree_genetree/combined_Btub_sequences.contree\n",
      "Reading input trees file iqtree_genetree/combined_Btub_sequences.contree\n",
      "Log-likelihood of consensus tree: -3974.351\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                iqtree_genetree/combined_Btub_sequences.iqtree\n",
      "  Maximum-likelihood tree:       iqtree_genetree/combined_Btub_sequences.treefile\n",
      "  Likelihood distances:          iqtree_genetree/combined_Btub_sequences.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          iqtree_genetree/combined_Btub_sequences.splits.nex\n",
      "  Consensus tree:                iqtree_genetree/combined_Btub_sequences.contree\n",
      "  Screen log file:               iqtree_genetree/combined_Btub_sequences.log\n",
      "\n",
      "Date and Time: Sun Jul 14 02:09:18 2024\n",
      "IQ-TREE multicore version 2.0.7 for Linux 64-bit built Jan 21 2022\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams.\n",
      "\n",
      "Host:    huanglabserver-w70 (AVX2, FMA3, 125 GB RAM)\n",
      "Command: iqtree2 -s trimmed/combined_RPBII_sequences_trimmed.fasta --prefix iqtree_genetree/combined_RPBII_sequences -B 1000 -nt AUTO -alrt 1000 --seed 5566\n",
      "Seed:    5566 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Jul 14 02:09:18 2024\n",
      "Kernel:  AVX+FMA - auto-detect threads (32 CPU cores detected)\n",
      "\n",
      "Reading alignment file trimmed/combined_RPBII_sequences_trimmed.fasta ... Fasta format detected\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 16 sequences with 1214 columns, 490 distinct patterns\n",
      "375 parsimony-informative, 181 singleton sites, 657 constant sites\n",
      "                                               Gap/Ambiguity  Composition  p-value\n",
      "   1  2021_000001                                      0.00%    passed     93.41%\n",
      "   2  2021_000003                                     68.78%    passed     13.30%\n",
      "   3  2021_000005                                      0.00%    passed     48.07%\n",
      "   4  2022_000206                                      0.08%    passed     60.88%\n",
      "   5  2022_000207                                      0.58%    failed      1.19%\n",
      "   6  2023_000248                                      0.33%    passed     52.02%\n",
      "   7  2023_000307                                      0.08%    passed     77.63%\n",
      "   8  2023_000566                                      0.00%    passed     60.93%\n",
      "   9  Raffaelea_arxii_CBS_273_70                      12.36%    passed     62.37%\n",
      "  10  Ophiostoma_novo_ulmi_subsp_novo_ulmi_CMW43870   15.57%    passed     99.82%\n",
      "  11  Heinzbutinia_microspora_CMW17152                12.11%    failed      0.76%\n",
      "  12  Heinzbutinia_grandicarpa_CMW1600                17.63%    passed     14.53%\n",
      "  13  Ophiostoma_quercus_CMW2465                      12.03%    passed     93.94%\n",
      "  14  Raffaelea_crossotarsa_CMW44793                  17.71%    passed     91.66%\n",
      "  15  Harringtonia_lauricola_CMW36261                 14.50%    failed      0.00%\n",
      "  16  Aspergillus_egyptiacus_CBS_656_73               23.15%    failed      1.00%\n",
      "WARNING: 1 sequences contain more than 50% gaps/ambiguity\n",
      "****  TOTAL                                           12.18%  4 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.001 seconds\n",
      "Measuring multi-threading efficiency up to 32 CPU cores\n",
      "Increase to 10 rounds for branch lengths\n",
      "3873 trees examined\n",
      "Threads: 1 / Time: 32.007 sec / Speedup: 1.000 / Efficiency: 100% / LogL: -8440\n",
      "Threads: 2 / Time: 41.204 sec / Speedup: 0.777 / Efficiency: 39% / LogL: -8440\n",
      "BEST NUMBER OF THREADS: 1\n",
      "\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -7041.114\n",
      "Optimal log-likelihood: -7041.025\n",
      "Rate parameters:  A-C: 1.23295  A-G: 3.08818  A-T: 0.58734  C-G: 1.72637  C-T: 8.67648  G-T: 1.00000\n",
      "Base frequencies:  A: 0.229  C: 0.276  G: 0.306  T: 0.189\n",
      "Proportion of invariable sites: 0.263\n",
      "Gamma shape alpha: 0.528\n",
      "Parameters optimization took 1 rounds (0.007 sec)\n",
      "Time for fast ML tree search: 0.074 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 4 MB RAM!\n",
      "ModelFinder will test up to 286 DNA models (sample size: 1214) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "  1  GTR+F         7466.474     37  15006.948    15009.339    15195.710\n",
      "  2  GTR+F+I       7110.468     38  14296.936    14299.458    14490.799\n",
      "  3  GTR+F+G4      7042.931     38  14161.861    14164.384    14355.725\n",
      "  4  GTR+F+I+G4    7040.972     39  14159.944    14162.602    14358.910\n",
      "  5  GTR+F+R2      7079.154     39  14236.308    14238.965    14435.273\n",
      "  6  GTR+F+R3      7064.145     41  14210.290    14213.229    14419.459\n",
      "  7  GTR+F+R4      7051.088     43  14188.177    14191.411    14407.549\n",
      "  8  GTR+F+R5      7047.345     45  14184.690    14188.234    14414.265\n",
      " 16  SYM+G4        7070.228     35  14210.456    14212.595    14389.015\n",
      " 17  SYM+I+G4      7069.040     36  14210.079    14212.342    14393.739\n",
      " 29  TVM+F+G4      7079.586     37  14233.172    14235.564    14421.934\n",
      " 30  TVM+F+I+G4    7079.596     38  14235.192    14237.714    14429.055\n",
      " 42  TVMe+G4       7090.250     34  14248.500    14250.519    14421.957\n",
      " 43  TVMe+I+G4     7090.422     35  14250.844    14252.983    14429.402\n",
      " 55  TIM3+F+G4     7046.541     36  14165.083    14167.346    14348.743\n",
      " 56  TIM3+F+I+G4   7044.759     37  14163.519    14165.910    14352.281\n",
      " 68  TIM3e+G4      7083.555     33  14233.111    14235.013    14401.466\n",
      " 69  TIM3e+I+G4    7082.421     34  14232.842    14234.861    14406.299\n",
      " 81  TIM2+F+G4     7049.153     36  14170.305    14172.568    14353.965\n",
      " 82  TIM2+F+I+G4   7046.976     37  14167.951    14170.342    14356.713\n",
      " 94  TIM2e+G4      7093.935     33  14253.871    14255.773    14422.226\n",
      " 95  TIM2e+I+G4    7092.877     34  14253.754    14255.773    14427.211\n",
      "107  TIM+F+G4      7050.784     36  14173.569    14175.832    14357.229\n",
      "108  TIM+F+I+G4    7048.969     37  14171.939    14174.330    14360.701\n",
      "120  TIMe+G4       7098.438     33  14262.876    14264.778    14431.231\n",
      "121  TIMe+I+G4     7097.862     34  14263.723    14265.742    14437.180\n",
      "133  TPM3u+F+G4    7083.259     35  14236.518    14238.658    14415.077\n",
      "134  TPM3u+F+I+G4  7083.375     36  14238.750    14241.013    14422.410\n",
      "146  TPM3+F+G4     7083.259     35  14236.518    14238.658    14415.077\n",
      "147  TPM3+F+I+G4   7083.375     36  14238.750    14241.013    14422.410\n",
      "159  TPM2u+F+G4    7086.091     35  14242.181    14244.320    14420.740\n",
      "160  TPM2u+F+I+G4  7086.334     36  14244.667    14246.931    14428.327\n",
      "172  TPM2+F+G4     7086.091     35  14242.181    14244.320    14420.740\n",
      "173  TPM2+F+I+G4   7086.334     36  14244.667    14246.931    14428.327\n",
      "185  K3Pu+F+G4     7087.548     35  14245.095    14247.235    14423.654\n",
      "186  K3Pu+F+I+G4   7088.075     36  14248.150    14250.413    14431.810\n",
      "198  K3P+G4        7117.500     32  14299.000    14300.788    14462.253\n",
      "199  K3P+I+G4      7118.370     33  14302.740    14304.642    14471.096\n",
      "211  TN+F+G4       7052.529     35  14175.059    14177.198    14353.617\n",
      "212  TN+F+I+G4     7050.449     36  14172.897    14175.161    14356.558\n",
      "224  TNe+G4        7105.531     32  14275.062    14276.850    14438.316\n",
      "225  TNe+I+G4      7104.590     33  14275.180    14277.081    14443.535\n",
      "237  HKY+F+G4      7089.568     34  14247.135    14249.154    14420.592\n",
      "238  HKY+F+I+G4    7089.859     35  14249.719    14251.858    14428.277\n",
      "250  K2P+G4        7124.881     31  14311.762    14313.440    14469.913\n",
      "251  K2P+I+G4      7125.446     32  14314.891    14316.680    14478.145\n",
      "263  F81+F+G4      7293.254     33  14652.507    14654.409    14820.862\n",
      "264  F81+F+I+G4    7296.935     34  14661.870    14663.889    14835.327\n",
      "276  JC+G4         7295.733     30  14651.465    14653.037    14804.515\n",
      "277  JC+I+G4       7299.194     31  14660.389    14662.067    14818.541\n",
      "Akaike Information Criterion:           GTR+F+I+G4\n",
      "Corrected Akaike Information Criterion: GTR+F+I+G4\n",
      "Bayesian Information Criterion:         TIM3+F+G4\n",
      "Best-fit model: TIM3+F+G4 chosen according to BIC\n",
      "\n",
      "All model information printed to iqtree_genetree/combined_RPBII_sequences.model.gz\n",
      "CPU time for ModelFinder: 115.051 seconds (0h:1m:55s)\n",
      "Wall-clock time for ModelFinder: 75.163 seconds (0h:1m:15s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 5566)...\n",
      "\n",
      "NOTE: 2 MB RAM (0 GB) is required!\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -7046.541\n",
      "Optimal log-likelihood: -7046.538\n",
      "Rate parameters:  A-C: 1.87628  A-G: 3.80507  A-T: 1.00000  C-G: 1.87628  C-T: 10.39610  G-T: 1.00000\n",
      "Base frequencies:  A: 0.229  C: 0.276  G: 0.306  T: 0.189\n",
      "Gamma shape alpha: 0.270\n",
      "Parameters optimization took 1 rounds (0.004 sec)\n",
      "Computing ML distances based on estimated model parameters... 0.002 sec\n",
      "Computing BIONJ tree...\n",
      "0.005 seconds\n",
      "Log-likelihood of BIONJ tree: -7045.255\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 98 parsimony trees... 0.084 second\n",
      "Computing log-likelihood of 98 initial trees ... 0.118 seconds\n",
      "Current best score: -7045.255\n",
      "\n",
      "Do NNI search on 20 best initial trees\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "BETTER TREE FOUND at iteration 1: -7044.979\n",
      "Iteration 10 / LogL: -7047.298 / Time: 0h:0m:0s\n",
      "Iteration 20 / LogL: -7047.515 / Time: 0h:0m:0s\n",
      "Finish initializing candidate tree set (4)\n",
      "Current best tree score: -7044.979 / CPU time: 0.592\n",
      "Number of iterations: 20\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 30 / LogL: -7045.396 / Time: 0h:0m:0s (0h:0m:2s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -7044.972\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -7044.851\n",
      "Iteration 40 / LogL: -7045.340 / Time: 0h:0m:1s (0h:0m:1s left)\n",
      "Iteration 50 / LogL: -7044.866 / Time: 0h:0m:1s (0h:0m:1s left)\n",
      "Log-likelihood cutoff on original alignment: -7068.905\n",
      "Iteration 60 / LogL: -7044.938 / Time: 0h:0m:2s (0h:0m:1s left)\n",
      "Iteration 70 / LogL: -7044.955 / Time: 0h:0m:2s (0h:0m:1s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -7044.842\n",
      "Iteration 80 / LogL: -7047.539 / Time: 0h:0m:3s (0h:0m:0s left)\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -7044.810\n",
      "Iteration 90 / LogL: -7044.810 / Time: 0h:0m:3s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -7046.151 / Time: 0h:0m:4s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -7068.905\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.999\n",
      "TREE SEARCH COMPLETED AFTER 102 ITERATIONS / Time: 0h:0m:4s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -7044.810\n",
      "Optimal log-likelihood: -7044.808\n",
      "Rate parameters:  A-C: 1.87596  A-G: 3.79532  A-T: 1.00000  C-G: 1.87596  C-T: 10.38504  G-T: 1.00000\n",
      "Base frequencies:  A: 0.229  C: 0.276  G: 0.306  T: 0.189\n",
      "Gamma shape alpha: 0.273\n",
      "Parameters optimization took 1 rounds (0.009 sec)\n",
      "BEST SCORE FOUND : -7044.808\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.360 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file iqtree_genetree/combined_RPBII_sequences.splits.nex\n",
      "Total tree length: 3.444\n",
      "\n",
      "Total number of iterations: 102\n",
      "CPU time used for tree search: 4.009 sec (0h:0m:4s)\n",
      "Wall-clock time used for tree search: 4.079 sec (0h:0m:4s)\n",
      "Total CPU time used: 4.444 sec (0h:0m:4s)\n",
      "Total wall-clock time used: 4.532 sec (0h:0m:4s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file iqtree_genetree/combined_RPBII_sequences.splits.nex...\n",
      "16 taxa and 81 splits.\n",
      "Consensus tree written to iqtree_genetree/combined_RPBII_sequences.contree\n",
      "Reading input trees file iqtree_genetree/combined_RPBII_sequences.contree\n",
      "Log-likelihood of consensus tree: -7044.812\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                iqtree_genetree/combined_RPBII_sequences.iqtree\n",
      "  Maximum-likelihood tree:       iqtree_genetree/combined_RPBII_sequences.treefile\n",
      "  Likelihood distances:          iqtree_genetree/combined_RPBII_sequences.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          iqtree_genetree/combined_RPBII_sequences.splits.nex\n",
      "  Consensus tree:                iqtree_genetree/combined_RPBII_sequences.contree\n",
      "  Screen log file:               iqtree_genetree/combined_RPBII_sequences.log\n",
      "\n",
      "Date and Time: Sun Jul 14 02:10:38 2024\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import subprocess\n",
    "\n",
    "# Create the output directory if it doesn't exist\n",
    "output_dir = 'iqtree_genetree'\n",
    "os.makedirs(output_dir, exist_ok=True)\n",
    "\n",
    "# Directory containing the trimmed sequences\n",
    "input_dir = 'trimmed'\n",
    "\n",
    "# Loop through each file in the input directory\n",
    "for filename in os.listdir(input_dir):\n",
    "    if filename.endswith('_trimmed.fasta'):\n",
    "        gene_name = filename.replace('_trimmed.fasta', '')\n",
    "        \n",
    "        # Construct the iqtree2 command\n",
    "        cmd = [\n",
    "            'iqtree2',\n",
    "            '-s', os.path.join(input_dir, filename),\n",
    "            '--prefix', os.path.join(output_dir, gene_name),\n",
    "            '-B', '1000',\n",
    "            '-nt', 'AUTO',\n",
    "            '-alrt', '1000',\n",
    "            '--seed', '5566'\n",
    "        ]\n",
    "        \n",
    "        # Run the command\n",
    "        subprocess.run(cmd)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "id": "complex-message",
   "metadata": {},
   "outputs": [],
   "source": [
    "!mkdir iqtree_genetree/astral/\n",
    "!cat iqtree_genetree/*.contree > iqtree_genetree/astral/all.contree"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "id": "protecting-clerk",
   "metadata": {},
   "outputs": [],
   "source": [
    "!astral \\\n",
    "-i iqtree_genetree/astral/all.contree \\\n",
    "-o iqtree_genetree/astral/all_astral.contree \\\n",
    "2> iqtree_genetree/astral/astral_out.log"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e346f6a0-0b8b-4e24-9487-2d5edfeba161",
   "metadata": {},
   "source": [
    "### 5.2 MrBayes\n",
    "\n",
    "Bayesian inference"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "id": "e111c847-1cc3-443f-96de-372a841049ec",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Function to translate model settings for MrBayes\n",
    "def mb_model(model, applyto):\n",
    "    r = \"\"\n",
    "    MRBAYES_AVAILABLE_MODEL = {\n",
    "        \"GTR\": {\n",
    "            \"nst\": \"6\",\n",
    "        },\n",
    "        \"SYM\": {\n",
    "            \"nst\": \"6\",\n",
    "            \"statefreqpr\": \"fixed(equal)\"\n",
    "        },\n",
    "        \"HKY\": {\n",
    "            \"nst\": \"2\",\n",
    "        },\n",
    "        \"K80\": {\n",
    "            \"nst\": \"2\",\n",
    "            \"statefreqpr\": \"fixed(equal)\"\n",
    "        },\n",
    "        \"F81\": {\n",
    "            \"nst\": \"1\",\n",
    "        },\n",
    "        \"JC\": {\n",
    "            \"nst\": \"1\",\n",
    "            \"statefreqpr\": \"fixed(equal)\"\n",
    "        },\n",
    "    }\n",
    "    rates = \"equal\"\n",
    "    if \"+I+G4\" in model:\n",
    "        rates = \"invgamma\"\n",
    "        model = model.replace(\"+I+G4\", \"\")\n",
    "    if \"+I\" in model:\n",
    "        rates = \"propinv\"\n",
    "        model = model.replace(\"+I\", \"\")\n",
    "    if \"+G4\" in model:\n",
    "        rates = \"gamma\"\n",
    "        model = model.replace(\"+G4\", \"\")\n",
    "    model_conf = MRBAYES_AVAILABLE_MODEL[model]\n",
    "    r += \"lset applyto=({}) nst={} rates={};\\n\".format(applyto, model_conf[\"nst\"], rates)\n",
    "    if \"statefreqpr\" in model_conf:\n",
    "        r += \"prset applyto=({}) statefreqpr={};\\n\".format(applyto, model_conf[\"statefreqpr\"])\n",
    "    return r"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "id": "d377ec5a-013b-441c-808a-1bdc42074501",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "nexus saved to mrbayes/mb.nex\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "\n",
    "# Set the input file paths\n",
    "fasta_file_path = 'concatenated.fasta'\n",
    "charset_file_path = 'modeltest_seqs/mb_charset.txt'\n",
    "lset_file_path = 'modeltest_seqs/mb_lset.txt'\n",
    "nexus_file_path = 'mrbayes/mb.nex'\n",
    "\n",
    "# Ensure the output directory exists\n",
    "output_directory = os.path.dirname(nexus_file_path)\n",
    "os.makedirs(output_directory, exist_ok=True)\n",
    "\n",
    "# Read sequences from concatenated.fasta\n",
    "with open(fasta_file_path, 'r') as fasta_file:\n",
    "    sequences = fasta_file.read()\n",
    "\n",
    "# Count the number of taxa and characters\n",
    "num_taxa = sequences.count('>')\n",
    "\n",
    "# Extract sequences and their lengths\n",
    "sequence_list = sequences.split('>')\n",
    "sequence_list = [seq for seq in sequence_list if seq]  # Remove empty entries\n",
    "sequences_dict = {}\n",
    "for seq in sequence_list:\n",
    "    lines = seq.split('\\n')\n",
    "    taxon = lines[0]\n",
    "    sequence = ''.join(lines[1:])\n",
    "    sequences_dict[taxon] = sequence\n",
    "\n",
    "num_characters = len(next(iter(sequences_dict.values())))\n",
    "\n",
    "# Read charset information from mb_charset.txt\n",
    "with open(charset_file_path, 'r') as charset_file:\n",
    "    charset_info = charset_file.read()\n",
    "\n",
    "# Read model settings from mb_lset.txt\n",
    "with open(lset_file_path, 'r') as lset_file:\n",
    "    lset_info = lset_file.read()\n",
    "\n",
    "# Extract partition information from charset_info\n",
    "partition_lines = charset_info.split('\\n')\n",
    "charset_lines = [line for line in partition_lines if line.startswith('charset')]\n",
    "partition_definition = [line for line in partition_lines if line.startswith('partition')][0]\n",
    "\n",
    "# Define the content of the NEXUS file\n",
    "nexus_content = f\"\"\"#NEXUS\n",
    "\n",
    "BEGIN DATA;\n",
    "    DIMENSIONS NTAX={num_taxa} NCHAR={num_characters};\n",
    "    FORMAT DATATYPE=DNA MISSING=? GAP=-;\n",
    "    MATRIX\n",
    "\"\"\"\n",
    "\n",
    "for taxon, sequence in sequences_dict.items():\n",
    "    nexus_content += f\"    {taxon}    {sequence}\\n\"\n",
    "\n",
    "nexus_content += \"\"\"    ;\n",
    "END;\n",
    "\n",
    "BEGIN SETS;\n",
    "\"\"\"\n",
    "\n",
    "for line in charset_lines:\n",
    "    nexus_content += f\"    {line}\\n\"\n",
    "\n",
    "nexus_content += \"\"\"END;\n",
    "\n",
    "BEGIN PARTITIONS;\n",
    "\"\"\"\n",
    "\n",
    "nexus_content += f\"    {partition_definition}\\n\"\n",
    "\n",
    "nexus_content += \"\"\"END;\n",
    "\n",
    "BEGIN MRBAYES;\n",
    "    set seed=5566;\n",
    "    set swapseed=5566;\n",
    "\"\"\"\n",
    "\n",
    "for line in charset_lines:\n",
    "    nexus_content += f\"    {line}\\n\"\n",
    "\n",
    "nexus_content += f\"\"\"\n",
    "    {partition_definition}\n",
    "    set partition=favored;\n",
    "\"\"\"\n",
    "\n",
    "# Append model settings using mb_model function\n",
    "lset_lines = lset_info.split('\\n')\n",
    "part_num = 1\n",
    "for line in lset_lines:\n",
    "    if line.strip():\n",
    "        parts = line.split()\n",
    "        model = parts[2].split('=')[1].strip(';')\n",
    "        nexus_content += mb_model(model, part_num)\n",
    "        part_num += 1\n",
    "\n",
    "nexus_content += \"\"\"\n",
    "    mcmcp ngen=1000000 nruns=2 samplefreq=100 savebrlens=yes nchains=4 filename=mb;\n",
    "    mcmc;\n",
    "    sumt filename=mb burnin=2000 contype=allcompat showtreeprobs=no;\n",
    "END;\n",
    "\"\"\"\n",
    "\n",
    "# Write the content to input.nex\n",
    "with open(nexus_file_path, 'w') as nex_file:\n",
    "    nex_file.write(nexus_content)\n",
    "\n",
    "print(f'nexus saved to {nexus_file_path}')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "id": "bafe4fc3-d64f-4759-9c52-21151a5c34d6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "                            MrBayes 3.2.7a x86_64\n",
      "\n",
      "                      (Bayesian Analysis of Phylogeny)\n",
      "\n",
      "              Distributed under the GNU General Public License\n",
      "\n",
      "\n",
      "               Type \"help\" or \"help <command>\" for information\n",
      "                     on the commands that are available.\n",
      "\n",
      "                   Type \"about\" for authorship and general\n",
      "                       information about the program.\n",
      "\n",
      "\n",
      "\n",
      "   Executing file \"mrbayes/mb.nex\"\n",
      "   UNIX line termination\n",
      "   Longest line length = 2620\n",
      "   Parsing file\n",
      "   Expecting NEXUS formatted file\n",
      "   Reading data block\n",
      "      Allocated taxon set\n",
      "      Allocated matrix\n",
      "      Defining new matrix with 50 taxa and 2566 characters\n",
      "      Data is Dna\n",
      "      Missing data coded as ?\n",
      "      Gaps coded as -\n",
      "      Taxon  1 -> Ophiostoma_quercus_CMW22909\n",
      "      Taxon  2 -> 2022_000207\n",
      "      Taxon  3 -> Heinzbutinia_microspora_CMW17152\n",
      "      Taxon  4 -> Leptographium_procerum_CMW_34542\n",
      "      Taxon  5 -> Heinzbutinia_microspora_CBS_440_69\n",
      "      Taxon  6 -> Harringtonia_lauricola_CBS_121567\n",
      "      Taxon  7 -> Raffaelea_tritirachium_CBS_726_69\n",
      "      Taxon  8 -> 2021_000001\n",
      "      Taxon  9 -> Harringtonia_lauricola_voucher_Hulcr7164\n",
      "      Taxon 10 -> Ophiostoma_noisomeae_CMW40326\n",
      "      Taxon 11 -> Ophiostoma_novo_ulmi_subsp_novo_ulmi_CMW43870\n",
      "      Taxon 12 -> Raffaelea_xyleborini_CMW_45859\n",
      "      Taxon 13 -> Heinzbutinia_grandicarpa_CMW41131\n",
      "      Taxon 14 -> Raffaelea_canadensis_CBS_168_66\n",
      "      Taxon 15 -> Leptographium_procerum_CMW45\n",
      "      Taxon 16 -> 2023_000307\n",
      "      Taxon 17 -> Ophiostoma_floccosum_CBS_799_73\n",
      "      Taxon 18 -> 2021_000003\n",
      "      Taxon 19 -> Raffaelea_canadensis_CBS_805_70\n",
      "      Taxon 20 -> 2023_000565\n",
      "      Taxon 21 -> 2021_000005\n",
      "      Taxon 22 -> Raffaelea_crossotarsa_CMW44793\n",
      "      Taxon 23 -> Ophiostoma_floccosum_CMW12623\n",
      "      Taxon 24 -> Aspergillus_egyptiacus_CBS_656_73\n",
      "      Taxon 25 -> Raffaelea_ambrosiae_CBS185_64\n",
      "      Taxon 26 -> Leptographium_procerum_CBS_594_69\n",
      "      Taxon 27 -> Raffaelea_sp_CMW51555\n",
      "      Taxon 28 -> Hawksworthiomyces_ciconiae_CBS_147964\n",
      "      Taxon 29 -> Raffaelea_cyclorhipidia_CMW44790\n",
      "      Taxon 30 -> Heinzbutinia_grandicarpa_CBS_250_88\n",
      "      Taxon 31 -> Ophiostoma_quercus_CMW2465\n",
      "      Taxon 32 -> Ophiostoma_quercus_CMW43227\n",
      "      Taxon 33 -> Ophiostoma_quercus_CMW40208\n",
      "      Taxon 34 -> Hawksworthiomyces_hibbettii_CMW_37663\n",
      "      Taxon 35 -> Esteya_vermicola_CBS_115803\n",
      "      Taxon 36 -> Leptographium_latens_CBS_124023\n",
      "      Taxon 37 -> Leptographium_procerum_CBS_929_69\n",
      "      Taxon 38 -> 2023_000248\n",
      "      Taxon 39 -> Harringtonia_lauricola_isolate_CV2018003\n",
      "      Taxon 40 -> Ophiostoma_ulmi_CBS_106_63\n",
      "      Taxon 41 -> Raffaelea_arxii_CBS_273_70\n",
      "      Taxon 42 -> 2023_000566\n",
      "      Taxon 43 -> Leptographium_procerum_CMW_29994\n",
      "      Taxon 44 -> 2022_000206\n",
      "      Taxon 45 -> Raffaelea_scolytodis_CBS_124461\n",
      "      Taxon 46 -> Ophiostoma_floccosum_CMW12622\n",
      "      Taxon 47 -> Ophiostoma_quercus_CMW41732\n",
      "      Taxon 48 -> Ophiostoma_ulmi_CBS_151_55\n",
      "      Taxon 49 -> Harringtonia_lauricola_CMW36261\n",
      "      Taxon 50 -> Heinzbutinia_grandicarpa_CMW1600\n",
      "      Successfully read matrix\n",
      "      Setting default partition (does not divide up characters)\n",
      "      Setting model defaults\n",
      "      Seed (for generating default start values) = 1720896239\n",
      "      Setting output file names to \"mrbayes/mb.nex.run<i>.<p|t>\"\n",
      "   Exiting data block\n",
      "   Skipping \"SETS\" block\n",
      "   Skipping \"PARTITIONS\" block\n",
      "   Reading mrbayes block\n",
      "      Setting seed to 5566\n",
      "      Setting swapseed to 5566\n",
      "      Defining charset called 'part0'\n",
      "      Defining charset called 'part1'\n",
      "      Defining charset called 'part2'\n",
      "      Defining partition called 'favored'\n",
      "      Setting favored as the partition, dividing characters into 3 parts.\n",
      "      Setting model defaults\n",
      "      Seed (for generating default start values) = 5566\n",
      "      Setting Nst to 6 for partition 1\n",
      "      Setting Rates to Invgamma for partition 1\n",
      "      Successfully set likelihood model parameters to\n",
      "      partition 1 (if applicable)\n",
      "      Setting Nst to 6 for partition 2\n",
      "      Setting Rates to Gamma for partition 2\n",
      "      Successfully set likelihood model parameters to\n",
      "      partition 2 (if applicable)\n",
      "      Setting Statefreqpr to Fixed(Equal) for partition 2\n",
      "      Successfully set prior model parameters to\n",
      "      partition 2 (if applicable)\n",
      "      Setting Nst to 6 for partition 3\n",
      "      Setting Rates to Gamma for partition 3\n",
      "      Successfully set likelihood model parameters to\n",
      "      partition 3 (if applicable)\n",
      "      Setting number of generations to 1000000\n",
      "      Setting number of runs to 2\n",
      "      Setting sample frequency to 100\n",
      "      Setting number of chains to 4\n",
      "      Setting chain output file names to \"mb.run<i>.<p/t>\"\n",
      "      Successfully set chain parameters\n",
      "      Running Markov chain\n",
      "      MCMC stamp = 3910121092\n",
      "      Seed = 273418564\n",
      "      Swapseed = 5566\n",
      "      Model settings:\n",
      "\n",
      "         Settings for partition 1 --\n",
      "            Datatype  = DNA\n",
      "            Nucmodel  = 4by4\n",
      "            Nst       = 6\n",
      "                        Substitution rates, expressed as proportions\n",
      "                        of the rate sum, have a Dirichlet prior\n",
      "                        (1.00,1.00,1.00,1.00,1.00,1.00)\n",
      "            Covarion  = No\n",
      "            # States  = 4\n",
      "                        State frequencies have a Dirichlet prior\n",
      "                        (1.00,1.00,1.00,1.00)\n",
      "            Rates     = Invgamma\n",
      "                        The distribution is approximated using 4 categories.\n",
      "                        Shape parameter is exponentially\n",
      "                        distributed with parameter (1.00).\n",
      "                        Proportion of invariable sites is uniformly dist-\n",
      "                        ributed on the interval (0.00,1.00).\n",
      "\n",
      "         Settings for partition 2 --\n",
      "            Datatype  = DNA\n",
      "            Nucmodel  = 4by4\n",
      "            Nst       = 6\n",
      "                        Substitution rates, expressed as proportions\n",
      "                        of the rate sum, have a Dirichlet prior\n",
      "                        (1.00,1.00,1.00,1.00,1.00,1.00)\n",
      "            Covarion  = No\n",
      "            # States  = 4\n",
      "                        State frequencies are fixed to be equal\n",
      "            Rates     = Gamma\n",
      "                        The distribution is approximated using 4 categories.\n",
      "                        Shape parameter is exponentially\n",
      "                        distributed with parameter (1.00).\n",
      "\n",
      "         Settings for partition 3 --\n",
      "            Datatype  = DNA\n",
      "            Nucmodel  = 4by4\n",
      "            Nst       = 6\n",
      "                        Substitution rates, expressed as proportions\n",
      "                        of the rate sum, have a Dirichlet prior\n",
      "                        (1.00,1.00,1.00,1.00,1.00,1.00)\n",
      "            Covarion  = No\n",
      "            # States  = 4\n",
      "                        State frequencies have a Dirichlet prior\n",
      "                        (1.00,1.00,1.00,1.00)\n",
      "            Rates     = Gamma\n",
      "                        The distribution is approximated using 4 categories.\n",
      "                        Shape parameter is exponentially\n",
      "                        distributed with parameter (1.00).\n",
      "\n",
      "      Active parameters: \n",
      "\n",
      "                             Partition(s)\n",
      "         Parameters          1  2  3\n",
      "         ---------------------------\n",
      "         Revmat              1  1  1\n",
      "         Statefreq           2  2  2\n",
      "         Shape               3  3  3\n",
      "         Pinvar              4  .  .\n",
      "         Ratemultiplier      5  5  5\n",
      "         Topology            6  6  6\n",
      "         Brlens              7  7  7\n",
      "         ---------------------------\n",
      "\n",
      "         Parameters can be linked or unlinked across partitions using 'link' and 'unlink'\n",
      "\n",
      "         1 --  Parameter  = Revmat{all}\n",
      "               Type       = Rates of reversible rate matrix\n",
      "               Prior      = Dirichlet(1.00,1.00,1.00,1.00,1.00,1.00)\n",
      "               Partitions = All\n",
      "\n",
      "         2 --  Parameter  = Pi{all}\n",
      "               Type       = Stationary state frequencies\n",
      "               Prior      = Dirichlet\n",
      "               Partitions = All\n",
      "\n",
      "         3 --  Parameter  = Alpha{all}\n",
      "               Type       = Shape of scaled gamma distribution of site rates\n",
      "               Prior      = Exponential(1.00)\n",
      "               Partitions = All\n",
      "\n",
      "         4 --  Parameter  = Pinvar{1}\n",
      "               Type       = Proportion of invariable sites\n",
      "               Prior      = Uniform(0.00,1.00)\n",
      "               Partition  = 1\n",
      "\n",
      "         5 --  Parameter  = Ratemultiplier{all}\n",
      "               Type       = Partition-specific rate multiplier\n",
      "               Prior      = Fixed(1.0)\n",
      "               Partitions = All\n",
      "\n",
      "         6 --  Parameter  = Tau{all}\n",
      "               Type       = Topology\n",
      "               Prior      = All topologies equally probable a priori\n",
      "               Partitions = All\n",
      "               Subparam.  = V{all}\n",
      "\n",
      "         7 --  Parameter  = V{all}\n",
      "               Type       = Branch lengths\n",
      "               Prior      = Unconstrained:GammaDir(1.0,0.1000,1.0,1.0)\n",
      "               Partitions = All\n",
      "\n",
      "\n",
      "\n",
      "      The MCMC sampler will use the following moves:\n",
      "         With prob.  Chain will use move\n",
      "            0.93 %   Dirichlet(Revmat{all})\n",
      "            0.93 %   Slider(Revmat{all})\n",
      "            0.93 %   Dirichlet(Pi{all})\n",
      "            0.93 %   Slider(Pi{all})\n",
      "            1.85 %   Multiplier(Alpha{all})\n",
      "            1.85 %   Slider(Pinvar{1})\n",
      "            9.26 %   ExtSPR(Tau{all},V{all})\n",
      "            9.26 %   ExtTBR(Tau{all},V{all})\n",
      "            9.26 %   NNI(Tau{all},V{all})\n",
      "            9.26 %   ParsSPR(Tau{all},V{all})\n",
      "           37.04 %   Multiplier(V{all})\n",
      "           12.96 %   Nodeslider(V{all})\n",
      "            5.56 %   TLMultiplier(V{all})\n",
      "\n",
      "      Division 1 has 514 unique site patterns\n",
      "      Division 2 has 379 unique site patterns\n",
      "      Division 3 has 490 unique site patterns\n",
      "      Initializing conditional likelihoods\n",
      "\n",
      "      Running benchmarks to automatically select fastest BEAGLE resource... \n",
      "      Selected resource is the CPU, changing to multi-instance BEAGLE mode\n",
      "\n",
      "      Running benchmarks to automatically select fastest BEAGLE resource... \n",
      "      Using BEAGLE v3.1.2 resource 0 for division 1:\n",
      "         Rsrc Name : CPU\n",
      "         Impl Name : CPU-4State-Single\n",
      "         Flags: PROCESSOR_CPU PRECISION_SINGLE COMPUTATION_SYNCH EIGEN_REAL\n",
      "                SCALING_MANUAL SCALERS_RAW VECTOR_NONE THREADING_CPP         MODEL STATES: 4\n",
      "\n",
      "      Running benchmarks to automatically select fastest BEAGLE resource... \n",
      "      Using BEAGLE v3.1.2 resource 0 for division 2:\n",
      "         Rsrc Name : CPU\n",
      "         Impl Name : CPU-4State-Single\n",
      "         Flags: PROCESSOR_CPU PRECISION_SINGLE COMPUTATION_SYNCH EIGEN_REAL\n",
      "                SCALING_MANUAL SCALERS_RAW VECTOR_NONE THREADING_CPP         MODEL STATES: 4\n",
      "\n",
      "      Running benchmarks to automatically select fastest BEAGLE resource... \n",
      "      Using BEAGLE v3.1.2 resource 0 for division 3:\n",
      "         Rsrc Name : CPU\n",
      "         Impl Name : CPU-4State-Single\n",
      "         Flags: PROCESSOR_CPU PRECISION_SINGLE COMPUTATION_SYNCH EIGEN_REAL\n",
      "                SCALING_MANUAL SCALERS_RAW VECTOR_NONE THREADING_CPP         MODEL STATES: 4\n",
      "      Initializing invariable-site conditional likelihoods\n",
      "\n",
      "      Initial log likelihoods and log prior probs for run 1:\n",
      "         Chain 1 -- -33543.544587 -- 113.409609\n",
      "         Chain 2 -- -32801.811635 -- 113.409609\n",
      "         Chain 3 -- -33962.515269 -- 113.409609\n",
      "         Chain 4 -- -32398.347315 -- 113.409609\n",
      "\n",
      "      Initial log likelihoods and log prior probs for run 2:\n",
      "         Chain 1 -- -33131.677338 -- 113.409609\n",
      "         Chain 2 -- -33441.520998 -- 113.409609\n",
      "         Chain 3 -- -33604.659173 -- 113.409609\n",
      "         Chain 4 -- -33110.147297 -- 113.409609\n",
      "\n",
      "\n",
      "      Using a relative burnin of 25.0 % for diagnostics\n",
      "\n",
      "      Chain results (1000000 generations requested):\n",
      "\n",
      "          0 -- [-33543.545] (-32801.812) (-33962.515) (-32398.347) * [-33131.677] (-33441.521) (-33604.659) (-33110.147) \n",
      "       1000 -- (-23841.988) (-22469.835) (-22949.319) [-22094.547] * (-22951.469) (-21475.312) [-21065.520] (-22684.192) -- 0:49:57\n",
      "       2000 -- (-20274.781) [-20033.265] (-20591.139) (-20076.509) * (-20062.633) [-19507.442] (-19660.644) (-20230.929) -- 0:58:13\n",
      "       3000 -- (-19234.865) [-19048.210] (-19503.839) (-19168.373) * (-19268.861) [-18996.644] (-19045.129) (-19459.842) -- 1:00:55\n",
      "       4000 -- (-18758.325) (-18742.557) (-18940.091) [-18735.241] * (-18902.812) (-18796.780) [-18664.262] (-19073.938) -- 0:58:06\n",
      "       5000 -- [-18533.762] (-18566.027) (-18631.652) (-18603.543) * (-18813.414) (-18634.242) [-18571.111] (-18855.404) -- 0:59:42\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.157036\n",
      "\n",
      "       6000 -- (-18516.158) (-18533.367) (-18498.478) [-18479.731] * (-18598.540) [-18535.535] (-18522.029) (-18637.738) -- 0:57:59\n",
      "       7000 -- (-18501.497) (-18496.784) (-18464.479) [-18434.883] * [-18500.482] (-18514.733) (-18457.254) (-18591.203) -- 0:59:06\n",
      "       8000 -- (-18487.095) (-18484.477) [-18459.709] (-18439.121) * [-18460.671] (-18503.929) (-18445.005) (-18554.781) -- 0:59:56\n",
      "       9000 -- (-18477.504) (-18495.972) [-18451.495] (-18453.740) * (-18447.383) (-18477.255) [-18435.265] (-18479.895) -- 1:00:33\n",
      "      10000 -- (-18486.029) (-18492.606) (-18464.361) [-18433.597] * [-18448.554] (-18477.975) (-18434.532) (-18468.161) -- 1:01:03\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.129970\n",
      "\n",
      "      11000 -- (-18465.367) (-18485.207) (-18450.630) [-18434.402] * (-18454.706) (-18492.466) [-18414.583] (-18466.920) -- 1:02:56\n",
      "      12000 -- (-18487.674) (-18459.464) (-18465.582) [-18434.281] * (-18461.589) (-18487.784) [-18412.499] (-18460.207) -- 1:03:07\n",
      "      13000 -- [-18472.651] (-18456.269) (-18479.875) (-18429.644) * (-18465.879) (-18494.356) [-18419.771] (-18450.289) -- 1:03:16\n",
      "      14000 -- (-18470.231) [-18428.837] (-18489.082) (-18440.454) * (-18460.752) (-18499.391) (-18424.125) [-18450.846] -- 1:04:33\n",
      "      15000 -- (-18469.257) [-18422.362] (-18471.238) (-18421.576) * (-18452.815) (-18492.212) [-18423.085] (-18458.136) -- 1:04:34\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.108035\n",
      "\n",
      "      16000 -- (-18448.535) (-18432.094) (-18478.494) [-18433.019] * (-18452.748) (-18476.271) [-18434.844] (-18471.484) -- 1:04:34\n",
      "      17000 -- (-18462.251) (-18456.395) (-18465.125) [-18422.390] * (-18446.984) (-18467.369) [-18433.742] (-18471.407) -- 1:04:34\n",
      "      18000 -- (-18456.502) (-18461.685) (-18462.760) [-18421.098] * (-18450.840) (-18485.391) [-18426.323] (-18455.484) -- 1:04:33\n",
      "      19000 -- (-18459.840) (-18439.579) (-18453.671) [-18421.225] * (-18460.707) (-18508.030) [-18431.644] (-18444.793) -- 1:04:32\n",
      "      20000 -- (-18465.438) (-18447.877) (-18449.351) [-18419.435] * (-18456.993) (-18496.717) [-18437.746] (-18448.641) -- 1:04:31\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.097283\n",
      "\n",
      "      21000 -- (-18474.796) (-18449.812) (-18448.373) [-18418.434] * (-18479.797) (-18485.428) (-18424.861) [-18432.782] -- 1:04:29\n",
      "      22000 -- (-18441.777) (-18451.650) (-18448.088) [-18422.824] * (-18484.068) (-18485.615) (-18439.955) [-18429.950] -- 1:04:27\n",
      "      23000 -- (-18442.054) (-18463.772) [-18439.575] (-18432.978) * (-18464.060) (-18483.600) (-18432.445) [-18437.187] -- 1:04:25\n",
      "      24000 -- (-18447.280) (-18473.764) [-18450.334] (-18453.422) * (-18482.000) (-18497.890) (-18440.625) [-18435.196] -- 1:05:04\n",
      "      25000 -- (-18437.701) (-18472.766) [-18432.347] (-18421.928) * (-18463.688) (-18468.350) (-18438.413) [-18448.157] -- 1:05:00\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.082634\n",
      "\n",
      "      26000 -- (-18454.970) (-18477.314) [-18430.763] (-18441.878) * (-18462.331) [-18458.614] (-18444.846) (-18468.564) -- 1:04:56\n",
      "      27000 -- (-18440.150) (-18475.844) (-18441.658) [-18433.294] * (-18441.677) (-18458.317) [-18446.827] (-18450.846) -- 1:04:52\n",
      "      28000 -- (-18449.375) (-18444.738) (-18457.826) [-18424.968] * [-18429.841] (-18463.043) (-18434.321) (-18438.026) -- 1:04:48\n",
      "      29000 -- (-18441.773) (-18450.187) (-18457.312) [-18435.728] * (-18438.752) (-18476.585) [-18430.103] (-18463.883) -- 1:04:44\n",
      "      30000 -- (-18450.800) (-18460.512) (-18451.012) [-18430.798] * (-18450.221) (-18478.269) [-18425.317] (-18494.983) -- 1:04:40\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.075667\n",
      "\n",
      "      31000 -- (-18446.627) (-18469.832) (-18443.313) [-18440.658] * (-18447.447) (-18477.382) [-18423.499] (-18476.258) -- 1:04:36\n",
      "      32000 -- (-18451.974) (-18461.979) (-18447.167) [-18440.536] * (-18449.024) (-18465.109) [-18419.936] (-18478.206) -- 1:04:32\n",
      "      33000 -- (-18459.624) (-18480.500) [-18433.179] (-18440.131) * (-18439.818) (-18452.262) [-18425.188] (-18458.447) -- 1:04:28\n",
      "      34000 -- (-18448.541) (-18459.513) (-18432.279) [-18434.178] * (-18444.563) [-18453.956] (-18445.454) (-18467.277) -- 1:04:24\n",
      "      35000 -- (-18435.231) (-18458.401) [-18428.465] (-18462.701) * (-18447.391) (-18452.926) [-18433.799] (-18475.340) -- 1:04:47\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.070745\n",
      "\n",
      "      36000 -- [-18440.627] (-18445.436) (-18429.840) (-18446.540) * (-18444.942) (-18448.554) [-18441.667] (-18491.333) -- 1:04:42\n",
      "      37000 -- (-18438.922) (-18450.969) [-18430.250] (-18459.211) * [-18438.305] (-18435.520) (-18435.303) (-18481.597) -- 1:04:38\n",
      "      38000 -- [-18440.153] (-18452.247) (-18462.065) (-18445.642) * (-18444.908) (-18434.176) [-18436.245] (-18473.346) -- 1:04:33\n",
      "      39000 -- [-18446.847] (-18445.588) (-18438.080) (-18460.388) * [-18448.340] (-18448.622) (-18466.338) (-18450.685) -- 1:04:28\n",
      "      40000 -- (-18462.321) (-18448.067) (-18434.185) [-18448.542] * (-18451.655) [-18448.462] (-18438.116) (-18451.267) -- 1:04:24\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.067378\n",
      "\n",
      "      41000 -- (-18458.700) (-18443.230) (-18428.797) [-18423.208] * (-18459.672) (-18458.327) [-18426.449] (-18454.781) -- 1:04:19\n",
      "      42000 -- (-18448.828) (-18443.452) (-18432.922) [-18429.331] * (-18456.064) (-18466.212) [-18421.273] (-18462.133) -- 1:04:14\n",
      "      43000 -- (-18447.329) (-18451.577) (-18425.333) [-18439.170] * (-18449.392) (-18456.286) [-18430.549] (-18463.839) -- 1:04:10\n",
      "      44000 -- (-18448.767) (-18453.134) (-18452.301) [-18430.371] * [-18438.315] (-18441.677) (-18436.788) (-18482.077) -- 1:04:27\n",
      "      45000 -- [-18428.811] (-18453.891) (-18447.661) (-18434.443) * [-18416.595] (-18444.605) (-18448.725) (-18467.010) -- 1:04:22\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.062721\n",
      "\n",
      "      46000 -- (-18446.538) (-18455.572) [-18437.767] (-18436.038) * [-18417.703] (-18447.336) (-18423.631) (-18464.230) -- 1:04:17\n",
      "      47000 -- (-18437.421) (-18453.392) (-18456.267) [-18428.721] * (-18422.778) (-18451.227) [-18419.409] (-18463.745) -- 1:04:12\n",
      "      48000 -- [-18431.995] (-18457.911) (-18468.478) (-18427.044) * (-18435.383) (-18461.894) [-18434.959] (-18449.263) -- 1:04:07\n",
      "      49000 -- (-18425.795) (-18452.757) (-18435.060) [-18422.735] * (-18432.033) (-18447.707) [-18424.357] (-18467.425) -- 1:04:02\n",
      "      50000 -- (-18438.666) (-18469.648) (-18431.841) [-18424.733] * (-18429.968) (-18437.640) [-18429.945] (-18473.527) -- 1:03:58\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.060743\n",
      "\n",
      "      51000 -- (-18437.119) (-18460.777) (-18452.381) [-18421.366] * (-18427.147) (-18438.641) [-18433.158] (-18474.832) -- 1:03:53\n",
      "      52000 -- (-18453.759) (-18448.004) (-18442.689) [-18417.376] * (-18431.357) (-18440.043) [-18430.930] (-18456.013) -- 1:04:06\n",
      "      53000 -- (-18444.771) (-18471.997) [-18433.832] (-18422.596) * [-18434.059] (-18432.214) (-18439.686) (-18460.617) -- 1:04:01\n",
      "      54000 -- (-18438.426) (-18453.971) [-18434.783] (-18447.492) * [-18430.601] (-18448.135) (-18431.225) (-18455.518) -- 1:03:56\n",
      "      55000 -- [-18442.067] (-18466.520) (-18466.830) (-18432.451) * [-18441.870] (-18452.911) (-18442.873) (-18461.063) -- 1:03:51\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.058893\n",
      "\n",
      "      56000 -- (-18441.033) (-18472.858) (-18449.534) [-18423.651] * (-18456.751) (-18458.350) [-18423.454] (-18448.192) -- 1:03:46\n",
      "      57000 -- (-18440.849) (-18474.897) (-18446.416) [-18429.242] * [-18428.215] (-18452.703) (-18430.574) (-18461.817) -- 1:03:41\n",
      "      58000 -- [-18421.536] (-18459.605) (-18450.101) (-18438.907) * (-18442.789) [-18458.104] (-18441.186) (-18446.714) -- 1:03:52\n",
      "      59000 -- (-18415.795) (-18480.698) [-18435.888] (-18454.422) * [-18427.187] (-18447.783) (-18446.395) (-18449.443) -- 1:03:47\n",
      "      60000 -- [-18405.913] (-18483.255) (-18424.802) (-18446.545) * (-18439.667) (-18470.961) [-18432.919] (-18462.222) -- 1:03:42\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.057137\n",
      "\n",
      "      61000 -- (-18423.492) (-18470.152) [-18416.774] (-18459.312) * (-18431.612) (-18467.850) [-18429.800] (-18479.461) -- 1:03:37\n",
      "      62000 -- (-18422.218) (-18454.715) [-18424.605] (-18456.222) * (-18427.432) (-18453.171) [-18432.092] (-18475.585) -- 1:03:32\n",
      "      63000 -- (-18437.555) (-18443.279) [-18426.843] (-18448.966) * (-18425.093) (-18458.060) [-18443.627] (-18470.166) -- 1:03:27\n",
      "      64000 -- (-18416.198) (-18446.389) (-18424.289) [-18428.316] * [-18433.160] (-18457.612) (-18440.070) (-18477.577) -- 1:03:22\n",
      "      65000 -- [-18428.928] (-18455.898) (-18439.414) (-18447.326) * [-18428.303] (-18467.054) (-18438.407) (-18455.114) -- 1:03:17\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.059810\n",
      "\n",
      "      66000 -- (-18448.263) (-18441.261) (-18441.148) [-18428.411] * [-18421.592] (-18469.399) (-18455.487) (-18441.025) -- 1:03:12\n",
      "      67000 -- [-18425.709] (-18433.886) (-18454.290) (-18438.992) * (-18426.876) (-18442.077) [-18437.821] (-18469.421) -- 1:03:07\n",
      "      68000 -- [-18435.888] (-18438.701) (-18437.524) (-18442.711) * (-18445.453) [-18439.241] (-18433.343) (-18466.291) -- 1:03:16\n",
      "      69000 -- [-18422.356] (-18451.904) (-18442.778) (-18451.433) * (-18433.063) (-18439.105) [-18413.449] (-18460.351) -- 1:03:11\n",
      "      70000 -- [-18421.998] (-18444.924) (-18458.871) (-18443.602) * (-18423.148) (-18450.858) [-18433.935] (-18466.463) -- 1:03:06\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.056753\n",
      "\n",
      "      71000 -- (-18430.371) (-18475.592) (-18457.132) [-18417.208] * (-18436.531) (-18441.945) [-18442.326] (-18458.673) -- 1:03:01\n",
      "      72000 -- (-18454.562) (-18475.452) [-18456.504] (-18433.530) * [-18443.249] (-18444.801) (-18444.998) (-18452.860) -- 1:02:56\n",
      "      73000 -- (-18459.613) (-18452.132) (-18447.432) [-18439.454] * [-18440.802] (-18468.693) (-18442.719) (-18459.236) -- 1:02:51\n",
      "      74000 -- (-18454.268) (-18455.946) [-18435.804] (-18443.420) * [-18430.222] (-18431.292) (-18450.508) (-18452.907) -- 1:02:59\n",
      "      75000 -- (-18444.953) (-18461.673) (-18457.135) [-18425.513] * [-18434.222] (-18453.212) (-18444.640) (-18453.970) -- 1:02:54\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.055454\n",
      "\n",
      "      76000 -- (-18443.152) (-18463.359) (-18457.848) [-18416.171] * [-18420.950] (-18447.205) (-18458.135) (-18469.358) -- 1:02:48\n",
      "      77000 -- (-18451.208) (-18447.769) (-18450.513) [-18422.866] * [-18442.606] (-18464.721) (-18454.669) (-18468.132) -- 1:02:43\n",
      "      78000 -- (-18452.778) (-18449.703) (-18451.556) [-18438.997] * [-18442.034] (-18457.011) (-18450.680) (-18453.870) -- 1:02:38\n",
      "      79000 -- (-18461.710) (-18443.174) (-18449.686) [-18425.823] * (-18455.581) (-18442.870) [-18435.267] (-18460.732) -- 1:02:33\n",
      "      80000 -- (-18438.929) (-18435.014) (-18443.480) [-18427.132] * [-18444.977] (-18456.772) (-18442.044) (-18461.241) -- 1:02:40\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.053695\n",
      "\n",
      "      81000 -- (-18463.183) (-18431.143) (-18444.563) [-18421.248] * (-18464.520) [-18441.587] (-18446.969) (-18465.301) -- 1:02:35\n",
      "      82000 -- (-18454.472) (-18439.366) [-18425.321] (-18440.879) * (-18456.048) (-18446.722) [-18444.206] (-18448.740) -- 1:02:30\n",
      "      83000 -- (-18434.150) (-18451.183) (-18436.862) [-18427.982] * (-18444.777) [-18447.724] (-18434.444) (-18467.868) -- 1:02:25\n",
      "      84000 -- (-18448.880) (-18460.339) (-18430.988) [-18429.641] * (-18444.370) [-18434.593] (-18446.389) (-18462.119) -- 1:02:20\n",
      "      85000 -- [-18439.185] (-18460.916) (-18452.140) (-18432.312) * (-18451.347) [-18435.714] (-18444.525) (-18438.647) -- 1:02:15\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.052621\n",
      "\n",
      "      86000 -- (-18433.291) (-18459.773) [-18425.502] (-18438.714) * (-18468.508) (-18444.697) [-18441.048] (-18458.803) -- 1:02:21\n",
      "      87000 -- (-18433.710) (-18453.698) (-18432.890) [-18429.057] * (-18469.355) [-18434.876] (-18431.011) (-18466.277) -- 1:02:15\n",
      "      88000 -- (-18441.860) (-18438.618) [-18431.639] (-18436.459) * (-18468.369) [-18441.860] (-18446.331) (-18461.792) -- 1:02:10\n",
      "      89000 -- [-18439.129] (-18456.532) (-18441.573) (-18445.516) * (-18451.547) (-18450.699) (-18433.925) [-18451.894] -- 1:02:05\n",
      "      90000 -- (-18461.802) (-18460.808) [-18436.300] (-18437.802) * [-18454.816] (-18437.008) (-18425.901) (-18451.582) -- 1:02:00\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.053199\n",
      "\n",
      "      91000 -- (-18447.794) (-18453.873) (-18439.197) [-18428.357] * (-18444.378) (-18460.904) [-18433.306] (-18470.141) -- 1:01:55\n",
      "      92000 -- (-18447.570) (-18469.323) (-18441.568) [-18440.752] * (-18453.319) (-18473.441) [-18444.098] (-18456.261) -- 1:02:00\n",
      "      93000 -- (-18435.205) (-18462.305) (-18433.509) [-18449.945] * (-18443.822) (-18459.776) [-18426.080] (-18453.503) -- 1:01:55\n",
      "      94000 -- (-18438.778) (-18454.504) (-18458.579) [-18435.597] * (-18441.474) (-18455.086) [-18430.675] (-18458.700) -- 1:01:50\n",
      "      95000 -- [-18423.736] (-18462.300) (-18447.323) (-18446.873) * (-18440.735) (-18473.261) [-18428.919] (-18444.515) -- 1:01:45\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.053706\n",
      "\n",
      "      96000 -- [-18430.323] (-18470.993) (-18434.312) (-18440.161) * (-18445.316) (-18475.772) [-18426.856] (-18443.820) -- 1:01:40\n",
      "      97000 -- (-18427.935) (-18459.159) [-18436.891] (-18455.365) * (-18440.771) (-18447.627) [-18438.721] (-18451.462) -- 1:01:35\n",
      "      98000 -- [-18432.417] (-18460.787) (-18441.314) (-18476.318) * (-18439.144) (-18471.680) [-18441.024] (-18463.082) -- 1:01:30\n",
      "      99000 -- (-18434.849) (-18442.759) [-18430.211] (-18446.739) * (-18438.482) (-18462.103) [-18447.744] (-18454.433) -- 1:01:35\n",
      "      100000 -- [-18421.230] (-18448.285) (-18451.749) (-18439.606) * (-18433.707) (-18476.141) [-18438.455] (-18442.055) -- 1:01:30\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.053163\n",
      "\n",
      "      101000 -- (-18426.046) (-18457.449) (-18469.598) [-18437.270] * (-18429.791) (-18467.643) [-18435.120] (-18447.101) -- 1:01:25\n",
      "      102000 -- [-18424.451] (-18456.383) (-18439.074) (-18448.178) * [-18425.912] (-18465.173) (-18430.726) (-18450.297) -- 1:01:20\n",
      "      103000 -- (-18427.538) (-18443.824) (-18440.303) [-18429.608] * (-18424.864) (-18472.306) [-18434.020] (-18455.784) -- 1:01:15\n",
      "      104000 -- [-18435.647] (-18443.755) (-18433.042) (-18440.977) * (-18441.748) (-18460.970) (-18437.598) [-18436.724] -- 1:01:10\n",
      "      105000 -- (-18433.290) (-18472.420) [-18434.124] (-18440.586) * [-18435.391] (-18461.376) (-18451.097) (-18448.985) -- 1:01:13\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.053840\n",
      "\n",
      "      106000 -- [-18424.822] (-18454.228) (-18437.476) (-18418.302) * [-18434.791] (-18450.802) (-18432.351) (-18477.420) -- 1:01:08\n",
      "      107000 -- (-18438.450) (-18448.786) (-18456.522) [-18427.231] * (-18436.944) (-18451.521) [-18431.933] (-18467.422) -- 1:01:03\n",
      "      108000 -- (-18436.944) (-18439.553) (-18440.545) [-18428.926] * [-18437.789] (-18451.750) (-18451.398) (-18458.845) -- 1:00:58\n",
      "      109000 -- (-18467.989) (-18446.260) (-18434.667) [-18437.966] * [-18426.219] (-18450.062) (-18455.457) (-18446.091) -- 1:00:53\n",
      "      110000 -- (-18440.189) (-18456.092) [-18429.469] (-18440.654) * [-18425.952] (-18469.913) (-18460.304) (-18458.494) -- 1:00:49\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.053433\n",
      "\n",
      "      111000 -- (-18463.373) (-18444.588) (-18435.361) [-18436.077] * [-18434.747] (-18473.233) (-18447.899) (-18450.868) -- 1:00:52\n",
      "      112000 -- (-18452.985) (-18474.571) (-18445.777) [-18430.407] * (-18454.104) (-18464.068) [-18431.165] (-18435.674) -- 1:00:47\n",
      "      113000 -- (-18443.903) [-18443.600] (-18459.264) (-18435.204) * (-18452.522) (-18446.014) (-18439.904) [-18430.307] -- 1:00:42\n",
      "      114000 -- (-18440.897) [-18441.617] (-18454.356) (-18464.516) * (-18454.492) [-18423.951] (-18456.645) (-18457.938) -- 1:00:37\n",
      "      115000 -- [-18428.042] (-18430.149) (-18462.436) (-18471.495) * [-18436.326] (-18450.501) (-18487.810) (-18449.788) -- 1:00:32\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.052998\n",
      "\n",
      "      116000 -- [-18427.450] (-18454.156) (-18441.314) (-18491.732) * (-18436.482) (-18449.506) (-18463.495) [-18431.998] -- 1:00:27\n",
      "      117000 -- [-18432.958] (-18444.734) (-18440.542) (-18472.419) * (-18442.497) (-18451.435) (-18475.778) [-18440.617] -- 1:00:22\n",
      "      118000 -- (-18440.466) [-18452.078] (-18447.175) (-18477.358) * [-18428.686] (-18452.912) (-18460.657) (-18460.652) -- 1:00:17\n",
      "      119000 -- (-18449.133) (-18438.104) (-18442.651) [-18447.422] * [-18439.037] (-18438.399) (-18470.963) (-18449.602) -- 1:00:12\n",
      "      120000 -- (-18453.418) (-18444.261) [-18436.483] (-18426.539) * (-18454.171) [-18433.822] (-18454.726) (-18453.156) -- 1:00:15\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.051254\n",
      "\n",
      "      121000 -- (-18435.571) (-18434.081) [-18439.593] (-18469.884) * (-18460.197) [-18438.997] (-18449.397) (-18447.720) -- 1:00:10\n",
      "      122000 -- [-18452.411] (-18454.548) (-18437.449) (-18447.518) * [-18441.822] (-18452.027) (-18444.770) (-18441.261) -- 1:00:05\n",
      "      123000 -- (-18438.256) [-18442.467] (-18448.838) (-18461.125) * (-18441.508) (-18451.727) [-18430.636] (-18454.328) -- 1:00:00\n",
      "      124000 -- (-18429.822) [-18459.981] (-18455.048) (-18455.343) * (-18451.207) (-18451.928) [-18435.063] (-18457.587) -- 0:59:55\n",
      "      125000 -- [-18425.073] (-18448.881) (-18458.155) (-18439.253) * [-18420.541] (-18438.052) (-18455.563) (-18467.638) -- 0:59:51\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.051844\n",
      "\n",
      "      126000 -- [-18438.187] (-18447.846) (-18444.916) (-18442.527) * (-18447.952) [-18432.540] (-18443.327) (-18467.658) -- 0:59:46\n",
      "      127000 -- (-18438.335) (-18455.834) (-18437.857) [-18431.381] * [-18444.705] (-18444.994) (-18440.749) (-18471.146) -- 0:59:48\n",
      "      128000 -- (-18433.400) [-18435.490] (-18462.128) (-18453.278) * (-18434.475) [-18432.462] (-18452.655) (-18454.996) -- 0:59:43\n",
      "      129000 -- (-18451.077) (-18438.929) [-18436.573] (-18486.702) * (-18438.788) [-18439.391] (-18459.349) (-18465.425) -- 0:59:38\n",
      "      130000 -- (-18441.293) [-18436.213] (-18441.448) (-18473.469) * (-18448.844) [-18432.226] (-18443.214) (-18488.865) -- 0:59:33\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.052412\n",
      "\n",
      "      131000 -- (-18453.684) (-18451.877) [-18444.172] (-18477.752) * [-18439.972] (-18441.129) (-18455.932) (-18468.094) -- 0:59:28\n",
      "      132000 -- [-18449.861] (-18434.818) (-18443.303) (-18472.549) * [-18433.410] (-18441.530) (-18463.488) (-18496.027) -- 0:59:24\n",
      "      133000 -- (-18476.130) (-18446.996) [-18431.535] (-18459.452) * [-18423.865] (-18430.959) (-18432.206) (-18488.928) -- 0:59:19\n",
      "      134000 -- (-18461.962) [-18439.211] (-18441.717) (-18450.577) * [-18418.757] (-18438.700) (-18423.603) (-18457.523) -- 0:59:14\n",
      "      135000 -- (-18444.954) (-18448.074) [-18433.651] (-18456.930) * [-18420.812] (-18448.282) (-18437.755) (-18454.142) -- 0:59:09\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.051689\n",
      "\n",
      "      136000 -- (-18450.154) (-18451.706) [-18437.530] (-18448.402) * [-18424.152] (-18438.706) (-18431.088) (-18461.472) -- 0:59:04\n",
      "      137000 -- (-18439.285) [-18434.928] (-18445.256) (-18446.232) * [-18443.541] (-18455.889) (-18441.652) (-18458.744) -- 0:59:06\n",
      "      138000 -- (-18443.820) [-18422.908] (-18432.884) (-18457.421) * (-18433.568) (-18442.562) [-18437.452] (-18461.990) -- 0:59:01\n",
      "      139000 -- (-18446.867) (-18438.616) [-18426.960] (-18464.686) * (-18456.013) [-18429.651] (-18441.940) (-18460.493) -- 0:58:56\n",
      "      140000 -- (-18440.479) [-18426.534] (-18426.561) (-18450.651) * (-18438.599) [-18448.317] (-18435.674) (-18460.917) -- 0:58:52\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.051062\n",
      "\n",
      "      141000 -- (-18421.696) (-18444.176) [-18431.715] (-18454.165) * (-18434.211) (-18457.199) [-18430.530] (-18453.371) -- 0:58:53\n",
      "      142000 -- (-18427.158) (-18443.765) [-18438.711] (-18441.304) * (-18433.844) (-18446.538) [-18433.374] (-18446.957) -- 0:58:48\n",
      "      143000 -- (-18428.862) (-18429.563) (-18451.044) [-18425.608] * [-18432.449] (-18457.847) (-18452.127) (-18451.006) -- 0:58:43\n",
      "      144000 -- [-18425.946] (-18446.141) (-18435.946) (-18445.384) * (-18435.915) (-18451.096) (-18440.109) [-18431.231] -- 0:58:39\n",
      "      145000 -- (-18430.372) [-18439.480] (-18447.849) (-18443.195) * (-18453.979) (-18446.834) [-18431.256] (-18433.569) -- 0:58:34\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.048687\n",
      "\n",
      "      146000 -- [-18420.351] (-18445.540) (-18443.589) (-18440.044) * (-18457.427) (-18457.359) [-18430.944] (-18444.268) -- 0:58:35\n",
      "      147000 -- (-18431.480) [-18457.976] (-18441.020) (-18447.655) * (-18468.285) (-18441.761) [-18425.928] (-18456.248) -- 0:58:30\n",
      "      148000 -- [-18421.771] (-18462.677) (-18432.299) (-18439.375) * (-18472.388) (-18458.923) [-18422.015] (-18457.493) -- 0:58:25\n",
      "      149000 -- [-18433.959] (-18477.456) (-18433.128) (-18438.083) * (-18446.759) (-18433.651) [-18417.034] (-18455.163) -- 0:58:26\n",
      "      150000 -- (-18443.643) (-18465.632) (-18433.654) [-18432.662] * (-18444.080) (-18443.058) [-18420.649] (-18470.056) -- 0:58:22\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.047016\n",
      "\n",
      "      151000 -- (-18440.746) (-18454.271) [-18434.141] (-18449.033) * [-18438.493] (-18440.015) (-18430.283) (-18459.457) -- 0:58:17\n",
      "      152000 -- [-18432.017] (-18449.641) (-18440.260) (-18456.862) * (-18452.616) (-18456.609) [-18421.625] (-18458.138) -- 0:58:12\n",
      "      153000 -- [-18431.294] (-18441.399) (-18438.521) (-18449.677) * (-18442.424) (-18460.151) [-18430.535] (-18444.938) -- 0:58:07\n",
      "      154000 -- (-18432.611) [-18431.141] (-18433.264) (-18444.788) * (-18455.675) (-18450.584) [-18438.661] (-18440.029) -- 0:58:08\n",
      "      155000 -- [-18437.201] (-18428.066) (-18437.162) (-18438.515) * [-18444.847] (-18457.882) (-18428.984) (-18451.389) -- 0:58:03\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.047406\n",
      "\n",
      "      156000 -- (-18440.903) [-18441.850] (-18446.076) (-18454.306) * [-18445.273] (-18444.422) (-18440.770) (-18462.690) -- 0:57:58\n",
      "      157000 -- (-18459.119) [-18423.503] (-18428.968) (-18448.656) * (-18440.669) (-18449.830) [-18430.493] (-18471.932) -- 0:57:54\n",
      "      158000 -- (-18430.339) (-18423.138) [-18427.372] (-18442.215) * [-18441.534] (-18460.858) (-18434.424) (-18445.422) -- 0:57:54\n",
      "      159000 -- (-18448.102) (-18438.342) (-18442.843) [-18427.226] * [-18441.882] (-18450.333) (-18419.057) (-18440.473) -- 0:57:49\n",
      "      160000 -- (-18457.171) [-18432.915] (-18440.919) (-18430.344) * [-18443.834] (-18448.409) (-18435.693) (-18434.509) -- 0:57:45\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.047514\n",
      "\n",
      "      161000 -- (-18463.611) (-18439.404) [-18421.997] (-18445.455) * (-18438.720) [-18437.797] (-18463.056) (-18457.031) -- 0:57:45\n",
      "      162000 -- (-18450.973) (-18445.083) [-18418.877] (-18440.535) * (-18434.294) [-18438.437] (-18450.054) (-18450.228) -- 0:57:40\n",
      "      163000 -- (-18437.543) (-18439.424) [-18419.390] (-18445.315) * (-18449.093) (-18454.286) [-18449.307] (-18455.190) -- 0:57:35\n",
      "      164000 -- (-18429.251) (-18460.558) [-18431.063] (-18449.981) * (-18464.991) (-18455.892) [-18425.530] (-18449.252) -- 0:57:31\n",
      "      165000 -- [-18432.265] (-18453.836) (-18441.050) (-18451.196) * (-18443.557) (-18472.345) [-18430.358] (-18466.524) -- 0:57:31\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.048857\n",
      "\n",
      "      166000 -- (-18440.871) (-18450.935) [-18431.231] (-18443.963) * (-18465.292) (-18456.267) [-18428.643] (-18474.024) -- 0:57:26\n",
      "      167000 -- (-18452.865) (-18443.594) [-18427.797] (-18451.167) * (-18459.883) (-18478.565) (-18433.039) [-18454.574] -- 0:57:21\n",
      "      168000 -- (-18436.541) (-18437.881) [-18434.997] (-18447.384) * (-18453.538) (-18481.949) [-18438.194] (-18448.920) -- 0:57:21\n",
      "      169000 -- (-18449.628) (-18438.107) (-18440.602) [-18437.141] * (-18470.487) (-18446.375) [-18441.855] (-18461.996) -- 0:57:17\n",
      "      170000 -- [-18422.250] (-18431.363) (-18446.324) (-18452.735) * [-18428.843] (-18468.334) (-18441.921) (-18443.458) -- 0:57:12\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.049791\n",
      "\n",
      "      171000 -- [-18433.714] (-18442.105) (-18444.177) (-18446.285) * (-18433.546) (-18466.276) [-18433.499] (-18458.696) -- 0:57:12\n",
      "      172000 -- (-18442.926) (-18450.417) [-18435.352] (-18456.861) * [-18432.290] (-18470.001) (-18429.734) (-18466.129) -- 0:57:07\n",
      "      173000 -- [-18421.766] (-18433.314) (-18432.514) (-18447.745) * (-18445.635) (-18460.614) (-18442.338) [-18455.824] -- 0:57:02\n",
      "      174000 -- [-18418.421] (-18437.951) (-18435.088) (-18449.778) * (-18439.872) (-18464.438) [-18433.439] (-18465.127) -- 0:56:57\n",
      "      175000 -- (-18433.382) [-18448.488] (-18431.812) (-18437.944) * (-18427.140) (-18454.332) [-18435.300] (-18470.857) -- 0:56:57\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.049853\n",
      "\n",
      "      176000 -- (-18436.807) (-18443.503) [-18428.091] (-18433.397) * (-18442.040) (-18466.265) [-18436.389] (-18469.043) -- 0:56:53\n",
      "      177000 -- (-18436.207) (-18435.979) (-18452.899) [-18431.216] * (-18438.555) (-18461.279) [-18428.734] (-18463.913) -- 0:56:48\n",
      "      178000 -- (-18439.672) [-18423.613] (-18443.628) (-18433.720) * [-18445.471] (-18477.363) (-18428.269) (-18459.252) -- 0:56:48\n",
      "      179000 -- (-18439.116) (-18441.465) (-18438.718) [-18428.119] * (-18444.028) (-18482.217) [-18434.384] (-18477.662) -- 0:56:43\n",
      "      180000 -- [-18448.830] (-18439.233) (-18447.530) (-18419.628) * (-18437.355) (-18452.541) [-18429.557] (-18462.197) -- 0:56:38\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.050942\n",
      "\n",
      "      181000 -- [-18423.971] (-18439.997) (-18436.280) (-18441.584) * (-18422.252) (-18453.196) [-18426.654] (-18465.805) -- 0:56:38\n",
      "      182000 -- (-18431.113) (-18458.143) [-18443.587] (-18447.523) * [-18423.848] (-18458.452) (-18450.282) (-18449.277) -- 0:56:33\n",
      "      183000 -- (-18434.356) (-18437.078) (-18464.500) [-18438.975] * [-18427.094] (-18451.122) (-18453.919) (-18452.587) -- 0:56:28\n",
      "      184000 -- (-18440.122) [-18446.670] (-18457.535) (-18430.458) * [-18422.718] (-18453.698) (-18428.096) (-18461.178) -- 0:56:28\n",
      "      185000 -- (-18436.020) (-18448.862) (-18467.580) [-18429.017] * (-18458.757) (-18449.301) [-18446.920] (-18450.093) -- 0:56:23\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.051633\n",
      "\n",
      "      186000 -- (-18438.878) [-18439.678] (-18467.961) (-18432.602) * (-18443.124) (-18446.811) (-18454.224) [-18442.444] -- 0:56:18\n",
      "      187000 -- [-18424.654] (-18443.680) (-18439.818) (-18430.947) * [-18434.486] (-18437.992) (-18452.366) (-18449.865) -- 0:56:13\n",
      "      188000 -- [-18423.329] (-18437.426) (-18440.825) (-18442.876) * [-18438.132] (-18461.669) (-18442.755) (-18446.645) -- 0:56:13\n",
      "      189000 -- (-18450.062) [-18421.953] (-18446.772) (-18444.139) * (-18446.148) (-18460.581) [-18442.266] (-18430.996) -- 0:56:08\n",
      "      190000 -- (-18447.375) [-18428.238] (-18455.979) (-18452.800) * (-18454.785) (-18442.553) [-18426.085] (-18431.469) -- 0:56:03\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.051706\n",
      "\n",
      "      191000 -- [-18448.559] (-18440.531) (-18454.813) (-18451.954) * (-18432.750) (-18436.699) [-18433.136] (-18453.978) -- 0:56:03\n",
      "      192000 -- (-18440.809) (-18449.473) (-18450.192) [-18449.473] * (-18474.634) [-18434.702] (-18439.457) (-18450.326) -- 0:55:58\n",
      "      193000 -- (-18449.777) [-18459.203] (-18447.207) (-18441.526) * (-18451.152) (-18449.118) [-18439.535] (-18456.895) -- 0:55:53\n",
      "      194000 -- [-18420.570] (-18447.231) (-18449.611) (-18443.181) * (-18459.097) (-18435.647) [-18426.325] (-18454.217) -- 0:55:48\n",
      "      195000 -- (-18432.051) (-18444.417) (-18452.020) [-18446.719] * (-18450.288) (-18449.428) [-18422.830] (-18457.859) -- 0:55:47\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.051323\n",
      "\n",
      "      196000 -- [-18419.104] (-18456.527) (-18459.079) (-18441.508) * (-18451.649) (-18479.601) [-18430.323] (-18439.328) -- 0:55:43\n",
      "      197000 -- [-18429.850] (-18445.314) (-18455.987) (-18448.699) * [-18432.601] (-18456.398) (-18440.463) (-18459.805) -- 0:55:38\n",
      "      198000 -- [-18422.992] (-18439.973) (-18456.977) (-18440.748) * (-18429.786) (-18450.577) [-18435.105] (-18453.088) -- 0:55:37\n",
      "      199000 -- (-18437.156) [-18449.100] (-18450.462) (-18451.663) * (-18437.302) (-18457.879) [-18435.206] (-18448.429) -- 0:55:32\n",
      "      200000 -- [-18445.022] (-18454.535) (-18452.736) (-18433.918) * [-18448.840] (-18453.870) (-18444.124) (-18445.992) -- 0:55:27\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.050746\n",
      "\n",
      "      201000 -- (-18439.522) (-18437.042) (-18459.266) [-18429.601] * (-18443.191) (-18458.320) [-18427.091] (-18459.264) -- 0:55:23\n",
      "      202000 -- (-18452.608) (-18441.907) (-18452.544) [-18444.640] * [-18431.566] (-18452.176) (-18435.625) (-18462.879) -- 0:55:22\n",
      "      203000 -- (-18452.279) (-18443.970) (-18456.546) [-18425.249] * (-18426.890) [-18429.243] (-18451.830) (-18467.426) -- 0:55:17\n",
      "      204000 -- (-18445.438) [-18445.344] (-18451.998) (-18431.643) * (-18424.486) [-18445.608] (-18448.079) (-18474.439) -- 0:55:12\n",
      "      205000 -- (-18438.947) (-18455.355) (-18454.615) [-18432.842] * (-18450.116) (-18464.001) [-18429.151] (-18451.958) -- 0:55:11\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.050889\n",
      "\n",
      "      206000 -- (-18454.959) (-18440.454) (-18434.940) [-18416.846] * (-18426.145) (-18462.717) [-18441.194] (-18447.874) -- 0:55:07\n",
      "      207000 -- (-18456.212) (-18429.585) (-18433.545) [-18422.979] * [-18430.710] (-18465.619) (-18435.859) (-18452.198) -- 0:55:02\n",
      "      208000 -- (-18468.144) [-18427.910] (-18432.651) (-18431.367) * (-18428.985) (-18471.509) [-18432.315] (-18462.568) -- 0:55:01\n",
      "      209000 -- (-18458.487) (-18432.240) (-18451.293) [-18442.603] * (-18441.770) (-18450.318) [-18431.598] (-18452.809) -- 0:54:56\n",
      "      210000 -- [-18446.300] (-18441.279) (-18454.919) (-18427.503) * (-18434.778) [-18430.729] (-18446.296) (-18458.082) -- 0:54:55\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.052805\n",
      "\n",
      "      211000 -- (-18436.063) (-18464.902) (-18436.622) [-18432.056] * (-18449.547) (-18432.127) [-18431.804] (-18455.106) -- 0:54:50\n",
      "      212000 -- (-18432.698) (-18473.451) [-18433.541] (-18435.193) * (-18441.410) (-18430.785) [-18431.424] (-18456.593) -- 0:54:45\n",
      "      213000 -- (-18440.471) [-18433.915] (-18452.549) (-18425.015) * (-18465.784) (-18433.972) [-18439.841] (-18465.379) -- 0:54:41\n",
      "      214000 -- (-18460.234) (-18452.286) [-18430.056] (-18427.069) * [-18437.708] (-18428.145) (-18433.588) (-18446.301) -- 0:54:39\n",
      "      215000 -- (-18425.039) (-18444.694) [-18444.038] (-18444.880) * (-18443.172) (-18446.030) [-18435.901] (-18459.874) -- 0:54:35\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.052015\n",
      "\n",
      "      216000 -- (-18439.142) (-18429.612) [-18428.146] (-18456.225) * (-18454.955) (-18442.314) [-18436.928] (-18468.008) -- 0:54:30\n",
      "      217000 -- (-18423.274) (-18442.283) [-18428.272] (-18449.255) * (-18458.801) (-18436.402) [-18428.793] (-18459.052) -- 0:54:29\n",
      "      218000 -- (-18443.377) (-18445.567) [-18438.659] (-18427.604) * (-18453.919) (-18448.843) [-18426.439] (-18465.882) -- 0:54:24\n",
      "      219000 -- [-18435.542] (-18444.643) (-18433.840) (-18442.789) * [-18447.078] (-18441.656) (-18420.279) (-18479.249) -- 0:54:19\n",
      "      220000 -- (-18439.734) [-18420.043] (-18452.531) (-18442.395) * (-18455.126) (-18447.675) [-18419.298] (-18471.161) -- 0:54:14\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.052606\n",
      "\n",
      "      221000 -- (-18449.790) (-18434.767) [-18440.142] (-18457.974) * [-18441.056] (-18431.974) (-18419.267) (-18479.894) -- 0:54:13\n",
      "      222000 -- (-18422.734) (-18431.193) [-18442.669] (-18453.228) * (-18422.846) [-18427.795] (-18431.304) (-18470.558) -- 0:54:08\n",
      "      223000 -- [-18415.605] (-18429.998) (-18452.024) (-18444.883) * [-18427.701] (-18444.663) (-18441.169) (-18460.900) -- 0:54:03\n",
      "      224000 -- (-18435.409) (-18448.736) (-18466.095) [-18427.584] * [-18420.316] (-18431.263) (-18449.690) (-18456.923) -- 0:53:59\n",
      "      225000 -- (-18443.085) (-18444.626) (-18445.258) [-18416.708] * (-18432.620) (-18442.537) [-18424.192] (-18450.157) -- 0:53:54\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.052101\n",
      "\n",
      "      226000 -- [-18438.228] (-18441.565) (-18465.457) (-18423.922) * (-18445.717) (-18442.201) [-18415.735] (-18477.409) -- 0:53:52\n",
      "      227000 -- (-18464.489) (-18455.270) (-18438.841) [-18423.217] * (-18445.141) [-18439.693] (-18420.893) (-18467.560) -- 0:53:48\n",
      "      228000 -- (-18459.138) (-18445.453) (-18441.378) [-18418.010] * (-18433.964) (-18438.576) [-18425.871] (-18468.967) -- 0:53:43\n",
      "      229000 -- (-18447.805) (-18443.782) [-18433.915] (-18419.529) * [-18436.057] (-18454.960) (-18437.829) (-18475.573) -- 0:53:42\n",
      "      230000 -- (-18450.397) (-18448.022) [-18435.604] (-18417.881) * (-18446.169) (-18444.078) [-18427.961] (-18459.222) -- 0:53:37\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.051442\n",
      "\n",
      "      231000 -- (-18467.536) (-18438.253) [-18437.831] (-18433.386) * (-18437.469) (-18438.312) [-18430.231] (-18456.913) -- 0:53:32\n",
      "      232000 -- (-18451.511) (-18466.924) [-18432.127] (-18465.281) * [-18435.862] (-18434.647) (-18421.011) (-18444.449) -- 0:53:27\n",
      "      233000 -- (-18446.531) (-18463.379) (-18458.128) [-18450.805] * (-18446.276) [-18440.907] (-18432.673) (-18464.336) -- 0:53:26\n",
      "      234000 -- (-18450.167) [-18443.418] (-18444.570) (-18452.127) * (-18467.144) (-18449.163) [-18423.154] (-18446.293) -- 0:53:21\n",
      "      235000 -- (-18438.691) (-18465.328) (-18445.211) [-18447.200] * [-18455.465] (-18460.326) (-18440.298) (-18457.401) -- 0:53:16\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.052948\n",
      "\n",
      "      236000 -- (-18445.602) (-18447.549) (-18446.337) [-18436.195] * [-18451.538] (-18452.943) (-18448.589) (-18464.849) -- 0:53:15\n",
      "      237000 -- (-18432.793) (-18476.320) [-18440.274] (-18431.744) * [-18439.673] (-18426.264) (-18444.924) (-18473.548) -- 0:53:10\n",
      "      238000 -- [-18446.584] (-18456.192) (-18453.489) (-18447.806) * [-18421.005] (-18455.839) (-18422.705) (-18470.944) -- 0:53:05\n",
      "      239000 -- (-18445.535) (-18452.831) [-18420.452] (-18454.390) * [-18411.340] (-18459.919) (-18434.924) (-18465.150) -- 0:53:04\n",
      "      240000 -- [-18444.111] (-18462.033) (-18428.411) (-18438.784) * (-18448.931) (-18449.051) [-18435.325] (-18465.245) -- 0:52:59\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.052775\n",
      "\n",
      "      241000 -- (-18453.841) (-18442.208) [-18434.212] (-18434.957) * (-18440.603) (-18449.326) [-18422.827] (-18475.567) -- 0:52:54\n",
      "      242000 -- (-18455.307) [-18433.423] (-18446.139) (-18450.629) * (-18467.863) (-18450.575) [-18415.662] (-18459.785) -- 0:52:49\n",
      "      243000 -- [-18427.387] (-18430.696) (-18464.621) (-18440.756) * (-18463.524) (-18448.169) [-18424.159] (-18460.065) -- 0:52:48\n",
      "      244000 -- [-18431.282] (-18440.891) (-18456.075) (-18433.082) * (-18465.586) (-18441.449) [-18428.113] (-18461.816) -- 0:52:43\n",
      "      245000 -- [-18432.766] (-18443.925) (-18438.938) (-18442.677) * (-18464.784) [-18429.921] (-18437.217) (-18456.067) -- 0:52:38\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.050688\n",
      "\n",
      "      246000 -- (-18433.472) [-18444.121] (-18454.147) (-18469.570) * (-18473.747) [-18429.389] (-18435.519) (-18465.352) -- 0:52:36\n",
      "      247000 -- [-18427.485] (-18447.524) (-18448.057) (-18452.243) * (-18463.623) [-18422.297] (-18437.364) (-18462.321) -- 0:52:32\n",
      "      248000 -- [-18423.852] (-18461.994) (-18434.637) (-18441.435) * (-18472.135) [-18426.789] (-18443.653) (-18445.015) -- 0:52:27\n",
      "      249000 -- [-18438.512] (-18476.107) (-18434.670) (-18459.184) * (-18475.011) [-18441.084] (-18441.208) (-18442.861) -- 0:52:25\n",
      "      250000 -- (-18443.146) (-18464.077) [-18432.398] (-18436.871) * (-18464.390) (-18434.990) (-18454.065) [-18436.886] -- 0:52:21\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.048601\n",
      "\n",
      "      251000 -- (-18446.148) [-18445.843] (-18442.141) (-18435.188) * (-18464.905) [-18423.456] (-18474.094) (-18438.645) -- 0:52:16\n",
      "      252000 -- [-18435.656] (-18446.302) (-18446.197) (-18453.256) * (-18454.008) (-18432.869) (-18486.050) [-18441.621] -- 0:52:14\n",
      "      253000 -- (-18442.456) [-18451.161] (-18439.674) (-18450.832) * (-18459.881) [-18438.414] (-18476.350) (-18443.296) -- 0:52:09\n",
      "      254000 -- (-18434.510) (-18447.679) [-18433.652] (-18440.830) * (-18455.280) [-18435.542] (-18471.870) (-18460.324) -- 0:52:04\n",
      "      255000 -- (-18451.791) [-18438.033] (-18439.671) (-18449.354) * (-18455.112) [-18429.106] (-18439.585) (-18479.367) -- 0:52:03\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.046095\n",
      "\n",
      "      256000 -- (-18444.259) [-18424.453] (-18430.640) (-18439.105) * (-18468.470) [-18436.639] (-18447.911) (-18462.253) -- 0:51:58\n",
      "      257000 -- (-18439.115) (-18444.069) [-18429.010] (-18436.196) * (-18457.375) [-18439.188] (-18423.035) (-18479.562) -- 0:51:53\n",
      "      258000 -- (-18465.280) (-18452.000) (-18431.433) [-18416.444] * (-18461.531) (-18438.338) [-18408.918] (-18438.586) -- 0:51:51\n",
      "      259000 -- (-18443.149) (-18453.627) [-18422.555] (-18447.495) * [-18454.180] (-18441.145) (-18449.380) (-18452.011) -- 0:51:47\n",
      "      260000 -- (-18442.358) (-18448.356) (-18434.242) [-18432.041] * (-18451.086) (-18450.652) [-18418.111] (-18439.311) -- 0:51:42\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.044779\n",
      "\n",
      "      261000 -- (-18482.041) [-18430.926] (-18447.466) (-18434.524) * (-18442.063) (-18448.795) [-18427.551] (-18457.375) -- 0:51:40\n",
      "      262000 -- (-18459.303) [-18430.098] (-18426.831) (-18450.328) * (-18435.018) (-18440.103) [-18420.902] (-18455.677) -- 0:51:35\n",
      "      263000 -- (-18486.556) [-18437.033] (-18433.935) (-18460.938) * (-18433.729) (-18426.836) [-18420.961] (-18443.645) -- 0:51:33\n",
      "      264000 -- (-18452.024) (-18462.553) (-18457.300) [-18425.657] * (-18437.878) (-18455.412) (-18423.042) [-18437.254] -- 0:51:28\n",
      "      265000 -- (-18438.872) (-18449.171) [-18443.142] (-18430.261) * (-18442.136) (-18442.648) (-18446.739) [-18439.834] -- 0:51:24\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.043620\n",
      "\n",
      "      266000 -- [-18420.594] (-18450.430) (-18423.497) (-18429.820) * [-18439.038] (-18447.551) (-18443.039) (-18438.758) -- 0:51:22\n",
      "      267000 -- (-18446.779) (-18446.233) [-18439.585] (-18423.394) * [-18418.517] (-18461.535) (-18446.888) (-18443.296) -- 0:51:17\n",
      "      268000 -- (-18468.634) (-18427.774) (-18445.891) [-18438.858] * (-18433.994) (-18475.074) [-18432.727] (-18434.687) -- 0:51:15\n",
      "      269000 -- (-18460.199) (-18434.128) (-18446.610) [-18430.416] * (-18450.395) (-18461.525) (-18422.354) [-18441.768] -- 0:51:10\n",
      "      270000 -- (-18470.088) (-18440.135) [-18433.274] (-18429.778) * (-18455.690) (-18449.279) [-18416.384] (-18438.197) -- 0:51:08\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.043042\n",
      "\n",
      "      271000 -- (-18454.766) [-18427.252] (-18435.573) (-18445.030) * (-18449.894) (-18430.197) [-18419.861] (-18439.459) -- 0:51:03\n",
      "      272000 -- [-18441.470] (-18447.634) (-18434.407) (-18429.230) * [-18418.490] (-18445.396) (-18427.297) (-18455.922) -- 0:50:59\n",
      "      273000 -- [-18434.454] (-18450.969) (-18437.205) (-18456.076) * [-18422.707] (-18451.077) (-18437.769) (-18449.349) -- 0:50:57\n",
      "      274000 -- (-18444.412) (-18459.216) [-18419.585] (-18444.553) * [-18432.844] (-18458.339) (-18435.408) (-18461.999) -- 0:50:52\n",
      "      275000 -- [-18431.405] (-18456.084) (-18432.009) (-18457.056) * [-18429.029] (-18451.466) (-18445.676) (-18451.750) -- 0:50:50\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.043373\n",
      "\n",
      "      276000 -- [-18415.144] (-18452.108) (-18432.420) (-18439.812) * [-18431.605] (-18464.828) (-18438.432) (-18454.909) -- 0:50:45\n",
      "      277000 -- [-18418.812] (-18449.814) (-18429.608) (-18451.295) * [-18426.711] (-18466.325) (-18422.812) (-18470.633) -- 0:50:43\n",
      "      278000 -- [-18423.237] (-18437.476) (-18438.905) (-18435.494) * (-18430.588) (-18464.272) [-18414.639] (-18460.965) -- 0:50:38\n",
      "      279000 -- [-18429.623] (-18445.641) (-18443.454) (-18436.491) * [-18424.170] (-18437.736) (-18443.549) (-18474.759) -- 0:50:33\n",
      "      280000 -- [-18421.416] (-18446.629) (-18442.928) (-18430.065) * [-18418.960] (-18426.564) (-18445.411) (-18454.557) -- 0:50:31\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.043222\n",
      "\n",
      "      281000 -- (-18441.392) [-18436.620] (-18455.837) (-18447.829) * (-18437.921) [-18428.043] (-18425.258) (-18453.553) -- 0:50:26\n",
      "      282000 -- (-18426.543) [-18439.401] (-18446.802) (-18435.146) * (-18443.002) (-18434.320) [-18416.698] (-18447.774) -- 0:50:22\n",
      "      283000 -- (-18448.054) [-18426.775] (-18430.340) (-18440.845) * (-18449.941) (-18430.811) [-18441.042] (-18457.764) -- 0:50:20\n",
      "      284000 -- (-18462.232) (-18422.584) (-18453.630) [-18439.321] * (-18446.315) [-18447.024] (-18438.725) (-18448.241) -- 0:50:15\n",
      "      285000 -- (-18448.936) [-18424.441] (-18442.458) (-18459.525) * (-18462.483) (-18446.364) [-18423.105] (-18438.835) -- 0:50:10\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.041218\n",
      "\n",
      "      286000 -- [-18425.218] (-18438.753) (-18444.062) (-18454.306) * (-18449.147) (-18452.562) (-18432.609) [-18440.531] -- 0:50:05\n",
      "      287000 -- (-18436.957) (-18450.434) [-18435.249] (-18452.239) * (-18431.429) [-18436.178] (-18446.007) (-18439.501) -- 0:50:03\n",
      "      288000 -- (-18433.032) [-18425.406] (-18443.159) (-18436.476) * (-18448.590) (-18431.285) [-18418.570] (-18453.761) -- 0:49:58\n",
      "      289000 -- (-18439.871) (-18431.163) (-18438.127) [-18440.962] * (-18432.792) [-18447.195] (-18435.322) (-18447.476) -- 0:49:54\n",
      "      290000 -- [-18418.851] (-18439.853) (-18457.377) (-18424.864) * (-18437.157) (-18440.653) [-18419.832] (-18461.411) -- 0:49:51\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.040144\n",
      "\n",
      "      291000 -- [-18434.105] (-18444.963) (-18450.279) (-18420.009) * (-18448.887) (-18441.544) [-18417.305] (-18459.368) -- 0:49:47\n",
      "      292000 -- (-18435.588) (-18450.909) (-18452.494) [-18427.217] * (-18437.225) (-18432.892) [-18417.172] (-18474.395) -- 0:49:42\n",
      "      293000 -- (-18440.292) (-18446.515) (-18454.778) [-18428.836] * (-18451.152) (-18442.634) [-18429.515] (-18459.937) -- 0:49:37\n",
      "      294000 -- (-18463.969) (-18445.768) (-18431.522) [-18433.331] * [-18426.367] (-18436.157) (-18418.526) (-18438.544) -- 0:49:32\n",
      "      295000 -- [-18442.688] (-18445.841) (-18429.397) (-18438.225) * (-18433.338) (-18444.440) [-18415.989] (-18458.670) -- 0:49:30\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.039975\n",
      "\n",
      "      296000 -- [-18433.375] (-18454.523) (-18441.929) (-18452.624) * (-18439.744) (-18454.749) [-18418.933] (-18454.876) -- 0:49:25\n",
      "      297000 -- (-18442.496) [-18430.407] (-18448.082) (-18455.248) * (-18437.987) (-18462.650) [-18454.311] (-18469.802) -- 0:49:21\n",
      "      298000 -- (-18446.698) [-18417.876] (-18443.246) (-18444.206) * (-18453.665) (-18441.617) [-18424.339] (-18473.143) -- 0:49:18\n",
      "      299000 -- [-18438.218] (-18429.609) (-18455.149) (-18454.281) * (-18449.468) (-18447.906) [-18421.881] (-18456.614) -- 0:49:14\n",
      "      300000 -- (-18457.520) [-18422.518] (-18439.839) (-18444.472) * (-18446.121) (-18446.848) [-18425.227] (-18450.329) -- 0:49:11\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.037645\n",
      "\n",
      "      301000 -- (-18448.089) [-18421.303] (-18442.374) (-18453.515) * [-18421.644] (-18443.086) (-18424.277) (-18447.934) -- 0:49:06\n",
      "      302000 -- [-18430.120] (-18443.007) (-18435.605) (-18448.896) * [-18432.097] (-18454.644) (-18433.956) (-18435.126) -- 0:49:02\n",
      "      303000 -- [-18423.583] (-18445.927) (-18435.713) (-18452.115) * (-18446.204) (-18459.825) [-18440.713] (-18443.286) -- 0:48:57\n",
      "      304000 -- (-18446.729) (-18437.221) [-18439.259] (-18437.291) * (-18453.235) (-18448.706) [-18415.077] (-18443.491) -- 0:48:55\n",
      "      305000 -- (-18446.277) (-18431.969) (-18456.411) [-18428.227] * (-18452.958) [-18454.524] (-18432.068) (-18483.156) -- 0:48:50\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.036428\n",
      "\n",
      "      306000 -- (-18440.107) (-18466.591) [-18428.286] (-18445.842) * [-18436.916] (-18446.772) (-18434.891) (-18463.156) -- 0:48:45\n",
      "      307000 -- (-18463.181) [-18438.319] (-18420.660) (-18439.929) * [-18435.224] (-18463.959) (-18428.117) (-18459.382) -- 0:48:43\n",
      "      308000 -- (-18457.129) (-18451.813) [-18432.545] (-18432.010) * (-18428.821) (-18444.292) [-18426.306] (-18472.427) -- 0:48:38\n",
      "      309000 -- (-18440.587) (-18463.375) (-18429.780) [-18437.961] * [-18428.463] (-18454.006) (-18450.502) (-18477.028) -- 0:48:33\n",
      "      310000 -- (-18449.549) (-18456.564) [-18428.068] (-18436.855) * [-18437.910] (-18462.994) (-18443.914) (-18438.291) -- 0:48:29\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.036584\n",
      "\n",
      "      311000 -- (-18450.642) (-18449.255) [-18431.013] (-18439.615) * (-18447.677) (-18450.609) [-18440.404] (-18453.666) -- 0:48:26\n",
      "      312000 -- (-18440.986) [-18436.371] (-18439.759) (-18437.170) * (-18463.539) [-18439.305] (-18443.123) (-18449.083) -- 0:48:21\n",
      "      313000 -- (-18446.082) [-18421.133] (-18457.776) (-18421.329) * [-18428.770] (-18449.142) (-18463.288) (-18456.203) -- 0:48:17\n",
      "      314000 -- [-18429.475] (-18439.609) (-18440.062) (-18435.017) * (-18446.591) (-18455.190) [-18437.296] (-18451.602) -- 0:48:12\n",
      "      315000 -- (-18435.085) [-18430.272] (-18441.595) (-18452.526) * (-18426.418) (-18465.999) [-18421.529] (-18465.385) -- 0:48:10\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.036442\n",
      "\n",
      "      316000 -- (-18450.136) [-18431.294] (-18454.255) (-18447.330) * [-18443.920] (-18449.695) (-18455.440) (-18479.880) -- 0:48:05\n",
      "      317000 -- (-18454.534) (-18446.374) (-18447.950) [-18435.354] * [-18438.758] (-18468.091) (-18459.147) (-18474.889) -- 0:48:00\n",
      "      318000 -- [-18422.884] (-18436.698) (-18439.609) (-18441.293) * [-18442.182] (-18456.262) (-18437.672) (-18445.139) -- 0:47:58\n",
      "      319000 -- [-18436.409] (-18431.947) (-18459.527) (-18468.383) * [-18439.024] (-18435.479) (-18430.478) (-18441.345) -- 0:47:53\n",
      "      320000 -- [-18431.684] (-18439.453) (-18451.122) (-18430.734) * (-18429.233) (-18425.867) (-18446.603) [-18452.207] -- 0:47:50\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.035774\n",
      "\n",
      "      321000 -- (-18415.747) [-18432.975] (-18447.440) (-18439.564) * (-18430.775) (-18448.979) (-18427.440) [-18437.346] -- 0:47:46\n",
      "      322000 -- (-18429.126) [-18446.078] (-18435.852) (-18453.245) * [-18441.267] (-18456.771) (-18424.308) (-18451.611) -- 0:47:41\n",
      "      323000 -- (-18441.978) (-18440.085) [-18428.568] (-18441.959) * (-18438.821) (-18453.692) [-18416.214] (-18457.329) -- 0:47:38\n",
      "      324000 -- (-18443.693) (-18432.546) [-18426.083] (-18449.613) * [-18436.052] (-18428.402) (-18451.781) (-18462.800) -- 0:47:34\n",
      "      325000 -- (-18454.416) [-18438.283] (-18434.135) (-18436.985) * (-18443.401) (-18424.483) [-18420.261] (-18443.123) -- 0:47:31\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.035028\n",
      "\n",
      "      326000 -- (-18463.424) (-18436.585) (-18437.707) [-18434.245] * (-18435.884) [-18417.469] (-18429.129) (-18462.699) -- 0:47:26\n",
      "      327000 -- (-18458.590) (-18431.652) [-18439.903] (-18443.850) * (-18451.376) [-18433.852] (-18429.567) (-18458.582) -- 0:47:22\n",
      "      328000 -- (-18470.025) [-18420.219] (-18445.382) (-18438.476) * (-18446.769) (-18462.945) [-18428.027] (-18447.377) -- 0:47:19\n",
      "      329000 -- (-18467.039) [-18426.920] (-18455.464) (-18428.948) * [-18433.168] (-18450.217) (-18447.987) (-18455.356) -- 0:47:14\n",
      "      330000 -- (-18469.518) (-18435.532) [-18432.630] (-18435.544) * [-18431.286] (-18448.614) (-18436.562) (-18465.958) -- 0:47:10\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.034856\n",
      "\n",
      "      331000 -- (-18474.418) (-18425.707) [-18431.536] (-18425.238) * [-18442.297] (-18447.140) (-18449.160) (-18470.948) -- 0:47:07\n",
      "      332000 -- (-18465.914) (-18433.951) (-18435.436) [-18418.021] * [-18427.587] (-18449.210) (-18443.950) (-18450.350) -- 0:47:02\n",
      "      333000 -- (-18458.278) (-18422.996) (-18426.705) [-18414.626] * [-18432.735] (-18467.023) (-18437.556) (-18441.945) -- 0:46:58\n",
      "      334000 -- (-18449.695) (-18439.623) (-18447.808) [-18422.566] * [-18433.442] (-18460.053) (-18449.651) (-18440.489) -- 0:46:55\n",
      "      335000 -- (-18438.593) (-18432.922) (-18443.019) [-18417.054] * [-18433.918] (-18454.171) (-18441.103) (-18458.807) -- 0:46:50\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.035687\n",
      "\n",
      "      336000 -- (-18439.260) (-18441.591) (-18436.527) [-18419.886] * [-18440.226] (-18439.814) (-18432.867) (-18455.921) -- 0:46:46\n",
      "      337000 -- (-18451.562) (-18476.843) [-18429.879] (-18434.094) * (-18438.470) [-18429.576] (-18436.054) (-18435.486) -- 0:46:43\n",
      "      338000 -- (-18460.157) (-18455.231) (-18446.314) [-18427.640] * (-18431.682) [-18433.318] (-18453.970) (-18451.998) -- 0:46:38\n",
      "      339000 -- (-18449.382) [-18434.116] (-18441.061) (-18427.749) * [-18437.583] (-18442.639) (-18420.848) (-18481.554) -- 0:46:34\n",
      "      340000 -- (-18442.279) (-18436.613) [-18459.163] (-18442.907) * (-18449.482) [-18422.382] (-18423.037) (-18452.253) -- 0:46:31\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.036641\n",
      "\n",
      "      341000 -- (-18441.471) (-18427.774) (-18464.227) [-18431.375] * (-18441.993) (-18440.045) [-18420.872] (-18434.922) -- 0:46:26\n",
      "      342000 -- [-18436.627] (-18430.598) (-18448.380) (-18453.424) * (-18456.555) [-18447.934] (-18437.140) (-18458.792) -- 0:46:22\n",
      "      343000 -- (-18448.742) [-18438.524] (-18446.900) (-18440.978) * (-18447.751) [-18451.631] (-18433.886) (-18454.073) -- 0:46:17\n",
      "      344000 -- [-18438.175] (-18444.808) (-18443.037) (-18427.517) * (-18447.700) (-18435.065) [-18431.059] (-18465.100) -- 0:46:14\n",
      "      345000 -- (-18478.585) (-18451.834) (-18446.447) [-18426.977] * [-18440.199] (-18452.492) (-18461.870) (-18474.570) -- 0:46:09\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.036371\n",
      "\n",
      "      346000 -- (-18439.333) (-18444.371) (-18461.105) [-18429.266] * (-18430.330) [-18443.587] (-18467.030) (-18475.095) -- 0:46:05\n",
      "      347000 -- [-18420.149] (-18442.526) (-18471.590) (-18428.197) * [-18432.872] (-18448.946) (-18432.271) (-18442.482) -- 0:46:02\n",
      "      348000 -- (-18435.749) (-18467.265) (-18445.062) [-18432.519] * [-18430.431] (-18433.435) (-18443.414) (-18457.970) -- 0:45:57\n",
      "      349000 -- [-18445.751] (-18454.991) (-18429.185) (-18452.893) * (-18453.173) [-18437.097] (-18436.980) (-18466.182) -- 0:45:53\n",
      "      350000 -- (-18429.175) (-18446.577) [-18428.905] (-18458.670) * (-18436.726) [-18432.848] (-18442.774) (-18463.608) -- 0:45:50\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.036470\n",
      "\n",
      "      351000 -- (-18431.158) (-18437.323) [-18429.918] (-18435.320) * (-18441.633) [-18430.556] (-18444.503) (-18444.789) -- 0:45:45\n",
      "      352000 -- (-18450.786) (-18454.414) [-18428.745] (-18452.580) * (-18443.528) (-18452.719) [-18428.714] (-18443.189) -- 0:45:41\n",
      "      353000 -- (-18437.852) (-18443.265) [-18437.101] (-18456.029) * (-18434.890) (-18450.760) (-18447.782) [-18441.748] -- 0:45:38\n",
      "      354000 -- (-18454.023) (-18431.568) [-18441.707] (-18460.669) * [-18425.071] (-18450.822) (-18452.362) (-18448.914) -- 0:45:33\n",
      "      355000 -- [-18434.215] (-18452.031) (-18450.441) (-18454.855) * [-18425.730] (-18452.286) (-18455.184) (-18459.304) -- 0:45:28\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.036824\n",
      "\n",
      "      356000 -- (-18441.968) (-18453.996) [-18431.953] (-18452.498) * [-18435.214] (-18429.180) (-18455.118) (-18454.219) -- 0:45:24\n",
      "      357000 -- (-18432.480) [-18438.178] (-18443.472) (-18457.200) * (-18452.041) [-18439.835] (-18441.654) (-18450.224) -- 0:45:19\n",
      "      358000 -- [-18436.639] (-18433.072) (-18447.765) (-18453.121) * (-18457.840) [-18430.326] (-18430.374) (-18456.608) -- 0:45:15\n",
      "      359000 -- (-18458.613) (-18421.611) [-18439.457] (-18441.690) * (-18437.848) [-18435.137] (-18441.750) (-18452.247) -- 0:45:12\n",
      "      360000 -- (-18453.541) (-18441.186) [-18419.136] (-18447.566) * (-18445.342) (-18449.476) [-18442.549] (-18462.232) -- 0:45:07\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.036045\n",
      "\n",
      "      361000 -- (-18440.083) (-18448.035) [-18420.431] (-18445.250) * (-18435.837) (-18469.791) [-18428.275] (-18446.257) -- 0:45:02\n",
      "      362000 -- (-18440.251) (-18462.983) [-18427.970] (-18427.878) * [-18437.641] (-18452.781) (-18434.505) (-18470.674) -- 0:45:00\n",
      "      363000 -- (-18429.610) (-18438.143) [-18412.951] (-18445.799) * [-18437.666] (-18469.926) (-18434.126) (-18461.129) -- 0:44:55\n",
      "      364000 -- (-18451.322) (-18434.521) (-18447.915) [-18427.798] * [-18447.232] (-18459.570) (-18453.327) (-18457.955) -- 0:44:50\n",
      "      365000 -- (-18446.531) (-18443.001) (-18459.597) [-18416.500] * [-18450.159] (-18438.564) (-18441.666) (-18461.521) -- 0:44:46\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.036155\n",
      "\n",
      "      366000 -- (-18430.802) [-18427.064] (-18446.552) (-18430.138) * [-18452.745] (-18433.578) (-18430.766) (-18471.934) -- 0:44:43\n",
      "      367000 -- [-18438.560] (-18433.523) (-18454.549) (-18445.055) * (-18433.474) (-18440.283) [-18445.055] (-18463.183) -- 0:44:38\n",
      "      368000 -- (-18444.566) (-18454.674) (-18430.551) [-18428.784] * [-18431.642] (-18425.207) (-18430.762) (-18457.305) -- 0:44:33\n",
      "      369000 -- (-18441.218) (-18450.044) [-18439.398] (-18445.997) * (-18429.554) (-18434.851) [-18439.987] (-18458.327) -- 0:44:31\n",
      "      370000 -- [-18431.106] (-18468.370) (-18446.885) (-18435.640) * [-18428.099] (-18439.158) (-18436.925) (-18453.855) -- 0:44:26\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.035554\n",
      "\n",
      "      371000 -- [-18423.759] (-18472.072) (-18425.032) (-18443.050) * (-18445.067) (-18421.856) [-18424.557] (-18450.317) -- 0:44:21\n",
      "      372000 -- [-18414.850] (-18451.183) (-18435.998) (-18431.194) * [-18425.868] (-18441.240) (-18436.899) (-18467.338) -- 0:44:18\n",
      "      373000 -- (-18436.256) (-18458.937) [-18420.408] (-18452.736) * [-18434.376] (-18447.362) (-18434.096) (-18465.356) -- 0:44:14\n",
      "      374000 -- (-18414.688) [-18444.430] (-18442.957) (-18450.951) * (-18440.350) [-18437.338] (-18438.625) (-18444.706) -- 0:44:09\n",
      "      375000 -- (-18445.954) (-18451.867) (-18463.179) [-18424.459] * (-18430.877) [-18436.668] (-18445.046) (-18432.128) -- 0:44:06\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.035267\n",
      "\n",
      "      376000 -- (-18434.275) (-18443.469) (-18456.714) [-18424.474] * (-18445.228) (-18440.426) (-18445.211) [-18435.462] -- 0:44:02\n",
      "      377000 -- (-18441.501) (-18453.586) (-18441.670) [-18418.918] * (-18440.042) [-18441.739] (-18431.291) (-18455.998) -- 0:43:59\n",
      "      378000 -- [-18435.895] (-18442.294) (-18448.667) (-18422.057) * (-18437.129) (-18453.569) [-18418.382] (-18453.980) -- 0:43:54\n",
      "      379000 -- (-18426.709) (-18443.805) (-18450.247) [-18414.526] * (-18447.857) (-18444.691) [-18428.363] (-18460.866) -- 0:43:49\n",
      "      380000 -- [-18426.373] (-18437.268) (-18444.626) (-18425.133) * (-18443.454) (-18447.134) (-18421.770) [-18437.312] -- 0:43:46\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.034720\n",
      "\n",
      "      381000 -- (-18441.988) [-18429.487] (-18449.185) (-18421.048) * (-18471.304) (-18454.425) [-18426.389] (-18456.543) -- 0:43:42\n",
      "      382000 -- (-18446.915) (-18441.458) (-18448.690) [-18431.064] * (-18470.092) (-18441.710) [-18437.680] (-18465.757) -- 0:43:39\n",
      "      383000 -- (-18449.690) (-18458.196) (-18453.797) [-18438.729] * (-18465.196) (-18435.104) [-18421.999] (-18483.315) -- 0:43:34\n",
      "      384000 -- (-18461.287) (-18449.344) [-18424.273] (-18431.680) * (-18464.531) [-18433.452] (-18418.448) (-18449.574) -- 0:43:29\n",
      "      385000 -- (-18451.068) (-18452.853) (-18446.245) [-18436.734] * (-18455.806) (-18430.448) [-18433.557] (-18462.149) -- 0:43:26\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.034411\n",
      "\n",
      "      386000 -- (-18472.032) (-18434.109) (-18436.060) [-18425.655] * (-18446.937) (-18445.327) [-18431.440] (-18452.729) -- 0:43:22\n",
      "      387000 -- (-18479.480) [-18422.052] (-18451.929) (-18435.821) * (-18445.777) [-18437.720] (-18449.605) (-18442.665) -- 0:43:17\n",
      "      388000 -- (-18453.015) [-18426.169] (-18445.731) (-18454.126) * (-18445.994) (-18445.076) [-18439.017] (-18453.912) -- 0:43:13\n",
      "      389000 -- (-18443.827) [-18430.093] (-18441.754) (-18464.518) * (-18456.670) (-18452.335) [-18428.508] (-18467.263) -- 0:43:10\n",
      "      390000 -- [-18445.072] (-18457.830) (-18438.153) (-18471.723) * (-18457.054) (-18440.274) [-18422.371] (-18474.750) -- 0:43:05\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.033923\n",
      "\n",
      "      391000 -- [-18439.273] (-18450.796) (-18448.480) (-18420.089) * [-18441.701] (-18445.179) (-18439.178) (-18458.585) -- 0:43:00\n",
      "      392000 -- (-18443.630) (-18430.081) (-18432.219) [-18424.369] * [-18436.407] (-18455.904) (-18442.728) (-18474.380) -- 0:42:57\n",
      "      393000 -- (-18446.243) [-18424.855] (-18460.439) (-18435.442) * [-18422.601] (-18456.042) (-18443.938) (-18450.817) -- 0:42:53\n",
      "      394000 -- (-18427.396) [-18423.303] (-18466.388) (-18436.590) * [-18430.394] (-18454.347) (-18440.085) (-18450.026) -- 0:42:48\n",
      "      395000 -- (-18448.809) [-18436.535] (-18465.608) (-18446.184) * (-18443.190) (-18460.722) (-18454.204) [-18417.794] -- 0:42:45\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.033168\n",
      "\n",
      "      396000 -- (-18434.695) [-18427.012] (-18462.095) (-18438.934) * (-18458.724) [-18438.077] (-18452.870) (-18441.492) -- 0:42:40\n",
      "      397000 -- (-18443.072) (-18444.314) [-18421.309] (-18458.893) * (-18456.228) (-18434.600) (-18445.586) [-18441.058] -- 0:42:37\n",
      "      398000 -- (-18453.109) [-18437.582] (-18436.192) (-18446.091) * (-18447.171) (-18422.332) [-18420.220] (-18451.023) -- 0:42:33\n",
      "      399000 -- (-18458.338) (-18440.799) (-18437.222) [-18429.717] * (-18474.658) [-18433.451] (-18433.925) (-18450.162) -- 0:42:30\n",
      "      400000 -- (-18454.546) [-18433.443] (-18472.423) (-18450.393) * (-18483.772) [-18430.707] (-18456.033) (-18432.179) -- 0:42:25\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.031923\n",
      "\n",
      "      401000 -- (-18438.301) [-18423.396] (-18438.252) (-18436.380) * (-18449.262) (-18429.681) (-18474.378) [-18439.418] -- 0:42:20\n",
      "      402000 -- (-18433.362) (-18440.908) [-18450.344] (-18449.426) * [-18429.766] (-18432.684) (-18442.296) (-18434.933) -- 0:42:17\n",
      "      403000 -- [-18439.339] (-18448.537) (-18443.095) (-18447.554) * (-18445.120) [-18432.473] (-18450.914) (-18453.706) -- 0:42:13\n",
      "      404000 -- [-18437.640] (-18447.201) (-18437.403) (-18440.795) * (-18436.970) [-18434.233] (-18456.819) (-18469.078) -- 0:42:10\n",
      "      405000 -- (-18448.617) [-18450.072] (-18450.567) (-18437.174) * [-18424.730] (-18429.718) (-18440.766) (-18492.620) -- 0:42:05\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.030930\n",
      "\n",
      "      406000 -- (-18468.345) [-18431.892] (-18452.433) (-18438.020) * (-18428.342) [-18428.285] (-18452.067) (-18458.959) -- 0:42:02\n",
      "      407000 -- (-18454.803) (-18450.130) (-18444.373) [-18429.410] * (-18449.769) [-18438.957] (-18457.033) (-18469.916) -- 0:41:57\n",
      "      408000 -- (-18456.210) (-18460.318) (-18441.868) [-18420.497] * (-18430.760) (-18454.797) (-18451.442) [-18444.695] -- 0:41:53\n",
      "      409000 -- (-18455.338) (-18452.404) (-18426.556) [-18434.454] * (-18440.852) [-18427.319] (-18447.100) (-18446.874) -- 0:41:49\n",
      "      410000 -- (-18451.915) (-18450.089) [-18429.102] (-18435.437) * (-18446.962) [-18432.969] (-18461.077) (-18445.488) -- 0:41:45\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.029687\n",
      "\n",
      "      411000 -- (-18475.299) [-18442.806] (-18436.828) (-18447.694) * (-18431.384) (-18431.910) [-18429.878] (-18462.435) -- 0:41:40\n",
      "      412000 -- (-18455.843) (-18455.733) [-18432.109] (-18420.820) * (-18438.167) (-18433.809) [-18419.539] (-18452.181) -- 0:41:37\n",
      "      413000 -- [-18440.878] (-18456.344) (-18431.465) (-18436.334) * [-18440.503] (-18425.458) (-18432.924) (-18460.336) -- 0:41:32\n",
      "      414000 -- (-18452.028) (-18438.394) (-18446.247) [-18429.404] * (-18450.639) [-18425.212] (-18435.007) (-18449.424) -- 0:41:28\n",
      "      415000 -- (-18420.882) (-18464.232) [-18452.499] (-18445.909) * (-18444.426) (-18432.099) [-18437.246] (-18458.322) -- 0:41:25\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.028554\n",
      "\n",
      "      416000 -- (-18463.932) (-18430.964) (-18439.283) [-18429.590] * (-18445.691) (-18433.522) [-18432.074] (-18459.520) -- 0:41:20\n",
      "      417000 -- (-18446.645) [-18425.519] (-18458.240) (-18441.868) * (-18441.177) (-18459.444) [-18445.084] (-18468.763) -- 0:41:16\n",
      "      418000 -- (-18433.668) [-18428.002] (-18442.535) (-18442.105) * (-18450.379) (-18448.806) [-18433.227] (-18485.918) -- 0:41:12\n",
      "      419000 -- (-18461.373) (-18439.146) (-18454.808) [-18427.841] * (-18441.538) (-18446.252) [-18423.905] (-18464.468) -- 0:41:08\n",
      "      420000 -- [-18438.787] (-18441.130) (-18438.808) (-18442.867) * (-18443.874) (-18452.549) (-18439.289) [-18445.911] -- 0:41:03\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.027251\n",
      "\n",
      "      421000 -- [-18429.999] (-18454.220) (-18442.001) (-18440.056) * [-18435.638] (-18446.598) (-18449.464) (-18472.081) -- 0:41:00\n",
      "      422000 -- (-18435.264) (-18455.737) [-18440.751] (-18443.787) * [-18420.992] (-18443.866) (-18446.791) (-18465.144) -- 0:40:55\n",
      "      423000 -- [-18419.109] (-18452.285) (-18445.029) (-18432.272) * [-18431.124] (-18430.836) (-18446.361) (-18440.825) -- 0:40:51\n",
      "      424000 -- (-18431.552) (-18487.897) [-18441.651] (-18443.600) * (-18436.147) (-18438.428) [-18445.136] (-18460.700) -- 0:40:46\n",
      "      425000 -- (-18447.469) (-18467.953) [-18424.978] (-18443.679) * (-18426.693) (-18426.347) [-18422.383] (-18460.149) -- 0:40:43\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.026881\n",
      "\n",
      "      426000 -- (-18437.201) (-18452.814) (-18440.316) [-18424.216] * [-18432.433] (-18442.726) (-18436.370) (-18471.500) -- 0:40:38\n",
      "      427000 -- (-18436.149) (-18445.863) (-18439.857) [-18431.564] * (-18457.645) [-18424.583] (-18443.460) (-18473.112) -- 0:40:34\n",
      "      428000 -- [-18436.189] (-18443.380) (-18455.689) (-18437.663) * (-18446.082) (-18440.297) [-18444.747] (-18462.021) -- 0:40:29\n",
      "      429000 -- [-18425.924] (-18456.518) (-18471.646) (-18440.892) * [-18430.817] (-18447.930) (-18443.714) (-18453.123) -- 0:40:26\n",
      "      430000 -- (-18430.319) (-18456.240) (-18462.532) [-18430.903] * [-18421.221] (-18433.125) (-18438.850) (-18455.135) -- 0:40:21\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.026192\n",
      "\n",
      "      431000 -- [-18429.293] (-18461.619) (-18456.241) (-18440.055) * (-18464.653) (-18425.869) [-18429.191] (-18449.947) -- 0:40:17\n",
      "      432000 -- (-18443.341) (-18448.675) (-18473.743) [-18427.062] * (-18460.653) [-18436.158] (-18438.489) (-18451.382) -- 0:40:12\n",
      "      433000 -- (-18436.500) [-18433.566] (-18455.011) (-18431.112) * (-18450.924) [-18437.814] (-18440.163) (-18440.539) -- 0:40:08\n",
      "      434000 -- [-18438.389] (-18451.841) (-18465.271) (-18439.044) * (-18439.343) [-18443.592] (-18444.650) (-18439.672) -- 0:40:04\n",
      "      435000 -- (-18447.166) (-18444.109) [-18445.005] (-18438.042) * (-18421.317) (-18447.363) [-18436.199] (-18445.211) -- 0:40:00\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025789\n",
      "\n",
      "      436000 -- [-18426.012] (-18457.351) (-18450.405) (-18438.739) * (-18422.074) (-18441.365) [-18432.736] (-18440.697) -- 0:39:55\n",
      "      437000 -- (-18456.584) (-18445.012) [-18446.707] (-18433.570) * [-18408.381] (-18438.808) (-18446.239) (-18454.767) -- 0:39:52\n",
      "      438000 -- (-18449.239) [-18444.469] (-18430.577) (-18437.419) * [-18434.250] (-18463.945) (-18459.453) (-18437.338) -- 0:39:47\n",
      "      439000 -- (-18445.124) [-18430.730] (-18437.201) (-18458.235) * (-18442.195) (-18465.968) (-18441.032) [-18441.630] -- 0:39:43\n",
      "      440000 -- (-18458.604) [-18430.396] (-18448.792) (-18451.689) * (-18449.896) (-18460.468) (-18435.619) [-18431.246] -- 0:39:40\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025497\n",
      "\n",
      "      441000 -- (-18448.110) [-18434.451] (-18450.437) (-18445.597) * (-18450.329) (-18453.043) [-18442.003] (-18445.704) -- 0:39:35\n",
      "      442000 -- (-18442.175) [-18434.097] (-18450.740) (-18426.637) * (-18459.643) [-18437.240] (-18428.280) (-18434.754) -- 0:39:30\n",
      "      443000 -- (-18443.330) [-18426.356] (-18454.653) (-18430.492) * (-18457.866) (-18458.428) (-18421.473) [-18432.419] -- 0:39:27\n",
      "      444000 -- [-18441.875] (-18440.741) (-18444.820) (-18441.187) * (-18451.429) (-18465.151) [-18423.335] (-18449.055) -- 0:39:23\n",
      "      445000 -- (-18443.506) [-18434.593] (-18433.149) (-18448.754) * (-18434.645) (-18450.724) [-18422.125] (-18439.021) -- 0:39:18\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025105\n",
      "\n",
      "      446000 -- [-18437.914] (-18457.793) (-18421.460) (-18485.228) * (-18431.269) (-18444.133) [-18435.552] (-18445.642) -- 0:39:15\n",
      "      447000 -- (-18448.564) [-18416.596] (-18439.515) (-18453.665) * [-18426.499] (-18438.620) (-18432.471) (-18435.411) -- 0:39:10\n",
      "      448000 -- (-18451.018) [-18424.420] (-18436.107) (-18464.301) * (-18440.885) (-18426.243) [-18437.791] (-18443.727) -- 0:39:07\n",
      "      449000 -- (-18436.014) (-18444.732) [-18410.454] (-18461.617) * [-18432.878] (-18442.850) (-18430.601) (-18451.452) -- 0:39:02\n",
      "      450000 -- (-18430.528) (-18454.022) [-18412.717] (-18451.161) * [-18434.194] (-18440.454) (-18435.245) (-18462.410) -- 0:38:58\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025013\n",
      "\n",
      "      451000 -- (-18430.623) (-18437.843) [-18424.015] (-18452.497) * (-18426.892) (-18456.339) (-18442.966) [-18445.490] -- 0:38:53\n",
      "      452000 -- [-18426.235] (-18442.520) (-18430.291) (-18451.934) * [-18431.135] (-18456.338) (-18442.302) (-18457.082) -- 0:38:49\n",
      "      453000 -- [-18426.051] (-18445.951) (-18468.870) (-18437.785) * [-18432.628] (-18448.786) (-18440.578) (-18458.316) -- 0:38:45\n",
      "      454000 -- (-18435.257) (-18429.908) (-18440.977) [-18432.851] * (-18443.501) (-18447.471) [-18423.755] (-18457.595) -- 0:38:41\n",
      "      455000 -- (-18427.851) [-18438.304] (-18467.023) (-18447.179) * [-18426.597] (-18432.847) (-18447.154) (-18468.687) -- 0:38:37\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025223\n",
      "\n",
      "      456000 -- (-18437.877) [-18438.062] (-18452.403) (-18441.714) * [-18432.564] (-18431.560) (-18454.351) (-18466.803) -- 0:38:33\n",
      "      457000 -- (-18423.764) (-18432.597) (-18447.963) [-18435.653] * [-18434.871] (-18428.372) (-18428.173) (-18449.495) -- 0:38:29\n",
      "      458000 -- (-18437.831) [-18418.533] (-18446.201) (-18442.077) * (-18473.291) (-18435.787) [-18429.784] (-18454.651) -- 0:38:25\n",
      "      459000 -- [-18422.446] (-18428.918) (-18436.480) (-18442.286) * (-18458.303) (-18430.448) (-18446.047) [-18449.626] -- 0:38:20\n",
      "      460000 -- [-18427.563] (-18453.561) (-18448.647) (-18431.039) * (-18459.752) (-18434.877) [-18440.635] (-18441.281) -- 0:38:17\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024735\n",
      "\n",
      "      461000 -- (-18436.923) [-18439.818] (-18422.666) (-18434.490) * (-18444.801) [-18443.833] (-18443.986) (-18451.240) -- 0:38:12\n",
      "      462000 -- (-18456.796) (-18447.939) [-18423.790] (-18446.516) * (-18469.975) [-18427.406] (-18435.146) (-18441.728) -- 0:38:08\n",
      "      463000 -- (-18452.306) (-18444.344) (-18434.942) [-18439.508] * (-18429.668) [-18420.217] (-18454.278) (-18442.737) -- 0:38:03\n",
      "      464000 -- (-18460.394) (-18433.179) [-18437.088] (-18435.820) * [-18422.671] (-18433.115) (-18460.091) (-18440.962) -- 0:38:00\n",
      "      465000 -- (-18460.135) (-18430.128) (-18431.589) [-18429.752] * [-18430.549] (-18452.542) (-18460.829) (-18465.792) -- 0:37:55\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024191\n",
      "\n",
      "      466000 -- (-18447.136) [-18423.263] (-18440.026) (-18437.260) * [-18439.002] (-18442.954) (-18456.130) (-18444.313) -- 0:37:51\n",
      "      467000 -- (-18456.826) [-18419.248] (-18442.956) (-18448.238) * (-18426.891) (-18452.715) (-18456.603) [-18431.720] -- 0:37:47\n",
      "      468000 -- (-18451.874) [-18420.535] (-18431.101) (-18446.303) * (-18448.752) (-18459.085) (-18467.147) [-18416.670] -- 0:37:43\n",
      "      469000 -- (-18439.315) [-18427.522] (-18433.079) (-18453.030) * (-18433.207) (-18448.237) [-18430.830] (-18444.820) -- 0:37:38\n",
      "      470000 -- (-18448.235) [-18422.544] (-18434.463) (-18437.535) * (-18444.074) [-18440.767] (-18459.932) (-18442.139) -- 0:37:35\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023613\n",
      "\n",
      "      471000 -- (-18450.828) (-18422.204) [-18430.012] (-18447.511) * (-18448.721) [-18426.105] (-18468.641) (-18437.240) -- 0:37:30\n",
      "      472000 -- (-18448.845) [-18434.492] (-18453.641) (-18450.110) * (-18451.460) [-18435.251] (-18453.004) (-18435.029) -- 0:37:26\n",
      "      473000 -- (-18456.611) [-18423.625] (-18444.239) (-18437.453) * (-18452.700) [-18431.602] (-18452.787) (-18429.928) -- 0:37:22\n",
      "      474000 -- (-18452.904) [-18435.717] (-18431.263) (-18427.964) * (-18440.742) [-18432.486] (-18435.658) (-18425.347) -- 0:37:18\n",
      "      475000 -- (-18460.533) (-18438.829) (-18435.632) [-18433.496] * [-18421.297] (-18435.688) (-18435.791) (-18428.762) -- 0:37:13\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022680\n",
      "\n",
      "      476000 -- (-18452.597) [-18429.738] (-18435.881) (-18431.432) * [-18427.603] (-18446.712) (-18440.504) (-18428.298) -- 0:37:10\n",
      "      477000 -- (-18439.867) [-18440.999] (-18433.653) (-18454.183) * [-18423.598] (-18447.360) (-18438.374) (-18423.068) -- 0:37:05\n",
      "      478000 -- (-18463.037) (-18440.027) (-18440.530) [-18427.314] * [-18424.578] (-18435.812) (-18455.506) (-18421.567) -- 0:37:01\n",
      "      479000 -- (-18449.786) (-18439.565) (-18446.473) [-18429.039] * [-18431.430] (-18429.588) (-18450.767) (-18442.779) -- 0:36:56\n",
      "      480000 -- (-18461.496) [-18438.380] (-18443.378) (-18448.746) * (-18426.599) (-18434.208) (-18459.350) [-18440.196] -- 0:36:53\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023055\n",
      "\n",
      "      481000 -- (-18425.954) (-18454.020) (-18435.905) [-18423.270] * [-18427.472] (-18445.543) (-18449.704) (-18442.745) -- 0:36:48\n",
      "      482000 -- (-18456.435) (-18436.895) (-18438.851) [-18428.445] * [-18440.255] (-18443.022) (-18445.037) (-18424.647) -- 0:36:44\n",
      "      483000 -- [-18420.839] (-18446.311) (-18459.881) (-18433.005) * (-18455.624) (-18438.738) [-18427.483] (-18445.084) -- 0:36:40\n",
      "      484000 -- [-18427.837] (-18447.010) (-18451.960) (-18432.423) * [-18449.380] (-18443.064) (-18443.188) (-18446.065) -- 0:36:36\n",
      "      485000 -- (-18435.760) (-18442.789) (-18432.165) [-18421.270] * (-18438.784) [-18423.144] (-18444.159) (-18429.225) -- 0:36:31\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023052\n",
      "\n",
      "      486000 -- (-18442.153) (-18449.166) (-18447.480) [-18434.073] * (-18450.168) [-18429.734] (-18436.502) (-18418.089) -- 0:36:27\n",
      "      487000 -- (-18436.410) (-18432.498) (-18429.322) [-18426.464] * [-18432.902] (-18455.664) (-18446.285) (-18423.319) -- 0:36:23\n",
      "      488000 -- (-18422.107) [-18437.348] (-18434.014) (-18442.895) * (-18432.879) (-18465.070) (-18436.006) [-18423.743] -- 0:36:19\n",
      "      489000 -- (-18433.038) [-18419.433] (-18441.604) (-18459.339) * (-18439.724) (-18452.004) (-18432.874) [-18427.293] -- 0:36:14\n",
      "      490000 -- (-18432.573) [-18427.596] (-18448.418) (-18444.925) * (-18444.562) (-18448.059) (-18454.327) [-18431.474] -- 0:36:10\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022496\n",
      "\n",
      "      491000 -- (-18463.920) [-18430.631] (-18454.479) (-18424.819) * (-18442.783) (-18432.634) (-18445.965) [-18429.062] -- 0:36:06\n",
      "      492000 -- (-18452.059) (-18448.528) (-18457.437) [-18424.226] * (-18446.411) (-18433.942) (-18442.982) [-18448.798] -- 0:36:02\n",
      "      493000 -- (-18436.612) (-18448.719) (-18443.524) [-18420.829] * (-18448.232) [-18425.717] (-18452.219) (-18436.289) -- 0:35:57\n",
      "      494000 -- (-18436.563) (-18442.645) (-18447.815) [-18434.138] * (-18439.534) (-18444.296) (-18432.802) [-18435.650] -- 0:35:54\n",
      "      495000 -- [-18437.667] (-18444.016) (-18462.726) (-18422.070) * [-18434.956] (-18452.843) (-18425.102) (-18423.153) -- 0:35:49\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022557\n",
      "\n",
      "      496000 -- [-18423.562] (-18462.714) (-18460.045) (-18452.097) * (-18429.432) (-18464.637) [-18437.083] (-18439.922) -- 0:35:45\n",
      "      497000 -- (-18420.254) [-18438.458] (-18445.324) (-18427.766) * (-18434.662) (-18460.418) [-18428.482] (-18421.826) -- 0:35:41\n",
      "      498000 -- (-18415.221) [-18428.227] (-18446.281) (-18449.710) * [-18434.274] (-18449.211) (-18434.372) (-18441.723) -- 0:35:37\n",
      "      499000 -- [-18411.315] (-18443.549) (-18425.913) (-18445.882) * [-18441.881] (-18445.389) (-18432.268) (-18458.942) -- 0:35:32\n",
      "      500000 -- (-18429.719) (-18428.215) [-18423.543] (-18452.065) * (-18437.009) (-18463.449) [-18440.648] (-18435.746) -- 0:35:29\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022766\n",
      "\n",
      "      501000 -- (-18421.403) (-18442.922) (-18437.749) [-18434.806] * [-18444.769] (-18454.993) (-18449.324) (-18461.311) -- 0:35:24\n",
      "      502000 -- (-18429.151) [-18450.931] (-18459.028) (-18451.992) * (-18450.754) [-18456.838] (-18457.392) (-18455.009) -- 0:35:19\n",
      "      503000 -- (-18423.508) (-18444.900) [-18450.706] (-18455.420) * (-18454.650) (-18443.413) (-18457.996) [-18434.172] -- 0:35:16\n",
      "      504000 -- [-18434.420] (-18432.228) (-18456.035) (-18476.775) * (-18442.864) (-18441.015) (-18454.712) [-18426.743] -- 0:35:11\n",
      "      505000 -- (-18453.695) [-18426.812] (-18445.766) (-18449.983) * (-18458.511) [-18416.568] (-18461.646) (-18435.653) -- 0:35:07\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022585\n",
      "\n",
      "      506000 -- (-18437.682) (-18435.407) [-18437.542] (-18447.252) * [-18439.890] (-18424.742) (-18445.952) (-18441.008) -- 0:35:03\n",
      "      507000 -- (-18428.544) [-18418.340] (-18445.309) (-18458.849) * (-18466.800) [-18422.965] (-18441.669) (-18447.588) -- 0:34:59\n",
      "      508000 -- (-18428.126) [-18430.819] (-18443.559) (-18455.937) * (-18456.965) (-18427.397) (-18471.074) [-18426.023] -- 0:34:54\n",
      "      509000 -- [-18424.952] (-18453.519) (-18438.144) (-18431.294) * (-18447.192) (-18437.470) [-18445.950] (-18455.316) -- 0:34:51\n",
      "      510000 -- (-18425.695) (-18433.078) [-18429.921] (-18429.778) * (-18426.917) [-18451.397] (-18438.355) (-18452.642) -- 0:34:46\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022465\n",
      "\n",
      "      511000 -- (-18444.035) [-18420.937] (-18420.014) (-18453.792) * (-18445.151) (-18456.018) [-18430.233] (-18454.564) -- 0:34:43\n",
      "      512000 -- (-18449.105) [-18424.471] (-18427.920) (-18455.274) * [-18426.590] (-18453.067) (-18443.667) (-18437.595) -- 0:34:38\n",
      "      513000 -- (-18454.888) [-18430.138] (-18457.156) (-18446.432) * [-18421.838] (-18454.339) (-18442.122) (-18447.987) -- 0:34:34\n",
      "      514000 -- (-18451.567) [-18424.782] (-18443.392) (-18445.296) * [-18427.373] (-18441.460) (-18438.626) (-18455.984) -- 0:34:29\n",
      "      515000 -- (-18451.414) [-18429.465] (-18431.834) (-18451.813) * [-18433.123] (-18446.644) (-18451.355) (-18471.410) -- 0:34:26\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022953\n",
      "\n",
      "      516000 -- (-18446.796) (-18440.567) (-18442.925) [-18432.864] * [-18439.465] (-18453.832) (-18439.298) (-18463.895) -- 0:34:21\n",
      "      517000 -- (-18446.478) (-18438.900) [-18441.553] (-18459.289) * [-18440.258] (-18438.985) (-18445.278) (-18450.313) -- 0:34:17\n",
      "      518000 -- (-18435.793) [-18423.094] (-18433.395) (-18459.259) * (-18447.609) (-18435.787) [-18441.242] (-18447.717) -- 0:34:13\n",
      "      519000 -- [-18427.759] (-18424.542) (-18448.470) (-18444.608) * (-18439.256) (-18434.040) (-18434.688) [-18441.480] -- 0:34:09\n",
      "      520000 -- (-18433.265) [-18425.094] (-18450.188) (-18447.725) * (-18448.581) (-18436.813) (-18436.586) [-18441.337] -- 0:34:04\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022863\n",
      "\n",
      "      521000 -- (-18429.415) [-18434.996] (-18454.431) (-18444.547) * [-18430.948] (-18442.934) (-18436.810) (-18455.396) -- 0:34:00\n",
      "      522000 -- (-18435.906) [-18424.010] (-18439.335) (-18432.613) * (-18438.118) (-18430.890) [-18430.877] (-18447.467) -- 0:33:56\n",
      "      523000 -- (-18430.438) (-18425.051) [-18422.706] (-18454.474) * (-18452.730) (-18450.395) [-18423.389] (-18453.560) -- 0:33:52\n",
      "      524000 -- (-18445.449) (-18445.003) [-18434.445] (-18470.552) * [-18441.309] (-18446.563) (-18437.746) (-18459.110) -- 0:33:47\n",
      "      525000 -- (-18445.576) [-18445.219] (-18443.866) (-18441.444) * (-18462.748) (-18451.099) (-18448.676) [-18443.847] -- 0:33:43\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023094\n",
      "\n",
      "      526000 -- [-18434.620] (-18456.920) (-18440.260) (-18440.637) * [-18439.653] (-18445.877) (-18466.642) (-18437.209) -- 0:33:39\n",
      "      527000 -- (-18443.836) [-18430.614] (-18465.378) (-18454.964) * (-18433.203) (-18468.701) (-18449.150) [-18419.592] -- 0:33:34\n",
      "      528000 -- (-18439.298) [-18418.769] (-18442.442) (-18461.820) * (-18443.116) (-18451.921) (-18438.213) [-18428.913] -- 0:33:31\n",
      "      529000 -- (-18445.102) [-18421.285] (-18439.438) (-18467.848) * (-18432.026) (-18439.502) (-18431.587) [-18422.084] -- 0:33:26\n",
      "      530000 -- [-18437.271] (-18449.678) (-18440.087) (-18434.512) * [-18429.816] (-18474.227) (-18447.369) (-18446.534) -- 0:33:22\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023204\n",
      "\n",
      "      531000 -- (-18422.288) (-18465.973) (-18427.119) [-18438.593] * [-18430.507] (-18460.166) (-18475.541) (-18439.899) -- 0:33:17\n",
      "      532000 -- [-18432.761] (-18463.010) (-18436.935) (-18447.980) * (-18446.118) (-18447.802) (-18466.005) [-18432.730] -- 0:33:14\n",
      "      533000 -- (-18443.681) (-18458.696) [-18437.745] (-18444.236) * [-18446.716] (-18441.361) (-18461.248) (-18434.531) -- 0:33:09\n",
      "      534000 -- (-18450.344) (-18442.941) (-18427.551) [-18425.509] * (-18447.496) (-18451.134) (-18472.105) [-18426.997] -- 0:33:05\n",
      "      535000 -- (-18433.374) (-18444.774) (-18445.242) [-18434.731] * (-18431.942) (-18451.428) (-18450.225) [-18434.304] -- 0:33:00\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023675\n",
      "\n",
      "      536000 -- (-18443.306) (-18441.587) (-18444.873) [-18412.093] * [-18437.385] (-18447.707) (-18427.572) (-18431.554) -- 0:32:57\n",
      "      537000 -- [-18429.614] (-18461.764) (-18447.145) (-18425.677) * (-18444.627) (-18441.101) [-18436.196] (-18450.493) -- 0:32:52\n",
      "      538000 -- (-18429.856) (-18442.844) (-18454.026) [-18417.092] * (-18439.112) (-18450.237) (-18441.634) [-18430.936] -- 0:32:48\n",
      "      539000 -- [-18420.448] (-18436.665) (-18453.103) (-18451.182) * (-18448.381) (-18451.726) [-18418.858] (-18440.579) -- 0:32:44\n",
      "      540000 -- (-18439.209) [-18433.873] (-18444.672) (-18451.140) * (-18465.771) [-18435.606] (-18426.081) (-18451.814) -- 0:32:40\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024197\n",
      "\n",
      "      541000 -- (-18442.435) (-18435.957) [-18431.467] (-18455.437) * (-18451.151) [-18436.073] (-18425.304) (-18447.411) -- 0:32:35\n",
      "      542000 -- (-18429.644) (-18451.241) [-18427.393] (-18449.708) * (-18439.224) (-18441.979) [-18427.546] (-18429.189) -- 0:32:31\n",
      "      543000 -- [-18433.006] (-18454.725) (-18455.423) (-18448.812) * [-18432.404] (-18447.863) (-18454.526) (-18448.433) -- 0:32:27\n",
      "      544000 -- [-18411.929] (-18456.093) (-18438.215) (-18444.086) * [-18412.417] (-18447.069) (-18440.355) (-18430.487) -- 0:32:23\n",
      "      545000 -- [-18420.774] (-18442.822) (-18439.465) (-18443.446) * [-18421.007] (-18438.544) (-18464.971) (-18454.604) -- 0:32:19\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024517\n",
      "\n",
      "      546000 -- [-18426.365] (-18434.805) (-18458.262) (-18445.082) * [-18431.191] (-18431.397) (-18474.247) (-18448.929) -- 0:32:14\n",
      "      547000 -- (-18444.700) [-18423.111] (-18433.839) (-18434.703) * [-18433.542] (-18443.453) (-18453.431) (-18466.326) -- 0:32:11\n",
      "      548000 -- (-18446.027) (-18428.630) [-18417.927] (-18431.691) * [-18428.657] (-18467.614) (-18448.730) (-18473.110) -- 0:32:06\n",
      "      549000 -- (-18438.417) [-18434.875] (-18439.245) (-18429.006) * [-18427.190] (-18446.957) (-18453.961) (-18458.084) -- 0:32:02\n",
      "      550000 -- (-18442.924) [-18424.537] (-18432.375) (-18413.074) * [-18420.432] (-18448.961) (-18448.793) (-18447.915) -- 0:31:58\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025405\n",
      "\n",
      "      551000 -- (-18452.270) [-18431.428] (-18439.401) (-18426.537) * (-18446.749) (-18451.775) (-18442.152) [-18429.707] -- 0:31:54\n",
      "      552000 -- (-18457.829) (-18447.674) (-18444.649) [-18430.825] * (-18444.633) (-18452.988) (-18428.601) [-18427.323] -- 0:31:49\n",
      "      553000 -- (-18442.684) (-18455.536) (-18438.452) [-18426.771] * (-18444.542) (-18453.902) (-18441.668) [-18423.517] -- 0:31:46\n",
      "      554000 -- [-18430.997] (-18430.765) (-18443.374) (-18444.809) * (-18431.400) (-18460.895) (-18455.475) [-18434.100] -- 0:31:41\n",
      "      555000 -- (-18446.901) (-18445.065) (-18455.519) [-18432.616] * (-18440.148) (-18444.886) (-18467.693) [-18435.899] -- 0:31:37\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025781\n",
      "\n",
      "      556000 -- (-18444.953) (-18435.959) (-18450.687) [-18431.930] * (-18461.723) [-18444.183] (-18446.499) (-18424.489) -- 0:31:33\n",
      "      557000 -- (-18465.049) (-18443.763) (-18444.195) [-18441.971] * (-18436.400) (-18484.580) [-18433.042] (-18439.562) -- 0:31:28\n",
      "      558000 -- [-18432.462] (-18443.323) (-18433.608) (-18455.223) * (-18448.315) (-18465.574) [-18430.759] (-18454.311) -- 0:31:24\n",
      "      559000 -- [-18436.665] (-18462.524) (-18434.501) (-18436.200) * [-18424.018] (-18458.026) (-18450.836) (-18448.518) -- 0:31:20\n",
      "      560000 -- (-18439.792) (-18431.253) (-18444.623) [-18430.714] * (-18426.427) (-18452.950) (-18462.923) [-18429.424] -- 0:31:16\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025942\n",
      "\n",
      "      561000 -- (-18452.265) (-18446.349) (-18450.273) [-18435.343] * (-18461.418) (-18434.932) [-18428.145] (-18441.844) -- 0:31:11\n",
      "      562000 -- [-18452.093] (-18435.859) (-18460.235) (-18435.388) * (-18456.682) (-18428.551) [-18424.649] (-18467.654) -- 0:31:07\n",
      "      563000 -- (-18455.298) (-18447.570) (-18448.050) [-18431.417] * (-18450.868) [-18442.611] (-18424.101) (-18449.245) -- 0:31:03\n",
      "      564000 -- (-18450.201) (-18454.169) [-18437.335] (-18441.290) * [-18449.610] (-18453.739) (-18431.724) (-18442.112) -- 0:30:59\n",
      "      565000 -- (-18450.699) [-18438.690] (-18435.418) (-18454.980) * [-18432.390] (-18455.093) (-18434.703) (-18431.196) -- 0:30:54\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.026259\n",
      "\n",
      "      566000 -- (-18431.429) [-18425.140] (-18430.387) (-18451.166) * (-18442.105) (-18451.460) [-18429.764] (-18441.273) -- 0:30:51\n",
      "      567000 -- (-18438.351) (-18434.971) (-18443.230) [-18449.185] * (-18447.834) (-18446.330) [-18433.751] (-18436.778) -- 0:30:46\n",
      "      568000 -- [-18431.437] (-18440.660) (-18449.467) (-18456.411) * (-18441.806) (-18456.699) [-18429.705] (-18433.346) -- 0:30:42\n",
      "      569000 -- (-18424.772) (-18448.829) (-18425.757) [-18429.728] * (-18451.358) (-18447.139) [-18430.270] (-18435.564) -- 0:30:37\n",
      "      570000 -- (-18415.138) (-18425.900) [-18412.990] (-18455.237) * (-18437.315) [-18444.333] (-18466.211) (-18432.663) -- 0:30:33\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.026629\n",
      "\n",
      "      571000 -- [-18423.533] (-18429.194) (-18446.053) (-18459.794) * [-18420.759] (-18444.786) (-18459.615) (-18428.731) -- 0:30:29\n",
      "      572000 -- [-18432.166] (-18425.651) (-18459.831) (-18451.100) * (-18438.170) (-18438.437) (-18445.495) [-18430.037] -- 0:30:24\n",
      "      573000 -- [-18428.997] (-18430.163) (-18455.661) (-18456.195) * (-18428.977) (-18447.717) (-18452.683) [-18432.431] -- 0:30:20\n",
      "      574000 -- (-18433.576) (-18442.877) [-18438.068] (-18446.817) * [-18434.753] (-18434.067) (-18459.461) (-18433.793) -- 0:30:16\n",
      "      575000 -- (-18439.759) [-18432.095] (-18457.527) (-18440.656) * (-18463.757) (-18434.480) (-18455.697) [-18434.640] -- 0:30:12\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.027164\n",
      "\n",
      "      576000 -- [-18431.302] (-18423.559) (-18454.461) (-18427.478) * (-18435.794) [-18431.888] (-18458.474) (-18447.672) -- 0:30:07\n",
      "      577000 -- [-18441.796] (-18434.697) (-18467.977) (-18438.369) * (-18444.197) (-18437.232) (-18434.779) [-18429.586] -- 0:30:04\n",
      "      578000 -- (-18435.493) (-18450.276) (-18452.643) [-18436.498] * [-18435.726] (-18444.435) (-18455.870) (-18432.820) -- 0:29:59\n",
      "      579000 -- (-18449.764) [-18421.478] (-18437.454) (-18434.917) * (-18430.855) (-18448.718) [-18426.597] (-18436.714) -- 0:29:55\n",
      "      580000 -- (-18430.849) (-18430.577) (-18435.738) [-18437.500] * (-18448.536) (-18449.182) [-18427.576] (-18450.915) -- 0:29:51\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.027549\n",
      "\n",
      "      581000 -- (-18430.890) [-18418.979] (-18449.118) (-18440.214) * [-18427.616] (-18448.801) (-18443.507) (-18444.307) -- 0:29:47\n",
      "      582000 -- (-18444.364) (-18424.614) (-18434.873) [-18425.398] * [-18429.026] (-18434.799) (-18443.748) (-18455.734) -- 0:29:42\n",
      "      583000 -- (-18460.545) [-18432.213] (-18434.445) (-18471.351) * [-18429.189] (-18438.135) (-18453.041) (-18461.230) -- 0:29:38\n",
      "      584000 -- (-18441.990) (-18446.997) [-18427.766] (-18441.930) * [-18429.401] (-18456.244) (-18433.920) (-18463.859) -- 0:29:34\n",
      "      585000 -- [-18430.737] (-18451.434) (-18448.105) (-18461.019) * [-18426.037] (-18444.405) (-18445.638) (-18444.028) -- 0:29:29\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.028073\n",
      "\n",
      "      586000 -- (-18466.222) (-18440.921) [-18418.683] (-18464.079) * (-18439.941) (-18451.550) (-18432.280) [-18434.252] -- 0:29:26\n",
      "      587000 -- (-18463.178) (-18435.259) (-18431.641) [-18443.839] * (-18446.124) (-18465.411) (-18410.309) [-18427.946] -- 0:29:21\n",
      "      588000 -- (-18425.491) [-18444.437] (-18437.596) (-18450.628) * (-18459.302) (-18463.748) [-18409.092] (-18435.924) -- 0:29:17\n",
      "      589000 -- [-18427.545] (-18432.255) (-18448.157) (-18424.654) * (-18448.106) (-18446.294) [-18422.407] (-18445.833) -- 0:29:13\n",
      "      590000 -- (-18446.039) (-18454.111) (-18431.828) [-18428.087] * (-18456.251) [-18445.912] (-18456.256) (-18458.108) -- 0:29:09\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.027830\n",
      "\n",
      "      591000 -- [-18429.158] (-18429.679) (-18456.582) (-18452.434) * (-18457.227) [-18443.576] (-18455.296) (-18456.364) -- 0:29:04\n",
      "      592000 -- [-18432.098] (-18432.784) (-18441.996) (-18464.007) * (-18437.712) (-18448.598) [-18449.998] (-18460.342) -- 0:29:00\n",
      "      593000 -- (-18439.960) [-18426.883] (-18444.269) (-18446.580) * [-18445.697] (-18448.112) (-18456.373) (-18462.417) -- 0:28:56\n",
      "      594000 -- (-18447.349) [-18427.841] (-18436.102) (-18451.801) * [-18442.622] (-18442.065) (-18456.751) (-18442.940) -- 0:28:51\n",
      "      595000 -- (-18453.702) [-18431.235] (-18441.143) (-18443.705) * (-18436.727) (-18437.632) [-18435.408] (-18439.635) -- 0:28:48\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.027500\n",
      "\n",
      "      596000 -- [-18429.945] (-18432.277) (-18431.176) (-18446.622) * (-18422.778) [-18436.033] (-18437.354) (-18468.021) -- 0:28:43\n",
      "      597000 -- (-18443.920) [-18425.719] (-18419.432) (-18440.965) * [-18423.875] (-18429.031) (-18433.953) (-18447.958) -- 0:28:39\n",
      "      598000 -- [-18432.067] (-18433.084) (-18446.529) (-18434.061) * (-18427.771) (-18430.050) [-18423.918] (-18444.871) -- 0:28:35\n",
      "      599000 -- (-18437.932) [-18435.208] (-18437.285) (-18432.121) * [-18419.748] (-18447.425) (-18428.575) (-18440.114) -- 0:28:31\n",
      "      600000 -- (-18455.746) (-18451.292) (-18427.440) [-18425.198] * (-18451.440) (-18434.957) [-18437.689] (-18428.479) -- 0:28:26\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.027483\n",
      "\n",
      "      601000 -- (-18450.192) (-18450.876) (-18445.896) [-18419.387] * (-18439.378) [-18430.614] (-18452.697) (-18434.009) -- 0:28:22\n",
      "      602000 -- (-18452.806) (-18434.429) (-18451.121) [-18420.227] * (-18464.827) [-18433.561] (-18450.874) (-18454.229) -- 0:28:18\n",
      "      603000 -- (-18465.584) (-18451.372) (-18430.781) [-18430.528] * (-18453.408) [-18432.649] (-18451.383) (-18459.745) -- 0:28:13\n",
      "      604000 -- (-18452.360) (-18448.272) [-18428.464] (-18421.465) * (-18443.860) [-18430.576] (-18446.073) (-18463.988) -- 0:28:10\n",
      "      605000 -- (-18444.059) (-18445.542) (-18431.262) [-18423.985] * (-18426.314) [-18427.749] (-18440.544) (-18452.551) -- 0:28:05\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.027558\n",
      "\n",
      "      606000 -- (-18449.108) (-18449.231) (-18423.987) [-18430.524] * (-18452.528) (-18436.928) [-18433.115] (-18444.189) -- 0:28:01\n",
      "      607000 -- (-18451.073) (-18458.719) [-18436.095] (-18421.788) * (-18449.165) (-18439.394) [-18434.079] (-18438.942) -- 0:27:57\n",
      "      608000 -- (-18441.645) (-18466.287) (-18452.116) [-18424.686] * (-18432.248) [-18430.908] (-18428.970) (-18454.218) -- 0:27:53\n",
      "      609000 -- (-18447.659) (-18449.301) (-18457.741) [-18428.094] * (-18445.775) (-18469.524) [-18436.910] (-18448.202) -- 0:27:49\n",
      "      610000 -- (-18424.546) (-18459.226) (-18447.219) [-18415.074] * (-18452.462) (-18487.416) [-18439.053] (-18439.023) -- 0:27:44\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.028004\n",
      "\n",
      "      611000 -- (-18460.507) (-18454.206) (-18426.859) [-18416.165] * (-18421.064) (-18458.870) (-18435.860) [-18429.753] -- 0:27:40\n",
      "      612000 -- (-18449.619) [-18428.864] (-18465.193) (-18436.902) * [-18427.247] (-18457.503) (-18445.204) (-18434.758) -- 0:27:35\n",
      "      613000 -- (-18431.701) [-18442.348] (-18439.719) (-18448.892) * (-18451.392) [-18423.079] (-18438.954) (-18454.480) -- 0:27:32\n",
      "      614000 -- (-18454.884) (-18445.273) (-18450.280) [-18437.451] * [-18458.505] (-18434.218) (-18452.025) (-18428.356) -- 0:27:27\n",
      "      615000 -- (-18438.260) (-18455.317) (-18446.492) [-18424.568] * (-18438.772) (-18439.532) (-18436.797) [-18415.258] -- 0:27:23\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.027974\n",
      "\n",
      "      616000 -- (-18455.985) [-18450.609] (-18448.611) (-18428.559) * (-18445.908) (-18441.727) (-18455.071) [-18419.051] -- 0:27:19\n",
      "      617000 -- (-18452.217) [-18459.240] (-18450.273) (-18428.597) * [-18414.974] (-18441.443) (-18453.970) (-18449.069) -- 0:27:15\n",
      "      618000 -- (-18461.791) (-18445.903) (-18465.653) [-18429.813] * [-18419.862] (-18441.753) (-18446.359) (-18441.909) -- 0:27:11\n",
      "      619000 -- [-18442.432] (-18446.997) (-18441.695) (-18437.315) * (-18429.712) [-18422.862] (-18434.820) (-18442.113) -- 0:27:06\n",
      "      620000 -- [-18423.705] (-18441.851) (-18438.640) (-18435.661) * (-18446.192) (-18440.415) [-18433.502] (-18446.416) -- 0:27:02\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.028084\n",
      "\n",
      "      621000 -- (-18433.473) (-18435.816) (-18451.481) [-18434.926] * (-18439.301) (-18433.484) (-18446.858) [-18436.619] -- 0:26:58\n",
      "      622000 -- (-18455.475) [-18435.425] (-18457.809) (-18435.891) * (-18443.898) (-18452.576) [-18439.319] (-18446.818) -- 0:26:54\n",
      "      623000 -- [-18443.442] (-18430.084) (-18452.299) (-18444.734) * [-18429.431] (-18432.188) (-18444.672) (-18467.468) -- 0:26:49\n",
      "      624000 -- (-18455.621) (-18429.037) (-18445.625) [-18447.581] * [-18424.269] (-18436.384) (-18435.127) (-18457.802) -- 0:26:45\n",
      "      625000 -- (-18452.268) (-18441.242) [-18430.645] (-18442.979) * [-18429.458] (-18449.682) (-18459.344) (-18452.215) -- 0:26:41\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.027934\n",
      "\n",
      "      626000 -- (-18452.784) (-18442.811) (-18448.897) [-18431.080] * [-18438.170] (-18439.818) (-18467.153) (-18451.125) -- 0:26:36\n",
      "      627000 -- (-18442.963) (-18452.719) (-18445.059) [-18441.201] * (-18434.022) (-18446.124) [-18437.982] (-18451.673) -- 0:26:32\n",
      "      628000 -- (-18444.233) (-18434.610) (-18441.833) [-18444.540] * [-18432.181] (-18440.239) (-18447.665) (-18444.742) -- 0:26:28\n",
      "      629000 -- (-18442.430) (-18437.940) [-18424.166] (-18447.566) * (-18416.140) [-18432.013] (-18455.648) (-18430.332) -- 0:26:24\n",
      "      630000 -- (-18435.141) [-18456.927] (-18424.499) (-18467.745) * [-18430.443] (-18428.593) (-18449.764) (-18439.455) -- 0:26:19\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.027455\n",
      "\n",
      "      631000 -- (-18450.827) (-18439.579) [-18420.354] (-18449.859) * (-18430.347) (-18452.291) (-18459.093) [-18440.826] -- 0:26:15\n",
      "      632000 -- (-18448.989) [-18422.171] (-18445.987) (-18452.571) * [-18423.089] (-18449.550) (-18453.264) (-18440.148) -- 0:26:11\n",
      "      633000 -- (-18449.376) (-18436.188) [-18436.751] (-18457.956) * [-18427.657] (-18439.851) (-18450.969) (-18427.430) -- 0:26:07\n",
      "      634000 -- (-18451.788) (-18432.817) [-18441.138] (-18440.152) * [-18425.293] (-18448.613) (-18448.589) (-18433.073) -- 0:26:02\n",
      "      635000 -- (-18428.379) (-18434.958) [-18421.206] (-18455.490) * (-18427.662) (-18451.274) [-18441.735] (-18430.659) -- 0:25:58\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.026643\n",
      "\n",
      "      636000 -- (-18478.061) [-18436.563] (-18434.857) (-18455.309) * [-18444.584] (-18443.106) (-18441.408) (-18443.707) -- 0:25:54\n",
      "      637000 -- (-18446.257) (-18433.886) [-18423.907] (-18449.816) * (-18444.890) (-18439.268) [-18427.803] (-18444.652) -- 0:25:50\n",
      "      638000 -- (-18460.101) [-18425.575] (-18432.239) (-18446.624) * [-18423.565] (-18438.296) (-18426.021) (-18455.159) -- 0:25:45\n",
      "      639000 -- (-18433.295) (-18452.476) [-18416.019] (-18448.155) * [-18428.500] (-18456.821) (-18446.614) (-18473.244) -- 0:25:41\n",
      "      640000 -- (-18452.707) (-18443.708) [-18421.479] (-18450.289) * [-18430.581] (-18442.784) (-18454.718) (-18446.049) -- 0:25:37\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.026667\n",
      "\n",
      "      641000 -- (-18439.491) (-18440.025) [-18425.136] (-18460.246) * [-18428.248] (-18454.731) (-18454.902) (-18452.847) -- 0:25:33\n",
      "      642000 -- [-18442.815] (-18436.993) (-18428.448) (-18438.120) * [-18435.987] (-18450.414) (-18461.240) (-18441.899) -- 0:25:29\n",
      "      643000 -- (-18434.991) (-18438.126) [-18425.115] (-18464.542) * [-18442.204] (-18458.542) (-18446.404) (-18447.520) -- 0:25:24\n",
      "      644000 -- (-18437.587) (-18441.219) [-18422.634] (-18445.124) * [-18431.856] (-18447.458) (-18461.710) (-18443.053) -- 0:25:20\n",
      "      645000 -- [-18431.071] (-18430.684) (-18432.620) (-18458.070) * (-18459.877) [-18431.886] (-18450.643) (-18424.851) -- 0:25:16\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.026342\n",
      "\n",
      "      646000 -- [-18433.025] (-18445.381) (-18445.643) (-18457.039) * (-18447.686) (-18440.962) (-18438.364) [-18427.518] -- 0:25:11\n",
      "      647000 -- [-18433.995] (-18445.156) (-18438.950) (-18433.695) * (-18460.368) (-18452.308) [-18435.527] (-18416.409) -- 0:25:08\n",
      "      648000 -- (-18442.022) (-18434.680) [-18440.356] (-18434.392) * (-18454.361) (-18457.360) [-18441.082] (-18423.691) -- 0:25:03\n",
      "      649000 -- (-18436.282) (-18436.529) (-18427.652) [-18419.705] * (-18442.636) [-18446.455] (-18453.988) (-18440.332) -- 0:24:59\n",
      "      650000 -- [-18449.268] (-18449.599) (-18429.617) (-18425.518) * (-18435.667) (-18443.219) (-18451.716) [-18433.825] -- 0:24:55\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025934\n",
      "\n",
      "      651000 -- (-18454.305) (-18441.123) (-18434.858) [-18432.747] * (-18438.840) (-18443.585) (-18435.555) [-18440.840] -- 0:24:50\n",
      "      652000 -- (-18436.980) [-18425.214] (-18425.863) (-18460.241) * (-18437.701) [-18431.887] (-18444.654) (-18443.050) -- 0:24:46\n",
      "      653000 -- (-18445.769) (-18435.395) [-18424.334] (-18460.634) * (-18437.022) (-18460.183) [-18435.567] (-18445.919) -- 0:24:42\n",
      "      654000 -- (-18472.703) (-18425.741) [-18423.725] (-18453.932) * (-18445.323) (-18451.351) (-18452.930) [-18420.700] -- 0:24:38\n",
      "      655000 -- (-18457.713) [-18431.965] (-18428.537) (-18460.454) * (-18434.281) (-18457.720) (-18435.910) [-18429.352] -- 0:24:33\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025854\n",
      "\n",
      "      656000 -- (-18452.969) [-18431.859] (-18440.590) (-18451.811) * [-18426.120] (-18457.215) (-18433.691) (-18441.199) -- 0:24:29\n",
      "      657000 -- (-18455.122) (-18438.054) [-18429.148] (-18458.534) * [-18426.779] (-18442.606) (-18438.911) (-18445.067) -- 0:24:25\n",
      "      658000 -- [-18434.015] (-18439.412) (-18444.446) (-18452.404) * (-18435.164) (-18430.813) (-18430.253) [-18439.648] -- 0:24:21\n",
      "      659000 -- (-18436.580) (-18454.786) [-18434.203] (-18444.057) * [-18434.963] (-18455.350) (-18449.806) (-18451.067) -- 0:24:17\n",
      "      660000 -- [-18431.142] (-18465.427) (-18432.537) (-18445.477) * (-18445.616) [-18424.601] (-18445.378) (-18428.058) -- 0:24:12\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025821\n",
      "\n",
      "      661000 -- [-18422.217] (-18459.146) (-18434.891) (-18436.415) * (-18437.304) (-18449.535) (-18431.870) [-18445.304] -- 0:24:08\n",
      "      662000 -- (-18433.567) (-18475.421) [-18441.046] (-18460.814) * (-18431.573) [-18428.698] (-18438.021) (-18437.772) -- 0:24:04\n",
      "      663000 -- [-18430.779] (-18452.360) (-18439.193) (-18465.860) * (-18442.682) (-18445.214) (-18448.917) [-18433.108] -- 0:24:00\n",
      "      664000 -- (-18432.777) [-18433.956] (-18431.017) (-18456.508) * (-18435.605) [-18436.831] (-18470.681) (-18435.672) -- 0:23:55\n",
      "      665000 -- [-18434.151] (-18452.584) (-18442.229) (-18452.553) * (-18445.554) [-18427.533] (-18456.053) (-18438.854) -- 0:23:51\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025548\n",
      "\n",
      "      666000 -- (-18412.117) (-18440.895) [-18431.908] (-18429.417) * (-18435.880) [-18441.074] (-18459.434) (-18436.321) -- 0:23:47\n",
      "      667000 -- [-18434.552] (-18436.554) (-18423.315) (-18438.170) * (-18454.530) [-18427.315] (-18455.906) (-18443.098) -- 0:23:42\n",
      "      668000 -- (-18439.855) (-18451.174) [-18431.282] (-18445.229) * (-18460.297) (-18422.255) (-18438.897) [-18427.137] -- 0:23:38\n",
      "      669000 -- (-18453.128) (-18440.772) (-18453.819) [-18442.346] * (-18442.052) (-18457.734) (-18431.853) [-18432.412] -- 0:23:34\n",
      "      670000 -- (-18458.983) (-18444.242) (-18435.038) [-18436.331] * (-18434.909) (-18459.432) (-18434.374) [-18432.975] -- 0:23:30\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025150\n",
      "\n",
      "      671000 -- (-18438.590) (-18468.812) (-18446.206) [-18440.638] * (-18431.577) (-18461.657) (-18432.308) [-18422.105] -- 0:23:26\n",
      "      672000 -- [-18417.370] (-18454.990) (-18428.056) (-18441.513) * [-18421.504] (-18448.727) (-18413.059) (-18440.121) -- 0:23:21\n",
      "      673000 -- (-18434.491) (-18448.079) [-18423.486] (-18434.145) * (-18433.696) [-18429.897] (-18430.143) (-18438.884) -- 0:23:17\n",
      "      674000 -- (-18460.098) (-18455.287) [-18420.065] (-18443.791) * [-18440.195] (-18425.234) (-18467.514) (-18437.010) -- 0:23:13\n",
      "      675000 -- (-18464.398) (-18448.816) (-18430.115) [-18426.649] * [-18426.400] (-18432.401) (-18449.934) (-18453.406) -- 0:23:09\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025090\n",
      "\n",
      "      676000 -- (-18465.764) [-18429.604] (-18441.780) (-18438.464) * (-18431.394) (-18438.473) [-18430.607] (-18432.366) -- 0:23:05\n",
      "      677000 -- (-18477.027) [-18435.612] (-18439.697) (-18440.740) * (-18421.119) (-18437.810) [-18438.121] (-18442.456) -- 0:23:00\n",
      "      678000 -- (-18461.557) (-18435.544) [-18425.050] (-18428.999) * [-18430.905] (-18452.498) (-18430.951) (-18455.109) -- 0:22:56\n",
      "      679000 -- (-18435.984) [-18438.364] (-18449.624) (-18441.034) * (-18438.357) (-18448.992) [-18429.004] (-18442.465) -- 0:22:52\n",
      "      680000 -- (-18446.081) (-18461.784) (-18442.738) [-18435.346] * (-18436.027) (-18439.441) [-18433.920] (-18453.128) -- 0:22:48\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025018\n",
      "\n",
      "      681000 -- [-18438.141] (-18450.414) (-18439.288) (-18436.058) * [-18436.450] (-18437.448) (-18433.915) (-18466.133) -- 0:22:43\n",
      "      682000 -- (-18442.989) (-18452.997) (-18462.697) [-18436.263] * [-18428.576] (-18438.841) (-18445.893) (-18454.714) -- 0:22:39\n",
      "      683000 -- (-18440.575) (-18450.101) (-18450.518) [-18427.715] * (-18442.161) (-18447.387) [-18431.342] (-18459.463) -- 0:22:35\n",
      "      684000 -- (-18459.818) [-18424.114] (-18444.703) (-18437.641) * (-18455.227) (-18449.784) [-18435.309] (-18460.575) -- 0:22:30\n",
      "      685000 -- (-18454.845) (-18429.598) (-18448.501) [-18435.890] * (-18448.521) (-18452.126) [-18442.288] (-18453.305) -- 0:22:26\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024926\n",
      "\n",
      "      686000 -- (-18458.535) [-18419.754] (-18464.267) (-18444.832) * [-18422.967] (-18448.156) (-18439.988) (-18451.840) -- 0:22:22\n",
      "      687000 -- (-18456.388) [-18421.530] (-18469.808) (-18443.809) * (-18438.074) (-18437.774) (-18449.005) [-18432.083] -- 0:22:18\n",
      "      688000 -- (-18465.394) [-18416.230] (-18446.903) (-18441.021) * [-18435.882] (-18445.832) (-18439.842) (-18424.353) -- 0:22:14\n",
      "      689000 -- [-18440.285] (-18462.266) (-18439.551) (-18445.566) * [-18434.440] (-18444.084) (-18451.885) (-18435.343) -- 0:22:09\n",
      "      690000 -- (-18426.221) (-18429.292) (-18438.871) [-18437.020] * (-18441.768) [-18430.563] (-18454.073) (-18429.740) -- 0:22:05\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024351\n",
      "\n",
      "      691000 -- (-18439.239) [-18435.843] (-18433.171) (-18459.133) * [-18427.636] (-18455.446) (-18443.961) (-18439.729) -- 0:22:01\n",
      "      692000 -- (-18451.594) [-18413.837] (-18453.180) (-18463.039) * [-18433.096] (-18451.223) (-18442.468) (-18431.960) -- 0:21:57\n",
      "      693000 -- (-18451.279) [-18436.271] (-18464.064) (-18445.635) * (-18451.039) (-18452.267) (-18451.638) [-18445.901] -- 0:21:52\n",
      "      694000 -- (-18457.178) (-18451.949) [-18436.316] (-18437.484) * (-18444.712) (-18428.636) (-18438.788) [-18433.602] -- 0:21:48\n",
      "      695000 -- (-18467.443) [-18417.254] (-18439.520) (-18434.185) * (-18454.781) [-18438.270] (-18441.510) (-18447.496) -- 0:21:44\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023945\n",
      "\n",
      "      696000 -- (-18450.375) [-18438.089] (-18439.213) (-18446.032) * [-18442.115] (-18431.872) (-18476.497) (-18445.915) -- 0:21:39\n",
      "      697000 -- (-18454.494) (-18438.131) (-18445.828) [-18443.422] * (-18438.161) [-18426.259] (-18447.818) (-18451.246) -- 0:21:35\n",
      "      698000 -- (-18440.719) (-18424.476) [-18420.085] (-18461.100) * (-18435.424) [-18437.083] (-18455.994) (-18455.218) -- 0:21:31\n",
      "      699000 -- (-18439.702) (-18430.449) [-18433.913] (-18442.283) * [-18429.642] (-18437.273) (-18442.542) (-18467.472) -- 0:21:27\n",
      "      700000 -- (-18447.708) (-18433.039) (-18432.025) [-18442.679] * (-18431.796) (-18449.370) [-18439.308] (-18465.298) -- 0:21:23\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024081\n",
      "\n",
      "      701000 -- (-18447.875) [-18446.408] (-18436.380) (-18438.599) * (-18442.408) [-18440.143] (-18437.200) (-18453.085) -- 0:21:18\n",
      "      702000 -- (-18450.949) (-18449.150) (-18442.407) [-18428.993] * (-18471.891) [-18442.339] (-18448.840) (-18434.556) -- 0:21:14\n",
      "      703000 -- (-18459.196) [-18440.639] (-18445.310) (-18443.271) * (-18463.303) (-18436.813) (-18434.107) [-18428.302] -- 0:21:09\n",
      "      704000 -- (-18464.981) (-18438.541) (-18457.538) [-18424.958] * (-18466.217) (-18445.682) (-18435.427) [-18431.734] -- 0:21:05\n",
      "      705000 -- (-18453.863) (-18445.256) [-18429.423] (-18445.921) * (-18451.859) [-18432.302] (-18443.686) (-18431.313) -- 0:21:01\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024112\n",
      "\n",
      "      706000 -- (-18438.108) [-18422.737] (-18436.626) (-18460.370) * (-18457.784) [-18427.109] (-18451.382) (-18434.560) -- 0:20:57\n",
      "      707000 -- (-18444.838) (-18447.741) [-18437.903] (-18452.842) * (-18439.493) (-18424.288) (-18445.690) [-18430.256] -- 0:20:53\n",
      "      708000 -- (-18450.860) (-18442.956) [-18454.196] (-18470.839) * (-18441.261) [-18411.479] (-18469.224) (-18433.828) -- 0:20:48\n",
      "      709000 -- (-18446.546) [-18438.751] (-18436.911) (-18460.794) * (-18455.586) [-18432.664] (-18458.550) (-18436.886) -- 0:20:44\n",
      "      710000 -- [-18438.126] (-18448.024) (-18440.139) (-18449.904) * (-18444.527) (-18440.922) (-18437.947) [-18420.330] -- 0:20:40\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023859\n",
      "\n",
      "      711000 -- (-18436.050) (-18445.954) [-18434.192] (-18451.535) * (-18435.123) [-18426.739] (-18435.256) (-18432.267) -- 0:20:36\n",
      "      712000 -- (-18431.953) (-18449.794) [-18422.807] (-18449.171) * (-18450.322) [-18437.668] (-18453.634) (-18426.689) -- 0:20:32\n",
      "      713000 -- (-18433.860) (-18462.491) (-18438.869) [-18432.766] * (-18435.470) (-18437.633) (-18428.923) [-18414.447] -- 0:20:27\n",
      "      714000 -- (-18449.043) (-18448.317) [-18432.512] (-18453.217) * (-18454.982) (-18439.599) (-18456.443) [-18423.200] -- 0:20:23\n",
      "      715000 -- (-18444.676) [-18432.288] (-18429.529) (-18452.259) * (-18441.041) [-18434.575] (-18447.742) (-18424.155) -- 0:20:19\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023635\n",
      "\n",
      "      716000 -- [-18436.855] (-18459.118) (-18455.245) (-18465.975) * (-18476.595) [-18429.097] (-18437.813) (-18441.436) -- 0:20:14\n",
      "      717000 -- (-18431.980) (-18450.640) [-18440.699] (-18442.583) * (-18461.177) [-18432.986] (-18431.293) (-18454.258) -- 0:20:10\n",
      "      718000 -- (-18444.397) (-18437.184) (-18441.089) [-18420.149] * (-18448.900) [-18428.242] (-18432.109) (-18442.681) -- 0:20:06\n",
      "      719000 -- (-18443.404) (-18440.395) (-18447.621) [-18425.066] * (-18447.269) [-18427.334] (-18424.456) (-18435.908) -- 0:20:02\n",
      "      720000 -- (-18474.781) (-18424.846) [-18425.352] (-18423.803) * (-18456.427) (-18442.622) (-18417.349) [-18426.081] -- 0:19:57\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023394\n",
      "\n",
      "      721000 -- (-18441.132) (-18458.831) [-18442.912] (-18429.400) * (-18473.786) [-18422.357] (-18432.647) (-18435.478) -- 0:19:53\n",
      "      722000 -- (-18440.175) (-18459.786) (-18437.900) [-18435.910] * (-18482.456) [-18425.146] (-18440.907) (-18440.311) -- 0:19:49\n",
      "      723000 -- (-18437.527) (-18462.620) (-18430.851) [-18434.545] * (-18443.571) [-18437.371] (-18434.571) (-18437.324) -- 0:19:45\n",
      "      724000 -- (-18429.778) (-18470.842) (-18444.401) [-18425.104] * [-18428.052] (-18432.192) (-18459.634) (-18431.705) -- 0:19:40\n",
      "      725000 -- (-18453.957) (-18456.489) (-18433.665) [-18427.771] * (-18428.860) (-18439.073) (-18442.482) [-18432.856] -- 0:19:36\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023198\n",
      "\n",
      "      726000 -- (-18441.363) (-18452.964) (-18456.354) [-18433.024] * [-18433.674] (-18443.792) (-18466.211) (-18435.792) -- 0:19:32\n",
      "      727000 -- (-18443.957) (-18448.005) (-18455.960) [-18434.257] * [-18437.544] (-18448.795) (-18455.033) (-18445.138) -- 0:19:27\n",
      "      728000 -- [-18428.217] (-18449.375) (-18436.777) (-18436.151) * (-18440.149) [-18432.487] (-18449.716) (-18431.475) -- 0:19:23\n",
      "      729000 -- [-18425.928] (-18458.248) (-18450.579) (-18447.761) * (-18432.177) [-18435.302] (-18438.944) (-18466.441) -- 0:19:19\n",
      "      730000 -- (-18442.924) [-18428.433] (-18450.209) (-18478.123) * (-18439.207) [-18437.365] (-18438.692) (-18446.404) -- 0:19:15\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023303\n",
      "\n",
      "      731000 -- [-18425.975] (-18442.617) (-18432.632) (-18449.840) * [-18440.242] (-18460.715) (-18422.267) (-18446.485) -- 0:19:10\n",
      "      732000 -- [-18429.316] (-18457.239) (-18442.220) (-18455.910) * [-18442.070] (-18451.058) (-18424.566) (-18439.256) -- 0:19:06\n",
      "      733000 -- [-18433.873] (-18459.213) (-18434.725) (-18459.309) * [-18446.017] (-18448.391) (-18427.785) (-18449.097) -- 0:19:01\n",
      "      734000 -- (-18449.264) (-18450.124) (-18446.133) [-18441.998] * (-18437.316) (-18428.465) [-18426.947] (-18467.604) -- 0:18:57\n",
      "      735000 -- (-18444.129) (-18434.995) (-18442.757) [-18442.918] * (-18451.614) (-18434.817) [-18415.812] (-18463.330) -- 0:18:53\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023179\n",
      "\n",
      "      736000 -- (-18458.036) (-18446.345) (-18436.427) [-18443.284] * (-18446.364) (-18444.188) [-18429.104] (-18452.406) -- 0:18:49\n",
      "      737000 -- (-18454.050) (-18448.370) (-18438.094) [-18438.359] * (-18445.488) (-18445.961) [-18424.679] (-18462.100) -- 0:18:44\n",
      "      738000 -- (-18448.953) [-18429.142] (-18433.918) (-18458.735) * (-18464.799) [-18429.279] (-18428.832) (-18426.907) -- 0:18:40\n",
      "      739000 -- (-18446.624) (-18445.488) [-18423.438] (-18440.623) * (-18456.490) [-18423.453] (-18444.673) (-18435.727) -- 0:18:36\n",
      "      740000 -- (-18431.194) (-18428.085) [-18435.372] (-18451.081) * (-18444.199) (-18440.868) (-18468.423) [-18438.582] -- 0:18:32\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022908\n",
      "\n",
      "      741000 -- (-18431.748) [-18428.543] (-18448.305) (-18450.570) * [-18436.861] (-18446.111) (-18470.539) (-18451.049) -- 0:18:27\n",
      "      742000 -- (-18432.251) [-18428.981] (-18442.907) (-18453.304) * (-18442.818) (-18446.178) (-18437.269) [-18436.556] -- 0:18:23\n",
      "      743000 -- (-18427.296) (-18437.143) [-18426.339] (-18435.825) * (-18440.751) (-18430.006) [-18423.361] (-18434.213) -- 0:18:19\n",
      "      744000 -- (-18469.997) (-18423.170) [-18442.575] (-18443.569) * (-18437.240) (-18439.717) [-18435.123] (-18443.874) -- 0:18:14\n",
      "      745000 -- (-18449.961) [-18433.103] (-18446.249) (-18450.358) * (-18446.995) (-18431.367) (-18468.654) [-18438.050] -- 0:18:10\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022731\n",
      "\n",
      "      746000 -- [-18433.018] (-18426.598) (-18435.481) (-18451.462) * [-18436.898] (-18431.278) (-18461.666) (-18440.846) -- 0:18:06\n",
      "      747000 -- (-18440.832) (-18426.007) [-18443.276] (-18454.163) * (-18430.366) [-18434.963] (-18432.237) (-18446.883) -- 0:18:02\n",
      "      748000 -- [-18437.098] (-18433.909) (-18439.191) (-18444.481) * (-18444.080) (-18449.194) [-18422.760] (-18437.167) -- 0:17:58\n",
      "      749000 -- (-18440.255) [-18436.118] (-18426.943) (-18441.848) * [-18435.304] (-18445.288) (-18416.062) (-18441.399) -- 0:17:53\n",
      "      750000 -- (-18441.875) (-18443.725) (-18433.547) [-18429.597] * (-18431.181) (-18448.081) [-18422.662] (-18437.785) -- 0:17:49\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022324\n",
      "\n",
      "      751000 -- (-18440.558) (-18452.640) (-18438.305) [-18433.418] * [-18443.697] (-18439.272) (-18422.644) (-18445.733) -- 0:17:44\n",
      "      752000 -- (-18446.260) (-18455.268) [-18426.654] (-18429.947) * (-18442.525) [-18434.695] (-18447.945) (-18452.939) -- 0:17:40\n",
      "      753000 -- (-18475.791) (-18438.147) [-18441.789] (-18437.803) * [-18448.167] (-18436.426) (-18441.052) (-18466.951) -- 0:17:36\n",
      "      754000 -- (-18456.756) (-18426.570) [-18430.725] (-18432.054) * (-18461.610) (-18433.800) [-18435.431] (-18429.739) -- 0:17:32\n",
      "      755000 -- (-18444.160) (-18459.297) (-18429.179) [-18435.429] * (-18436.471) [-18436.487] (-18437.049) (-18435.953) -- 0:17:27\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022153\n",
      "\n",
      "      756000 -- (-18451.340) (-18440.934) [-18427.674] (-18445.222) * (-18429.018) [-18437.557] (-18443.304) (-18442.367) -- 0:17:23\n",
      "      757000 -- (-18440.934) (-18445.379) [-18444.406] (-18447.385) * [-18412.904] (-18451.102) (-18443.772) (-18457.200) -- 0:17:19\n",
      "      758000 -- (-18440.651) (-18441.866) [-18433.248] (-18444.365) * (-18426.175) [-18431.500] (-18458.847) (-18436.671) -- 0:17:15\n",
      "      759000 -- (-18451.327) (-18446.457) (-18433.399) [-18430.522] * [-18425.372] (-18433.475) (-18450.764) (-18426.641) -- 0:17:10\n",
      "      760000 -- (-18432.806) (-18436.024) [-18425.803] (-18446.134) * (-18429.658) (-18456.195) (-18435.613) [-18453.974] -- 0:17:06\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022238\n",
      "\n",
      "      761000 -- (-18465.535) (-18437.782) [-18433.285] (-18429.247) * [-18428.340] (-18443.636) (-18444.983) (-18431.453) -- 0:17:02\n",
      "      762000 -- (-18449.135) [-18425.879] (-18463.884) (-18424.086) * (-18454.629) [-18440.522] (-18444.185) (-18431.404) -- 0:16:58\n",
      "      763000 -- (-18452.206) (-18428.835) (-18451.064) [-18410.785] * (-18450.754) (-18440.588) [-18434.159] (-18432.059) -- 0:16:53\n",
      "      764000 -- (-18443.163) [-18441.642] (-18444.576) (-18418.777) * (-18441.403) (-18453.849) (-18431.714) [-18421.504] -- 0:16:49\n",
      "      765000 -- (-18432.760) (-18432.743) [-18433.993] (-18452.566) * (-18443.388) (-18443.817) (-18442.585) [-18418.812] -- 0:16:45\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022238\n",
      "\n",
      "      766000 -- (-18426.982) (-18447.371) (-18458.697) [-18440.539] * (-18434.758) (-18455.285) (-18449.907) [-18427.294] -- 0:16:41\n",
      "      767000 -- (-18445.330) (-18448.676) (-18441.441) [-18434.354] * [-18433.238] (-18460.788) (-18438.963) (-18441.725) -- 0:16:36\n",
      "      768000 -- (-18452.971) [-18429.306] (-18436.762) (-18456.579) * (-18429.743) (-18452.556) (-18435.388) [-18432.963] -- 0:16:32\n",
      "      769000 -- (-18433.943) [-18428.765] (-18457.123) (-18455.525) * [-18421.894] (-18450.855) (-18431.792) (-18434.163) -- 0:16:28\n",
      "      770000 -- [-18428.063] (-18430.716) (-18444.548) (-18457.279) * [-18438.645] (-18441.411) (-18453.652) (-18423.752) -- 0:16:24\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022242\n",
      "\n",
      "      771000 -- [-18422.367] (-18432.864) (-18439.497) (-18445.913) * [-18430.177] (-18463.859) (-18447.788) (-18442.436) -- 0:16:19\n",
      "      772000 -- (-18461.324) (-18431.589) (-18448.904) [-18428.014] * (-18451.658) (-18429.828) [-18427.309] (-18460.221) -- 0:16:15\n",
      "      773000 -- (-18435.320) (-18441.329) (-18442.810) [-18432.583] * (-18452.634) [-18427.613] (-18422.954) (-18448.442) -- 0:16:11\n",
      "      774000 -- (-18446.516) (-18457.701) [-18436.114] (-18449.276) * (-18431.458) (-18450.555) [-18418.776] (-18439.316) -- 0:16:07\n",
      "      775000 -- (-18442.835) [-18437.630] (-18447.720) (-18439.943) * [-18440.035] (-18441.070) (-18433.694) (-18436.863) -- 0:16:02\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022480\n",
      "\n",
      "      776000 -- (-18449.692) [-18423.127] (-18446.710) (-18451.862) * (-18448.325) (-18458.579) [-18420.479] (-18452.833) -- 0:15:58\n",
      "      777000 -- (-18449.304) [-18426.567] (-18460.320) (-18440.258) * (-18434.726) (-18447.398) [-18416.830] (-18459.119) -- 0:15:54\n",
      "      778000 -- (-18428.867) (-18458.357) (-18452.471) [-18426.422] * [-18432.522] (-18440.597) (-18434.920) (-18429.445) -- 0:15:49\n",
      "      779000 -- (-18460.208) (-18441.359) (-18437.385) [-18428.401] * (-18451.765) (-18445.080) [-18445.976] (-18428.310) -- 0:15:45\n",
      "      780000 -- (-18464.053) [-18422.489] (-18440.111) (-18440.177) * [-18443.680] (-18467.385) (-18440.583) (-18429.184) -- 0:15:41\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022617\n",
      "\n",
      "      781000 -- (-18458.371) (-18433.992) [-18425.586] (-18449.366) * (-18441.629) (-18442.308) (-18451.562) [-18420.535] -- 0:15:37\n",
      "      782000 -- (-18470.517) (-18439.184) [-18431.274] (-18452.614) * (-18449.923) (-18456.596) (-18441.038) [-18439.974] -- 0:15:33\n",
      "      783000 -- (-18442.251) (-18450.169) (-18429.327) [-18426.088] * [-18439.032] (-18457.528) (-18431.210) (-18437.200) -- 0:15:28\n",
      "      784000 -- [-18455.251] (-18440.285) (-18424.100) (-18419.373) * [-18444.297] (-18453.257) (-18439.211) (-18429.833) -- 0:15:24\n",
      "      785000 -- (-18446.805) (-18446.894) (-18426.494) [-18429.671] * [-18442.452] (-18444.534) (-18445.549) (-18439.092) -- 0:15:19\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022599\n",
      "\n",
      "      786000 -- (-18452.458) (-18445.642) (-18425.684) [-18431.487] * (-18458.718) (-18450.269) [-18431.850] (-18427.021) -- 0:15:15\n",
      "      787000 -- (-18444.059) (-18449.086) [-18439.130] (-18454.372) * (-18450.919) (-18462.516) (-18437.613) [-18426.771] -- 0:15:11\n",
      "      788000 -- (-18463.937) (-18439.579) [-18436.068] (-18445.163) * [-18436.377] (-18455.802) (-18445.343) (-18434.088) -- 0:15:07\n",
      "      789000 -- [-18458.174] (-18441.400) (-18441.902) (-18449.933) * (-18457.592) (-18422.507) (-18439.563) [-18419.376] -- 0:15:03\n",
      "      790000 -- (-18456.102) (-18456.521) [-18432.789] (-18454.889) * (-18440.062) [-18428.671] (-18434.449) (-18419.352) -- 0:14:58\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022491\n",
      "\n",
      "      791000 -- (-18458.441) (-18441.820) [-18441.269] (-18439.045) * (-18432.004) (-18451.901) [-18422.445] (-18426.465) -- 0:14:54\n",
      "      792000 -- (-18472.823) [-18433.152] (-18434.133) (-18440.540) * (-18427.861) (-18439.980) [-18420.182] (-18449.391) -- 0:14:50\n",
      "      793000 -- (-18460.325) (-18430.834) [-18431.599] (-18460.573) * (-18441.492) [-18428.241] (-18443.251) (-18440.013) -- 0:14:46\n",
      "      794000 -- (-18447.560) (-18434.403) [-18429.989] (-18462.617) * (-18441.093) [-18439.142] (-18436.413) (-18455.975) -- 0:14:42\n",
      "      795000 -- (-18450.453) [-18427.368] (-18420.721) (-18463.783) * (-18449.166) (-18440.567) [-18429.735] (-18456.808) -- 0:14:38\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.022573\n",
      "\n",
      "      796000 -- (-18454.828) (-18443.515) (-18420.481) [-18434.883] * (-18460.306) (-18466.758) [-18420.979] (-18447.740) -- 0:14:33\n",
      "      797000 -- (-18468.149) (-18441.651) (-18445.210) [-18426.992] * (-18436.121) (-18457.822) [-18424.955] (-18438.141) -- 0:14:29\n",
      "      798000 -- (-18447.699) (-18449.581) (-18433.352) [-18419.261] * [-18427.581] (-18474.082) (-18421.708) (-18435.658) -- 0:14:24\n",
      "      799000 -- (-18433.476) (-18439.972) (-18447.289) [-18425.069] * [-18429.699] (-18471.644) (-18460.193) (-18436.094) -- 0:14:20\n",
      "      800000 -- (-18450.435) (-18427.761) (-18473.305) [-18419.537] * [-18438.594] (-18461.653) (-18438.819) (-18440.231) -- 0:14:16\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023057\n",
      "\n",
      "      801000 -- (-18446.963) [-18442.359] (-18462.361) (-18431.342) * [-18434.069] (-18440.619) (-18425.852) (-18448.432) -- 0:14:12\n",
      "      802000 -- (-18447.388) (-18458.489) (-18458.064) [-18427.252] * (-18438.666) (-18440.529) (-18442.175) [-18451.352] -- 0:14:08\n",
      "      803000 -- (-18463.523) (-18466.736) (-18459.821) [-18434.760] * (-18441.878) [-18433.679] (-18432.048) (-18448.677) -- 0:14:03\n",
      "      804000 -- [-18430.787] (-18461.231) (-18459.251) (-18413.483) * (-18446.673) [-18421.601] (-18438.390) (-18439.238) -- 0:13:59\n",
      "      805000 -- [-18421.788] (-18449.409) (-18453.655) (-18440.134) * (-18438.535) [-18431.067] (-18446.314) (-18436.774) -- 0:13:55\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023427\n",
      "\n",
      "      806000 -- (-18424.378) [-18432.555] (-18465.014) (-18443.781) * [-18431.134] (-18437.187) (-18423.559) (-18451.100) -- 0:13:50\n",
      "      807000 -- [-18433.619] (-18422.170) (-18469.273) (-18441.942) * (-18427.581) (-18445.583) [-18439.680] (-18457.983) -- 0:13:46\n",
      "      808000 -- (-18439.046) [-18422.365] (-18477.246) (-18433.135) * (-18441.287) (-18453.020) [-18436.823] (-18461.444) -- 0:13:42\n",
      "      809000 -- [-18441.095] (-18425.421) (-18455.318) (-18421.739) * [-18424.429] (-18436.654) (-18437.694) (-18458.928) -- 0:13:38\n",
      "      810000 -- (-18441.885) [-18430.302] (-18441.181) (-18453.139) * (-18430.278) (-18450.161) (-18467.823) [-18430.707] -- 0:13:33\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023685\n",
      "\n",
      "      811000 -- (-18454.226) (-18429.787) [-18444.958] (-18458.823) * (-18433.592) (-18442.949) (-18466.785) [-18416.589] -- 0:13:29\n",
      "      812000 -- (-18435.561) [-18425.400] (-18432.958) (-18460.022) * (-18430.904) (-18437.835) (-18459.077) [-18423.666] -- 0:13:25\n",
      "      813000 -- (-18445.815) [-18434.617] (-18444.528) (-18441.563) * (-18465.286) [-18435.221] (-18476.695) (-18435.332) -- 0:13:20\n",
      "      814000 -- (-18441.801) [-18441.438] (-18436.864) (-18443.765) * (-18444.909) [-18429.553] (-18449.988) (-18430.921) -- 0:13:16\n",
      "      815000 -- (-18460.106) [-18423.806] (-18435.965) (-18440.706) * (-18450.965) (-18424.682) [-18425.404] (-18464.012) -- 0:13:12\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023537\n",
      "\n",
      "      816000 -- (-18440.128) [-18426.516] (-18436.882) (-18438.519) * (-18450.186) (-18444.033) [-18436.791] (-18450.825) -- 0:13:08\n",
      "      817000 -- [-18430.477] (-18428.009) (-18444.107) (-18450.953) * (-18432.056) [-18436.274] (-18434.734) (-18451.866) -- 0:13:03\n",
      "      818000 -- (-18432.247) (-18439.399) [-18426.094] (-18455.113) * (-18444.380) (-18424.640) [-18428.476] (-18442.992) -- 0:12:59\n",
      "      819000 -- (-18459.229) (-18464.490) [-18429.198] (-18436.335) * (-18456.504) [-18426.958] (-18445.554) (-18434.505) -- 0:12:55\n",
      "      820000 -- (-18435.341) (-18450.229) [-18443.199] (-18454.294) * (-18447.728) [-18427.927] (-18449.023) (-18443.651) -- 0:12:51\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023740\n",
      "\n",
      "      821000 -- (-18442.295) (-18432.953) [-18442.159] (-18440.298) * (-18445.305) (-18438.242) [-18438.207] (-18446.019) -- 0:12:46\n",
      "      822000 -- (-18419.323) (-18434.411) (-18447.399) [-18430.192] * (-18433.533) (-18453.941) [-18435.030] (-18455.825) -- 0:12:42\n",
      "      823000 -- (-18426.010) [-18430.673] (-18460.485) (-18427.805) * (-18436.597) (-18461.554) [-18423.649] (-18459.052) -- 0:12:38\n",
      "      824000 -- [-18443.563] (-18432.972) (-18445.399) (-18458.129) * (-18419.158) (-18454.598) (-18436.586) [-18449.367] -- 0:12:33\n",
      "      825000 -- (-18440.359) (-18448.809) (-18451.945) [-18441.016] * [-18427.152] (-18446.977) (-18435.488) (-18452.556) -- 0:12:29\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023923\n",
      "\n",
      "      826000 -- (-18434.026) (-18468.176) (-18441.802) [-18444.374] * (-18424.397) (-18450.563) [-18420.367] (-18440.619) -- 0:12:25\n",
      "      827000 -- [-18425.153] (-18458.316) (-18431.647) (-18431.461) * (-18433.854) (-18452.815) [-18435.339] (-18434.494) -- 0:12:21\n",
      "      828000 -- [-18431.398] (-18446.832) (-18450.897) (-18439.381) * (-18467.231) (-18460.567) [-18441.676] (-18437.430) -- 0:12:17\n",
      "      829000 -- [-18424.567] (-18466.655) (-18458.505) (-18445.403) * (-18435.707) (-18430.982) [-18437.672] (-18432.757) -- 0:12:12\n",
      "      830000 -- (-18445.528) (-18435.081) [-18428.377] (-18448.937) * (-18441.539) (-18425.071) [-18424.739] (-18462.001) -- 0:12:08\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.023965\n",
      "\n",
      "      831000 -- (-18443.965) (-18453.889) [-18442.398] (-18455.885) * (-18418.516) [-18413.358] (-18436.928) (-18454.463) -- 0:12:03\n",
      "      832000 -- (-18437.974) [-18446.886] (-18432.246) (-18468.559) * (-18419.815) (-18435.284) (-18465.537) [-18435.328] -- 0:11:59\n",
      "      833000 -- (-18438.740) [-18444.350] (-18439.725) (-18460.981) * (-18419.293) (-18424.770) [-18435.112] (-18448.233) -- 0:11:55\n",
      "      834000 -- (-18446.533) (-18454.372) [-18445.776] (-18456.886) * (-18435.867) [-18418.244] (-18442.981) (-18449.982) -- 0:11:51\n",
      "      835000 -- (-18435.282) [-18433.834] (-18452.641) (-18442.282) * [-18443.510] (-18436.867) (-18446.112) (-18456.830) -- 0:11:47\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024141\n",
      "\n",
      "      836000 -- [-18438.591] (-18444.146) (-18456.735) (-18436.132) * (-18429.995) (-18430.760) [-18432.657] (-18451.088) -- 0:11:42\n",
      "      837000 -- (-18451.354) (-18448.105) (-18448.217) [-18438.040] * [-18432.058] (-18440.889) (-18451.444) (-18445.501) -- 0:11:38\n",
      "      838000 -- (-18431.817) (-18457.068) (-18439.369) [-18433.181] * (-18437.619) [-18425.266] (-18441.837) (-18437.655) -- 0:11:34\n",
      "      839000 -- (-18428.702) (-18459.421) (-18429.802) [-18434.532] * [-18434.113] (-18432.158) (-18457.524) (-18442.251) -- 0:11:29\n",
      "      840000 -- (-18424.376) (-18454.965) (-18439.680) [-18431.519] * (-18456.471) [-18437.867] (-18456.012) (-18437.134) -- 0:11:25\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024288\n",
      "\n",
      "      841000 -- [-18428.594] (-18455.838) (-18458.582) (-18441.823) * [-18429.127] (-18431.493) (-18458.709) (-18443.947) -- 0:11:21\n",
      "      842000 -- (-18432.543) (-18437.125) (-18460.277) [-18440.577] * [-18444.632] (-18452.679) (-18433.770) (-18435.400) -- 0:11:16\n",
      "      843000 -- (-18452.588) [-18430.649] (-18468.708) (-18432.679) * [-18430.323] (-18441.141) (-18443.689) (-18447.802) -- 0:11:12\n",
      "      844000 -- [-18439.690] (-18456.043) (-18469.994) (-18448.617) * [-18426.948] (-18462.785) (-18420.171) (-18442.235) -- 0:11:08\n",
      "      845000 -- (-18431.425) (-18442.345) (-18464.183) [-18434.666] * (-18429.514) (-18449.650) [-18423.185] (-18442.885) -- 0:11:04\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024414\n",
      "\n",
      "      846000 -- (-18439.242) [-18456.108] (-18461.759) (-18447.212) * (-18436.646) (-18450.373) [-18414.085] (-18449.362) -- 0:10:59\n",
      "      847000 -- (-18429.610) (-18450.653) [-18431.613] (-18443.339) * (-18454.450) (-18429.683) [-18425.754] (-18437.623) -- 0:10:55\n",
      "      848000 -- (-18444.547) (-18428.671) [-18419.704] (-18442.050) * (-18449.874) [-18433.037] (-18435.130) (-18436.553) -- 0:10:51\n",
      "      849000 -- (-18437.740) (-18430.168) [-18431.593] (-18439.643) * (-18458.570) (-18463.018) [-18424.392] (-18440.535) -- 0:10:47\n",
      "      850000 -- (-18465.856) [-18427.522] (-18451.395) (-18434.926) * (-18431.109) (-18453.484) (-18430.064) [-18425.551] -- 0:10:42\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024605\n",
      "\n",
      "      851000 -- [-18435.539] (-18427.818) (-18428.838) (-18450.727) * (-18441.037) (-18444.534) [-18430.840] (-18431.811) -- 0:10:38\n",
      "      852000 -- (-18426.042) (-18445.346) [-18416.558] (-18450.473) * (-18438.926) (-18443.460) (-18434.726) [-18421.931] -- 0:10:34\n",
      "      853000 -- (-18434.728) (-18431.766) [-18420.852] (-18467.195) * (-18463.024) (-18448.537) (-18430.183) [-18412.903] -- 0:10:29\n",
      "      854000 -- [-18428.207] (-18441.297) (-18425.831) (-18451.986) * [-18440.765] (-18461.904) (-18449.723) (-18436.438) -- 0:10:25\n",
      "      855000 -- [-18445.575] (-18437.367) (-18437.607) (-18471.538) * [-18434.615] (-18472.945) (-18442.357) (-18441.301) -- 0:10:21\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.024935\n",
      "\n",
      "      856000 -- (-18455.658) (-18434.488) (-18438.681) [-18444.877] * (-18439.257) (-18454.847) (-18463.106) [-18437.177] -- 0:10:17\n",
      "      857000 -- (-18455.806) (-18424.633) [-18455.798] (-18469.354) * (-18458.661) (-18447.639) [-18428.664] (-18442.203) -- 0:10:12\n",
      "      858000 -- (-18453.478) (-18446.762) (-18438.750) [-18426.367] * [-18425.287] (-18465.352) (-18436.031) (-18439.714) -- 0:10:08\n",
      "      859000 -- (-18454.023) (-18442.000) (-18460.269) [-18432.519] * [-18423.267] (-18443.047) (-18444.049) (-18443.735) -- 0:10:04\n",
      "      860000 -- (-18465.496) (-18446.649) (-18452.290) [-18438.179] * (-18442.315) [-18432.078] (-18437.112) (-18433.956) -- 0:09:59\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025422\n",
      "\n",
      "      861000 -- (-18444.049) (-18451.231) (-18447.558) [-18430.778] * (-18462.204) (-18439.068) [-18423.921] (-18428.255) -- 0:09:55\n",
      "      862000 -- (-18450.294) (-18436.767) (-18454.775) [-18435.398] * (-18460.570) [-18436.303] (-18456.051) (-18431.442) -- 0:09:51\n",
      "      863000 -- (-18449.017) (-18461.739) (-18445.192) [-18419.019] * (-18457.868) (-18423.532) (-18434.150) [-18418.461] -- 0:09:47\n",
      "      864000 -- (-18463.461) (-18469.485) (-18441.443) [-18423.006] * (-18449.102) [-18442.700] (-18433.650) (-18428.118) -- 0:09:42\n",
      "      865000 -- (-18458.457) (-18460.235) (-18426.443) [-18416.939] * (-18471.032) (-18424.578) (-18464.265) [-18433.590] -- 0:09:38\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025482\n",
      "\n",
      "      866000 -- [-18441.428] (-18461.431) (-18431.797) (-18434.653) * [-18445.795] (-18434.958) (-18458.191) (-18445.638) -- 0:09:34\n",
      "      867000 -- [-18438.274] (-18430.620) (-18445.492) (-18433.405) * (-18436.453) (-18451.429) [-18442.404] (-18427.560) -- 0:09:30\n",
      "      868000 -- (-18448.308) (-18439.566) (-18417.523) [-18432.552] * (-18438.541) [-18431.650] (-18462.757) (-18425.293) -- 0:09:25\n",
      "      869000 -- (-18439.508) (-18459.537) [-18417.463] (-18447.861) * [-18438.850] (-18447.867) (-18462.761) (-18459.326) -- 0:09:21\n",
      "      870000 -- (-18449.996) (-18450.326) [-18419.729] (-18450.170) * [-18435.343] (-18452.590) (-18457.493) (-18425.155) -- 0:09:17\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.025833\n",
      "\n",
      "      871000 -- [-18433.039] (-18438.057) (-18435.790) (-18443.094) * [-18438.984] (-18462.005) (-18463.161) (-18432.283) -- 0:09:12\n",
      "      872000 -- (-18447.765) (-18425.174) (-18447.247) [-18438.597] * (-18469.072) [-18424.516] (-18467.538) (-18430.462) -- 0:09:08\n",
      "      873000 -- [-18440.340] (-18432.001) (-18460.921) (-18446.453) * [-18442.694] (-18445.620) (-18478.833) (-18428.294) -- 0:09:04\n",
      "      874000 -- [-18431.162] (-18439.293) (-18449.193) (-18453.228) * (-18441.842) [-18430.007] (-18433.227) (-18458.104) -- 0:09:00\n",
      "      875000 -- (-18445.405) (-18443.136) [-18444.950] (-18448.975) * (-18446.522) (-18434.127) [-18433.939] (-18441.659) -- 0:08:55\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.026312\n",
      "\n",
      "      876000 -- [-18440.818] (-18453.931) (-18433.925) (-18446.780) * (-18435.133) (-18450.625) [-18439.127] (-18443.546) -- 0:08:51\n",
      "      877000 -- (-18432.905) (-18446.715) (-18444.839) [-18438.431] * (-18448.365) [-18428.600] (-18435.385) (-18453.083) -- 0:08:47\n",
      "      878000 -- [-18440.570] (-18433.651) (-18422.108) (-18448.613) * [-18420.338] (-18429.990) (-18434.943) (-18449.673) -- 0:08:42\n",
      "      879000 -- (-18448.047) (-18440.351) [-18423.295] (-18461.417) * (-18434.409) [-18418.464] (-18437.005) (-18461.720) -- 0:08:38\n",
      "      880000 -- [-18444.733] (-18442.670) (-18429.952) (-18458.603) * [-18434.009] (-18427.551) (-18441.426) (-18467.450) -- 0:08:34\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.026735\n",
      "\n",
      "      881000 -- (-18431.685) [-18440.529] (-18447.048) (-18454.751) * (-18451.742) [-18432.201] (-18447.769) (-18445.703) -- 0:08:30\n",
      "      882000 -- (-18437.819) [-18433.201] (-18437.011) (-18460.284) * [-18430.110] (-18443.003) (-18438.573) (-18440.091) -- 0:08:25\n",
      "      883000 -- (-18448.561) (-18425.098) [-18428.968] (-18440.488) * [-18434.514] (-18439.414) (-18446.568) (-18471.237) -- 0:08:21\n",
      "      884000 -- (-18453.451) (-18436.056) (-18449.013) [-18435.634] * [-18432.551] (-18437.540) (-18436.371) (-18469.826) -- 0:08:17\n",
      "      885000 -- (-18458.146) [-18435.133] (-18444.134) (-18443.518) * [-18432.487] (-18421.864) (-18436.108) (-18459.923) -- 0:08:12\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.027259\n",
      "\n",
      "      886000 -- (-18450.543) (-18453.244) (-18449.219) [-18431.007] * (-18441.746) (-18431.772) [-18425.346] (-18472.053) -- 0:08:08\n",
      "      887000 -- (-18427.905) (-18475.467) (-18442.006) [-18435.051] * (-18439.152) (-18429.987) [-18426.064] (-18458.250) -- 0:08:04\n",
      "      888000 -- (-18424.869) (-18442.336) [-18439.534] (-18436.324) * [-18443.540] (-18414.580) (-18432.426) (-18460.266) -- 0:08:00\n",
      "      889000 -- (-18441.745) (-18443.735) [-18435.329] (-18439.104) * [-18440.329] (-18431.237) (-18440.028) (-18449.964) -- 0:07:55\n",
      "      890000 -- (-18457.040) (-18448.260) [-18441.329] (-18441.072) * [-18431.576] (-18425.996) (-18444.342) (-18443.518) -- 0:07:51\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.027752\n",
      "\n",
      "      891000 -- (-18446.341) (-18451.090) [-18444.292] (-18439.936) * [-18438.617] (-18458.794) (-18435.374) (-18443.080) -- 0:07:47\n",
      "      892000 -- (-18432.934) (-18471.338) [-18440.591] (-18451.492) * (-18433.210) (-18449.338) [-18444.359] (-18434.972) -- 0:07:42\n",
      "      893000 -- [-18428.032] (-18447.897) (-18433.210) (-18465.519) * (-18446.469) (-18446.921) [-18441.914] (-18437.086) -- 0:07:38\n",
      "      894000 -- [-18428.781] (-18420.738) (-18433.335) (-18476.185) * (-18441.216) [-18434.011] (-18456.904) (-18438.959) -- 0:07:34\n",
      "      895000 -- [-18432.672] (-18443.336) (-18433.069) (-18459.710) * (-18449.207) (-18424.924) (-18435.929) [-18441.308] -- 0:07:30\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.028343\n",
      "\n",
      "      896000 -- [-18424.773] (-18460.818) (-18416.840) (-18465.191) * (-18446.083) [-18417.003] (-18456.521) (-18434.393) -- 0:07:25\n",
      "      897000 -- (-18434.678) (-18454.420) [-18423.938] (-18452.724) * (-18437.498) [-18429.653] (-18456.402) (-18430.233) -- 0:07:21\n",
      "      898000 -- [-18437.249] (-18440.557) (-18428.107) (-18474.861) * (-18453.115) [-18422.212] (-18441.028) (-18439.481) -- 0:07:17\n",
      "      899000 -- (-18456.706) (-18430.365) [-18443.498] (-18447.426) * (-18447.222) (-18444.303) (-18452.820) [-18429.871] -- 0:07:12\n",
      "      900000 -- (-18466.394) [-18438.772] (-18446.903) (-18450.386) * (-18460.394) (-18469.629) [-18431.012] (-18429.893) -- 0:07:08\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.028325\n",
      "\n",
      "      901000 -- (-18448.922) (-18422.965) [-18434.803] (-18447.835) * (-18428.991) (-18435.483) (-18447.248) [-18420.206] -- 0:07:04\n",
      "      902000 -- (-18435.179) (-18436.604) (-18466.562) [-18429.701] * (-18462.530) (-18449.677) [-18448.545] (-18450.887) -- 0:07:00\n",
      "      903000 -- (-18437.841) [-18425.557] (-18454.877) (-18455.359) * (-18444.466) (-18453.122) [-18438.551] (-18458.252) -- 0:06:55\n",
      "      904000 -- [-18427.706] (-18444.661) (-18446.995) (-18456.326) * (-18444.173) [-18422.050] (-18458.715) (-18431.559) -- 0:06:51\n",
      "      905000 -- (-18446.844) (-18438.377) (-18456.464) [-18430.413] * (-18460.373) [-18435.304] (-18449.271) (-18454.376) -- 0:06:47\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.028624\n",
      "\n",
      "      906000 -- (-18441.084) (-18452.792) (-18438.350) [-18437.101] * (-18452.744) [-18427.574] (-18448.361) (-18446.201) -- 0:06:42\n",
      "      907000 -- (-18463.315) (-18426.680) [-18424.094] (-18443.832) * (-18449.474) [-18422.180] (-18445.501) (-18447.419) -- 0:06:38\n",
      "      908000 -- (-18457.840) [-18438.360] (-18431.499) (-18449.567) * (-18440.014) (-18432.538) (-18444.521) [-18427.493] -- 0:06:34\n",
      "      909000 -- (-18453.923) (-18434.168) [-18430.916] (-18453.110) * (-18434.123) [-18421.964] (-18444.515) (-18421.284) -- 0:06:30\n",
      "      910000 -- (-18450.116) (-18423.418) [-18424.663] (-18446.212) * [-18444.043] (-18430.375) (-18448.294) (-18432.698) -- 0:06:25\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.028821\n",
      "\n",
      "      911000 -- (-18453.228) (-18453.738) [-18425.454] (-18435.665) * [-18458.497] (-18439.493) (-18457.869) (-18450.669) -- 0:06:21\n",
      "      912000 -- (-18447.729) (-18428.988) [-18428.565] (-18429.863) * (-18457.685) [-18427.903] (-18449.642) (-18443.380) -- 0:06:17\n",
      "      913000 -- (-18458.336) [-18419.925] (-18448.307) (-18448.911) * (-18438.875) [-18431.511] (-18450.807) (-18444.298) -- 0:06:12\n",
      "      914000 -- (-18455.897) (-18439.882) (-18439.168) [-18432.284] * (-18453.751) [-18421.587] (-18446.818) (-18446.546) -- 0:06:08\n",
      "      915000 -- (-18446.861) (-18451.480) (-18457.940) [-18428.858] * (-18451.448) (-18454.472) [-18435.982] (-18448.378) -- 0:06:04\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.028873\n",
      "\n",
      "      916000 -- (-18437.349) (-18452.090) (-18455.737) [-18420.134] * [-18430.077] (-18459.926) (-18451.877) (-18455.308) -- 0:06:00\n",
      "      917000 -- [-18431.125] (-18446.882) (-18442.232) (-18447.278) * [-18436.256] (-18452.306) (-18435.787) (-18450.772) -- 0:05:55\n",
      "      918000 -- [-18431.272] (-18437.301) (-18449.977) (-18446.632) * [-18436.268] (-18441.111) (-18463.328) (-18452.821) -- 0:05:51\n",
      "      919000 -- (-18429.968) (-18469.675) (-18449.369) [-18428.018] * [-18436.553] (-18432.682) (-18465.738) (-18459.015) -- 0:05:47\n",
      "      920000 -- [-18429.016] (-18448.050) (-18447.178) (-18451.158) * [-18425.658] (-18436.561) (-18452.529) (-18439.510) -- 0:05:42\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.029106\n",
      "\n",
      "      921000 -- [-18423.285] (-18431.303) (-18463.409) (-18434.674) * [-18431.599] (-18428.681) (-18447.832) (-18423.442) -- 0:05:38\n",
      "      922000 -- [-18428.123] (-18440.332) (-18462.692) (-18445.858) * (-18452.145) [-18426.840] (-18449.255) (-18426.706) -- 0:05:34\n",
      "      923000 -- [-18421.143] (-18455.728) (-18448.903) (-18452.428) * (-18443.092) [-18426.465] (-18470.421) (-18429.626) -- 0:05:30\n",
      "      924000 -- [-18427.461] (-18451.048) (-18453.058) (-18466.766) * (-18438.996) (-18431.034) (-18452.386) [-18419.822] -- 0:05:25\n",
      "      925000 -- [-18426.441] (-18458.687) (-18463.638) (-18437.958) * (-18430.490) (-18464.938) (-18444.236) [-18423.935] -- 0:05:21\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.029280\n",
      "\n",
      "      926000 -- [-18427.095] (-18469.999) (-18429.013) (-18436.496) * (-18425.996) [-18446.390] (-18444.144) (-18447.969) -- 0:05:17\n",
      "      927000 -- (-18433.342) (-18457.528) [-18424.411] (-18461.549) * [-18426.258] (-18476.419) (-18453.429) (-18448.014) -- 0:05:12\n",
      "      928000 -- [-18425.514] (-18472.872) (-18427.809) (-18446.294) * [-18433.135] (-18442.237) (-18450.685) (-18448.562) -- 0:05:08\n",
      "      929000 -- [-18418.010] (-18465.209) (-18436.877) (-18442.089) * (-18441.725) (-18456.042) (-18444.821) [-18439.584] -- 0:05:04\n",
      "      930000 -- [-18429.942] (-18454.901) (-18437.699) (-18450.525) * [-18438.654] (-18443.022) (-18442.435) (-18436.541) -- 0:05:00\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.029367\n",
      "\n",
      "      931000 -- (-18430.090) (-18467.535) [-18436.457] (-18440.208) * (-18441.722) [-18446.546] (-18445.315) (-18445.859) -- 0:04:55\n",
      "      932000 -- [-18441.652] (-18459.247) (-18453.234) (-18434.978) * (-18444.300) (-18446.588) [-18436.557] (-18425.485) -- 0:04:51\n",
      "      933000 -- [-18433.449] (-18467.314) (-18433.923) (-18438.992) * (-18439.051) (-18435.258) (-18452.636) [-18431.331] -- 0:04:47\n",
      "      934000 -- (-18425.636) (-18456.929) [-18427.861] (-18437.714) * (-18426.301) [-18433.492] (-18461.534) (-18446.157) -- 0:04:42\n",
      "      935000 -- (-18445.136) (-18454.020) [-18431.704] (-18444.945) * (-18442.302) [-18434.881] (-18444.876) (-18438.920) -- 0:04:38\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.029473\n",
      "\n",
      "      936000 -- (-18461.626) (-18461.998) [-18426.042] (-18438.725) * (-18432.159) [-18430.320] (-18438.211) (-18437.172) -- 0:04:34\n",
      "      937000 -- (-18447.256) (-18463.963) (-18432.118) [-18423.875] * (-18436.226) (-18434.114) [-18441.728] (-18462.504) -- 0:04:30\n",
      "      938000 -- (-18458.480) (-18464.765) [-18430.996] (-18433.594) * (-18434.452) [-18427.948] (-18426.782) (-18469.176) -- 0:04:25\n",
      "      939000 -- (-18461.200) (-18472.228) [-18430.991] (-18428.973) * [-18421.154] (-18432.535) (-18437.857) (-18464.563) -- 0:04:21\n",
      "      940000 -- (-18455.493) (-18452.076) [-18424.083] (-18435.435) * (-18445.080) [-18437.662] (-18465.724) (-18448.594) -- 0:04:17\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.029569\n",
      "\n",
      "      941000 -- (-18438.246) (-18462.241) [-18432.278] (-18442.410) * (-18443.014) [-18437.212] (-18452.698) (-18443.295) -- 0:04:12\n",
      "      942000 -- [-18439.998] (-18456.903) (-18442.349) (-18445.872) * [-18429.599] (-18437.546) (-18459.036) (-18446.631) -- 0:04:08\n",
      "      943000 -- (-18437.242) (-18441.414) [-18437.530] (-18450.710) * (-18430.341) [-18429.570] (-18461.708) (-18449.770) -- 0:04:04\n",
      "      944000 -- [-18425.310] (-18444.878) (-18423.770) (-18453.147) * [-18424.442] (-18441.962) (-18441.603) (-18448.391) -- 0:04:00\n",
      "      945000 -- [-18424.916] (-18430.548) (-18443.662) (-18442.426) * [-18432.910] (-18447.531) (-18464.391) (-18486.646) -- 0:03:55\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.029630\n",
      "\n",
      "      946000 -- [-18445.376] (-18433.341) (-18453.958) (-18445.369) * [-18422.916] (-18459.276) (-18454.442) (-18449.849) -- 0:03:51\n",
      "      947000 -- [-18440.848] (-18434.463) (-18447.832) (-18455.309) * [-18420.825] (-18445.414) (-18443.663) (-18453.031) -- 0:03:47\n",
      "      948000 -- [-18449.989] (-18439.576) (-18441.168) (-18439.541) * (-18437.733) (-18437.209) (-18458.633) [-18445.895] -- 0:03:42\n",
      "      949000 -- (-18453.090) (-18425.184) [-18439.152] (-18443.885) * [-18441.505] (-18430.188) (-18449.624) (-18442.533) -- 0:03:38\n",
      "      950000 -- (-18477.048) [-18430.520] (-18442.338) (-18453.788) * (-18436.080) [-18428.207] (-18431.765) (-18438.937) -- 0:03:34\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.029714\n",
      "\n",
      "      951000 -- (-18458.001) (-18443.587) [-18432.546] (-18462.585) * (-18464.234) [-18420.853] (-18454.235) (-18432.569) -- 0:03:30\n",
      "      952000 -- (-18443.275) [-18426.239] (-18446.399) (-18440.209) * (-18473.929) [-18422.806] (-18447.372) (-18447.737) -- 0:03:25\n",
      "      953000 -- (-18442.936) (-18449.964) [-18430.256] (-18458.833) * (-18450.117) [-18440.382] (-18440.994) (-18434.329) -- 0:03:21\n",
      "      954000 -- (-18425.583) (-18451.794) [-18428.609] (-18452.464) * (-18444.206) [-18426.964] (-18447.998) (-18451.759) -- 0:03:17\n",
      "      955000 -- (-18456.274) (-18433.311) (-18434.249) [-18433.092] * (-18439.313) [-18427.026] (-18452.406) (-18449.963) -- 0:03:12\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.029546\n",
      "\n",
      "      956000 -- (-18446.321) (-18424.480) [-18432.167] (-18441.833) * (-18437.174) (-18424.983) [-18425.936] (-18438.675) -- 0:03:08\n",
      "      957000 -- (-18450.751) [-18433.179] (-18424.393) (-18441.115) * (-18444.265) (-18437.614) [-18431.967] (-18441.020) -- 0:03:04\n",
      "      958000 -- [-18443.022] (-18428.628) (-18442.988) (-18434.908) * [-18432.220] (-18447.396) (-18436.556) (-18455.976) -- 0:03:00\n",
      "      959000 -- (-18459.813) [-18432.052] (-18439.628) (-18424.865) * [-18449.978] (-18456.992) (-18442.938) (-18459.919) -- 0:02:55\n",
      "      960000 -- (-18442.116) [-18422.643] (-18447.400) (-18443.722) * (-18454.195) (-18456.433) [-18428.315] (-18450.959) -- 0:02:51\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.029586\n",
      "\n",
      "      961000 -- (-18447.122) (-18429.575) (-18437.207) [-18440.932] * (-18436.534) [-18444.267] (-18445.593) (-18453.892) -- 0:02:47\n",
      "      962000 -- [-18436.602] (-18440.390) (-18448.606) (-18454.200) * (-18449.046) [-18449.556] (-18444.240) (-18446.368) -- 0:02:42\n",
      "      963000 -- (-18444.289) [-18435.246] (-18445.592) (-18444.667) * (-18450.160) (-18438.450) (-18465.022) [-18454.456] -- 0:02:38\n",
      "      964000 -- (-18445.974) (-18435.670) (-18445.892) [-18450.050] * (-18452.464) [-18433.653] (-18458.034) (-18447.138) -- 0:02:34\n",
      "      965000 -- (-18455.205) (-18449.753) [-18424.891] (-18428.676) * (-18431.244) (-18445.949) (-18460.431) [-18444.913] -- 0:02:30\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.029345\n",
      "\n",
      "      966000 -- (-18445.196) (-18455.451) (-18442.750) [-18424.819] * (-18427.403) [-18447.610] (-18444.782) (-18439.795) -- 0:02:25\n",
      "      967000 -- (-18431.532) (-18452.895) [-18435.438] (-18446.398) * [-18430.968] (-18442.792) (-18456.129) (-18449.783) -- 0:02:21\n",
      "      968000 -- [-18450.244] (-18455.406) (-18431.643) (-18439.023) * (-18440.215) (-18451.737) (-18460.978) [-18466.317] -- 0:02:17\n",
      "      969000 -- [-18446.992] (-18454.668) (-18439.227) (-18444.428) * (-18436.179) (-18452.222) [-18432.246] (-18468.960) -- 0:02:12\n",
      "      970000 -- (-18456.855) (-18459.263) [-18440.940] (-18438.583) * [-18425.561] (-18473.915) (-18443.904) (-18454.249) -- 0:02:08\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.029779\n",
      "\n",
      "      971000 -- (-18455.144) (-18451.377) [-18455.218] (-18439.286) * (-18433.779) (-18447.696) [-18430.689] (-18450.028) -- 0:02:04\n",
      "      972000 -- (-18432.374) (-18443.988) (-18452.616) [-18430.218] * (-18447.904) (-18438.655) [-18425.611] (-18444.594) -- 0:02:00\n",
      "      973000 -- (-18469.594) [-18434.024] (-18429.487) (-18432.737) * [-18423.184] (-18427.692) (-18433.575) (-18445.865) -- 0:01:55\n",
      "      974000 -- (-18479.473) (-18447.757) [-18425.375] (-18425.493) * [-18421.766] (-18435.389) (-18453.399) (-18438.136) -- 0:01:51\n",
      "      975000 -- (-18460.258) (-18451.836) [-18419.123] (-18440.951) * [-18433.921] (-18447.933) (-18471.063) (-18433.948) -- 0:01:47\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.030310\n",
      "\n",
      "      976000 -- (-18451.844) (-18456.413) [-18429.784] (-18423.960) * (-18442.970) [-18452.948] (-18452.526) (-18462.161) -- 0:01:42\n",
      "      977000 -- (-18456.904) (-18438.911) (-18427.596) [-18417.206] * (-18430.454) (-18434.498) (-18444.790) [-18434.698] -- 0:01:38\n",
      "      978000 -- (-18464.330) [-18426.619] (-18428.029) (-18431.637) * (-18449.707) [-18438.768] (-18447.062) (-18448.450) -- 0:01:34\n",
      "      979000 -- (-18448.385) [-18428.411] (-18441.321) (-18430.030) * (-18442.786) (-18442.936) (-18438.383) [-18433.698] -- 0:01:30\n",
      "      980000 -- (-18448.898) [-18435.479] (-18462.145) (-18422.634) * (-18441.666) (-18453.438) (-18444.472) [-18431.682] -- 0:01:25\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.030383\n",
      "\n",
      "      981000 -- (-18444.978) (-18451.006) (-18434.366) [-18428.613] * (-18421.096) (-18447.620) (-18457.585) [-18432.938] -- 0:01:21\n",
      "      982000 -- (-18451.084) (-18447.588) [-18445.740] (-18449.501) * [-18416.337] (-18454.672) (-18421.910) (-18434.410) -- 0:01:17\n",
      "      983000 -- (-18446.157) (-18445.325) (-18443.990) [-18438.266] * (-18435.699) (-18456.065) [-18425.927] (-18447.575) -- 0:01:12\n",
      "      984000 -- (-18467.321) (-18447.385) [-18429.935] (-18429.436) * (-18435.672) (-18435.682) [-18420.986] (-18447.973) -- 0:01:08\n",
      "      985000 -- (-18460.272) (-18437.456) [-18420.624] (-18438.803) * (-18447.532) (-18436.383) [-18426.832] (-18435.615) -- 0:01:04\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.030549\n",
      "\n",
      "      986000 -- (-18460.326) (-18440.653) [-18419.053] (-18455.849) * (-18453.681) [-18448.116] (-18433.068) (-18443.188) -- 0:01:00\n",
      "      987000 -- (-18431.727) (-18437.345) [-18415.735] (-18461.800) * (-18462.274) (-18447.400) (-18448.529) [-18438.975] -- 0:00:55\n",
      "      988000 -- (-18468.506) (-18440.022) [-18426.126] (-18433.091) * (-18451.447) (-18446.920) (-18448.249) [-18434.248] -- 0:00:51\n",
      "      989000 -- (-18449.096) (-18442.604) [-18423.068] (-18426.965) * (-18458.059) [-18438.354] (-18426.138) (-18437.192) -- 0:00:47\n",
      "      990000 -- (-18462.082) (-18461.180) (-18433.261) [-18419.487] * (-18446.045) (-18439.029) [-18415.969] (-18442.682) -- 0:00:42\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.030776\n",
      "\n",
      "      991000 -- (-18464.044) (-18436.749) [-18423.088] (-18447.142) * (-18453.885) (-18445.649) (-18422.644) [-18431.887] -- 0:00:38\n",
      "      992000 -- (-18461.059) [-18455.099] (-18437.099) (-18425.434) * (-18455.022) (-18443.034) [-18414.161] (-18450.529) -- 0:00:34\n",
      "      993000 -- (-18464.869) (-18460.720) [-18429.847] (-18438.783) * (-18456.890) [-18422.755] (-18419.971) (-18434.716) -- 0:00:30\n",
      "      994000 -- (-18441.820) (-18454.092) [-18418.945] (-18442.921) * (-18459.980) [-18429.472] (-18424.869) (-18432.458) -- 0:00:25\n",
      "      995000 -- (-18448.285) (-18477.011) [-18412.670] (-18454.753) * (-18431.876) [-18417.804] (-18446.400) (-18441.666) -- 0:00:21\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.031196\n",
      "\n",
      "      996000 -- (-18461.193) (-18460.055) (-18423.927) [-18437.063] * (-18443.171) (-18427.444) (-18445.553) [-18431.820] -- 0:00:17\n",
      "      997000 -- (-18446.154) (-18435.117) [-18426.156] (-18447.785) * (-18450.051) [-18437.186] (-18451.336) (-18458.307) -- 0:00:12\n",
      "      998000 -- (-18455.556) [-18437.234] (-18427.319) (-18457.575) * (-18459.414) [-18432.358] (-18441.072) (-18464.682) -- 0:00:08\n",
      "      999000 -- (-18484.810) (-18441.416) [-18438.389] (-18445.825) * [-18436.850] (-18432.861) (-18434.068) (-18458.311) -- 0:00:04\n",
      "      1000000 -- [-18447.281] (-18443.694) (-18440.121) (-18424.981) * (-18425.272) [-18440.178] (-18437.889) (-18438.499) -- 0:00:00\n",
      "\n",
      "      Average standard deviation of split frequencies: 0.031494\n",
      "\n",
      "      Analysis completed in 1 hours 11 mins 31 seconds\n",
      "      Analysis used 9170.32 seconds of CPU time\n",
      "      Likelihood of best state for \"cold\" chain of run 1 was -18403.56\n",
      "      Likelihood of best state for \"cold\" chain of run 2 was -18403.56\n",
      "\n",
      "      Acceptance rates for the moves in the \"cold\" chain of run 1:\n",
      "         With prob.   (last 100)   chain accepted proposals by move\n",
      "            19.1 %     ( 28 %)     Dirichlet(Revmat{all})\n",
      "            31.5 %     ( 20 %)     Slider(Revmat{all})\n",
      "             7.1 %     ( 16 %)     Dirichlet(Pi{all})\n",
      "            20.6 %     ( 23 %)     Slider(Pi{all})\n",
      "            24.1 %     ( 26 %)     Multiplier(Alpha{all})\n",
      "            25.7 %     ( 18 %)     Slider(Pinvar{1})\n",
      "            21.3 %     ( 27 %)     ExtSPR(Tau{all},V{all})\n",
      "            10.5 %     ( 16 %)     ExtTBR(Tau{all},V{all})\n",
      "            20.8 %     ( 16 %)     NNI(Tau{all},V{all})\n",
      "             5.8 %     (  5 %)     ParsSPR(Tau{all},V{all})\n",
      "            26.4 %     ( 24 %)     Multiplier(V{all})\n",
      "            38.5 %     ( 47 %)     Nodeslider(V{all})\n",
      "            23.4 %     ( 35 %)     TLMultiplier(V{all})\n",
      "\n",
      "      Acceptance rates for the moves in the \"cold\" chain of run 2:\n",
      "         With prob.   (last 100)   chain accepted proposals by move\n",
      "            18.5 %     ( 23 %)     Dirichlet(Revmat{all})\n",
      "            31.4 %     ( 30 %)     Slider(Revmat{all})\n",
      "             8.0 %     ( 12 %)     Dirichlet(Pi{all})\n",
      "            20.4 %     ( 31 %)     Slider(Pi{all})\n",
      "            23.6 %     ( 26 %)     Multiplier(Alpha{all})\n",
      "            25.7 %     ( 29 %)     Slider(Pinvar{1})\n",
      "            21.1 %     ( 20 %)     ExtSPR(Tau{all},V{all})\n",
      "            10.5 %     ( 13 %)     ExtTBR(Tau{all},V{all})\n",
      "            20.8 %     ( 24 %)     NNI(Tau{all},V{all})\n",
      "             5.6 %     (  9 %)     ParsSPR(Tau{all},V{all})\n",
      "            26.4 %     ( 20 %)     Multiplier(V{all})\n",
      "            38.7 %     ( 40 %)     Nodeslider(V{all})\n",
      "            23.1 %     ( 32 %)     TLMultiplier(V{all})\n",
      "\n",
      "      Chain swap information for run 1:\n",
      "\n",
      "                   1       2       3       4 \n",
      "           ----------------------------------\n",
      "         1 |            0.53    0.24    0.09 \n",
      "         2 |  166967            0.57    0.27 \n",
      "         3 |  167037  166806            0.59 \n",
      "         4 |  166675  165786  166729         \n",
      "\n",
      "      Chain swap information for run 2:\n",
      "\n",
      "                   1       2       3       4 \n",
      "           ----------------------------------\n",
      "         1 |            0.53    0.23    0.08 \n",
      "         2 |  166702            0.55    0.26 \n",
      "         3 |  167104  166464            0.58 \n",
      "         4 |  166665  166638  166427         \n",
      "\n",
      "      Upper diagonal: Proportion of successful state exchanges between chains\n",
      "      Lower diagonal: Number of attempted state exchanges between chains\n",
      "\n",
      "      Chain information:\n",
      "\n",
      "        ID -- Heat \n",
      "       -----------\n",
      "         1 -- 1.00  (cold chain)\n",
      "         2 -- 0.91 \n",
      "         3 -- 0.83 \n",
      "         4 -- 0.77 \n",
      "\n",
      "      Heat = 1 / (1 + T * (ID - 1))\n",
      "         (where T = 0.10 is the temperature and ID is the chain number)\n",
      "\n",
      "      Setting sumt filename and outputname to mb\n",
      "      Setting urn-in to 2000\n",
      "      Setting sumt contype to Allcompat\n",
      "      Setting showtreeprobs to no\n",
      "      Summarizing trees in files \"mb.run1.t\" and \"mb.run2.t\"\n",
      "      Using relative burnin ('relburnin=yes'), discarding the first 25 % of sampled trees\n",
      "      Writing statistics to files mb.<parts|tstat|vstat|trprobs|con>\n",
      "      Examining first file ...\n",
      "      Found one tree block in file \"mb.run1.t\" with 10001 trees in last block\n",
      "      Expecting the same number of trees in the last tree block of all files\n",
      "\n",
      "      Tree reading status:\n",
      "\n",
      "      0      10      20      30      40      50      60      70      80      90     100\n",
      "      v-------v-------v-------v-------v-------v-------v-------v-------v-------v-------v\n",
      "      *********************************************************************************\n",
      "\n",
      "      Read a total of 20002 trees in 2 files (sampling 15002 of them)\n",
      "         (Each file contained 10001 trees of which 7501 were sampled)\n",
      "                                                                                   \n",
      "      General explanation:                                                          \n",
      "                                                                                   \n",
      "      In an unrooted tree, a taxon bipartition (split) is specified by removing a   \n",
      "      branch, thereby dividing the species into those to the left and those to the  \n",
      "      right of the branch. Here, taxa to one side of the removed branch are denoted \n",
      "      '.' and those to the other side are denoted '*'. Specifically, the '.' symbol \n",
      "      is used for the taxa on the same side as the outgroup.                        \n",
      "                                                                                   \n",
      "      In a rooted or clock tree, the tree is rooted using the model and not by      \n",
      "      reference to an outgroup. Each bipartition therefore corresponds to a clade,  \n",
      "      that is, a group that includes all the descendants of a particular branch in  \n",
      "      the tree.  Taxa that are included in each clade are denoted using '*', and    \n",
      "      taxa that are not included are denoted using the '.' symbol.                  \n",
      "                                                                                   \n",
      "      The output first includes a key to all the bipartitions with frequency larger \n",
      "      or equual to (Minpartfreq) in at least one run. Minpartfreq is a parameter to \n",
      "      sumt command and currently it is set to 0.10.  This is followed by a table  \n",
      "      with statistics for the informative bipartitions (those including at least    \n",
      "      two taxa), sorted from highest to lowest probability. For each bipartition,   \n",
      "      the table gives the number of times the partition or split was observed in all\n",
      "      runs (#obs) and the posterior probability of the bipartition (Probab.), which \n",
      "      is the same as the split frequency. If several runs are summarized, this is   \n",
      "      followed by the minimum split frequency (Min(s)), the maximum frequency       \n",
      "      (Max(s)), and the standard deviation of frequencies (Stddev(s)) across runs.  \n",
      "      The latter value should approach 0 for all bipartitions as MCMC runs converge.\n",
      "                                                                                   \n",
      "      This is followed by a table summarizing branch lengths, node heights (if a    \n",
      "      clock model was used) and relaxed clock parameters (if a relaxed clock model  \n",
      "      was used). The mean, variance, and 95 % credible interval are given for each \n",
      "      of these parameters. If several runs are summarized, the potential scale      \n",
      "      reduction factor (PSRF) is also given; it should approach 1 as runs converge. \n",
      "      Node heights will take calibration points into account, if such points were   \n",
      "      used in the analysis.                                                         \n",
      "                                                                                    \n",
      "      Note that Stddev may be unreliable if the partition is not present in all     \n",
      "      runs (the last column indicates the number of runs that sampled the partition \n",
      "      if more than one run is summarized). The PSRF is not calculated at all if     \n",
      "      the partition is not present in all runs.The PSRF is also sensitive to small  \n",
      "      sample sizes and it should only be considered a rough guide to convergence    \n",
      "      since some of the assumptions allowing one to interpret it as a true potential\n",
      "      scale reduction factor are violated in MrBayes.                               \n",
      "                                                                                    \n",
      "      List of taxa in bipartitions:                                                 \n",
      "                                                                                   \n",
      "         1 -- Ophiostoma_quercus_CMW22909\n",
      "         2 -- 2022_000207\n",
      "         3 -- Heinzbutinia_microspora_CMW17152\n",
      "         4 -- Leptographium_procerum_CMW_34542\n",
      "         5 -- Heinzbutinia_microspora_CBS_440_69\n",
      "         6 -- Harringtonia_lauricola_CBS_121567\n",
      "         7 -- Raffaelea_tritirachium_CBS_726_69\n",
      "         8 -- 2021_000001\n",
      "         9 -- Harringtonia_lauricola_voucher_Hulcr7164\n",
      "        10 -- Ophiostoma_noisomeae_CMW40326\n",
      "        11 -- Ophiostoma_novo_ulmi_subsp_novo_ulmi_CMW43870\n",
      "        12 -- Raffaelea_xyleborini_CMW_45859\n",
      "        13 -- Heinzbutinia_grandicarpa_CMW41131\n",
      "        14 -- Raffaelea_canadensis_CBS_168_66\n",
      "        15 -- Leptographium_procerum_CMW45\n",
      "        16 -- 2023_000307\n",
      "        17 -- Ophiostoma_floccosum_CBS_799_73\n",
      "        18 -- 2021_000003\n",
      "        19 -- Raffaelea_canadensis_CBS_805_70\n",
      "        20 -- 2023_000565\n",
      "        21 -- 2021_000005\n",
      "        22 -- Raffaelea_crossotarsa_CMW44793\n",
      "        23 -- Ophiostoma_floccosum_CMW12623\n",
      "        24 -- Aspergillus_egyptiacus_CBS_656_73\n",
      "        25 -- Raffaelea_ambrosiae_CBS185_64\n",
      "        26 -- Leptographium_procerum_CBS_594_69\n",
      "        27 -- Raffaelea_sp_CMW51555\n",
      "        28 -- Hawksworthiomyces_ciconiae_CBS_147964\n",
      "        29 -- Raffaelea_cyclorhipidia_CMW44790\n",
      "        30 -- Heinzbutinia_grandicarpa_CBS_250_88\n",
      "        31 -- Ophiostoma_quercus_CMW2465\n",
      "        32 -- Ophiostoma_quercus_CMW43227\n",
      "        33 -- Ophiostoma_quercus_CMW40208\n",
      "        34 -- Hawksworthiomyces_hibbettii_CMW_37663\n",
      "        35 -- Esteya_vermicola_CBS_115803\n",
      "        36 -- Leptographium_latens_CBS_124023\n",
      "        37 -- Leptographium_procerum_CBS_929_69\n",
      "        38 -- 2023_000248\n",
      "        39 -- Harringtonia_lauricola_isolate_CV2018003\n",
      "        40 -- Ophiostoma_ulmi_CBS_106_63\n",
      "        41 -- Raffaelea_arxii_CBS_273_70\n",
      "        42 -- 2023_000566\n",
      "        43 -- Leptographium_procerum_CMW_29994\n",
      "        44 -- 2022_000206\n",
      "        45 -- Raffaelea_scolytodis_CBS_124461\n",
      "        46 -- Ophiostoma_floccosum_CMW12622\n",
      "        47 -- Ophiostoma_quercus_CMW41732\n",
      "        48 -- Ophiostoma_ulmi_CBS_151_55\n",
      "        49 -- Harringtonia_lauricola_CMW36261\n",
      "        50 -- Heinzbutinia_grandicarpa_CMW1600\n",
      "\n",
      "      Key to taxon bipartitions (saved to file \"mb.parts\"):\n",
      "\n",
      "       ID -- Partition\n",
      "      ---------------------------------------------------------\n",
      "        1 -- .*************************************************\n",
      "        2 -- .*................................................\n",
      "        3 -- ..*...............................................\n",
      "        4 -- ...*..............................................\n",
      "        5 -- ....*.............................................\n",
      "        6 -- .....*............................................\n",
      "        7 -- ......*...........................................\n",
      "        8 -- .......*..........................................\n",
      "        9 -- ........*.........................................\n",
      "       10 -- .........*........................................\n",
      "       11 -- ..........*.......................................\n",
      "       12 -- ...........*......................................\n",
      "       13 -- ............*.....................................\n",
      "       14 -- .............*....................................\n",
      "       15 -- ..............*...................................\n",
      "       16 -- ...............*..................................\n",
      "       17 -- ................*.................................\n",
      "       18 -- .................*................................\n",
      "       19 -- ..................*...............................\n",
      "       20 -- ...................*..............................\n",
      "       21 -- ....................*.............................\n",
      "       22 -- .....................*............................\n",
      "       23 -- ......................*...........................\n",
      "       24 -- .......................*..........................\n",
      "       25 -- ........................*.........................\n",
      "       26 -- .........................*........................\n",
      "       27 -- ..........................*.......................\n",
      "       28 -- ...........................*......................\n",
      "       29 -- ............................*.....................\n",
      "       30 -- .............................*....................\n",
      "       31 -- ..............................*...................\n",
      "       32 -- ...............................*..................\n",
      "       33 -- ................................*.................\n",
      "       34 -- .................................*................\n",
      "       35 -- ..................................*...............\n",
      "       36 -- ...................................*..............\n",
      "       37 -- ....................................*.............\n",
      "       38 -- .....................................*............\n",
      "       39 -- ......................................*...........\n",
      "       40 -- .......................................*..........\n",
      "       41 -- ........................................*.........\n",
      "       42 -- .........................................*........\n",
      "       43 -- ..........................................*.......\n",
      "       44 -- ...........................................*......\n",
      "       45 -- ............................................*.....\n",
      "       46 -- .............................................*....\n",
      "       47 -- ..............................................*...\n",
      "       48 -- ...............................................*..\n",
      "       49 -- ................................................*.\n",
      "       50 -- .................................................*\n",
      "       51 -- .******.**.*****.**.**.*******...******.*.***...**\n",
      "       52 -- ................*..*..*......................*....\n",
      "       53 -- .....*..*........*....................*.........*.\n",
      "       54 -- ...*..........*.....*....*..........*.....*.......\n",
      "       55 -- .......*..*.......................................\n",
      "       56 -- ...*..........*..........*..........*.....*.......\n",
      "       57 -- ............*................*...................*\n",
      "       58 -- ..*.*.......*................*...................*\n",
      "       59 -- ..*.*.......*................*.............*.....*\n",
      "       60 -- ...............*.....*....*.......................\n",
      "       61 -- ...*..........*.....*....*.........**.....*.......\n",
      "       62 -- .**.***.**.***.*.**..*.**.****...**..**.*..**...**\n",
      "       63 -- ...........................*.....*................\n",
      "       64 -- .............*....*...............................\n",
      "       65 -- .....................................*..*.........\n",
      "       66 -- .*****************************...********.****.***\n",
      "       67 -- .******************************..*****************\n",
      "       68 -- ........*........*....................*...........\n",
      "       69 -- .******************************..********.********\n",
      "       70 -- ...............*..........*.......................\n",
      "       71 -- .....*..*........*....................*...........\n",
      "       72 -- .......*..*............................*.......*..\n",
      "       73 -- ......*....*...*.....*..*.*.......................\n",
      "       74 -- .......*..*.....*..*..*................*.....*.*..\n",
      "       75 -- ..*.*....*..*..............*.*...*.........*.....*\n",
      "       76 -- ......*....*............*.........................\n",
      "       77 -- ..*.*.......*..............*.*...*.........*.....*\n",
      "       78 -- .....................................*..*...*.....\n",
      "       79 -- ..*.*.............................................\n",
      "       80 -- .*...**.*..*.*.*.**..*..*.*.*.....*..**.*...*...*.\n",
      "       81 -- .*...........*....*.........*.....*...............\n",
      "       82 -- .*...*..*....*...**.........*.....*..**.*...*...*.\n",
      "       83 -- .*...**.*..*.*.*.**..*.**.*.*.....*..**.*...*...*.\n",
      "       84 -- .............*....*.........*.....*...............\n",
      "       85 -- ..*.*....*..*..........*...*.*...*.........*.....*\n",
      "       86 -- .............*....*.........*.....................\n",
      "       87 -- .*****************************...********.********\n",
      "       88 -- .*...*..*........*................*..**.*...*...*.\n",
      "       89 -- .......*..*....................................*..\n",
      "       90 -- ......*.................*.........................\n",
      "       91 -- .....*..*........*...................**.*...*...*.\n",
      "       92 -- .******************************.******************\n",
      "       93 -- .*................................*..*..*...*.....\n",
      "       94 -- ............*....................................*\n",
      "       95 -- .............................*...................*\n",
      "       96 -- .******************************..********.****.***\n",
      "       97 -- ........*........*................................\n",
      "       98 -- .......................................*.......*..\n",
      "       99 -- ..................................*..*..*...*.....\n",
      "      100 -- ....*.......*................*...................*\n",
      "      101 -- ................*..*..............................\n",
      "      102 -- ........*.............................*...........\n",
      "      103 -- .................*....................*...........\n",
      "      104 -- ......*....*......................................\n",
      "      105 -- ......*....*.*.*..*..*.**.*.*.....................\n",
      "      106 -- ...*..........*.....................*.....*.......\n",
      "      107 -- .*******************************.*****************\n",
      "      108 -- ............................*.....*...............\n",
      "      109 -- ...................*..*......................*....\n",
      "      110 -- ...............................**.................\n",
      "      111 -- ...*..........*...........................*.......\n",
      "      112 -- ...........*............*.........................\n",
      "      113 -- ............*................*....................\n",
      "      114 -- ...*..........*..........*................*.......\n",
      "      115 -- ......*....*.*.*..*..*..*.*.*.....................\n",
      "      116 -- ...*......................................*.......\n",
      "      117 -- ..............*.....................*.............\n",
      "      118 -- ..............*...........................*.......\n",
      "      119 -- ......................*......................*....\n",
      "      120 -- ................*..*.........................*....\n",
      "      121 -- .........*.................*.....*................\n",
      "      122 -- ................*..*..*...........................\n",
      "      123 -- ...................*.........................*....\n",
      "      124 -- ...................*..*...........................\n",
      "      125 -- ......*....*...*.....*....*.......................\n",
      "      126 -- ..............................*...............*...\n",
      "      127 -- .******.**.*******************...******.*.****..**\n",
      "      128 -- .......*..*............................*..........\n",
      "      129 -- ..................................*.........*.....\n",
      "      130 -- .........*.............*..........................\n",
      "      131 -- .............*....*...............*...............\n",
      "      132 -- .............*....*.....*...*.....................\n",
      "      133 -- .*................................*...............\n",
      "      134 -- .*...........*....*.........*.....*..*..*...*.....\n",
      "      135 -- .........................*..........*.............\n",
      "      136 -- ................*............................*....\n",
      "      137 -- ................*.....*...........................\n",
      "      138 -- ...*..........*...................................\n",
      "      139 -- .............*....*....*....*.....................\n",
      "      140 -- ......*....*...*.....*.**.*.......................\n",
      "      141 -- ..............*..........*........................\n",
      "      142 -- ................*.....*......................*....\n",
      "      143 -- ..............*.....................*.....*.......\n",
      "      144 -- ..................................*..*..*.........\n",
      "      145 -- ...*.....................*........................\n",
      "      ---------------------------------------------------------\n",
      "\n",
      "      Summary statistics for informative taxon bipartitions\n",
      "         (saved to file \"mb.tstat\"):\n",
      "\n",
      "       ID   #obs     Probab.     Sd(s)+      Min(s)      Max(s)   Nruns \n",
      "      ------------------------------------------------------------------\n",
      "       51  15002    1.000000    0.000000    1.000000    1.000000    2\n",
      "       52  15002    1.000000    0.000000    1.000000    1.000000    2\n",
      "       53  15002    1.000000    0.000000    1.000000    1.000000    2\n",
      "       54  15002    1.000000    0.000000    1.000000    1.000000    2\n",
      "       55  15002    1.000000    0.000000    1.000000    1.000000    2\n",
      "       56  15002    1.000000    0.000000    1.000000    1.000000    2\n",
      "       57  14998    0.999733    0.000000    0.999733    0.999733    2\n",
      "       58  14991    0.999267    0.001037    0.998534    1.000000    2\n",
      "       59  14982    0.998667    0.001885    0.997334    1.000000    2\n",
      "       60  14949    0.996467    0.002545    0.994667    0.998267    2\n",
      "       61  14916    0.994267    0.008107    0.988535    1.000000    2\n",
      "       62  14898    0.993068    0.009804    0.986135    1.000000    2\n",
      "       63  14841    0.989268    0.000283    0.989068    0.989468    2\n",
      "       64  14718    0.981069    0.003205    0.978803    0.983336    2\n",
      "       65  14620    0.974537    0.013575    0.964938    0.984135    2\n",
      "       66  14438    0.962405    0.016214    0.950940    0.973870    2\n",
      "       67  14426    0.961605    0.000189    0.961472    0.961738    2\n",
      "       68  14147    0.943008    0.000471    0.942674    0.943341    2\n",
      "       69  14027    0.935009    0.001603    0.933875    0.936142    2\n",
      "       70  13745    0.916211    0.000660    0.915745    0.916678    2\n",
      "       71  13542    0.902680    0.018288    0.889748    0.915611    2\n",
      "       72  13167    0.877683    0.012915    0.868551    0.886815    2\n",
      "       73  12637    0.842354    0.138669    0.744301    0.940408    2\n",
      "       74  12216    0.814291    0.014140    0.804293    0.824290    2\n",
      "       75  11993    0.799427    0.005373    0.795627    0.803226    2\n",
      "       76  11967    0.797694    0.160728    0.684042    0.911345    2\n",
      "       77  11331    0.755299    0.076452    0.701240    0.809359    2\n",
      "       78  11176    0.744967    0.085407    0.684575    0.805359    2\n",
      "       79   8809    0.587188    0.035539    0.562058    0.612318    2\n",
      "       80   8320    0.554593    0.065988    0.507932    0.601253    2\n",
      "       81   7839    0.522530    0.126225    0.433276    0.611785    2\n",
      "       82   7554    0.503533    0.167232    0.385282    0.621784    2\n",
      "       83   7225    0.481602    0.041761    0.452073    0.511132    2\n",
      "       84   7185    0.478936    0.136406    0.382482    0.575390    2\n",
      "       85   7121    0.474670    0.048925    0.440075    0.509265    2\n",
      "       86   6837    0.455739    0.035539    0.430609    0.480869    2\n",
      "       87   6747    0.449740    0.007259    0.444607    0.454873    2\n",
      "       88   6699    0.446540    0.118118    0.363018    0.530063    2\n",
      "       89   6504    0.433542    0.000566    0.433142    0.433942    2\n",
      "       90   6465    0.430943    0.093231    0.365018    0.496867    2\n",
      "       91   6045    0.402946    0.101338    0.331289    0.474603    2\n",
      "       92   5939    0.395881    0.000660    0.395414    0.396347    2\n",
      "       93   5690    0.379283    0.117647    0.296094    0.462472    2\n",
      "       94   5678    0.378483    0.007541    0.373150    0.383815    2\n",
      "       95   5454    0.363552    0.002828    0.361552    0.365551    2\n",
      "       96   5447    0.363085    0.001414    0.362085    0.364085    2\n",
      "       97   5304    0.353553    0.012066    0.345021    0.362085    2\n",
      "       98   5218    0.347820    0.013009    0.338622    0.357019    2\n",
      "       99   4925    0.328290    0.102281    0.255966    0.400613    2\n",
      "      100   4792    0.319424    0.025452    0.301426    0.337422    2\n",
      "      101   4762    0.317424    0.004902    0.313958    0.320891    2\n",
      "      102   4714    0.314225    0.004148    0.311292    0.317158    2\n",
      "      103   4641    0.309359    0.004619    0.306093    0.312625    2\n",
      "      104   4632    0.308759    0.152526    0.200907    0.416611    2\n",
      "      105   4563    0.304159    0.040064    0.275830    0.332489    2\n",
      "      106   4510    0.300627    0.008673    0.294494    0.306759    2\n",
      "      107   4436    0.295694    0.001885    0.294361    0.297027    2\n",
      "      108   4378    0.291828    0.121418    0.205973    0.377683    2\n",
      "      109   4377    0.291761    0.005939    0.287562    0.295961    2\n",
      "      110   4320    0.287962    0.002828    0.285962    0.289961    2\n",
      "      111   4109    0.273897    0.001980    0.272497    0.275297    2\n",
      "      112   3882    0.258765    0.059766    0.216504    0.301027    2\n",
      "      113   3868    0.257832    0.010181    0.250633    0.265031    2\n",
      "      114   3769    0.251233    0.002168    0.249700    0.252766    2\n",
      "      115   3652    0.243434    0.079940    0.186908    0.299960    2\n",
      "      116   3645    0.242968    0.003676    0.240368    0.245567    2\n",
      "      117   3458    0.230503    0.002828    0.228503    0.232502    2\n",
      "      118   3416    0.227703    0.008861    0.221437    0.233969    2\n",
      "      119   3349    0.223237    0.002734    0.221304    0.225170    2\n",
      "      120   3290    0.219304    0.006033    0.215038    0.223570    2\n",
      "      121   3205    0.213638    0.077960    0.158512    0.268764    2\n",
      "      122   3034    0.202240    0.008296    0.196374    0.208106    2\n",
      "      123   3015    0.200973    0.000471    0.200640    0.201306    2\n",
      "      124   2967    0.197774    0.001791    0.196507    0.199040    2\n",
      "      125   2745    0.182976    0.153186    0.074657    0.291294    2\n",
      "      126   2702    0.180109    0.010181    0.172910    0.187308    2\n",
      "      127   2579    0.171910    0.011595    0.163712    0.180109    2\n",
      "      128   2570    0.171310    0.010370    0.163978    0.178643    2\n",
      "      129   2262    0.150780    0.053733    0.112785    0.188775    2\n",
      "      130   2257    0.150447    0.009898    0.143448    0.157446    2\n",
      "      131   2255    0.150313    0.003488    0.147847    0.152780    2\n",
      "      132   2225    0.148314    0.136029    0.052126    0.244501    2\n",
      "      133   2170    0.144647    0.007919    0.139048    0.150247    2\n",
      "      134   2072    0.138115    0.041667    0.108652    0.167578    2\n",
      "      135   1902    0.126783    0.003205    0.124517    0.129049    2\n",
      "      136   1822    0.121450    0.003394    0.119051    0.123850    2\n",
      "      137   1796    0.119717    0.000943    0.119051    0.120384    2\n",
      "      138   1689    0.112585    0.004242    0.109585    0.115585    2\n",
      "      139   1627    0.108452    0.001791    0.107186    0.109719    2\n",
      "      140   1600    0.106652    0.039781    0.078523    0.134782    2\n",
      "      141   1595    0.106319    0.003488    0.103853    0.108785    2\n",
      "      142   1592    0.106119    0.012632    0.097187    0.115051    2\n",
      "      143   1588    0.105853    0.002640    0.103986    0.107719    2\n",
      "      144   1564    0.104253    0.031674    0.081856    0.126650    2\n",
      "      145   1522    0.101453    0.003771    0.098787    0.104119    2\n",
      "      ------------------------------------------------------------------\n",
      "      + Convergence diagnostic (standard deviation of split frequencies)\n",
      "        should approach 0.0 as runs converge.\n",
      "\n",
      "\n",
      "      Summary statistics for branch and node parameters\n",
      "         (saved to file \"mb.vstat\"):\n",
      "\n",
      "                                                    95% HPD Interval\n",
      "                                                  --------------------\n",
      "      Parameter            Mean       Variance     Lower       Upper       Median     PSRF+  Nruns\n",
      "      --------------------------------------------------------------------------------------------\n",
      "      length{all}[1]      0.001395    0.000002    0.000000    0.004312    0.000929    1.000    2\n",
      "      length{all}[2]      0.075178    0.000727    0.021499    0.126520    0.075124    1.011    2\n",
      "      length{all}[3]      0.074369    0.000485    0.030298    0.113390    0.076878    1.002    2\n",
      "      length{all}[4]      0.000785    0.000001    0.000000    0.002358    0.000545    1.000    2\n",
      "      length{all}[5]      0.028371    0.000287    0.000032    0.056083    0.029086    1.001    2\n",
      "      length{all}[6]      0.027827    0.000074    0.012210    0.044812    0.027040    1.000    2\n",
      "      length{all}[7]      0.061893    0.000893    0.000081    0.106074    0.066434    1.007    2\n",
      "      length{all}[8]      0.000902    0.000001    0.000000    0.002549    0.000634    1.000    2\n",
      "      length{all}[9]      0.001974    0.000004    0.000002    0.005920    0.001386    1.000    2\n",
      "      length{all}[10]     0.181453    0.001994    0.084795    0.271692    0.182856    1.002    2\n",
      "      length{all}[11]     0.000659    0.000000    0.000000    0.001973    0.000445    1.001    2\n",
      "      length{all}[12]     0.047349    0.000436    0.000011    0.080719    0.048685    1.008    2\n",
      "      length{all}[13]     0.001672    0.000003    0.000000    0.004980    0.001174    1.000    2\n",
      "      length{all}[14]     0.001754    0.000003    0.000000    0.005222    0.001221    1.001    2\n",
      "      length{all}[15]     0.008237    0.000021    0.000091    0.016702    0.007541    1.000    2\n",
      "      length{all}[16]     0.026651    0.000068    0.010946    0.042702    0.025991    1.000    2\n",
      "      length{all}[17]     0.002838    0.000004    0.000027    0.006838    0.002380    1.000    2\n",
      "      length{all}[18]     0.001765    0.000003    0.000000    0.005274    0.001211    1.000    2\n",
      "      length{all}[19]     0.001768    0.000003    0.000000    0.005318    0.001202    1.000    2\n",
      "      length{all}[20]     0.002365    0.000003    0.000002    0.005995    0.001949    1.000    2\n",
      "      length{all}[21]     0.013346    0.000028    0.003684    0.023470    0.012986    1.000    2\n",
      "      length{all}[22]     0.056344    0.000197    0.029764    0.084493    0.055203    1.000    2\n",
      "      length{all}[23]     0.000957    0.000001    0.000000    0.002868    0.000676    1.000    2\n",
      "      length{all}[24]     1.088897    0.011547    0.882727    1.303972    1.082491    1.001    2\n",
      "      length{all}[25]     0.059972    0.002225    0.000009    0.155825    0.048945    1.001    2\n",
      "      length{all}[26]     0.001452    0.000002    0.000001    0.004310    0.001029    1.000    2\n",
      "      length{all}[27]     0.026375    0.000097    0.006370    0.046800    0.025957    1.000    2\n",
      "      length{all}[28]     0.085142    0.000301    0.052135    0.118554    0.084795    1.002    2\n",
      "      length{all}[29]     0.073293    0.001159    0.000019    0.122817    0.078715    1.012    2\n",
      "      length{all}[30]     0.001680    0.000003    0.000000    0.005054    0.001185    1.000    2\n",
      "      length{all}[31]     0.020118    0.000032    0.009654    0.031586    0.019742    1.000    2\n",
      "      length{all}[32]     0.001410    0.000002    0.000000    0.004252    0.000971    1.000    2\n",
      "      length{all}[33]     0.001021    0.000001    0.000000    0.002969    0.000740    1.000    2\n",
      "      length{all}[34]     0.042307    0.000163    0.017998    0.067087    0.041309    1.001    2\n",
      "      length{all}[35]     0.101587    0.006471    0.000118    0.253146    0.078781    1.000    2\n",
      "      length{all}[36]     0.177379    0.001108    0.116901    0.242960    0.175420    1.005    2\n",
      "      length{all}[37]     0.002988    0.000005    0.000002    0.007534    0.002449    1.000    2\n",
      "      length{all}[38]     0.005044    0.000010    0.000008    0.011173    0.004627    1.002    2\n",
      "      length{all}[39]     0.002346    0.000006    0.000001    0.006909    0.001641    1.000    2\n",
      "      length{all}[40]     0.006627    0.000010    0.001479    0.012955    0.006165    1.000    2\n",
      "      length{all}[41]     0.009905    0.000017    0.001924    0.017901    0.009687    1.001    2\n",
      "      length{all}[42]     0.001141    0.000001    0.000000    0.003508    0.000788    1.000    2\n",
      "      length{all}[43]     0.001499    0.000002    0.000000    0.004560    0.001044    1.001    2\n",
      "      length{all}[44]     0.091388    0.000170    0.063564    0.115375    0.091072    1.004    2\n",
      "      length{all}[45]     0.126365    0.001272    0.042928    0.193941    0.128069    1.014    2\n",
      "      length{all}[46]     0.001019    0.000001    0.000000    0.003044    0.000716    1.000    2\n",
      "      length{all}[47]     0.002893    0.000005    0.000002    0.007226    0.002372    1.000    2\n",
      "      length{all}[48]     0.014736    0.000020    0.006516    0.023647    0.014225    1.000    2\n",
      "      length{all}[49]     0.002625    0.000006    0.000000    0.007470    0.001891    1.000    2\n",
      "      length{all}[50]     0.002459    0.000006    0.000001    0.007387    0.001733    1.001    2\n",
      "      length{all}[51]     0.143675    0.000266    0.111584    0.174525    0.143182    1.000    2\n",
      "      length{all}[52]     0.082098    0.000149    0.057382    0.106116    0.082413    1.000    2\n",
      "      length{all}[53]     0.103849    0.000226    0.075634    0.133365    0.103636    1.000    2\n",
      "      length{all}[54]     0.093836    0.000281    0.061745    0.127352    0.093114    1.000    2\n",
      "      length{all}[55]     0.024646    0.000076    0.008514    0.041727    0.024081    1.005    2\n",
      "      length{all}[56]     0.038551    0.000071    0.022268    0.054431    0.038160    1.003    2\n",
      "      length{all}[57]     0.036370    0.000091    0.017504    0.055230    0.036636    1.000    2\n",
      "      length{all}[58]     0.041649    0.000113    0.020027    0.061612    0.041362    1.000    2\n",
      "      length{all}[59]     0.077368    0.000296    0.044993    0.112398    0.077113    1.000    2\n",
      "      length{all}[60]     0.073810    0.000489    0.033278    0.118884    0.073393    1.001    2\n",
      "      length{all}[61]     0.079224    0.000288    0.048316    0.114093    0.078364    1.001    2\n",
      "      length{all}[62]     0.061185    0.000156    0.036617    0.084883    0.060675    1.000    2\n",
      "      length{all}[63]     0.075088    0.000529    0.028884    0.119355    0.074156    1.002    2\n",
      "      length{all}[64]     0.051943    0.000490    0.000113    0.089408    0.052118    1.002    2\n",
      "      length{all}[65]     0.050848    0.000343    0.014146    0.089394    0.050686    1.001    2\n",
      "      length{all}[66]     0.014302    0.000040    0.002459    0.026713    0.013949    1.001    2\n",
      "      length{all}[67]     0.003084    0.000003    0.000294    0.006373    0.002789    1.002    2\n",
      "      length{all}[68]     0.020938    0.000143    0.000108    0.043168    0.019691    1.002    2\n",
      "      length{all}[69]     0.005316    0.000007    0.000526    0.010389    0.005047    1.000    2\n",
      "      length{all}[70]     0.037538    0.000260    0.006750    0.069468    0.037087    1.000    2\n",
      "      length{all}[71]     0.022400    0.000174    0.000035    0.045363    0.021640    1.000    2\n",
      "      length{all}[72]     0.016482    0.000075    0.000087    0.031894    0.015795    1.002    2\n",
      "      length{all}[73]     0.077290    0.000526    0.035231    0.122301    0.077953    1.002    2\n",
      "      length{all}[74]     0.016409    0.000040    0.005100    0.029591    0.015983    1.000    2\n",
      "      length{all}[75]     0.060890    0.000659    0.008656    0.109741    0.060116    1.007    2\n",
      "      length{all}[76]     0.079026    0.000586    0.032144    0.128858    0.078860    1.001    2\n",
      "      length{all}[77]     0.050156    0.000287    0.018870    0.083589    0.049045    1.000    2\n",
      "      length{all}[78]     0.186971    0.002857    0.024708    0.260052    0.198522    1.020    2\n",
      "      length{all}[79]     0.025826    0.000222    0.000022    0.050829    0.025289    1.000    2\n",
      "      length{all}[80]     0.030727    0.000155    0.007009    0.055052    0.030533    1.001    2\n",
      "      length{all}[81]     0.076972    0.000502    0.033542    0.122371    0.076232    1.000    2\n",
      "      length{all}[82]     0.042805    0.000122    0.021822    0.064549    0.042516    1.001    2\n",
      "      length{all}[83]     0.028784    0.000173    0.001577    0.052742    0.028278    1.002    2\n",
      "      length{all}[84]     0.095661    0.001468    0.022689    0.171354    0.094756    1.001    2\n",
      "      length{all}[85]     0.051089    0.000658    0.001689    0.095339    0.050413    1.002    2\n",
      "      length{all}[86]     0.112384    0.003308    0.000002    0.200739    0.123079    1.015    2\n",
      "      length{all}[87]     0.004780    0.000014    0.000009    0.011744    0.003996    1.000    2\n",
      "      length{all}[88]     0.042049    0.000147    0.019442    0.065588    0.042089    1.001    2\n",
      "      length{all}[89]     0.006185    0.000025    0.000001    0.016151    0.004943    1.002    2\n",
      "      length{all}[90]     0.047579    0.000674    0.000021    0.091469    0.047066    1.000    2\n",
      "      length{all}[91]     0.018714    0.000099    0.000002    0.036298    0.017655    1.003    2\n",
      "      length{all}[92]     0.001413    0.000002    0.000001    0.004324    0.000944    1.000    2\n",
      "      length{all}[93]     0.044339    0.000259    0.011716    0.076824    0.044838    1.001    2\n",
      "      length{all}[94]     0.002429    0.000006    0.000000    0.007261    0.001717    1.000    2\n",
      "      length{all}[95]     0.002427    0.000006    0.000001    0.007052    0.001714    1.000    2\n",
      "      length{all}[96]     0.003896    0.000009    0.000008    0.009932    0.003308    1.002    2\n",
      "      length{all}[97]     0.002147    0.000005    0.000001    0.006251    0.001539    1.000    2\n",
      "      length{all}[98]     0.003351    0.000007    0.000001    0.008237    0.002733    1.000    2\n",
      "      length{all}[99]     0.175037    0.001354    0.101966    0.251918    0.178356    1.000    2\n",
      "      length{all}[100]    0.014990    0.000095    0.000002    0.032711    0.013704    1.004    2\n",
      "      length{all}[101]    0.001816    0.000002    0.000000    0.004958    0.001400    1.001    2\n",
      "      length{all}[102]    0.002037    0.000004    0.000002    0.005956    0.001454    1.000    2\n",
      "      length{all}[103]    0.001777    0.000003    0.000001    0.005317    0.001253    1.000    2\n",
      "      length{all}[104]    0.061961    0.000987    0.000601    0.109953    0.065126    1.095    2\n",
      "      length{all}[105]    0.040312    0.000232    0.011779    0.069938    0.039469    1.009    2\n",
      "      length{all}[106]    0.002769    0.000008    0.000001    0.008544    0.001893    1.004    2\n",
      "      length{all}[107]    0.000995    0.000001    0.000001    0.002891    0.000701    1.001    2\n",
      "      length{all}[108]    0.063448    0.001007    0.000235    0.114395    0.064529    1.000    2\n",
      "      length{all}[109]    0.002187    0.000004    0.000001    0.006258    0.001575    1.000    2\n",
      "      length{all}[110]    0.001013    0.000001    0.000000    0.002948    0.000707    1.002    2\n",
      "      length{all}[111]    0.002692    0.000007    0.000000    0.008081    0.001839    1.001    2\n",
      "      length{all}[112]    0.035280    0.000401    0.000028    0.069135    0.034375    1.001    2\n",
      "      length{all}[113]    0.001718    0.000003    0.000000    0.004960    0.001189    1.003    2\n",
      "      length{all}[114]    0.003085    0.000008    0.000003    0.009225    0.002285    1.003    2\n",
      "      length{all}[115]    0.033327    0.000366    0.000187    0.062821    0.034049    1.079    2\n",
      "      length{all}[116]    0.001026    0.000001    0.000000    0.003099    0.000723    1.001    2\n",
      "      length{all}[117]    0.002340    0.000004    0.000002    0.006586    0.001840    1.000    2\n",
      "      length{all}[118]    0.001551    0.000003    0.000000    0.004656    0.001079    1.000    2\n",
      "      length{all}[119]    0.001080    0.000001    0.000000    0.003257    0.000707    1.000    2\n",
      "      length{all}[120]    0.001283    0.000002    0.000000    0.003840    0.000883    1.000    2\n",
      "      length{all}[121]    0.060947    0.000489    0.019894    0.102949    0.059807    1.000    2\n",
      "      length{all}[122]    0.001256    0.000002    0.000000    0.003744    0.000865    1.000    2\n",
      "      length{all}[123]    0.001279    0.000002    0.000001    0.003901    0.000870    1.001    2\n",
      "      length{all}[124]    0.001281    0.000002    0.000000    0.003753    0.000913    1.000    2\n",
      "      length{all}[125]    0.053897    0.000392    0.006788    0.087492    0.054233    1.017    2\n",
      "      length{all}[126]    0.002441    0.000005    0.000009    0.007040    0.001808    1.000    2\n",
      "      length{all}[127]    0.015003    0.000048    0.002173    0.028195    0.014354    1.000    2\n",
      "      length{all}[128]    0.003649    0.000012    0.000008    0.010544    0.002584    1.000    2\n",
      "      length{all}[129]    0.058225    0.001350    0.000130    0.124487    0.055795    1.000    2\n",
      "      length{all}[130]    0.076941    0.002096    0.000073    0.159444    0.076884    1.004    2\n",
      "      length{all}[131]    0.043168    0.000587    0.000313    0.085167    0.042524    1.001    2\n",
      "      length{all}[132]    0.127408    0.001251    0.045477    0.193322    0.130143    1.037    2\n",
      "      length{all}[133]    0.068388    0.000655    0.019256    0.117210    0.070905    1.013    2\n",
      "      length{all}[134]    0.015578    0.000110    0.000005    0.034955    0.013495    1.000    2\n",
      "      length{all}[135]    0.001455    0.000002    0.000001    0.004449    0.000978    1.003    2\n",
      "      length{all}[136]    0.000980    0.000001    0.000000    0.002945    0.000682    0.999    2\n",
      "      length{all}[137]    0.000971    0.000001    0.000000    0.002754    0.000698    0.999    2\n",
      "      length{all}[138]    0.000761    0.000001    0.000000    0.002308    0.000528    1.000    2\n",
      "      length{all}[139]    0.038499    0.000979    0.000294    0.097523    0.031104    1.000    2\n",
      "      length{all}[140]    0.026309    0.000507    0.000002    0.075316    0.020275    0.999    2\n",
      "      length{all}[141]    0.001426    0.000002    0.000004    0.004473    0.000959    1.001    2\n",
      "      length{all}[142]    0.001004    0.000001    0.000000    0.002985    0.000701    0.999    2\n",
      "      length{all}[143]    0.000974    0.000001    0.000001    0.002928    0.000658    1.000    2\n",
      "      length{all}[144]    0.037795    0.000468    0.000064    0.071141    0.037881    0.999    2\n",
      "      length{all}[145]    0.000824    0.000001    0.000001    0.002435    0.000604    0.999    2\n",
      "      --------------------------------------------------------------------------------------------\n",
      "      + Convergence diagnostic (PSRF = Potential Scale Reduction Factor; Gelman\n",
      "        and Rubin, 1992) should approach 1.0 as runs converge. NA is reported when\n",
      "        deviation of parameter values within all runs is 0 or when a parameter\n",
      "        value (a branch length, for instance) is not sampled in all runs.\n",
      "\n",
      "\n",
      "      Summary statistics for partitions with frequency >= 0.10 in at least one run:\n",
      "          Average standard deviation of split frequencies = 0.031494\n",
      "          Maximum standard deviation of split frequencies = 0.167232\n",
      "          Average PSRF for parameter values (excluding NA and >10.0) = 1.003\n",
      "          Maximum PSRF for parameter values = 1.095\n",
      "\n",
      "\n",
      "      Clade credibility values:\n",
      "\n",
      "      Subtree rooted at node 84:\n",
      "\n",
      "                                           /--------------------- 2022_000207 (2)\n",
      "                                           |                                          \n",
      "                                           |               /----- Raffaelea_cana~ (14)\n",
      "                                           |         /--98-+                          \n",
      "                                /----52----+         |     \\----- Raffaelea_cana~ (19)\n",
      "                                |          |    /-46-+                                \n",
      "                                |          |    |    \\----------- Raffaelea_cycl~ (29)\n",
      "                                |          \\-48-+                                     \n",
      "                                |               \\---------------- Esteya_vermico~ (35)\n",
      "                                |                                                     \n",
      "                                |               /---------------- Harringtonia_la~ (6)\n",
      "                                |               |                                     \n",
      "                           /-50-+               |          /----- Harringtonia_la~ (9)\n",
      "                           |    |          /-90-+    /--35-+                          \n",
      "                           |    |          |    |    |     \\----- 2021_000003 (18)\n",
      "                           |    |          |    \\-94-+                                \n",
      "                           |    |     /-100+         \\----------- Harringtonia_l~ (39)\n",
      "                           |    |     |    |                                          \n",
      "                           |    |     |    \\--------------------- Harringtonia_l~ (49)\n",
      "                           |    |     |                                               \n",
      "                           |    \\--40-+                    /----- 2023_000248 (38)\n",
      "                           |          |              /--97-+                          \n",
      "                      /-55-+          |              |     \\----- Raffaelea_arxi~ (41)\n",
      "                      |    |          \\------74------+                                \n",
      "                      |    |                         \\----------- Raffaelea_scol~ (45)\n",
      "                      |    |                                                          \n",
      "                      |    |                               /----- Raffaelea_triti~ (7)\n",
      "                      |    |                         /--43-+                          \n",
      "                      |    |                         |     \\----- Raffaelea_ambr~ (25)\n",
      "                      |    |                    /-80-+                                \n",
      "                 /-48-+    |                    |    \\----------- Raffaelea_xyle~ (12)\n",
      "                 |    |    |                    |                                     \n",
      "                 |    |    \\---------84---------+          /----- 2023_000307 (16)\n",
      "                 |    |                         |    /--92-+                          \n",
      "                 |    |                         |    |     \\----- Raffaelea_sp_C~ (27)\n",
      "                 |    |                         \\-100+                                \n",
      "                 |    |                              \\----------- Raffaelea_cros~ (22)\n",
      "                 |    |                                                               \n",
      "                 |    \\------------------------------------------ Aspergillus_eg~ (24)\n",
      "                 |                                                                    \n",
      "                 |                                         /----- Heinzbutinia_mi~ (3)\n",
      "                 |                              /----59----+                          \n",
      "           /--99-+                              |          \\----- Heinzbutinia_mi~ (5)\n",
      "           |     |                              |                                     \n",
      "           |     |                         /-100+          /----- Heinzbutinia_g~ (13)\n",
      "           |     |                         |    |    /--38-+                          \n",
      "           |     |                         |    |    |     \\----- Heinzbutinia_g~ (50)\n",
      "           |     |                    /-100+    \\-100+                                \n",
      "           |     |                    |    |         \\----------- Heinzbutinia_g~ (30)\n",
      "           |     |                    |    |                                          \n",
      "           |     |              /--76-+    \\--------------------- 2022_000206 (44)\n",
      "           |     |              |     |                                               \n",
      "           |     |              |     |                    /----- Hawksworthiomy~ (28)\n",
      "           |     \\------80------+     \\---------99---------+                          \n",
      "           |                    |                          \\----- Hawksworthiomy~ (34)\n",
      "           |                    |                                                     \n",
      "      --100+                    \\-------------------------------- Ophiostoma_noi~ (10)\n",
      "           |                                                                          \n",
      "           |                                               /----- Leptographium_p~ (4)\n",
      "           |                                         /--24-+                          \n",
      "           |                                         |     \\----- Leptographium_~ (43)\n",
      "           |                                    /-27-+                                \n",
      "           |                                    |    \\----------- Leptographium_~ (15)\n",
      "           |                               /-30-+                                     \n",
      "           |                               |    \\---------------- Leptographium_~ (37)\n",
      "           |                          /-100+                                          \n",
      "           |                          |    \\--------------------- Leptographium_~ (26)\n",
      "           |                    /-100-+                                               \n",
      "           |                    |     \\-------------------------- 2021_000005 (21)\n",
      "           \\---------99---------+                                                     \n",
      "                                \\-------------------------------- Leptographium_~ (36)\n",
      "                                                                                      \n",
      "      Root part of tree:\n",
      "\n",
      "      /---------------------------------------------------------- Ophiostoma_quer~ (1)\n",
      "      |                                                                               \n",
      "      |---------------------------------------------------------- Ophiostoma_que~ (32)\n",
      "      |                                                                               \n",
      "      |                            /----------------------------- (84)\n",
      "      |                            |                                                  \n",
      "      |                            |                      /------ 2021_000001 (8)\n",
      "      |                            |                /-100-+                           \n",
      "      |                            |                |     \\------ Ophiostoma_nov~ (11)\n",
      "      |                      /--96-+           /-43-+                                 \n",
      "      |                      |     |           |    \\------------ Ophiostoma_ulm~ (48)\n",
      "      |                      |     |     /--88-+                                      \n",
      "      |                      |     |     |     \\----------------- Ophiostoma_ulm~ (40)\n",
      "      +                      |     |     |                                            \n",
      "      |                      |     \\--81-+                /------ Ophiostoma_flo~ (17)\n",
      "      |                      |           |          /--32-+                           \n",
      "      |                /--45-+           |          |     \\------ 2023_000565 (20)\n",
      "      |                |     |           \\----100---+                                 \n",
      "      |                |     |                      |     /------ Ophiostoma_flo~ (23)\n",
      "      |                |     |                      \\--22-+                           \n",
      "      |           /-94-+     |                            \\------ Ophiostoma_flo~ (46)\n",
      "      |           |    |     |                                                        \n",
      "      |           |    |     \\----------------------------------- Ophiostoma_que~ (47)\n",
      "      |     /--96-+    |                                                              \n",
      "      |     |     |    \\----------------------------------------- Ophiostoma_que~ (31)\n",
      "      |     |     |                                                                   \n",
      "      \\--40-+     \\---------------------------------------------- 2023_000566 (42)\n",
      "            |                                                                         \n",
      "            \\---------------------------------------------------- Ophiostoma_que~ (33)\n",
      "                                                                                      \n",
      "\n",
      "      Phylogram (based on average branch lengths):\n",
      "\n",
      "      / Ophiostoma_quer~ (1)\n",
      "      |                                                                               \n",
      "      | Ophiostoma_que~ (32)\n",
      "      |                                                                               \n",
      "      |                 /--- 2022_000207 (2)\n",
      "      |                 |                                                             \n",
      "      |                 |          / Raffaelea_cana~ (14)\n",
      "      |                 |        /-+                                                  \n",
      "      |             /---+        | \\ Raffaelea_cana~ (19)\n",
      "      |             |   |   /----+                                                    \n",
      "      |             |   |   |    \\--- Raffaelea_cycl~ (29)\n",
      "      |             |   \\---+                                                         \n",
      "      |             |       \\--- Esteya_vermico~ (35)\n",
      "      |             |                                                                 \n",
      "      |             |      /- Harringtonia_la~ (6)\n",
      "      |             |      |                                                          \n",
      "      |            /+      |- Harringtonia_la~ (9)\n",
      "      |            ||     /+                                                          \n",
      "      |            ||     ||- 2021_000003 (18)\n",
      "      |            ||     ||                                                          \n",
      "      |            ||/----+\\ Harringtonia_l~ (39)\n",
      "      |            |||    |                                                           \n",
      "      |            |||    \\ Harringtonia_l~ (49)\n",
      "      |            |||                                                                \n",
      "      |            |\\+          / 2023_000248 (38)\n",
      "      |            | |        /-+                                                     \n",
      "      |          /-+ |        | \\ Raffaelea_arxi~ (41)\n",
      "      |          | | \\--------+                                                       \n",
      "      |          | |          \\----- Raffaelea_scol~ (45)\n",
      "      |          | |                                                                  \n",
      "      |          | |       /--- Raffaelea_triti~ (7)\n",
      "      |          | |     /-+                                                          \n",
      "      |          | |     | \\-- Raffaelea_ambr~ (25)\n",
      "      |          | |  /--+                                                            \n",
      "      |         /+ |  |  \\-- Raffaelea_xyle~ (12)\n",
      "      |         || |  |                                                               \n",
      "      |         || \\--+    /- 2023_000307 (16)\n",
      "      |         ||    |  /-+                                                          \n",
      "      |         ||    |  | \\- Raffaelea_sp_C~ (27)\n",
      "      |         ||    \\--+                                                            \n",
      "      |         ||       \\-- Raffaelea_cros~ (22)\n",
      "      |         ||                                                                    \n",
      "      |         |\\----------------------------------------------- Aspergillus_eg~ (24)\n",
      "      |         |                                                                     \n",
      "      |         |          /--- Heinzbutinia_mi~ (3)\n",
      "      |         |         /+                                                          \n",
      "      |      /--+         |\\- Heinzbutinia_mi~ (5)\n",
      "      |      |  |         |                                                           \n",
      "      +      |  |       /-+/- Heinzbutinia_g~ (13)\n",
      "      |      |  |       | ||                                                          \n",
      "      |      |  |       | ||- Heinzbutinia_g~ (50)\n",
      "      |      |  |    /--+ \\+                                                          \n",
      "      |      |  |    |  |  \\ Heinzbutinia_g~ (30)\n",
      "      |      |  |    |  |                                                             \n",
      "      |      |  |  /-+  \\---- 2022_000206 (44)\n",
      "      |      |  |  | |                                                                \n",
      "      |      |  |  | |  /---- Hawksworthiomy~ (28)\n",
      "      |      |  \\--+ \\--+                                                             \n",
      "      |      |     |    \\-- Hawksworthiomy~ (34)\n",
      "      |      |     |                                                                  \n",
      "      |/-----+     \\------- Ophiostoma_noi~ (10)\n",
      "      ||     |                                                                        \n",
      "      ||     |         / Leptographium_p~ (4)\n",
      "      ||     |         |                                                              \n",
      "      ||     |         | Leptographium_~ (43)\n",
      "      ||     |        /+                                                              \n",
      "      ||     |        |\\ Leptographium_~ (15)\n",
      "      ||     |        |                                                               \n",
      "      ||     |        |- Leptographium_~ (37)\n",
      "      ||     |       /+                                                               \n",
      "      ||     |       |\\ Leptographium_~ (26)\n",
      "      ||     |   /---+                                                                \n",
      "      ||     |   |   \\ 2021_000005 (21)\n",
      "      ||     \\---+                                                                    \n",
      "      ||         \\------- Leptographium_~ (36)\n",
      "      ||                                                                              \n",
      "      ||  / 2021_000001 (8)\n",
      "      || /+                                                                           \n",
      "      || |\\ Ophiostoma_nov~ (11)\n",
      "      || |                                                                            \n",
      "      || | Ophiostoma_ulm~ (48)\n",
      "      ||/+                                                                            \n",
      "      |+|\\ Ophiostoma_ulm~ (40)\n",
      "      |||                                                                             \n",
      "      ||+  /- Ophiostoma_flo~ (17)\n",
      "      |||  |                                                                          \n",
      "      |||  |- 2023_000565 (20)\n",
      "      ||\\--+                                                                          \n",
      "      ||   | Ophiostoma_flo~ (23)\n",
      "      ||   |                                                                          \n",
      "      ||   \\ Ophiostoma_flo~ (46)\n",
      "      ||                                                                              \n",
      "      |\\ Ophiostoma_que~ (47)\n",
      "      |                                                                               \n",
      "      |- Ophiostoma_que~ (31)\n",
      "      |                                                                               \n",
      "      | 2023_000566 (42)\n",
      "      |                                                                               \n",
      "      \\ Ophiostoma_que~ (33)\n",
      "                                                                                      \n",
      "      |-------| 0.200 expected changes per site\n",
      "\n",
      "      Calculating tree probabilities...\n",
      "\n",
      "      Credible sets of trees (14939 trees sampled):\n",
      "         50 % credible set contains 7438 trees\n",
      "         90 % credible set contains 13439 trees\n",
      "         95 % credible set contains 14189 trees\n",
      "         99 % credible set contains 14789 trees\n",
      "\n",
      "   Exiting mrbayes block\n",
      "   Reached end of file\n",
      "\n",
      "   Tasks completed, exiting program because mode is noninteractive\n",
      "   To return control to the command line after completion of file processing, \n",
      "   set mode to interactive with 'mb -i <filename>' (i is for interactive)\n",
      "   or use 'set mode=interactive'\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!mb mrbayes/mb.nex"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "id": "17f796a8-39ba-4fe5-87dc-82e527f026cc",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Mrbayes outputs saved in mrbayes\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import shutil\n",
    "\n",
    "# Move all files starting with \"mb\" to the \"mrbayes\" folder\n",
    "current_directory = os.getcwd()\n",
    "mrbayes_directory = os.path.dirname(nexus_file_path)\n",
    "\n",
    "for filename in os.listdir(current_directory):\n",
    "    if filename.startswith(\"mb\"):\n",
    "        shutil.move(os.path.join(current_directory, filename), os.path.join(mrbayes_directory, filename))\n",
    "\n",
    "print(f'Mrbayes outputs saved in {mrbayes_directory}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "01e1a14b-0034-4d26-9cd4-1df700907041",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
